---
layout: page
title: About Me
menu: main
permalink: /about/
---

<img src="{{ site.url }}/assets/fong-internet.jpg" alt="Fong Chun Chan" style="width: 300px;" class="floatright" />

I am currently a Principal Bioinformatics Scientist at [Achilles Therapeutics](http://achillestx.com/). My primary focus has been to develop an industry-grade immunotherapy/immunogenomics pipeline (based on the TRACERx pipeline pioneered from Charles Swanton's lab) that identifies clonal neo-antigens that can elicit antitumor responses. In particular, I am spearheading the research and improvements on copy number profiling, clonal mutation detection, and neoantigen generation.

I completed my Bioinformatics PhD in 2017 under the co-supervision of [Dr. Sohrab Shah](http://compbio.bccrc.ca/) and [Dr. Christian Steidl](http://steidllab.med.ubc.ca/) studying tumour heterogeneity and its implications on disease progression in B-cell lymphomas. In particular, how tumour evolution plays a role in treatment resistance and how understanding this process may aid in the determination of relevant and precise therapeutic approaches for each cancer patient. Throughout my PhD and industry experience, I have acquired experience/skills in the following specific areas:

* Cancer genomic analyses that include prediction of somatic single nucleotide variants, small insertions/deletions, genomic breakpoints, gene-fusions, and copy number calling.
* Cancer evolution analyses that include the deconvolution of clonal composition, both at the single region and multi-regional level, and tracking of tumor evolution over time through serial sampling (e.g. primary vs. relapse)
* Prognostic biomarker discovery and modeling (e.g. multi-variate elastic-net Cox regression)
* Immunogenomics analyses that include HLA typing, and neo-antigen predictions

I have a passion for applying statistical and machine learning approaches to big data, in particular genomics data (e.g. sequencing data). In terms of technical skills:

* I have expertise in dealing with big data in the R programming language (e.g. data.table, tidyr, dplyr) and a thorough understanding of the core principles around R
* I have extensive experience in doing reproducible research through interactive applications and D3 web reporting (e.g. Rmarkdown, knitr, pandoc, git, shiny)
* I have expertise in developing/managing big data processing pipelines through the Make engine with experience in using ruffus, bpipe, snakemake, and nextflow. 
* I have experience in continuous integration/unit testing in production based workflows.

<h3>Selected Publications <small><a href="https://scholar.google.co.in/citations?user=BbZVuYMAAAAJ&hl=en&oi=ao" target="_new">[Google Scholar]</a></small>
</h3>

<div class="list-group">
  <div class="list-group-item">
    <h4 class="list-group-item-heading">Prognostic Model to Predict Post-Autologous Stem-Cell Transplantation Outcomes in Classical Hodgkin Lymphoma</h4>
    <div>Fong Chun Chan*, Anja Mottok*, Aline Gerrie, <i>et al.</i> 2017. <a href="http://ascopubs.org/journal/jco">Journal of Clinical Oncology</a>.</div>
    <div>* Equal Contribution</div>
    <div>
      <ul class="list-inline">
         <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/28898161"><span class="label label-success">Pubmed</span></a></li>
      </ul>
    </div>
  </div>
  <div class="list-group-item">
    <h4 class="list-group-item-heading">Observing Clonal Dynamics Across Spatiotemporal Axes: A Prelude to Quantitative Fitness Models for Cancer</h4>
    <div>Andrew McPherson, Fong Chun Chan, and Sohrab Shah <i>et al.</i> 2016. <a href="http://perspectivesinmedicine.cshlp.org/">Cold Spring Harb Perspect Med</a>.</div>
    <div>
      <ul class="list-inline">
         <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/28630229"><span class="label label-success">Pubmed</span></a></li>
      </ul>
    </div>
  </div>
  <div class="list-group-item">
    <h4 class="list-group-item-heading">Histological Transformation and Progression in Follicular Lymphoma: a Clonal Evolution Study</h4>
    <div>Robert Kridel*, Fong Chun Chan*, Anja Mottok, Merrill Boyle, <i>et al.</i> 2016. <a href="http://journals.plos.org/plosmedicine/">PLOS Medicine</a>.</div>
    <div>* Equal Contribution</div>
    <div>
      <ul class="list-inline">
         <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/27959929"><span class="label label-success">Pubmed</span></a></li>
         <li><a href="https://www.ebi.ac.uk/ega/studies/EGAS00001001709"><span class="label label-info">Data</span></a></li>
      </ul>
    </div>
  </div>
  <div class="list-group-item">
    <h4 class="list-group-item-heading">An RCOR1 loss-associated gene expression signature identifies a prognostically significant DLBCL subgroup</h4>
    <div>Fong Chun Chan, Adele Telenius, Shannon Healy, Susana Ben-Neriah, <i>et al.</i> 2015. <a href="http://www.bloodjournal.org/">Blood</a>.</div>
    <div>
      <ul class="list-inline">
         <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25395426"><span class="label label-success">PubMed</span></a></li>
         <li><a href="https://www.ebi.ac.uk/ega/studies/EGAS00001001000"><span class="label label-info">Data</span></a></li>
      </ul>
    </div>
  </div>
  <div class="list-group-item">
    <h4 class="list-group-item-heading">Recurrent somatic mutations of PTPN1 in primary mediastinal B cell lymphoma and Hodgkin lymphoma</h4>
    <div>Jay Gunawardana, Fong Chun Chan, Adele Telenius, Bruce Woolcock, <i>et al.</i> 2014. <a href="http://www.nature.com/ng/index.html">Nature Genetics</a>.</div>
    <div>
      <ul class="list-inline">
         <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24531327"><span class="label label-success">PubMed</span></a></li>
         <li><a href="https://www.ebi.ac.uk/ega/studies/EGAS00001000554"><span class="label label-info">Data</span></a></li>
      </ul>
    </div>
  </div>
  <div class="list-group-item">
    <h4 class="list-group-item-heading">Genomic rearrangements involving programmed death ligands are recurrent in primary mediastinal large B-cell lymphoma</h4>
    <div>David D W Twa, Fong Chun Chan, Susana Ben-Neriah, Bruce W Woolcock, <i>et al.</i> 2014. <a href="http://www.bloodjournal.org/">Blood</a>.</div>
    <div>
      <ul class="list-inline">
         <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24497532"><span class="label label-success">PubMed</span></a></li>
      </ul>
    </div>
  </div>
  <div class="list-group-item">
    <h4 class="list-group-item-heading">Gene expression-based model using formalin-fixed paraffin-embedded biopsies predicts overall survival in advanced-stage classical Hodgkin lymphoma</h4>
    <div>David W Scott, Fong Chun Chan, Fangxin Hong, Sanja Rogic, <i>et al.</i> 2013. <a href="http://ascopubs.org/journal/jco">Journal of Clinical Oncology</a>.</div>
    <div>
      <ul class="list-inline">
         <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23182984"><span class="label label-success">PubMed</span></a></li>
         <li><a href="https://github.com/tinyheero/CHL26predictor"><span class="label label-primary">Software</span></a></li>
      </ul>
    </div>
</div>

<h3>Software</h3>

<div class="list-group">
  <div class="list-group-item">
    <h4 class="list-group-item-heading">cofeatureR</h4>
    <p class="list-group-item-text">An R package for generating cofeature (feature by sample) matrices. The package utilizies ggplot2::geom_tile to generate the matrix allowing for easy additions from the base matrix.</p>
		<br />
		<ul class="list-inline">
		  <li><i class="fa fa-github fa-lg"></i> <a href="https://github.com/tinyheero/cofeatureR">GitHub</a></li>
			<li><a href="https://cran.rstudio.com/web/packages/cofeatureR"><img src="http://www.r-pkg.org/badges/version/cofeatureR" /></a></li>
      <li><a href="https://cran.rstudio.com/web/packages/cofeatureR"><img src="http://cranlogs.r-pkg.org/badges/cofeatureR" /></a></li>
      <li><a href="https://cran.rstudio.com/web/packages/cofeatureR"><img src="https://cranlogs.r-pkg.org/badges/grand-total/cofeatureR" /></a></li>
			<li><a href="https://anaconda.org/fongchun/r-cofeaturer"> <img src="https://anaconda.org/fongchun/r-cofeaturer/badges/version.svg" /> </a></li>
		</ul>
  </div>
  <div class="list-group-item">
    <h4 class="list-group-item-heading">survutils</h4>
    <p class="list-group-item-text">This package uses functional programming principles to iteratively run Cox regression and plot its results. The results are reported in tidy data frames. Additional utility functions are available for working with other aspects of survival analysis such as survival curves, C-statistics, etc.</p>
		<br />
		<ul class="list-inline">
		  <li><i class="fa fa-github fa-lg"></i> <a href="https://github.com/tinyheero/survutils">GitHub</a></li>
			<li><a href="https://cran.rstudio.com/web/packages/survutils"> <img src="http://www.r-pkg.org/badges/version/survutils" /></a></li>
      <li><a href="https://cran.rstudio.com/web/packages/survutils"> <img src="http://cranlogs.r-pkg.org/badges/survutils" /></a></li>
      <li><a href="https://cran.rstudio.com/web/packages/survutils"> <img src="https://cranlogs.r-pkg.org/badges/grand-total/survutils" /></a></li>
      <li><a href="https://anaconda.org/fongchun/r-survutils"> <img src="https://anaconda.org/fongchun/r-survutils/badges/version.svg" /></a></li>
		</ul>
  </div>
  <div class="list-group-item">
    <h4 class="list-group-item-heading">CHL26predictor</h4>
		<p class="list-group-item-text">An R Package for the Classical Hodgkin Lymphoma (CHL) 26 Gene Overall Survival Predictor. This is the companion R package for the predictor that has been <a href="http://www.ncbi.nlm.nih.gov/pubmed/23182984">published</a>.</p>
		<br />
		<ul class="list-inline">
		  <li><i class="fa fa-github fa-lg"></i> <a href="https://github.com/tinyheero/CHL26predictor">GitHub</a></li>
      <li><a href="https://anaconda.org/fongchun/chl26predictor"> <img src="https://anaconda.org/fongchun/chl26predictor/badges/version.svg" /> </a></li>
		</ul>
  </div>
</div>

<h3>Teaching</h3>

<div class="list-group">
  <div class="list-group-item">
    <h4 class="list-group-item-heading">Variant Calling in Cancer Genomes</h4>
    <div>Workshop on how calling somatic single nucleotide variants in cancer genome data</div>
		<br />
    <div>
      <ul class="list-inline">
        <li><i class="fa fa-github fa-lg"></i> <a href="https://github.com/tinyheero/variant_calling_in_cancer_genomes_seminar">GitHub</a></li>
      </ul>
    </div>
  </div>
  <div class="list-group-item">
    <h4 class="list-group-item-heading">Generating Heatmaps in R - Workshop</h4>
    <div>UBC R Study Group Workshop on generating Heatmaps in R</div>
		<br />
    <div>
      <ul class="list-inline">
        <li><i class="fa fa-github fa-lg"></i> <a href="https://github.com/tinyheero/R-Heatmaps">Github</a></li>
      </ul>
    </div>
</div>

<h3>Theses</h3>

<div class="list-group">
  <div class="list-group-item">
    <h4 class="list-group-item-heading">Clinical Implications of inter-tumour, intra-tumour, and tumour microenvironment heterogeneity in B-cell lymphomas</h4>
    <div>PhD Thesis. 2012 - 2017.</div>
    <div>
      <ul class="list-inline">
         <li><a href="http://hdl.handle.net/2429/61022"><span class="label label-success">UBC Circle</span></a></li>
      </ul>
    </div>
  </div>
  <div class="list-group-item">
    <h4 class="list-group-item-heading">Detection of differentially expressed alternative transcripts using conventional microarrays : with application to diffuse large B-cell lymphoma</h4>
    <div>MSc Thesis. 2009 - 2011.</div>
    <div>
      <ul class="list-inline">
         <li><a href="http://hdl.handle.net/2429/39846"><span class="label label-success">UBC Circle</span></a></li>
      </ul>
    </div>
</div>
