---
layout: post
title: "Multiple alignment of mammalian PrP amino acid sequences"
author: ericminikel
date: 2012-11-14 00:44:37
---
<p>Here is a multiple alignment of the PrP amino acid sequence from several different mammals:</p>
<p><a href="/wp-content/uploads/2012/11/t-coffee-mammal-prnp.html"><img class="alignnone size-full wp-image-1003" title="t-coffee-mammal-prnp" src="/wp-content/uploads/2012/11/t-coffee-mammal-prnp1.jpg"/></a></p>
<p>At the end of this post I will give quick instructions on how to do this.  First, some discussion.  The multiple alignment allowed me to see visually several things I&#8217;d heard for a while.  Notice that mouse has one fewer glycine than human right here:</p>
<p><a href="/wp-content/uploads/2012/11/t-coffee-human-mouse-deletion.jpg"><img class="alignnone size-full wp-image-995" title="t-coffee-human-mouse-deletion" src="/wp-content/uploads/2012/11/t-coffee-human-mouse-deletion.jpg"/></a></p>
<p>That&#8217;s why all the amino acids after that are numbered differently in mice (e.g. A117V in humans is A116V in mouse, D178N in humans is D177N in mice, etc.).  Similarly, you can see how cows have one extra copy of the octapeptide repeat PHGGGWGQ compared to all the other mammals I&#8217;m looking at:</p>
<p><a href="/wp-content/uploads/2012/11/t-coffee-cow-octapeptide.jpg"><img class="alignnone size-full wp-image-996" title="t-coffee-cow-octapeptide" src="/wp-content/uploads/2012/11/t-coffee-cow-octapeptide.jpg"/></a></p>
<p>(And apparently opossums have a decapeptide repeat instead!)</p>
<p>Codon 178 (right, below) is completely conserved as D in all these mammals, and even codon 129 (which has a non-disease-causing polymorphism in humans) is M in the reference genome for all of these as well:</p>
<p><a href="/wp-content/uploads/2012/11/t-coffee-129-178.jpg"><img class="alignnone size-full wp-image-997" title="t-coffee-129-178" src="/wp-content/uploads/2012/11/t-coffee-129-178.jpg"/></a></p>
<p>Overall, you can see just how much red is present throughout the alignment.  (In T-Coffee output, red doesn&#8217;t necessarily mean perfect amino acid match&#8211; if you look carefully at the above you&#8217;ll see some functionally similar amino acids e.g. the hydrophobic I, M, V, L substituted for one another). PRNP is a very highly conserved gene among mammals.  It&#8217;s quite peculiar actually: you&#8217;d expect highly conserved genes to be vitally important and if you knock them out, the organism would be severely affected and maybe not viable.  That&#8217;s the case with <a href="http://en.wikipedia.org/wiki/Huntingtin">HTT</a>.  But not with PRNP: it&#8217;s highly conserved and yet knockouts have no really severe or obvious phenotype [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1373228">Bueler 1992</a>] (later studies have shown there are <em>some</em> phenotypes though they are subtle; topic for a future post).</p>
<p>Notice I said PRNP was highly conserved &#8220;among mammals.&#8221;  Here&#8217;s a multiple alignment of human, mouse, chicken and zebrafish.  <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0013573">Zebrafish has three paralogs of PRNP</a> so all are included here.</p>
<p><a href="/wp-content/uploads/2012/11/t-coffee-human-mouse-chicken-zebrafish.html"><img class="alignnone size-full wp-image-999" title="t-coffee-human-mouse-chicken-zebrafish" src="/wp-content/uploads/2012/11/t-coffee-human-mouse-chicken-zebrafish.jpg"/></a></p>
<p>Notice how much poorer the alignment got when we added these non-mammals: barely any red in that graphic.  Though there is one stretch (from about 114 to 150 in human codon numbering) that is pretty highly conserved.  The stretch around human codon 178 also shows pretty decent homology with the chicken and zebrafish, though few of the amino acids are actually identical.</p>
<p>So PRNP is highly conserved among mammals and not-so-conserved among vertebrates more broadly.  Although the sequence is not very conserved, if you look at <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?position=chr10:20709904-20713769&amp;mrna=pack&amp;hgsid=310893295">Zebrafish PrP in the Genome Browser</a>, you&#8217;ll see it does have the same exon structure as its human homologue: two exons, with all the coding sequence in the second exon.</p>
<p><span style="text-decoration: underline;">Appendix: How to create a multiple alignment like this</span></p>
<ol>
<li>Go to the UCSC Genome Browser, select Human and search for PRNP.  Scroll down to the list of RefSeq genes and click on the first (or any) hit.  This will get you to a view kind of like <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?position=chr20:4666797-4682234&amp;hgsid=310893295&amp;refGene=pack&amp;hgFind.matches=NM_000311,">this</a>.</li>
<li>In the leftmost column of the browser, click on &#8220;PRNP&#8221; icon:<img title="ucsc-step-2" src="/wp-content/uploads/2012/11/ucsc-step-2.jpg"/></li>
<li> Scroll down and click on the &#8220;Predicted Protein&#8221; link.  This will give you the <a href="http://genome.ucsc.edu/cgi-bin/hgc?hgsid=310893295&amp;g=htcTranslatedProtein&amp;i=NP_000302&amp;c=chr20&amp;l=4666796&amp;r=4682234&amp;o=refPep&amp;table=refGene">amino acid sequence</a>.  This is the protein as translated directly from mRNA, so for PrP that includes N and C terminal sequences that later get cleaved off during post-translational modification.</li>
<li>Repeat starting from step 1 for other organisms you want to include, and paste all the results into a text file.*</li>
<li>Paste the contents of your text file into your favorite multiple alignment tool.  I am partial to <a href="http://tcoffee.crg.cat/apps/tcoffee/do:regular">T-Coffee</a> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3125728/">Di Tommaso 2011</a>]; some people prefer <a href="http://www.ebi.ac.uk/Tools/msa/clustalo/">Clustal Omega</a>.</li>
</ol>
<p>*Am I making step 4 more difficult than it needs to be?  If you click on the mRNAs for other organisms in the human genome browser you can get the nucleotide sequence but I don&#8217;t see a link for the protein, and for the limited number I was doing I figured it would be quicker to just click through for all of them than to copy and paste each nt sequence into another tool to translate to amino acids.  Alternately you can just take the first amino acid sequence you get from UCSC, <a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a> it and copy the top results from BLAST.  One advantage of that approach is that you&#8217;ll pick up orthologs that aren&#8217;t named &#8220;prnp&#8221;.  Yet another approach is to just use the multiple alignment that UCSC produces for you.  To do that, from Step 2 just click on &#8220;CDS FASTA alignment&#8221; and you&#8217;ll get to something like <a href="http://genome.ucsc.edu/cgi-bin/hgPal?g=refGene&amp;i=NM_000311&amp;c=chr20&amp;l=4666796&amp;r=4682234">this</a>.   This UCSC tool is wonderfully easy to use: you can just check boxes for the different species you want included or not.  However: it only shows deletions relative to the human sequence (or whatever sequence you started with), not insertions: &#8220;Because the CDS FASTA alignments are based on one reference genome, any amino acids or nucleotides that are not in the reference genome are not displayed.&#8221; - <a href="http://genome.ucsc.edu/goldenPath/help/hgTablesHelp.html#FASTA">User Guide</a>.  So if you want a true multiple alignment including insertions and deletions of everything relative to everything else, you&#8217;ve got to gather all the amino acid sequences separately and then do a multiple alignment with a separate tool.</p>
