---
layout: post
title: "How PCR duplicates arise in next-generation sequencing"
author: ericminikel
date: 2012-12-11 19:28:40
---
<p>PCR duplicates are an everyday annoyance in sequencing.  You spend hundreds or thousands of dollars to get sequencing done, and after you get the reads back, you find that several percent, sometimes even 30% or 70% of your reads are identical copies of each other.  These are called PCR duplicates and most sequencing pipelines recommend removing them or at least marking them (<a href="http://picard.sourceforge.net/command-line-overview.shtml#MarkDuplicates">Picard&#8217;s MarkDuplicates</a> or <a href="http://samtools.sourceforge.net/samtools.shtml">samtools rmdup</a> are two available tools).</p>
<p>Ever since I got into sequencing I wanted to know <em>how </em>these duplicates arise &#8211; what does it <em>mean</em> for a read to be a PCR duplicate and why do we ignore them?  I&#8217;ve Googled this over and over and never found a satisfying explanation.  So after hearing <a href="http://www.hudsonalpha.org/shawn-levy-phd">Shawn Levy</a> give an excellent introduction to the biochemistry and technology of sequencing at the <a title="UAB day 1: intro to sequencing" href="/2012/12/11/uab-day-1-intro-to-sequencing/">first day of the UAB sequencing course</a>, I asked him how PCR duplicates arise in next-generation sequencing.  In this post I&#8217;ll explain the answer he gave me.</p>
<p>First, let&#8217;s put this in context of the basic steps of library prep and sequencing:</p>
<ol>
<li>Shatter genomic DNA, e.g. with a sonicator.</li>
<li>Ligate adapters to both ends of the fragments.</li>
<li>PCR amplify the fragments with adapters</li>
<li>Create an oil-water emulsion of micrometer beads and DNA molecules (for Roche or Ion Torrent technologies) or spread DNA molecules across flowcells (for Illumina technology).  Goal is to get exactly one DNA molecule per bead or per flowcell lawn of primers.  This depends purely on probability, based on the concentration of DNA.</li>
<li>Use bridge PCR to amplify the single molecule on each bead or each lawn so that you can get a strong enough signal (whether light or pH) to detect.  Usually this requires several hundred or low thousands of molecules.</li>
<li>Sequence by synthesis of complementary strand: <a href="http://en.wikipedia.org/wiki/Pyrosequencing">pyrosequencing</a>  (Roche), <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2581791/">reversible terminator chemistry</a> (Illumina), or <a href="http://en.wikipedia.org/wiki/Ion_semiconductor_sequencing">ion semiconductor</a> (Ion Torrent).</li>
</ol>
<p>PCR duplicates arise in step 3.  In step 3 you are <em>intentionally</em> creating multiple copies of each original genomic DNA molecule so that you have enough of them, therefore <em>some</em> amount of PCR duplication is inevitable.  PCR duplicates occur when two copies of the same original molecule get onto different beads or different primer lawns in a flowcell.  Ideally in step 3 you are amplifying only ~64-fold (Shawn Levy said he prefers to do no more than 6 rounds of PCR, hence 2<sup>6</sup>) and so the library still has high &#8220;complexity&#8221;, i.e. information entropy.  This will let you have the fraction of your final reads that are PCR duplicates can be as low as 4%.  Higher rates of PCR duplicates e.g. 30% arise when people have too little starting material such that greater amplification of the library is needed in step 3, or when people have too great a variance in fragment size, such that smaller fragments, which are easier to PCR amplify, end up over-represented.</p>
<p>This whole explanation depends upon the random assignment of molecules to beads, beads to molecules that occurs in step 4.  You can only generate interpretable reads if you have <em>monoclonal </em>beads or clusters by the time you get to step 5.  Polyclonal beads &#8211; where more than one distinct DNA molecule arrived in step 4 &#8211; always arise in sequencing, and sequencers have software for masking out and ignoring these picotiter wells or these clusters on a flowcell.  Similarly, software will ignore empty wells or empty clusters where <em>no</em> DNA molecule wound up.  Sometimes more than one copy of the same original molecule (say, 2 out of the 64 copies you made of each molecule) which each hybridize to a different bead, and so you&#8217;ll be reading the same DNA in two different wells/clusters &#8211; those are your PCR duplicates.  But the goal is to have the concentrations of DNA and of beads/lawns just right in step 4 such that it&#8217;s likely that the majority of beads/clusters will get exactly one DNA molecule, and very few unique DNA molecules will be represented more than once.</p>
<p>Does that make sense?  I had to sit down and work through the probability myself.  The random hybridization of DNA molecules to beads is kind of complicated: it&#8217;s like drawing different-colored balls from different bins and then dropping those balls into different bins.  As a simplifying assumption, let&#8217;s suppose all beads will end up monoclonal, and only ask the question of how many DNA molecules will end up represented 0, 1, 2&#8230; n times in the resulting reads.</p>
<p>Here&#8217;s some quick back-of-the-envelope: say your NGS technology requries 1μg of DNA as input (figure quoted <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3246199/">here</a>).  If, as Shawn Levy recommends, you&#8217;ve done 6 PCR cycles, then that means you must have started with 1μg/(2<sup>6</sup>) = 15.6 ng of DNA isolated from your sample.  Molecular weight of DNA averages around 660g/mol/bp, and let&#8217;s say your fragments average 200bp long, so you&#8217;ve got 15.6ng/(660e9ng/mol/bp*200bp)*(6.022e23molecules/mol) = ~7e10 unique molecules.  If you&#8217;re doing 20x whole genome sequencing with 100bp reads, you&#8217;ll want 20x*3e9b/100b = 600M reads.  Since we&#8217;re simplifying and assuming for now that all beads will be monoclonal, for now let&#8217;s assume 600M reads means 600M = 6e8 beads.</p>
<p>So far you&#8217;ve got 64 copies each of 7e10 unique molecules and you want to hybridize them to 6e8 beads.  The expected number of copies of each molecule represented in your reads will be 6e8/7e10 = .0085.  In order to figure out the PCR duplicate rate, it would be nice to know the fraction of the 7e10 unique molecules that will be represented 0, 1, 2, &#8230; n times in the output reads.  You could model this as a <a href="http://en.wikipedia.org/wiki/Binomial_distribution">binomial distribution</a> with p = 1/(7e10) and n = 6e8, but since the probability is very low and number of trials very high, this will be better modeled by the asymptotic limit of the binomial: the <a href="http://en.wikipedia.org/wiki/Poisson_distribution">Poisson distribution</a>.  The <a href="http://en.wikipedia.org/wiki/Probability_density_function">PDF</a> of the Poisson distribution is given by λ<sup>k</sup>e<sup>-λ</sup>/k! where k = 0,1,2,&#8230;n is the number of copies of the molecule drawn, and the parameter lambda is equal to the expected number of copies of each molecule, which as I&#8217;ve said above is ~ .0085.</p>
<p>With these parameters, I generate in R a histogram of the probability of a given molecule being represented 0, 1, 2, etc. times:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">x <span style="color: #666666;">=</span> seq(<span style="color: #666666;">0</span>,<span style="color: #666666;">10</span>,<span style="color: #666666;">1</span>)
xnames <span style="color: #666666;">=</span> as.character(x)
xlab <span style="color: #666666;">=</span> <span style="color: #ba2121;">"Number of times a given molecule is represented in reads"</span>
ylab <span style="color: #666666;">=</span> <span style="color: #ba2121;">"Probability"</span>
barplot(dpois(x,lambda<span style="color: #666666;">=.0085</span>),names.arg<span style="color: #666666;">=</span>xnames,xlab<span style="color: #666666;">=</span>xlab,ylab<span style="color: #666666;">=</span>ylab)</pre>
</div>
<p>&nbsp;</p>
<p><a href="/wp-content/uploads/2012/12/pois_histo_11.png"><img class="alignnone size-full wp-image-1242" title="pois_histo_1" alt="" src="/wp-content/uploads/2012/12/pois_histo_11.png"/></a></p>
<p>As you&#8217;d expect, since we have far more molecules than beads, most reads get represented 0 times.  That&#8217;s fine &#8211; we started with many copies of the genome extracted from many cells, and we&#8217;ll get plenty of coverage from the handful of reads that are represented.  The important thing from this histogram is the very high height of the &#8220;1&#8243; bar  compared to all the ones that come after it, which is difficult to see in the above histogram because everything is dwarfed by the &#8220;0&#8243; bar.  Let&#8217;s exclude that bar:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">x1 <span style="color: #666666;">=</span> seq(<span style="color: #666666;">1</span>,<span style="color: #666666;">10</span>,<span style="color: #666666;">1</span>)
x1names <span style="color: #666666;">=</span> as.character(x1)
x1lab <span style="color: #666666;">=</span> <span style="color: #ba2121;">"Number of times a given molecule is represented in reads"</span>
ylab <span style="color: #666666;">=</span> <span style="color: #ba2121;">"Probability"</span>
barplot(dpois(x1,lambda<span style="color: #666666;">=.0085</span>),names.arg<span style="color: #666666;">=</span>x1names,xlab<span style="color: #666666;">=</span>x1lab,ylab<span style="color: #666666;">=</span>ylab)</pre>
</div>
<p><a href="/wp-content/uploads/2012/12/pois_histo_21.png"><img class="alignnone size-full wp-image-1243" title="pois_histo_2" alt="" src="/wp-content/uploads/2012/12/pois_histo_21.png"/></a></p>
<p>This histogram looks really good: given that a DNA molecule is represented in a read at all (since we&#8217;ve filtered out the k=0 molecules), it&#8217;s overwhelmingly likely that it&#8217;s represented exactly once.  And luckily, even if it&#8217;s represented twice, that&#8217;s not two PCR duplicates, just one &#8211; we&#8217;ll still use one copy and throw the other out.  Therefore we can quantify the fraction of PCR duplicates with this expression:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">sum(dpois(x1,lambda<span style="color: #666666;">=.0085</span>)<span style="color: #666666;">/</span>x1)<span style="color: #666666;">/</span>(<span style="color: #666666;">1-</span>dpois(<span style="color: #666666;">0</span>,lambda<span style="color: #666666;">=.0085</span>))</pre>
</div>
<p>Just to explain: <code>sum(dpois(x1,lambda=.0085)/x1)</code> is just count(1)/1 + count(2)/2 &#8230; i.e. summing the number of unique reads, and dividing <code>(1-dpois(0,lambda=.0085))</code> is just normalizing for the fraction of molecules that get sequenced at all.</p>
<p>The expression evaluates to .9979, so only 0.21% PCR duplicates.  The fact that this is different from Shawn Levy&#8217;s quoted figure of 4% under ideal conditions is due to some combination of (1) my back-of-the envelope numbers were off by a bit, and (2) even if you do everything right you&#8217;ll have some bias where by short length, low GC-content fragments are overrepresented, whereas I&#8217;ve assumed uniformly amplification of every starting molecule.</p>
<p>This model looks useful, so let&#8217;s continue with it.  Now suppose that instead of 7e10 unique molecules amplified 64-fold, you started with a smaller sample of just ~9e9 molecules and ran 9 PCR cycles, thus amplifying them 2<sup>9</sup> = 512-fold.  Now the lambda parameter in your Poisson distribution has risen to 6e8/9e9 = 0.067.  You started with so few unique molecules that on expectation each one will be represented 0.067 times.</p>
<p>Plugging this into our expression:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">sum(dpois(x1,lambda<span style="color: #666666;">=.067</span>)<span style="color: #666666;">/</span>x1)<span style="color: #666666;">/</span>(<span style="color: #666666;">1-</span>dpois(<span style="color: #666666;">0</span>,lambda<span style="color: #666666;">=.067</span>))</pre>
</div>
<p>We get .983, so a 1.7% PCR duplicate rate, still small but much higher than before. It&#8217;s easy to see why this occurs by looking at the histogram:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">barplot(dpois(x1,lambda<span style="color: #666666;">=.067</span>),names.arg<span style="color: #666666;">=</span>x1names,xlab<span style="color: #666666;">=</span>x1lab,ylab<span style="color: #666666;">=</span>ylab)</pre>
</div>
<p><a href="/wp-content/uploads/2012/12/pois_histo_31.png"><img class="alignnone size-full wp-image-1244" title="pois_histo_3" alt="" src="/wp-content/uploads/2012/12/pois_histo_31.png"/></a></p>
<p>You can see that the bar for k=2 is subtly taller than it was in the histogram with lambda = .0085.  Due to our lower number of unique molecules to start with, more molecules are getting represented twice, meaning you get more reads that are PCR duplicates.</p>
<p>We can look at an even worse scenario: maybe you started with just 1e9 unique molecules and did 12 PCR cycles, thus 2<sup>12</sup> = 4096 copies of each molecule.  Now your Poisson lambda is 6e8/1e9 = 0.6.  Plugging that in we see:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">sum(dpois(x1,lambda<span style="color: #666666;">=.6</span>)<span style="color: #666666;">/</span>x1)<span style="color: #666666;">/</span>(<span style="color: #666666;">1-</span>dpois(<span style="color: #666666;">0</span>,lambda<span style="color: #666666;">=.6</span>))</pre>
</div>
<p>= .85, implying 15% of reads will be PCR duplicates. And this is even easier to see on the barplot &#8211;  the k=2, 3, 4 etc. bars are very visibly higher:</p>
<pre>barplot(dpois(x1,lambda=.6),names.arg=x1names,xlab=x1lab,ylab=ylab)</pre>
<p><a href="/wp-content/uploads/2012/12/pois_histo_4.png"><img class="alignnone size-full wp-image-1245" title="pois_histo_4" alt="" src="/wp-content/uploads/2012/12/pois_histo_4.png"/></a></p>
<p>This whole analysis, while simplified, illustrates at least two important points about this random bead-DNA selection step 4 which arise from the Poisson distribution:</p>
<ul>
<li>The lower your ratio of <em>unique</em> molecules to beads, the more PCR duplicates you will have, <em>even before</em> your ratio begins to approach 1.0, where PCR duplicates would be required in order to saturate the beads with DNA molecules.</li>
<li>Even with 64 copies each of the unique molecules, PCR duplicates are very rare by pure chance.  Each unique molecule&#8217;s chance of getting represented even once (let alone twice) is small.</li>
</ul>
<div>What this model does not account for is that PCR amplification introduces bias.  This model predicts 0.21% duplicates after 6 PCR cycles, while in practice at least 4% is more common.  Most of that difference is probably due to the fact that PCR preferentially enriches smaller and more GC-poor molecules. Having an uneven distribution of copies of unique molecules in your library (which I haven&#8217;t modeled) has one important thing in common with not having enough unique molecules (which I have modeled): they both result in having a large number of copies of some molecules, making it likely that those will be duplicated.</div>
<div></div>
<div>This simplified Poisson model does not account for the other type of selection going on, whereby DNA molecules &#8220;choose&#8221; the beads, and the beads wind up blank (k=0) vs. monoclonal (k=1) vs. polyclonal (k&gt;=2).  Shawn Levy stated that 50-80% of wells in Roche or Ion Torrent sequencing will be occupied by a monoclonal bead, which means we can&#8217;t well model that process as a Poisson distribution.  If you play with a Poisson distribution you&#8217;ll discover there is no lambda value that gets you more than 1/e = ~37% of the probability density on k = 1, and his figure would imply that about 50-80% of the probability density lies there.  Presumably there is also some chemical magic going on here which makes monoclonality more likely &#8211; maybe having to do with the water/oil emulsion.</div>
<div></div>
<div>I&#8217;ve talked almost exclusively in terms of beads, which are the way that <a href="http://en.wikipedia.org/wiki/Pyrosequencing">Roche 454</a> and <a href="http://en.wikipedia.org/wiki/Ion_semiconductor_sequencing">Ion Torrent</a> sequencing work.  I don&#8217;t know as much about how Illumina technology gets molecules distributed into distinct clusters, but I&#8217;m assuming that the model I&#8217;ve outlined above also has some relevance to that technology.</div>
<div></div>
<div>This model also breaks down if you try to use it on very low-information libraries, like if you have twice as many beads as unique molecules.  What the Poisson distribution is really giving us here is the probability that a given copy of a unique molecule will be a unique read.  At low lamdba, that can be used to approximate the PCR duplicate rate, but at higher lambda (&gt;~1) it breaks down and you&#8217;ll get nonsensical answers that imply you&#8217;d get more unique reads than you have unique molecules.</div>
<div></div>
<div>There&#8217;s still a lot here I don&#8217;t totally understand, and I didn&#8217;t think of a good way to model amplification bias and see how that impacts the duplicate rate.  Still, this simple model has been really helpful to me in understanding why PCR duplicates arise, why (with good library prep) they are rare even though we <em>intentionally </em>make many copies of each molecule, why limited library complexity leads to greater duplicate rates.  The moral of the story for those doing library prep is: extract more DNA as starting material so that you get more unique molecules, and try to get as tight a distribution of fragment sizes as possible with your sonicator so that amplification bias is minimized.</div>
<div></div>
<div>If you found this explanation useful as well, or if you have an improvement or correction, drop me a comment to let me know.</div>
<div></div>
<div><strong>update 2012-12-17:</strong>  In light of the above post, today some colleagues and I were discussing the question: &#8220;then why wouldn&#8217;t you just start with way more DNA and not do <em>any</em> PCR?&#8221;  <a href="http://talkowski.mgh.harvard.edu/people/">Mike Talkowski</a> chimed in with an answer that highlights an important fact that was missing from my above explanation of PCR duplicates.  In step 2 when you ligate adapters, only a small fraction of DNA fragments will have an adapter successfully ligate to them.  In step 3, your PCR primers are based on the adapter sequence, so you&#8217;re only amplifying the successfully ligated molecules, leaving the unligated ones as a tiny fraction of the total.  Therefore by the time you get to step 4, the overwhelming majority of your DNA fragments will have adapters capable of hybridizing to the beads/flowcells.  If you didn&#8217;t do any PCR, you&#8217;d be putting in mostly unligated DNA fragments.</div>
