---
layout: post
title: "Improvements in GATK 2.x"
author: ericminikel
date: 2012-09-07 15:43:27
---
<p>Yesterday I attended a lecture by the folks at the Broad Institute about the new release of <a href="http://www.broadinstitute.org/gatk/">Genome Analysis Toolkit (GATK)</a> 2.x.  For those who don&#8217;t know, GATK (together with <a href="http://bio-bwa.sourceforge.net/">BWA</a> and <a href="http://picard.sourceforge.net/">Picard</a>, with which it comes bundled) is an end-to-end solution for sequencing, going from raw FASTQ reads coming off of a sequencer all the way to finished genotypes or haplotypes.</p>
<p>I have been working on a pipeline for analyzing exome data and, after many software problems, had initially given up on GATK and used an alternative approach (blog post forthcoming).  However this workshop has given me enough reason to go back and try again to get GATK working on my data.  First, it&#8217;s emerging as a gold standard: apparently it now boasts 6,000 users, making it far and away the most widely used solution.  Second, improvements are coming along fast, and that is much of what this talk was about.  Here are a few highlights of 2.x as opposed to 1.x:</p>
<ul>
<li><strong>Reduced reads.  </strong>GATK can take a BAM file and retain the full diversity of original reads only in places where the reads disagree with each other; for any bases where all (actually the default value is 95% or more) of the reads agree with each other (whether or not they agree with the reference sequence), it compresses them down to one read with metadata indicating how many reads there originally were.  This results in 20x &#8211; 100x compression, though obviously only on deep coverage files (any regions with 1x coverage obviously can&#8217;t be compressed at all).   This means it&#8217;s easier to move files around, and that variant calling will be several times faster.</li>
<li><strong>Base quality recalibration</strong> has been improved.  Base quality scores from sequencing companies tend to be inflated&#8211; ex. a PHRED score of 20 means 99% accuracy, yet in practice it is found that more than 1% of bases with score 20 are actually wrong.  GATK can recalibrate the scores to make them more accurate (though still not perfect).</li>
<li><strong>Combined SNP, indel and structural variant calling.</strong>  This is all in one step instead of separate steps.</li>
<li><strong>Better indel calling.</strong>  Indels are still a lot harder to call than SNPs and more of them fail validation, but it&#8217;s improving.</li>
</ul>
<p>Another highlight was something I believe is not actually new, but that I didn&#8217;t know about before: <a href="http://www.broadinstitute.org/gatk/guide/topic?name=developer-zone">Queue</a>.  There are a LOT of steps in GATK.  To wit:</p>
<p><a href="/wp-content/uploads/2012/09/depristo-2011-fig-11.jpg"><img class="alignnone size-large wp-image-723" title="depristo_figure1" src="/wp-content/uploads/2012/09/depristo-2011-fig-11-1024x758.jpg"/></a></p>
<p>That&#8217;s Figure 1 from <a href="http://ukpmc.ac.uk/articles/PMC3083463/">DePristo 2011</a>&#8216;s paper on GATK.  So naturally it becomes a pain having to pipeline all these steps yourself, and deal with job scheduling, memory requirements, intermediate files, etc.  Queue is supposed to take care of that for you.  They also gave us an order of magnitude estimate of time savings which is always helpful &#8212; a UnifiedGenotyper job which would take 10 days to run in series can be parallelized to run in 6 hours using Queue.</p>
