---
layout: post
title: "Power for GWAS and extreme phenotype studies"
author: ericminikel
date: 2012-12-05 23:07:52
---
<p><strong>Motivation</strong></p>
<p>Today many researchers are simultaneously pursuing ever-larger genome-wide association studies (GWAS) using SNP chips, while also pursuing exome sequencing studies to try to find rarer variants than can be found in GWAS.  Rare variants are less likely to be well-tagged by GWAS SNPs, so it&#8217;s believed that exome sequencing will have better power for rare variants, though as of today there are precious few examples of significant associations being found (one still-debated example is <a href="/2012/08/17/exome-sequencing-identifies-disease-modifiers-in-cystic-fibrosis/">Emond&#8217;s cystic fibrosis study</a>).</p>
<p>GWAS have poorer power to detect rare variants <em>per individual genotyped</em>, but at $50 or $100 per chip you can genotype a lot of people.  Exome sequencing still costs several hundred dollars, and inevitably exome studies are much smaller than GWAS.</p>
<p>In terms of strategic planning, then, an important question is: &#8220;how many people do I need to include in my exome study in order to have power to detect something I won&#8217;t already find anyway through my GWAS?&#8221;</p>
<p>This post aims to show how to answer this question.  Since I work on Huntington&#8217;s Disease age of onset, I&#8217;ll present the issue in that light.  First, here is some background information.</p>
<p><strong>Background on quantitative phenotypes and models</strong></p>
<p>Much of the literature on association studies deals with case/control studies.  &#8221;Quantitative&#8221; here is the opposite of &#8220;case/control&#8221;.  Quantitative means continuous as opposed to discrete; numeric as opposed to binary or dichotomous.  For instance, <a href="http://en.wikipedia.org/wiki/Low-density_lipoprotein#Normal_ranges">LDL levels</a> which vary continuously from ~25 to ~200 are a quantitative phenotype, while &#8220;people with schizophrenia&#8221; vs. &#8220;people who don&#8217;t have schizophrenia&#8221; is a case/control phenotype.</p>
<p><a title="Background on heritability in Huntington’s Disease age of onset" href="/2012/11/28/background-on-heritability-in-huntingtons-disease-age-of-onset/">HD residual age of onset</a> is an intrinsically quantitative phenotype and deserves slightly different treatment than many other phenotypes.  Some other studies of quantitative phenotypes still speak of an <a href="http://en.wikipedia.org/wiki/Odds_ratio">odds ratio</a> or other such terminology really intended for dichotomous case/control comparisons.  They do that because for many quantitative phenotypes such as blood lipid levels, clinicians still have some cutoff they use to decide when a person is &#8220;affected&#8221; and an intervention is needed.  Other times the phenotype starts out dichotomous and researchers then subtract out effects of non-genetic factors to get a &#8220;liability&#8221; score (see <a href="http://coruscant.itmat.upenn.edu/pubs/GueyEA_GenEpi_Extremes.pdf">Guey 2011</a> for an example).  But <em>all</em> of the HD patients are affected, it&#8217;s just a matter of when.  There is no meaningful cutoff.  So instead of odds ratios, effect sizes are better thought of in terms of years or standard deviations.</p>
<p>HD residual (log) age of onset is created by subtracting <em>predicted</em> log age of onset (predicted based on CAG length, which controls ~70% of variance in overall HD age of onset) from actual log age of onset.  This gives you a residual which is uncorrelated with CAG length.  This residual looks vaguely normally distributed, though it&#8217;s skewed a bit left (i.e. the mean is left of the median) and is way excessively kurtotic (<a href="http://en.wikipedia.org/wiki/Kurtosis#Terminology_and_examples">leptokurtic</a> to be fancy), with a kurtosis of about 5.3 using <code>kurtosis()</code> from the <a href="http://math.furman.edu/~dcs/courses/math47/R/library/fBasics/html/015A-BasicStatistics.html">R <code>fBasics</code> package</a>):</p>
<p><a href="/wp-content/uploads/2012/12/residual.histo_1.png"><img class="alignnone size-medium wp-image-1190" title="residual.histo" alt="" src="/wp-content/uploads/2012/12/residual.histo_1-300x300.png"/></a></p>
<p>For a moment I&#8217;ll just assume this is normal anyway (I&#8217;ll un-assume this later) and I&#8217;ll <a href="http://en.wikipedia.org/wiki/Standard_score">z-score</a> it, so that I&#8217;m using a standardized residual log age of onset.</p>
<p>As mentioned <a title="Background on heritability in Huntington’s Disease age of onset" href="/2012/11/28/background-on-heritability-in-huntingtons-disease-age-of-onset/">previously</a>, we think that ~40% of the variance in this residual is heritable [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC373491/">U.S.-Venezuela Collaborative Research Project 2003</a>] and our goal is to find the genetic variations that make up that heritability.</p>
<p>One simple way to model the effect of a hypothetical causal variant is to assume there&#8217;s an underlying normal distribution of phenotypes with an effect due to the causal variant &#8220;spiked in&#8221;.  The causal variant has a given frequency we&#8217;ll call <a href="http://en.wikipedia.org/wiki/Minor_allele_frequency">maf</a>, and an effect size we&#8217;ll call es.  It&#8217;s random whether any person gets a causal variant, but if they do, then it adds its effect size to their phenotype.  Here&#8217;s an example in R for a hypothetical causal SNP with frequency 3% and effect size equal to 0.6 standard deviations:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">N <span style="color: #666666;">=</span> <span style="color: #666666;">6000</span>
maf <span style="color: #666666;">=</span> <span style="color: #666666;">.03</span>
es <span style="color: #666666;">=</span> <span style="color: #666666;">.6</span>
allele_counts <span style="color: #666666;">=</span> rbinom(n<span style="color: #666666;">=</span>N,prob<span style="color: #666666;">=</span>maf,size<span style="color: #666666;">=2</span>) <span style="color: #408080; font-style: italic;"># size=2 because we're diploid</span>
phenotypes <span style="color: #666666;">=</span> rnorm(n<span style="color: #666666;">=</span>N,m<span style="color: #666666;">=0</span>,sd<span style="color: #666666;">=1</span>) <span style="color: #666666;">+</span> es<span style="color: #666666;">*</span>allele_counts</pre>
</div>
<p>Note that maf and effect size together will determine the proportion of overall variance explained by the variant.  Variance explained <em>in the population</em> is approximately equal to (es)<sup>2</sup>*maf.  For the above example, it should be about (.6)2*.03 = .0108, i.e. about 1% of variance explained.  If you actually run the above code and do a linear model <code>lm(phenotypes~allele_counts)</code> you&#8217;ll tend to get a smaller number because this is just a model of a small sample, but if you pick much bigger numbers to simulate &#8220;the population&#8221; you&#8217;ll get something fairly close &#8211; try this:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">ph <span style="color: #666666;">=</span> c(rnorm(<span style="color: #666666;">970000</span>,m<span style="color: #666666;">=0</span>,sd<span style="color: #666666;">=1</span>),rnorm(<span style="color: #666666;">30000</span>,m<span style="color: #666666;">=.6</span>,sd<span style="color: #666666;">=1</span>))
ge <span style="color: #666666;">=</span> c(rep(<span style="color: #666666;">0</span>,<span style="color: #666666;">970000</span>),rep(<span style="color: #666666;">1</span>,<span style="color: #666666;">30000</span>))
summary(lm(ph~ge))</pre>
</div>
<p>(Admittedly a simplification ignoring homozygous individuals).  But I get .0104, which is pretty close to .0108.</p>
<p>Now here I&#8217;ve already started doing linear models.  That&#8217;s one way to associate phenotype with genotype, supported by the <a href="http://pngu.mgh.harvard.edu/~purcell/plink/">PLINK</a> <code>--linear</code> feature and easy to do in R, and <a href="http://www.biostat.washington.edu/~yanez/b578D/materials/dupontplummer.pdf">Dupont 1998</a> discusses how to calculate power for this model, though not specifically in a genetic context.  Perhaps more widely used in genetic association studies is the <a href="http://en.wikipedia.org/wiki/Likelihood-ratio_test">likelihood ratio test</a>.  To review, this consists of the following:</p>
<p>L<sub>a</sub> is the likelihood of the given data under an alternative hypothesis (i.e. &#8220;this SNP has an effect&#8221;, with the effect size estimated based on the data)</p>
<p>L<sub>0</sub> is the likelihood of the given data under the null hypothesis (&#8220;this SNP has no effect&#8221;)</p>
<p>D = 2(ln(L<sub>a</sub>)-ln(L<sub>0</sub>))</p>
<p>D is the likelihood ratio test statistic.  Compare it to the <a href="http://en.wikipedia.org/wiki/Chi-squared_distribution">chi-squared distribution</a> with degrees of freedom df<sub>a</sub> &#8211; df<sub>0</sub> in order to determine a p value.</p>
<p>A formulation which is only slightly different is the <a href="http://en.wikipedia.org/wiki/Wald_test">Wald test</a>, which computes D from the model parameters rather than likelihood.</p>
<p>θ<sub>a</sub> = maximum likelihood estimate of model parameter</p>
<p>θ<sub>0</sub> = model parameter under the null</p>
<p>D = (θ<sub>a</sub> &#8211; θ<sub>0</sub>)<sup>2</sup> / var(θ<sub>a</sub>)</p>
<p>And as above, D is compared to the chi-squared distribution.</p>
<p>The likelihood ratio and Wald tests are asymptotically equivalent.  If you&#8217;ve ever used <a href="http://pngu.mgh.harvard.edu/~purcell/plink/anal.shtml#qt">PLINK&#8217;s <code>--assoc</code> feature with a quantitative phenotype</a>, it assumes asymptotic behavior and so is using either/both of these.  If you aren&#8217;t sure that your data meet asymptotic assumptions, you can use <a href="http://pngu.mgh.harvard.edu/~purcell/plink/perm.shtml#mperm">max(T) permutation with the <code>--mperm</code> command</a>, and that uses the likelihood ratio test.</p>
<p>It seems people are using both likelihood ratio and/or linear models in the literature for quantitative trait loci (QTLs, i.e. variants that impact quantitative phenotypes).  As one example, <a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000072">Melzer 2008</a> does initial association using linear models in PLINK but then validates the most significant SNPs with permutation using max(T) permutation, thus the likelihood ratio test.</p>
<p><strong>How to compute power for a GWAS on a quantitative phenotype</strong></p>
<p>You could imagine simulating power for a GWAS linear model by creating the normally distributed phenotype and spiking in effect size as I&#8217;ve done above, running a linear model, and repeating 100 or 1000 times to see how often the coefficient for your SNP has a p value less than the threshold you want.  But hold on there, a power calculation for GWAS is not quite that straightforward.  Say our array has a million SNPs on it; it might happen to include the causal SNP, or it might just include a bunch of SNPs that are close enough to the causal SNP to be in <a href="http://en.wikipedia.org/wiki/Linkage_disequilibrium">linkage disequilibrium</a> (LD).  So you have to consider the extent of LD, and then not only the minor allele frequency of the causal SNP but also the &#8220;marker&#8221; SNP that might end up being a proxy for it.</p>
<p>I looked on the web for some guidance as to what assumptions I can make about LD if you assume, say, ~1M SNPs on a chip (thus an average of 1 SNP per 3000 bases).   The answer: it&#8217;s really complicated.  <a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.0020142">Eberle 2006</a> separately analyzes groups of unrelated Europeans, Africans and Asians and presents <a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.0020142?imageURI=info:doi/10.1371/journal.pgen.0020142.g001">a nice graph</a> of how LD between two common SNPs decays as the SNPs get further apart, depending on whether or not you only look at frequency-matched SNPs.  <a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000477">Spencer 2009</a> argues that LD patterns in the human genome are too complex for there to be any closed-form solution for calculating power for GWAS: instead it can only be done by simulation.  Spencer presents the <a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/gwas.html"><code>HAPGEN</code> and <code>GWAPOWER</code> R packages</a> for doing these simulations, but they only work for case/control studies whereas I&#8217;m dealing with quantitative phenotype.</p>
<p>Any power calculation will rest on a variety of assumptions, so I decided I&#8217;m willing to accept some assumptions about LD for now, knowing that I can vary that assumption and see how power is affected.  To get started, let&#8217;s do some back-of-the-envelope here.  If the 1M SNPs are uniformly scattered across the genome, they occur every 3kb and so the farthest that one ungenotyped SNP can get from its nearest genotyped SNP is 1.5kb.   And to the extent that the SNPs <em>aren&#8217;t</em> uniformly scattered, it&#8217;s probably because they&#8217;re concentrated in regions of functional importance and sequence conservation, so it&#8217;s not clear that power is actually reduced.   Eberle&#8217;s <a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.0020142?imageURI=info:doi/10.1371/journal.pgen.0020142.g001">graphs</a> are on the scale of 200kb; if you squint your eyes to see what&#8217;s going on at 1.5kb, LD is up around 90% of theoretical maximum.  That&#8217;s for common SNPs (&gt; 10%); Eberle is also looking at frequency-matched SNPs but my understanding is that this is accounted for if you use <a href="http://en.wikipedia.org/wiki/Linkage_disequilibrium#Definition">D&#8217;</a>, which is adjusted for frequency.  I decided to start by assuming LD of 0.8 between a theoretical causal SNP and its nearest marker.</p>
<p>I asked around for what tools people use to calculate power for GWAS, and I learned that <a href="http://pngu.mgh.harvard.edu/~purcell/gpc/">Shaun Purcell&#8217;s GPC</a> is the most widely used tool for calculating power for association studies.  It&#8217;s really stood the test of time, having been launched in 2002, which is ancient history in bioinformatics time.  It can calculate power for a number of different types of studies; the <a href="http://pngu.mgh.harvard.edu/~purcell/gpc/qtlassoc.html">QTL association feature</a> works for quantitative phenotypes.  This feature calculates power based on a likelihood ratio test - <a href="http://bioinformatics.oxfordjournals.org/content/19/1/149.long">Purcell 2003</a> cites <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1378020/pdf/10762547.pdf">Sham 2000</a> for the method.</p>
<p>GPC is much more elegant and straightforward than the deluge of similar tools that have come out since it was first released.  I&#8217;m interested in calculating power for a variety of different effect sizes and minor allele frequencies and creating a sort of &#8220;heat map&#8221; of what sort of variants we are powered to detect in our GWAS.  GPC has no web service so this would require a lot of clicking back and forth.  I also, after reading <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1378020/pdf/10762547.pdf">Sham 2000</a>, couldn&#8217;t wrap my head around the method GPC is using and why it didn&#8217;t match results I was getting in a simpler simulation.  Motivated by the desire to automate this process as well as my desire to understand any tool I&#8217;m going to use rather than just black boxing it, I got in touch with Shaun Purcell, who was nice enough to share the source code for the QTL association feature with me so I could port it to R.  Here&#8217;s my R implementation along with a few tests you can check against <a href="http://pngu.mgh.harvard.edu/~purcell/gpc/qtlassoc.html">GPC itself</a> to see that you get the same result:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;"><span style="color: #408080; font-style: italic;"># Eric Minikel - CureFFI.org</span>
<span style="color: #408080; font-style: italic;"># 2012-12-05</span>
<span style="color: #408080; font-style: italic;"># R implementation of QTL association as featured in Genetic Power Calculator:</span>
<span style="color: #408080; font-style: italic;"># http://pngu.mgh.harvard.edu/~purcell/gpc/</span>
<span style="color: #408080; font-style: italic;"># based on C++ source code generously provided by Shaun Purcell</span>
<span style="color: #408080; font-style: italic;"># who in turn based it on method in Sham 2000:</span>
<span style="color: #408080; font-style: italic;"># http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1378020/</span>

likelihood.ratio.test.power <span style="color: #666666;">=</span> <span style="color: #008000; font-weight: bold;">function</span>(alpha,df,ncp) {
    <span style="color: #008000; font-weight: bold;">return</span> ( <span style="color: #666666;">1-</span>pchisq(q<span style="color: #666666;">=</span>qchisq(p<span style="color: #666666;">=1-</span>alpha,df<span style="color: #666666;">=</span>df,ncp<span style="color: #666666;">=0</span>),df<span style="color: #666666;">=</span>df,ncp<span style="color: #666666;">=</span>ncp) )
}

<span style="color: #408080; font-style: italic;"># qtl.assoc.power</span>
<span style="color: #408080; font-style: italic;"># calculates power for specified n</span>
<span style="color: #408080; font-style: italic;"># parameters, their names in GPC, and explanations:</span>
<span style="color: #408080; font-style: italic;"># n = Sample Size</span>
<span style="color: #408080; font-style: italic;"># vq = Total QTL Variance = fraction of trait variance explained by this SNP</span>
<span style="color: #408080; font-style: italic;"># p = QTL increaser allele frequency = MAF of causal SNP</span>
<span style="color: #408080; font-style: italic;"># m1 = Marker M1 allele frequency = MAF of marker SNP, proxy for causal SNP</span>
<span style="color: #408080; font-style: italic;"># dprime = Linkage disequilibrium (D-prime)</span>
<span style="color: #408080; font-style: italic;"># alpha = User-defined type I error rate</span>
<span style="color: #408080; font-style: italic;"># da = Dominance : additive QTL effects = set nodom=FALSE to use this</span>
<span style="color: #408080; font-style: italic;"># nodom = No dominance = flag to ignore da and assume purely allelic model</span>
<span style="color: #408080; font-style: italic;"># parent = Both parents genotyped = flag for whether parents are included</span>
<span style="color: #408080; font-style: italic;"># sibr = Sibling correlation </span>
<span style="color: #408080; font-style: italic;"># s = Sibship Size = 1 for unrelated individuals, 2 for sib pairs, etc.</span>
<span style="color: #408080; font-style: italic;"># whichpower = which power calc you want: "overall", "between", "within"</span>
<span style="color: #408080; font-style: italic;"># for more details see notes: http://pngu.mgh.harvard.edu/~purcell/gpc/#qtl_ins</span>
qtl.assoc.power <span style="color: #666666;">=</span> <span style="color: #008000; font-weight: bold;">function</span>(n,vq,p,m1,dprime,alpha,da<span style="color: #666666;">=0</span>,nodom<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">TRUE</span>,parent<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">FALSE</span>,sibr<span style="color: #666666;">=1</span>,s<span style="color: #666666;">=1</span>,whichpower<span style="color: #666666;">=</span><span style="color: #ba2121;">"overall"</span>) {
    <span style="color: #008000; font-weight: bold;">if</span>(nodom) {
        tdf <span style="color: #666666;">=</span> <span style="color: #666666;">1</span> <span style="color: #408080; font-style: italic;"># 1 degree of freedom without dominance</span>
        <span style="color: #408080; font-style: italic;"># allele count is assumed to be linear with effect, so the only thing</span>
        <span style="color: #408080; font-style: italic;"># that can vary is the slope</span>
    } <span style="color: #008000; font-weight: bold;">else</span> {
        tdf <span style="color: #666666;">=</span> <span style="color: #666666;">2</span> <span style="color: #408080; font-style: italic;"># 2 degrees of freedom with dominance</span>
        <span style="color: #408080; font-style: italic;"># allele counts 0, 1, 2 vs. their effects are not collinear</span>
    }

    <span style="color: #408080; font-style: italic;"># allele frequencies</span>
    q <span style="color: #666666;">=</span> <span style="color: #666666;">1</span> <span style="color: #666666;">-</span> p <span style="color: #408080; font-style: italic;"># major AF of causal SNP</span>
    m2 <span style="color: #666666;">=</span> <span style="color: #666666;">1</span> <span style="color: #666666;">-</span> m1 <span style="color: #408080; font-style: italic;"># major AF of marker SNP</span>

    <span style="color: #408080; font-style: italic;"># recover un-normalized linkage disequilibrium</span>
    <span style="color: #408080; font-style: italic;"># back-calculate regular D based on D' which is passed to this function</span>
    <span style="color: #408080; font-style: italic;"># see http://en.wikipedia.org/wiki/Linkage_disequilibrium#Definition</span>
    dmax <span style="color: #666666;">=</span> p<span style="color: #666666;">*</span>m2
    <span style="color: #008000; font-weight: bold;">if</span> (q<span style="color: #666666;">*</span>m1 <span style="color: #666666;">&lt;</span> dmax) {
        dmax <span style="color: #666666;">=</span> q<span style="color: #666666;">*</span>m1
    }
    ld <span style="color: #666666;">=</span> dprime <span style="color: #666666;">*</span> dmax

    <span style="color: #408080; font-style: italic;"># calculate haplotype frequencies</span>
    h00 <span style="color: #666666;">=</span> p<span style="color: #666666;">*</span>m1 <span style="color: #666666;">+</span> ld <span style="color: #408080; font-style: italic;"># based on D from above, get haplotype frequencies</span>
    h01 <span style="color: #666666;">=</span> p<span style="color: #666666;">*</span>m2 <span style="color: #666666;">-</span> ld
    h10 <span style="color: #666666;">=</span> q<span style="color: #666666;">*</span>m1 <span style="color: #666666;">-</span> ld
    h11 <span style="color: #666666;">=</span> q<span style="color: #666666;">*</span>m2 <span style="color: #666666;">+</span> ld

    <span style="color: #408080; font-style: italic;"># genetic values</span>
    vtot <span style="color: #666666;">=</span> <span style="color: #666666;">1</span>
    a <span style="color: #666666;">=</span> sqrt( (vq<span style="color: #666666;">*</span>vtot) <span style="color: #666666;">/</span> ( (<span style="color: #666666;">2*</span>p<span style="color: #666666;">*</span>q)<span style="color: #666666;">*</span>(<span style="color: #666666;">1+</span>da<span style="color: #666666;">*</span>(q<span style="color: #666666;">-</span>p))<span style="color: #666666;">**2</span> <span style="color: #666666;">+</span> (<span style="color: #666666;">2*</span>p<span style="color: #666666;">*</span>q<span style="color: #666666;">*</span>da)<span style="color: #666666;">**2</span> ) )
    d <span style="color: #666666;">=</span> da<span style="color: #666666;">*</span>a

    <span style="color: #408080; font-style: italic;"># variance components</span>
    va <span style="color: #666666;">=</span> <span style="color: #666666;">2</span> <span style="color: #666666;">*</span> p <span style="color: #666666;">*</span> q <span style="color: #666666;">*</span> ( a <span style="color: #666666;">+</span> d<span style="color: #666666;">*</span>(q<span style="color: #666666;">-</span>p))<span style="color: #666666;">**2</span>
    vd <span style="color: #666666;">=</span> (<span style="color: #666666;">2*</span>p<span style="color: #666666;">*</span>q<span style="color: #666666;">*</span>d)<span style="color: #666666;">*</span>(<span style="color: #666666;">2*</span>p<span style="color: #666666;">*</span>q<span style="color: #666666;">*</span>d)
    vs <span style="color: #666666;">=</span> sibr<span style="color: #666666;">*</span>vtot <span style="color: #666666;">-</span> va<span style="color: #666666;">/2</span> <span style="color: #666666;">-</span> vd<span style="color: #666666;">/4</span>
    vn <span style="color: #666666;">=</span> (<span style="color: #666666;">1-</span>sibr)<span style="color: #666666;">*</span>vtot <span style="color: #666666;">-</span> va<span style="color: #666666;">/2</span> <span style="color: #666666;">-</span> <span style="color: #666666;">3*</span>vd<span style="color: #666666;">/4</span>

    <span style="color: #408080; font-style: italic;"># attenuation factor due to incomplete LD for association test</span>
    phisq <span style="color: #666666;">=</span>         ((h00 <span style="color: #666666;">-</span> m1<span style="color: #666666;">*</span>p)<span style="color: #666666;">**2/</span>(m1<span style="color: #666666;">*</span>p)) 
    phisq <span style="color: #666666;">=</span> phisq <span style="color: #666666;">+</span> ((h10 <span style="color: #666666;">-</span> m1<span style="color: #666666;">*</span>q)<span style="color: #666666;">**2/</span>(m1<span style="color: #666666;">*</span>q))
    phisq <span style="color: #666666;">=</span> phisq <span style="color: #666666;">+</span> ((h01 <span style="color: #666666;">-</span> m2<span style="color: #666666;">*</span>p)<span style="color: #666666;">**2/</span>(m2<span style="color: #666666;">*</span>p)) 
    phisq <span style="color: #666666;">=</span> phisq <span style="color: #666666;">+</span> ((h11 <span style="color: #666666;">-</span> m2<span style="color: #666666;">*</span>q)<span style="color: #666666;">**2/</span>(m2<span style="color: #666666;">*</span>q))
    vam <span style="color: #666666;">=</span> phisq <span style="color: #666666;">*</span> va
    vdm <span style="color: #666666;">=</span> phisq<span style="color: #666666;">**2</span> <span style="color: #666666;">*</span> vd
    vsm <span style="color: #666666;">=</span> vs <span style="color: #666666;">+</span> (((<span style="color: #666666;">1-</span>phisq)<span style="color: #666666;">*</span>va)<span style="color: #666666;">/2</span>) <span style="color: #666666;">+</span> (((<span style="color: #666666;">1-</span>phisq<span style="color: #666666;">**2</span>)<span style="color: #666666;">*</span>vd)<span style="color: #666666;">/4</span>)
    vnm <span style="color: #666666;">=</span> vn <span style="color: #666666;">+</span> (((<span style="color: #666666;">1-</span>phisq)<span style="color: #666666;">*</span>va)<span style="color: #666666;">/2</span>) <span style="color: #666666;">+</span> ((<span style="color: #666666;">3*</span>(<span style="color: #666666;">1-</span>phisq<span style="color: #666666;">**2</span>)<span style="color: #666666;">*</span>vd)<span style="color: #666666;">/4</span>)

    <span style="color: #408080; font-style: italic;"># compute expected non-centrality parameters </span>
    <span style="color: #408080; font-style: italic;"># ncp_b = between sibships</span>
    <span style="color: #408080; font-style: italic;"># ncp_w = within sibships</span>
    <span style="color: #008000; font-weight: bold;">if</span> (parent) {
        ncp_b <span style="color: #666666;">=</span> log(vnm <span style="color: #666666;">+</span> s<span style="color: #666666;">*</span>vsm <span style="color: #666666;">+</span> (s<span style="color: #666666;">/2</span>)<span style="color: #666666;">*</span>vam <span style="color: #666666;">+</span> (s<span style="color: #666666;">/4</span>)<span style="color: #666666;">*</span>vdm) <span style="color: #666666;">-</span> log(vnm <span style="color: #666666;">+</span> s<span style="color: #666666;">*</span>vsm)
        ncp_w <span style="color: #666666;">=</span> log((vnm<span style="color: #666666;">+0.5*</span>vam<span style="color: #666666;">+0.75*</span>vdm)<span style="color: #666666;">**</span>(s<span style="color: #666666;">-1</span>)) <span style="color: #666666;">-</span>
                log(vnm<span style="color: #666666;">**</span>(s<span style="color: #666666;">-1</span>)) <span style="color: #666666;">+</span>
                log(vnm <span style="color: #666666;">+</span> s<span style="color: #666666;">*</span>vsm <span style="color: #666666;">+</span> <span style="color: #666666;">0.5*</span>vam <span style="color: #666666;">+</span> <span style="color: #666666;">0.75*</span>vdm) <span style="color: #666666;">-</span> 
                log(vnm <span style="color: #666666;">+</span> s<span style="color: #666666;">*</span>vsm)
    } <span style="color: #008000; font-weight: bold;">else</span> {
        ncp_b <span style="color: #666666;">=</span> log(vnm <span style="color: #666666;">+</span> s<span style="color: #666666;">*</span>vsm <span style="color: #666666;">+</span> ((s<span style="color: #666666;">+1</span>)<span style="color: #666666;">/2</span>)<span style="color: #666666;">*</span>vam <span style="color: #666666;">+</span> ((s<span style="color: #666666;">+3</span>)<span style="color: #666666;">/4</span>)<span style="color: #666666;">*</span>vdm) <span style="color: #666666;">-</span> 
                log(vnm <span style="color: #666666;">+</span> s<span style="color: #666666;">*</span>vsm)
        ncp_w <span style="color: #666666;">=</span> log((vnm <span style="color: #666666;">+</span> <span style="color: #666666;">0.5*</span>vam <span style="color: #666666;">+</span> <span style="color: #666666;">0.75*</span>vdm)<span style="color: #666666;">**</span>(s<span style="color: #666666;">-1</span>))<span style="color: #666666;">-</span>log(vnm<span style="color: #666666;">**</span>(s<span style="color: #666666;">-1</span>))
    }

    <span style="color: #408080; font-style: italic;"># take sample size into account</span>
    ncp_b <span style="color: #666666;">=</span> ncp_b <span style="color: #666666;">*</span> n
    ncp_w <span style="color: #666666;">=</span> ncp_w <span style="color: #666666;">*</span> n

    between.sibships.power <span style="color: #666666;">=</span> likelihood.ratio.test.power(alpha,tdf,ncp_b)
    within.sibships.power <span style="color: #666666;">=</span> likelihood.ratio.test.power(alpha,tdf,ncp_w)
    overall.power <span style="color: #666666;">=</span> likelihood.ratio.test.power(alpha,tdf,ncp_b<span style="color: #666666;">+</span>ncp_w)

    <span style="color: #008000; font-weight: bold;">if</span> (whichpower <span style="color: #666666;">==</span> <span style="color: #ba2121;">"overall"</span>) {
        <span style="color: #008000; font-weight: bold;">return</span>(overall.power)
    } <span style="color: #008000; font-weight: bold;">else</span> <span style="color: #008000; font-weight: bold;">if</span> (whichpower <span style="color: #666666;">==</span> <span style="color: #ba2121;">"between"</span>) {
        <span style="color: #008000; font-weight: bold;">return</span>(between.sibships.power)
    } <span style="color: #008000; font-weight: bold;">else</span> <span style="color: #008000; font-weight: bold;">if</span> (whichpower <span style="color: #666666;">==</span> <span style="color: #ba2121;">"within"</span>) {
        <span style="color: #008000; font-weight: bold;">return</span>(within.sibships.power)
    } <span style="color: #008000; font-weight: bold;">else</span> {
        stop(paste(<span style="color: #ba2121;">"invalid whichvalue parameter:"</span>,whichpower))
    }

}

<span style="color: #408080; font-style: italic;"># a few tests</span>
qtl.assoc.power(n<span style="color: #666666;">=6000</span>,vq<span style="color: #666666;">=.01</span>,p<span style="color: #666666;">=.01</span>,m1<span style="color: #666666;">=.01</span>,dprime<span style="color: #666666;">=.8</span>,alpha<span style="color: #666666;">=5</span>e<span style="color: #666666;">-08</span>)
qtl.assoc.power(n<span style="color: #666666;">=6000</span>,vq<span style="color: #666666;">=.001</span>,p<span style="color: #666666;">=.01</span>,m1<span style="color: #666666;">=.01</span>,dprime<span style="color: #666666;">=.8</span>,alpha<span style="color: #666666;">=5</span>e<span style="color: #666666;">-08</span>)
qtl.assoc.power(n<span style="color: #666666;">=6000</span>,vq<span style="color: #666666;">=.01</span>,p<span style="color: #666666;">=.01</span>,m1<span style="color: #666666;">=.01</span>,dprime<span style="color: #666666;">=.8</span>,alpha<span style="color: #666666;">=5</span>e<span style="color: #666666;">-08</span>,sibr<span style="color: #666666;">=.5</span>,s<span style="color: #666666;">=2</span>)
qtl.assoc.power(n<span style="color: #666666;">=6000</span>,vq<span style="color: #666666;">=.01</span>,p<span style="color: #666666;">=.01</span>,m1<span style="color: #666666;">=.01</span>,dprime<span style="color: #666666;">=.8</span>,alpha<span style="color: #666666;">=5</span>e<span style="color: #666666;">-08</span>,sibr<span style="color: #666666;">=.5</span>,s<span style="color: #666666;">=2</span>,parent<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">TRUE</span>)
qtl.assoc.power(n<span style="color: #666666;">=6000</span>,vq<span style="color: #666666;">=.01</span>,p<span style="color: #666666;">=.01</span>,m1<span style="color: #666666;">=.01</span>,dprime<span style="color: #666666;">=.8</span>,alpha<span style="color: #666666;">=5</span>e<span style="color: #666666;">-08</span>,sibr<span style="color: #666666;">=.5</span>,s<span style="color: #666666;">=2</span>,parent<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">TRUE</span>,nodom<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">FALSE</span>,da<span style="color: #666666;">=0.5</span>,whichpower<span style="color: #666666;">=</span><span style="color: #ba2121;">"between"</span>)
qtl.assoc.power(n<span style="color: #666666;">=6000</span>,vq<span style="color: #666666;">=.01</span>,p<span style="color: #666666;">=.01</span>,m1<span style="color: #666666;">=.01</span>,dprime<span style="color: #666666;">=.8</span>,alpha<span style="color: #666666;">=5</span>e<span style="color: #666666;">-08</span>,sibr<span style="color: #666666;">=.5</span>,s<span style="color: #666666;">=2</span>,parent<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">TRUE</span>,nodom<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">FALSE</span>,da<span style="color: #666666;">=0.5</span>,whichpower<span style="color: #666666;">=</span><span style="color: #ba2121;">"within"</span>)
qtl.assoc.power(n<span style="color: #666666;">=6000</span>,vq<span style="color: #666666;">=.01</span>,p<span style="color: #666666;">=.01</span>,m1<span style="color: #666666;">=.01</span>,dprime<span style="color: #666666;">=.8</span>,alpha<span style="color: #666666;">=5</span>e<span style="color: #666666;">-08</span>,sibr<span style="color: #666666;">=.5</span>,s<span style="color: #666666;">=2</span>,parent<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">TRUE</span>,nodom<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">FALSE</span>,da<span style="color: #666666;">=0.5</span>,whichpower<span style="color: #666666;">=</span><span style="color: #ba2121;">"overall"</span>)</pre>
</div>
<p>You&#8217;ll notice that the GPC (and thus this code) expects &#8220;Total QTL variance&#8221; as one of its inputs (<code>vq</code>).  This is the proportion of phenotypic variance explained by the putative causal variant (basically an R<sup>2</sup>).  To me, that&#8217;s a bit non-intuitive: I&#8217;d rather think of R<sup>2</sup> as a dependent variable that&#8217;s a function of allele frequency and effect size.  If you&#8217;re assuming normality and expressing effect sizes in standard deviations, then you can calculate vq = es<sup>2</sup>*maf.</p>
<p>To create the &#8220;heat map&#8221; that I wanted, I wrote this code to test a range of effect sizes from .1 to 1 standard deviation and minor allele frequencies from 1% to 10%.  To be conservative, I assume all unrelated individuals; in reality we&#8217;ll have some sibships as well and that will increase power somewhat.</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;"><span style="color: #408080; font-style: italic;"># parameters to run simulation</span>
mafs <span style="color: #666666;">=</span> seq(<span style="color: #666666;">.01</span>,<span style="color: #666666;">.1</span>,<span style="color: #666666;">.01</span>) <span style="color: #408080; font-style: italic;"># minor allele frequencies to test</span>
ess <span style="color: #666666;">=</span> seq(<span style="color: #666666;">.1</span>,<span style="color: #666666;">1</span>,<span style="color: #666666;">.1</span>) <span style="color: #408080; font-style: italic;"># effect sizes to test</span>
N <span style="color: #666666;">=</span> <span style="color: #666666;">6000</span>
dprime <span style="color: #666666;">=</span> <span style="color: #666666;">.8</span>
alpha <span style="color: #666666;">=</span> <span style="color: #666666;">5</span>e<span style="color: #666666;">-08</span> <span style="color: #408080; font-style: italic;"># genome-wide significance</span>
results <span style="color: #666666;">=</span> matrix(nrow<span style="color: #666666;">=</span>length(ess)<span style="color: #666666;">*</span>length(mafs),ncol <span style="color: #666666;">=</span> <span style="color: #666666;">3</span>) <span style="color: #408080; font-style: italic;"># initialize matrix to hold results</span>
counter <span style="color: #666666;">=</span> <span style="color: #666666;">0</span>
<span style="color: #008000; font-weight: bold;">for</span> (es in ess) {
    <span style="color: #008000; font-weight: bold;">for</span> (maf in mafs) {
        counter <span style="color: #666666;">=</span> counter <span style="color: #666666;">+</span> <span style="color: #666666;">1</span>
        vq <span style="color: #666666;">=</span> maf<span style="color: #666666;">*</span>(es<span style="color: #666666;">**2</span>) <span style="color: #408080; font-style: italic;"># calculate variance explained</span>
        results[counter,] <span style="color: #666666;">&lt;-</span> c(maf,es,qtl.assoc.power(n<span style="color: #666666;">=</span>N,vq<span style="color: #666666;">=</span>vq,p<span style="color: #666666;">=</span>maf,m1<span style="color: #666666;">=</span>maf,
            dprime<span style="color: #666666;">=</span>dprime,alpha<span style="color: #666666;">=</span>alpha))
    }
}
colnames(results) <span style="color: #666666;">=</span> c(<span style="color: #ba2121;">"MAF"</span>,<span style="color: #ba2121;">"ES"</span>,<span style="color: #ba2121;">"PWR"</span>) <span style="color: #408080; font-style: italic;"># name columns</span>
rdf <span style="color: #666666;">=</span> data.frame(results) <span style="color: #408080; font-style: italic;"># convert result matrix to data frame</span>
library(reshape) <span style="color: #408080; font-style: italic;"># reshape library lets you cast the results into a matrix for visual inspection </span>
temp <span style="color: #666666;">&lt;-</span> melt.data.frame(rdf,id.vars<span style="color: #666666;">=</span>c(<span style="color: #ba2121;">"MAF"</span>,<span style="color: #ba2121;">"ES"</span>),measure.vars<span style="color: #666666;">=</span>c(<span style="color: #ba2121;">"PWR"</span>)) <span style="color: #408080; font-style: italic;"># melt the data to unique id of (MAF, ES)</span>
cast(temp,ES~MAF,mean) <span style="color: #408080; font-style: italic;"># cast into matrix and display results</span>
k<span style="color: #666666;">=</span> colorRampPalette(c(<span style="color: #ba2121;">"yellow"</span>,<span style="color: #ba2121;">"blue"</span>))(<span style="color: #666666;">100</span>) <span style="color: #408080; font-style: italic;"># generate a yellow-blue color ramp</span>
image(mafs,ess,matrix(rdf$PWR,nrow<span style="color: #666666;">=</span>length(mafs)),col<span style="color: #666666;">=</span>k) <span style="color: #408080; font-style: italic;"># plot maf vs. es vs. power</span></pre>
</div>
<p>It&#8217;s trivial to change the LD assumption; for LD = .8 and N = 6000 as I have above, this is the heat map I get:</p>
<p><a href="/wp-content/uploads/2012/12/gwas.power_.N6000.LD_.8.alpha5e-08.png"><img class="alignnone size-full wp-image-1197" title="GWAS QTL association power heat map for N=6000 LD=0.8 and alpha=5e-08" alt="" src="/wp-content/uploads/2012/12/gwas.power_.N6000.LD_.8.alpha5e-08.png"/></a></p>
<p>With that, I&#8217;m halfway there: now I just need to know the power for extreme phenotype sequencing.</p>
<p><strong>Power for extreme phenotype studies</strong></p>
<p>The premier citation for how to calculate power for an extreme phenotype sequencing study is <a href="http://coruscant.itmat.upenn.edu/pubs/GueyEA_GenEpi_Extremes.pdf">Guey 2011</a>, from Ben Voight&#8217;s group at Broad.  It presents a simple and powerful simulation approach to calculating power to (1) discover a causal variant, and (2) associate a causal variant with a phenotype, under varying assumptions about MAF, effect size, and how extreme of phenotypes you ascertain from how large a population.</p>
<p>The paper considers &#8220;liability&#8221;, i.e. an originally-dichotomous phenotype with the phenotypic contribution of known risk factors subtracted to get a continuously varying phenotype that reflects the individual&#8217;s supposed genetic liability for the disease.  So in that sense it&#8217;s a bit different from my study.  Whereas I am dealing with a purely quantitative phenotype, Guey still has an underlying affected/unaffected status and so can speak in terms of odds ratios.</p>
<p>Puzzled as to how to adapt this approach to my study, I got in touch with <a href="http://coruscant.itmat.upenn.edu/">Ben Voight</a>, who very generously spent some time on the phone with me understanding my study and bouncing some ideas back and forth.</p>
<p>The following pseudocode sums up, at a very loose level, my understanding of how the Guey 2011 paper simulates genotype and phenotype.  (The way I&#8217;m modeling log probability of affected status is probably not exactly right, but you get the general picture):</p>
<pre>for all individuals i:
    genotype[i] = bernoulli_random_variable(p=freq)
    log_probability_of_being_affected[i] = a_priori_disease_prevalence + or_age(age[i]) + or_bmi(bmi[i]) + or_sex(sex[i]) + or_genetic(genotype[i])

phenotype[i] = bernoulli_random_variable(p=exp(log_probability_of_being_affected[i]))

liability[i] = phenotype[i] - age_effect(age[i]) - bmi_effect(bmi[i]) - sex_effect(sex[i])

cases = all i where liability[i] &gt; 98th percentile
controls = all i where liability[i] &lt; 2nd percentile

fisher_exact_test(genotype vs. phenotype, data = controls and cases)</pre>
<p>Notice that liabilities are just used for ascertaining extreme cases and controls; the actual association testing uses <a href="http://udel.edu/~mcdonald/statfishers.html">Fisher&#8217;s exact test</a>.  The odds ratios they refer to in the paper are the <code>or_genetic</code> term, the amount that the genetic risk factor changes the odds of being affected vs. unaffected.  (As opposed to the odds of being extreme case vs. extreme control).  And notice that in this simulation, the randomness (unexplained variance in phenotype) is introduced through the fact that the phenotype is decided by a <a href="http://en.wikipedia.org/wiki/Bernoulli_distribution">Bernoulli</a> coin flip, where the genetic liability only influences the probability.  That corresponds to, in my conception of HD age of onset, the way that I generate an underlying normal distribution and then just spike in the genetic effect.</p>
<p>To adapt this general approach to my residual age of onset quantitative phenotype, I can basically just generate the normally distributed phenotype with spiked genetic effect like I discussed above, then ascertain extreme cases and extreme controls like Guey has done, and simulate doing a <a href="http://udel.edu/~mcdonald/statfishers.html">Fisher</a> or <a href="http://udel.edu/~mcdonald/statchiind.html">χ<sup>2</sup> test</a>.  And because this is sequencing, you don&#8217;t need to worry about <a href="http://en.wikipedia.org/wiki/Linkage_disequilibrium">LD</a> &#8211; just assume if the variant is present in your data you&#8217;ll find the variant itself.</p>
<p>At Ben&#8217;s suggestion I also played with the extra step of modeling effect size in years of life rather than standard deviations.  This is a much more intuitive way to discuss effect size.  Because this requires working in the space of actual ages of onset rather than log ages of onset, and because I wanted to get away from the assumption of normally distributed residuals anyway, I felt it made the most sense to do this based on sampling (using R&#8217;s <code>sample</code>) from my actual data rather than trying to model the distributions of CAG, residuals, etc. none of which are well approximated by canonical distributions (exponential, normal etc).</p>
<p>I actually ran the simulation both ways &#8211; with effect size in standard deviations (of residual log age of onset) while assuming normality, or in years of life while sampling without a normality assumption.  I&#8217;m varying two different things here (the distribution and the way of spiking in genetic effect) so it&#8217;s not a good experiment.  That said, I don&#8217;t think the years of life vs. standard deviations model of effect size makes much of a difference, but I did notice my simulations gave me slightly higher power when I sampled rather than assuming normality.  I believe this is because my actual residuals have positive kurtosis, meaning they&#8217;re more peaked at the mean than a normal distribution, which means that a spiked-in genetic effect is more &#8220;noticeable&#8221; as it pushes an individual towards the edge of the distribution.</p>
<p>After all that, here&#8217;s my implementation of this simulation in R:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;"><span style="color: #408080; font-style: italic;"># N = number of individuals for whom to generate genotype and phenotype</span>
<span style="color: #408080; font-style: italic;"># es = effect size. in standard deviations (if how = "normal") or years of life (if how = "sample")</span>
<span style="color: #408080; font-style: italic;"># maf = minor allele frequency of causal snp</span>
<span style="color: #408080; font-style: italic;"># how = how to generate. "normal" = assume residuals are normally distributed. "sample" = sample from actual data stored as global variables</span>
generate.genotype.and.phenotype <span style="color: #666666;">=</span> <span style="color: #008000; font-weight: bold;">function</span>(N,es,maf,how<span style="color: #666666;">=</span><span style="color: #ba2121;">"normal"</span>) {
    <span style="color: #008000; font-weight: bold;">if</span>(how<span style="color: #666666;">==</span><span style="color: #ba2121;">"normal"</span>) { <span style="color: #408080; font-style: italic;"># assume normally distributed residuals; effect size es is in standard deviations</span>
        allele_count <span style="color: #666666;">=</span> rbinom(n<span style="color: #666666;">=</span>N,size<span style="color: #666666;">=2</span>,prob<span style="color: #666666;">=</span>maf)
        phenotype <span style="color: #666666;">=</span> (rnorm(n<span style="color: #666666;">=</span>N,mean<span style="color: #666666;">=0</span>,sd<span style="color: #666666;">=1</span>) <span style="color: #666666;">+</span> es<span style="color: #666666;">*</span>allele_count) <span style="color: #408080; font-style: italic;"># normal dist with effect from alleles spiked in</span>
        <span style="color: #008000; font-weight: bold;">return</span> (data.frame(<span style="color: #ba2121;">"allele_count"</span><span style="color: #666666;">=</span>allele_count,<span style="color: #ba2121;">"phenotype"</span><span style="color: #666666;">=</span>phenotype))
    } <span style="color: #008000; font-weight: bold;">else</span> <span style="color: #008000; font-weight: bold;">if</span>(how<span style="color: #666666;">==</span><span style="color: #ba2121;">"sample"</span>) { <span style="color: #408080; font-style: italic;"># sample from actual data; effect size es is in years of life</span>
        allele_count <span style="color: #666666;">=</span> rbinom(n<span style="color: #666666;">=</span>N,size<span style="color: #666666;">=2</span>,prob<span style="color: #666666;">=</span>maf)
        cag.sim <span style="color: #666666;">=</span> sample(real.data.cag1,size<span style="color: #666666;">=</span>N,replace<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">TRUE</span>) <span style="color: #408080; font-style: italic;"># sample a distribution of CAG lengths</span>
        res.log.ao.sim <span style="color: #666666;">=</span> sample(real.data.res.log.ao,size<span style="color: #666666;">=</span>N,replace<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">TRUE</span>) <span style="color: #408080; font-style: italic;"># sample a distribution of residuals</span>
        pred.log.ao <span style="color: #666666;">=</span> model.intercept <span style="color: #666666;">+</span> model.coefficient<span style="color: #666666;">*</span>cag.sim <span style="color: #408080; font-style: italic;"># create a predicted log age of onset based on CAG</span>
        actual.ao <span style="color: #666666;">=</span> exp(pred.log.ao<span style="color: #666666;">+</span>res.log.ao.sim) <span style="color: #666666;">+</span> es<span style="color: #666666;">*</span>allele_count <span style="color: #408080; font-style: italic;"># create an actual age of onset based on CAG, residual and genetic effect of causal SNP</span>
        <span style="color: #408080; font-style: italic;"># when simulating using actual data sometimes you get a combination of cag and residual where actual age of onset </span>
        <span style="color: #408080; font-style: italic;"># ends up being is negative or longer than a normal human life span; identify and exclude these outliers so you</span>
        <span style="color: #408080; font-style: italic;"># don't get NA when you take log(actual.ao)</span>
        outliers <span style="color: #666666;">=</span> (actual.ao <span style="color: #666666;">&gt;</span> <span style="color: #666666;">100</span> <span style="color: #666666;">|</span> actual.ao <span style="color: #666666;">&lt;</span> <span style="color: #666666;">2</span>) <span style="color: #408080; font-style: italic;"># otherwise in large samples by chance you get negative or too high</span>
        actual.ao <span style="color: #666666;">=</span> actual.ao[!outliers] <span style="color: #408080; font-style: italic;"># clip outliers out of all these vectors</span>
        pred.log.ao <span style="color: #666666;">=</span> pred.log.ao[!outliers]
        allele_count <span style="color: #666666;">=</span> allele_count[!outliers]
        res.log.ao <span style="color: #666666;">=</span> log(actual.ao) <span style="color: #666666;">-</span> pred.log.ao <span style="color: #408080; font-style: italic;"># create a new residual which includes the genetic effect</span>
        phenotype <span style="color: #666666;">=</span> res.log.ao
        <span style="color: #008000; font-weight: bold;">return</span> (data.frame(<span style="color: #ba2121;">"allele_count"</span><span style="color: #666666;">=</span>allele_count,<span style="color: #ba2121;">"phenotype"</span><span style="color: #666666;">=</span>phenotype))
    }
}

<span style="color: #408080; font-style: italic;"># maf: minor allele frequency of causal SNP</span>
<span style="color: #408080; font-style: italic;"># es: effect size (meaning depends on "how" parameter below)</span>
<span style="color: #408080; font-style: italic;"># Ntot: number of patients from whom you will ascertain cases and controls</span>
<span style="color: #408080; font-style: italic;"># Nseq: number of people you will actually sequence, in *each* group, so total number sequenced is 2*Nseq</span>
<span style="color: #408080; font-style: italic;"># alpha: already-Bonferroni-corrected threshold at which you want significance, e.g. 5e-08</span>
<span style="color: #408080; font-style: italic;"># niter: number of iterations to use to estimate power</span>
<span style="color: #408080; font-style: italic;"># how: how to simulate phenotype and genotype, "normal" = assume residuals are normally distributed. "sample" = sample from actual data stored as global variables</span>
extreme.phenotype.power.sim <span style="color: #666666;">=</span> <span style="color: #008000; font-weight: bold;">function</span> (maf,es,Ntot,Nseq,alpha,niter,how<span style="color: #666666;">=</span><span style="color: #ba2121;">"normal"</span>) {
    numsuccess <span style="color: #666666;">=</span> <span style="color: #666666;">0</span>
    <span style="color: #008000; font-weight: bold;">for</span> (iter in <span style="color: #666666;">1</span>:niter) {
        sim <span style="color: #666666;">=</span> generate.genotype.and.phenotype(N<span style="color: #666666;">=</span>Ntot,es<span style="color: #666666;">=</span>es,maf<span style="color: #666666;">=</span>maf,how<span style="color: #666666;">=</span>how)
        phenotype <span style="color: #666666;">=</span> sim$phenotype
        allele_count <span style="color: #666666;">=</span> sim$allele_count
        <span style="color: #408080; font-style: italic;"># ascertain extreme cases and controls</span>
        excontrols <span style="color: #666666;">=</span> (phenotype <span style="color: #666666;">&gt;=</span> quantile(phenotype,<span style="color: #666666;">1-</span>Nseq<span style="color: #666666;">/</span>Ntot)) <span style="color: #408080; font-style: italic;"># top percentiles</span>
        excases <span style="color: #666666;">=</span> (phenotype <span style="color: #666666;">&lt;=</span> quantile(phenotype,Nseq<span style="color: #666666;">/</span>Ntot)) <span style="color: #408080; font-style: italic;"># bottom percentiles</span>
        <span style="color: #408080; font-style: italic;"># association testing</span>
        contingency_table <span style="color: #666666;">=</span> matrix(c(sum(allele_count[excases]),<span style="color: #666666;">2*</span>sum(excases)<span style="color: #666666;">-</span>sum(allele_count[excases]),sum(allele_count[excontrols]),<span style="color: #666666;">2*</span>sum(excontrols)<span style="color: #666666;">-</span>sum(allele_count[excontrols])),nrow<span style="color: #666666;">=2</span>)
        f <span style="color: #666666;">=</span> fisher.test(contingency_table,alternative<span style="color: #666666;">=</span><span style="color: #ba2121;">"two.sided"</span>)
        p <span style="color: #666666;">=</span> f$p.value
        <span style="color: #008000; font-weight: bold;">if</span> (p <span style="color: #666666;">&lt;</span> alpha) {
            numsuccess <span style="color: #666666;">=</span> numsuccess <span style="color: #666666;">+</span> <span style="color: #666666;">1</span>
        }
    }
    <span style="color: #008000; font-weight: bold;">return</span> (numsuccess<span style="color: #666666;">/</span>niter)
}

<span style="color: #408080; font-style: italic;"># parameters to run simulation</span>
mafs <span style="color: #666666;">=</span> seq(<span style="color: #666666;">.01</span>,<span style="color: #666666;">.1</span>,<span style="color: #666666;">.01</span>) <span style="color: #408080; font-style: italic;"># minor allele frequencies to test</span>
ess <span style="color: #666666;">=</span> seq(<span style="color: #666666;">.1</span>,<span style="color: #666666;">1</span>,<span style="color: #666666;">.1</span>) <span style="color: #408080; font-style: italic;"># effect sizes to test</span>
niter <span style="color: #666666;">=</span> <span style="color: #666666;">100</span>
Ntot <span style="color: #666666;">=</span> <span style="color: #666666;">6000</span>
Nseq <span style="color: #666666;">=</span> <span style="color: #666666;">400</span> <span style="color: #408080; font-style: italic;"># vary this to see effects of choosing larger or smaller number of individuals from same cohort</span>
alpha <span style="color: #666666;">=</span> <span style="color: #666666;">5</span>e<span style="color: #666666;">-08</span> <span style="color: #408080; font-style: italic;"># genome-wide significance</span>
results <span style="color: #666666;">=</span> matrix(nrow<span style="color: #666666;">=</span>length(ess)<span style="color: #666666;">*</span>length(mafs),ncol <span style="color: #666666;">=</span> <span style="color: #666666;">3</span>) <span style="color: #408080; font-style: italic;"># initialize matrix to hold results</span>
counter <span style="color: #666666;">=</span> <span style="color: #666666;">0</span>
<span style="color: #008000; font-weight: bold;">for</span> (es in ess) {
    <span style="color: #008000; font-weight: bold;">for</span> (maf in mafs) {
        counter <span style="color: #666666;">=</span> counter <span style="color: #666666;">+</span> <span style="color: #666666;">1</span>
        results[counter,] <span style="color: #666666;">&lt;-</span> c(maf,es,extreme.phenotype.power.sim(maf,es,Ntot,Nseq,alpha,niter,how<span style="color: #666666;">=</span><span style="color: #ba2121;">"normal"</span>))
    }
}
colnames(results) <span style="color: #666666;">=</span> c(<span style="color: #ba2121;">"MAF"</span>,<span style="color: #ba2121;">"ES"</span>,<span style="color: #ba2121;">"PWR"</span>) <span style="color: #408080; font-style: italic;"># name columns</span>
rdf <span style="color: #666666;">=</span> data.frame(results) <span style="color: #408080; font-style: italic;"># convert result matrix to data frame</span>
library(reshape) <span style="color: #408080; font-style: italic;"># reshape library lets you cast the results into a matrix for visual inspection </span>
temp <span style="color: #666666;">&lt;-</span> melt.data.frame(rdf,id.vars<span style="color: #666666;">=</span>c(<span style="color: #ba2121;">"MAF"</span>,<span style="color: #ba2121;">"ES"</span>),measure.vars<span style="color: #666666;">=</span>c(<span style="color: #ba2121;">"PWR"</span>)) <span style="color: #408080; font-style: italic;"># melt the data to unique id of (MAF, ES)</span>
cast(temp,ES~MAF,mean) <span style="color: #408080; font-style: italic;"># cast into matrix and display results</span>
k<span style="color: #666666;">=</span> colorRampPalette(c(<span style="color: #ba2121;">"yellow"</span>,<span style="color: #ba2121;">"blue"</span>))(<span style="color: #666666;">100</span>) <span style="color: #408080; font-style: italic;"># generate a yellow-blue color ramp</span>
image(mafs,ess,matrix(rdf$PWR,nrow<span style="color: #666666;">=</span>length(mafs)),col<span style="color: #666666;">=</span>k) <span style="color: #408080; font-style: italic;"># plot maf vs. es vs. power</span></pre>
</div>
<p>And here are results for power when you assume normality and ascertain 25, 100 and 400 cases (and equal number of controls) from a cohort of 6000:</p>
<p><a href="/wp-content/uploads/2012/12/exphen.power_.Ntot6000.Nseq25.alpha5e-08.png"><img class="alignnone  wp-image-1191" title="25 extreme cases vs. 25 extreme controls" alt="25 extreme cases vs. 25 extreme controls" src="/wp-content/uploads/2012/12/exphen.power_.Ntot6000.Nseq25.alpha5e-08.png"/></a></p>
<p><a href="/wp-content/uploads/2012/12/exphen.power_.Ntot6000.Nseq100.alpha5e-08.png"><img class="alignnone size-full wp-image-1192" title="100 extreme cases vs. extreme 100 controls" alt="100 extreme cases vs. 100 extreme controls" src="/wp-content/uploads/2012/12/exphen.power_.Ntot6000.Nseq100.alpha5e-08.png"/></a></p>
<p><a href="/wp-content/uploads/2012/12/exphen.power_.Ntot6000.Nseq400.alpha5e-08.png"><img class="alignnone size-full wp-image-1193" title="400 extreme cases vs. 400 extreme controls" alt="400 extreme cases vs. 400 extreme controls" src="/wp-content/uploads/2012/12/exphen.power_.Ntot6000.Nseq400.alpha5e-08.png"/></a></p>
<p>You&#8217;ll notice that there&#8217;s not that big a gain in power going from 100 to 400 people per group.  That&#8217;s because we have a fixed cohort size of 6000, so we are getting less extreme of individuals as we expand our sample size.  But if you increase the cohort size at the same time as you increase the number of individuals sequenced, you&#8217;ll get more and more power.</p>
<p><strong>Comparison of GWAS vs. extreme phenotype exome sequencing</strong></p>
<p>So far I&#8217;ve got heat maps of power for GWAS and for extreme phenotype exome sequencing.  Now to see the difference in power, I just need to run the two under the same assumptions and subtract the power.  If I want to do this with my sampling-and-modeling-effect-size-in-years approach, I need to be able to calculate <code>vq</code> (QTL variance explained) for the given effect size empirically, so I added this function to my quiver:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">sample.variance.explained <span style="color: #666666;">=</span> <span style="color: #008000; font-weight: bold;">function</span>(es,maf) {
    N <span style="color: #666666;">=</span> <span style="color: #666666;">100000</span> <span style="color: #408080; font-style: italic;"># large N to calculate variance explained</span>
    sim <span style="color: #666666;">=</span> generate.genotype.and.phenotype(N,es,maf,how<span style="color: #666666;">=</span><span style="color: #ba2121;">"sample"</span>)
    m <span style="color: #666666;">=</span> lm(sim$phenotype ~ sim$allele_count)
    <span style="color: #008000; font-weight: bold;">return</span> (summary(m)$r.squared)
}</pre>
</div>
<p>With that, I can create a similar heat map where I subtract the two powers to show in what range of MAF and effect size the extreme phenotype study gives a gain in power over the GWAS.  Here&#8217;s code &#8212; this time using sampling and years of effect size, but you can change two lines to <code>how="normal"</code> and <code>es=seq(.1,1,.1)</code> if you want to switch back to &#8220;normal&#8221; mode.</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;"><span style="color: #408080; font-style: italic;"># compare GWAS power to exphen power</span>
<span style="color: #408080; font-style: italic;"># parameters to run simulation</span>
mafs <span style="color: #666666;">=</span> seq(<span style="color: #666666;">.01</span>,<span style="color: #666666;">.1</span>,<span style="color: #666666;">.01</span>) <span style="color: #408080; font-style: italic;"># minor allele frequencies to test</span>
ess <span style="color: #666666;">=</span> seq(<span style="color: #666666;">1</span>,<span style="color: #666666;">13</span>,<span style="color: #666666;">13/10</span>) <span style="color: #408080; font-style: italic;"># effect sizes to test</span>
Ntot <span style="color: #666666;">=</span> <span style="color: #666666;">6000</span>
Nseq <span style="color: #666666;">=</span> <span style="color: #666666;">400</span>
niter <span style="color: #666666;">=</span> <span style="color: #666666;">100</span>
dprime <span style="color: #666666;">=</span> <span style="color: #666666;">.5</span>
alpha <span style="color: #666666;">=</span> <span style="color: #666666;">5</span>e<span style="color: #666666;">-08</span> <span style="color: #408080; font-style: italic;"># genome-wide significance</span>
results <span style="color: #666666;">=</span> matrix(nrow<span style="color: #666666;">=</span>length(ess)<span style="color: #666666;">*</span>length(mafs),ncol <span style="color: #666666;">=</span> <span style="color: #666666;">3</span>) <span style="color: #408080; font-style: italic;"># initialize matrix to hold results</span>
counter <span style="color: #666666;">=</span> <span style="color: #666666;">0</span>
how <span style="color: #666666;">=</span> <span style="color: #ba2121;">"normal"</span> <span style="color: #408080; font-style: italic;"># what model to assume: normality or sampling from </span>
<span style="color: #008000; font-weight: bold;">for</span> (es in ess) {
    <span style="color: #008000; font-weight: bold;">for</span> (maf in mafs) {
        counter <span style="color: #666666;">=</span> counter <span style="color: #666666;">+</span> <span style="color: #666666;">1</span>
        <span style="color: #408080; font-style: italic;"># calculate variance explained</span>
        <span style="color: #008000; font-weight: bold;">if</span> (how <span style="color: #666666;">==</span> <span style="color: #ba2121;">"normal"</span>) { <span style="color: #408080; font-style: italic;"># this is the setting I'm using here</span>
            vq <span style="color: #666666;">=</span> maf<span style="color: #666666;">*</span>(es<span style="color: #666666;">**2</span>) 
        } <span style="color: #008000; font-weight: bold;">else</span> <span style="color: #008000; font-weight: bold;">if</span> (how <span style="color: #666666;">==</span> <span style="color: #ba2121;">"sample"</span>) { <span style="color: #408080; font-style: italic;"># more on this option later</span>
            vq <span style="color: #666666;">=</span> sample.variance.explained(es,maf)
        }
        gwas <span style="color: #666666;">=</span> qtl.assoc.power(n<span style="color: #666666;">=</span>Ntot,vq<span style="color: #666666;">=</span>vq,p<span style="color: #666666;">=</span>maf,m1<span style="color: #666666;">=</span>maf,dprime<span style="color: #666666;">=</span>dprime,alpha<span style="color: #666666;">=</span>alpha)
        exphen <span style="color: #666666;">=</span> extreme.phenotype.power.sim(maf,es,Ntot,Nseq,alpha,niter,how<span style="color: #666666;">=</span><span style="color: #ba2121;">"sample"</span>)
        results[counter,] <span style="color: #666666;">&lt;-</span> c(maf,es,max(exphen<span style="color: #666666;">-</span>gwas,<span style="color: #666666;">0</span>))
    }
}
colnames(results) <span style="color: #666666;">=</span> c(<span style="color: #ba2121;">"MAF"</span>,<span style="color: #ba2121;">"ES"</span>,<span style="color: #ba2121;">"PWR"</span>) <span style="color: #408080; font-style: italic;"># name columns</span>
rdf <span style="color: #666666;">=</span> data.frame(results) <span style="color: #408080; font-style: italic;"># convert result matrix to data frame</span>
library(reshape) <span style="color: #408080; font-style: italic;"># reshape library lets you cast the results into a matrix for visual inspection </span>
temp <span style="color: #666666;">&lt;-</span> melt.data.frame(rdf,id.vars<span style="color: #666666;">=</span>c(<span style="color: #ba2121;">"MAF"</span>,<span style="color: #ba2121;">"ES"</span>),measure.vars<span style="color: #666666;">=</span>c(<span style="color: #ba2121;">"PWR"</span>)) <span style="color: #408080; font-style: italic;"># melt the data to unique id of (MAF, ES)</span>
cast(temp,ES~MAF,mean) <span style="color: #408080; font-style: italic;"># cast into matrix and display results</span>
k<span style="color: #666666;">=</span> colorRampPalette(c(<span style="color: #ba2121;">"yellow"</span>,<span style="color: #ba2121;">"blue"</span>))(<span style="color: #666666;">100</span>) <span style="color: #408080; font-style: italic;"># generate a yellow-blue color ramp</span>
image(mafs,ess,matrix(rdf$PWR,nrow<span style="color: #666666;">=</span>length(mafs)),col<span style="color: #666666;">=</span>k) <span style="color: #408080; font-style: italic;"># plot maf vs. es vs. power gained</span></pre>
</div>
<p>And here are results for LD = 0.5 and number of exomes sequenced = 400.</p>
<p><a href="/wp-content/uploads/2012/12/gwas.v.exphen.Ntot6000.Nseq400.alpha5e-08.LD_.5.cagsim.png"><img class="alignnone size-full wp-image-1198" title="gwas.v.exphen.Ntot6000.Nseq400.alpha5e-08.LD.5.cagsim" alt="" src="/wp-content/uploads/2012/12/gwas.v.exphen.Ntot6000.Nseq400.alpha5e-08.LD_.5.cagsim.png"/></a></p>
<p>You can see an appreciable gain in power for a narrow swath of effect size and MAF combinations.  For instance, you&#8217;re pretty strongly powered for a 1% MAF SNP with an effect size of 8 years in an extreme phenotype study and not at all in a GWAS, according to this model and these assumptions.</p>
<p><strong>Discussion</strong></p>
<p>All of the simulating and plotting above assumes <em>one</em> causal variant.  You could and should ask, what about gene score models?  Doesn&#8217;t exome sequencing offer extra power not captured by the simulation presented here, through the fact that it allows you to group variants by gene and look for gene-wise association?  Yes, it probably does.  I have yet to see any papers in the literature that have quantified <em>how much</em> power is gained; if you the reader know of any, please comment on this post!  It seems pretty muddled: gene score models that assume unidirectional effect (e.g. <a href="/2012/09/25/implementation-of-rvt1-in-r-and-sql/">RVT1 and RVT2</a>) get diluted if the gene also contains variants with the opposite phenotypic effect; gene score models that allow bidirectional effects (e.g. <a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001322">C-Alpha</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3135811/">SKAT</a>) get diluted if the gene also contains neutral variants.  This means that you&#8217;ll have even more assumptions to tweak.</p>
<p>One way of thinking about the gain in power from gene score models is that it just lets you group SNPs up to greater frequency.  So if you&#8217;ve got two causal 1% MAF SNPs, then at best it&#8217;s kind of like having one 2% MAF SNP.  But by the way, this logic works in GWAS too: if there are two rare causal SNPs in the same gene, then as long as they&#8217;re in LD with different marker SNPs or opposite values of the same marker SNP, they&#8217;ll both contribute to the height of that association peak on your <a href="http://en.wikipedia.org/wiki/File:Manhattan_Plot.png">Manhattan plot</a>.</p>
<p>Personally, I&#8217;m not inclined to include any boost in power due to gene score modeling in my power calculation for exome sequencing.  Having been through the process once, it seems to me that gene score models will give pretty dramatically different results depending on the model used, the functional filters applied (e.g. including only missense, nonsense, frameshift and splice sites) and the MAF filters applied (most models call for only including variants below a certain MAF, say, 5%).  And though I&#8217;ve ignored this issue throughout this whole post, exome is exome and GWAS is genome-wide, so this whole comparison assumes that the causal variant is exonic.</p>
<p>You&#8217;ll notice that for the final graphic above, I resorted to the rather conservative assumption of LD = 0.5 and I assumed sequencing 400 people per group for a total of 800 exomes.  I chose those assumptions for this graphic because under many other sets of assumptions, say, LD = 0.8 and number of exomes sequenced = 100, the extreme phenotype study is actually less powered than the GWAS at <em>every</em> effect size and MAF value.</p>
<p>And that&#8217;s where I see the value in this comparison as lying.  People are not going to agree on any one set of assumptions &#8212; will LD between the hypothetical causal SNP and its nearest marker SNP be 0.8 or 0.5? &#8212; but this kind of approach at least lets you see how extreme your assumptions would need to be in order for it to be worth sequencing a given number of exomes.  Through that you can hopefully rule out some possibilities, ex. &#8220;well we&#8217;re definitely not going to bother sequencing just 100 exomes.&#8221;</p>
<p>Of course, there is value to having exomes sequenced besides power to associate variants with phenotype, which is all I&#8217;ve addressed above.  <a href="http://coruscant.itmat.upenn.edu/pubs/GueyEA_GenEpi_Extremes.pdf">Guey 2011</a> also explores &#8220;power to discover&#8221;, i.e. to even happen upon one individual with a given variant and thus know it exists and could be a genetic modifier.  If you read the paper you&#8217;ll see that power to discover was much higher than power to associate.  That could be useful if, say, you want to be able to look under linkage peaks you already <em>know</em> are associated with phenotype, and try to figure out which variant in that region is actually causal.  <strong>update 2012-12-06: </strong>An important point of the Guey&#8217;s paper is that power-to-associate is generally pretty low in these sorts of extreme sequencing studies, so the key to getting value out of your study is figuring out how to transfer the information from sequencing to a larger sample.  One way is to use sequencing to identify candidate genes for followup genotyping; another way is to use <a href="ftp://ftp.sanger.ac.uk/pub4/theses/minichiello/chapter6.pdf">imputation</a>.  (Thanks to Ben Voight for clarifying these points!)</p>
<p>Just as with the assumptions regarding LD and so on, reasonable people will disagree about the value of &#8220;power to discover&#8221; versus &#8220;power to associate.&#8221;  But at a minimum, power calculations should be part of the discussion, and this approach can serve as a decision support tool.</p>
<p><strong>update 2012-12-07:</strong> Thinking about this more and looking at Shaun Purcell&#8217;s code, it occurred to me that GWAS power depends on absolute linkage disequilibrium (D), not normalized linkage disequilibrium (D&#8217;) (see <a href="http://en.wikipedia.org/wiki/Linkage_disequilibrium#Definition">LD definition on Wikipedia</a>).  That means you get greatly reduced power if your causal SNP and marker SNP are of different frequencies.  All my calculations above assume equal frequencies.  How much of a difference does it make?  A lot &#8211; I ran a little &#8220;experiment&#8221; assuming a normally distributed phenotype, an effect size of one standard deviation, and a causal SNP MAF of 1%, varying the marker SNP MAF between 1%, 2%, 5%, and 10%.  Look at what a difference it makes:</p>
<pre>&gt; es = 1 # standard deviations
&gt; maf = .01
&gt; vq = maf*(es**2)
&gt; qtl.assoc.power(n=6000,vq=vq,p=maf,m1=maf,dprime=.8,alpha=5e-08)
[1] 0.7749992
&gt; qtl.assoc.power(n=6000,vq=vq,p=maf,m1=.02,dprime=.8,alpha=5e-08)
[1] 0.1382406
&gt; qtl.assoc.power(n=6000,vq=vq,p=maf,m1=.05,dprime=.8,alpha=5e-08)
[1] 0.003111842
&gt; qtl.assoc.power(n=6000,vq=vq,p=maf,m1=.10,dprime=.8,alpha=5e-08)
[1] 0.0001700689</pre>
<p>When you consider that most SNPs on a SNP array used for GWAS will be relatively common (unless you&#8217;re using <a href="http://genome.sph.umich.edu/wiki/Exome_Chip_Design">the exome chip</a>), this all means that the gain in power from exome sequencing as opposed to GWAS will be much larger for rare variants than I&#8217;ve given it credit for in the brief analysis above.  When doing an actual analysis to inform a decision about what study to devote time and money to, it would probably pay to first get a sense of the range of MAFs represented on the chip you&#8217;re using in GWAS, and then ask questions about power for rare variants based on that knowledge.  <strong>end update</strong></p>
