---
layout: post
title: "Exome sequencing reveals disease modifiers for cystic fibrosis"
author: ericminikel
date: 2012-08-17 00:51:50
---
<p>In <a href="/2012/08/15/qq-plots-with-matplotlib/">yesterday&#8217;s post on QQ plots</a>, I showed a simulated QQ plot from &#8220;the GWAS of your dreams&#8221; in which all observed p values clung tightly to expectation (indicating good, unbiased data) except for a few p values that stood far above the curve (indicating you&#8217;ve discovered genes or variants that actually matter).  Apparently sometimes GWAS dreams do come true:</p>
<p><a href="/wp-content/uploads/2012/08/emond-2012-fig1a.png"><img class="alignnone  wp-image-703" title="emond-2012-fig1a" src="/wp-content/uploads/2012/08/emond-2012-fig1a.png"/></a></p>
<p>That&#8217;s Figure 1a from <a href="http://www.ncbi.nlm.nih.gov/pubmed/22772370">Emond&#8217;s &#8220;Exome sequencing of extreme phenotypes identifies DCTN4 as a modifier of chronic Pseudomonas aeruginosa infection in cystic fibrosis&#8221;</a>, <a href="http://www.nature.com/ng/journal/v44/n8/extref/ng.2344-S1.pdf">[supplement]</a> published in Nature Genetics last month.  The paper is just as stunning as the graphic in its clarity and conclusiveness.  <a href="http://en.wikipedia.org/wiki/Cystic_fibrosis#Chronic_infections">Pseudomonas aeruginosa infection</a> is a major part of the pathophysiology of cystic fibrosis, and Emond discovered that rare variants in DCTN4 are significantly associated with earlier and more chronic infections.  The paper is an excellent tour of the methodology needed for this sort of study, so this post will walk through the &#8220;how-to&#8221;.</p>
<p>In terms of general approach, there are two things really notable about Emond&#8217;s approach as distinct from the other GWAS I&#8217;ve reviewed on this blog:</p>
<ol>
<li>Emond chooses extreme phenotypes (43 early, 48 late onset) to increase statistical power.</li>
<li>Emond looks specifically at rare variants (for the initial screen, those with <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/docs/rs_attributes.html">minor allele frequency</a> less than 12.5%, though she also re-runs the analysis with other thresholds; Fig 1a is based on a 5% threshold).</li>
</ol>
<p>Why rare variants?  Emond explains this choice in the supplement:</p>
<blockquote><p>The general thrust of the recent strategies to analyze rare variants is to exclude common variants from the collapsed (by gene) score given to each individual for each gene in an association analysis. The reasons for this are that common variants can usually be tested for association via a SNP chip based GWAS, as long as the sample size and MAF combination are large enough to achieve reasonable power for the test. Common variants are also more likely to have no association with phenotype, and including such variants in the collapsed score would have a detrimental effect on power to detect any association between the phenotype and the rare variants.</p></blockquote>
<p>Consider also that because common variants are common, in order to detect a causal one in a GWAS you&#8217;d need either a large sample size (larger than the 91 people in this study) <em>or</em> the variant would need to have a very large effect size so that it segregates very cleanly between case and control (ex. at this locus, all 43 early patients have C, all 48 late patients have G).  Despite what <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3400344/">Pasaniuc</a> has to say, a lot of people still think that such common variants are better elucidated through SNP array-based GWAS, while the comparative advantage of exome sequencing in association studies lies in its ability to detect rare variants that don&#8217;t make it onto SNP arrays.</p>
<p>Now, when you&#8217;re looking at rare variants, you can&#8217;t afford to do the analysis at the variant level, because you&#8217;d find a whole lot of variants present in 1 case and 0 controls.  That&#8217;s never going to be significant.  But it could be the case that several of your cases each have a <em>different</em> rare variant in the same gene (or same pathway, etc.) leading to the same (or similar) loss of function and the same phenotype.  So you group the variants by what <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2962811/">Morris and Zeggini 2010</a> call a &#8220;functional unit&#8221; &#8211; most likely a gene, but a pathway could work too.</p>
<p>When you group by gene, there are a couple of different ways to pose your question: &#8220;are cases more likely to have <em>more</em> rare variants in this gene than controls?&#8221;  or &#8220;are cases more likely to have <em>at least one</em> rare variant in this gene than controls?&#8221;.  These, in essence, are models RVT1 and RVT2 (respectively) as introduced by Morris and Zeggini.  These authors run both models on a simulation and find that RVT1, based on &#8220;the proportion of rare variants at which an individual carries minor alleles&#8221;, is more powerful across the board&#8211; even when the phenotype is actually caused by having at least one rare variant, which is the case RVT2 is designed to detect. Therfore &#8220;RVT1 is awesome&#8221; seems to be the take-home message from Morris and Zeggini&#8217;s paper and that&#8217;s the method Emond uses.  It&#8217;s a logistic regression on the proportion of minor alleles to fit a coefficient indicating the effect of that proportion on phenotype.  Here is how Morris and Zeggini introduce it:</p>
<blockquote>
<p id="__p7">Consider a sample of unrelated individuals, phenotyped for a normally distributed trait, and typed for rare variants in a gene or small genomic region. Let <em>n<sub>i</sub></em> denote the number of rare variants for which the<em>i</em>th individual has been successfully genotyped, and let <em>r<sub>i</sub></em> denote the number of these variants at which they carry at least one copy of the minor allele. We can model the phenotype, <em>y<sub>i</sub></em>, of the <em>i</em>th individual in a linear regression framework, given by <em>y<sub>i</sub></em> = E[<em>y<sub>i</sub></em>]+ɛ<sub><em>i</em></sub>, where ɛ<sub><em>i</em></sub>,σ<sub><em>E</em></sub>), and</p>
<div id="__p8">
<div id="m1">
<div><img title="equation image" src="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2962811/bin/gepi0034-0188-m1.jpg" alt="equation image" width="127" height="29"/></div>
</div>
</div>
<p id="__p9">In this expression, <em>x<sub>i</sub></em> denotes a vector of covariate measurements for the <em>i</em>th individual, with corresponding regression coefficients β. The parameter λ is the expected increase in the phenotype for an individual carrying a full complement of minor alleles at rare variants compared to an individual carrying none.</p>
</blockquote>
<p>That&#8217;s basically the model that Emond runs to find DCTN4.  Emond found 11,542 genes meeting their criteria: MAF &lt; .125 and &#8220;the distribution of variants was not collinear with the risk group variable&#8221; (there are two risk categories for cystic fibrosis, so any variants that segregate with the lower or higher risk group would have muddied the results).  So that means 11,542 p values from 11,542 regressions, each performed across the 91 individuals in the study.  The model is just slightly more complicated than the above, by the inclusion of a couple of other controls: Emond also controlled for risk group, and did a <a href="http://en.wikipedia.org/wiki/Principal_component_analysis">principal component analysis</a> to find three principal components associated with ancestry and control for those as well.  After getting the 11,542 p values, Emond applied a <a href="http://en.wikipedia.org/wiki/Bonferroni_correction">Bonferroni correction</a> and found that DCTN4 was the sole significant gene (naive p = 2.2e-6, corrected p = .025).</p>
<p>Inspection of the DCTN4 variants in the samples revealed that there were two different rare variants: of the 43 early onset patients, 9 had a  variant at <a href="http://opensnp.org/snps/rs11954652"> rs11954652</a> (pretty rare) and 3 had a variant at <a href="http://opensnp.org/snps/rs35772018">rs35772018</a> (very rare and highly conserved).</p>
<p>Next, Emond double-checked this result in just about every way imaginable, and it stood up to every test.  It is remarkable how thoroughly this result was validated&#8211; both in additional statistical models and in additional patients&#8211; before it went to publication.  Here are the steps Emond took to validate her finding:</p>
<ul>
<li>Re-ran the RVT1 analysis with only variants with MAF &lt; .05</li>
<li>Ran a SKAT-O test (due to <a href="http://www.ncbi.nlm.nih.gov/pubmed/22863193">Lee, S 2012</a>; see p.8 and p.25 of the <a href="/wp-content/uploads/2012/08/emond-2012-exome-sequencing-of-extreme-phenotypes-supplement.pdf">supplement</a>).  According to Emond, this test:<br/>
<blockquote><p>uses a variance components approach that can capture genes that have variants at different sites that contributing to different extremes of phenotype. (Such variants will make the overall “burden” in the cases and controls look similar and result in a high p-value for the MZ test, which does not account for the direction of effect.)</p></blockquote>
</li>
<li>Ran an EREC test (due to <a href="http://www.sciencedirect.com/science/article/pii/S0002929711003090">Lin &amp; Tang 2011</a>; see p. 9 and p. 25 of the <a href="/wp-content/uploads/2012/08/emond-2012-exome-sequencing-of-extreme-phenotypes-supplement.pdf">supplement</a>).  Like the SKAT-O, this also accounts for genes with multiple variants pushing in opposite phenotypic directions.  Emond explains that it:<br/>
<blockquote><p>uses partially estimated weights for the variants and can also detect genes with different variants that have opposing effects on the phenotype</p></blockquote>
</li>
<li>Ran 10 million MZ <a href="http://en.wikipedia.org/wiki/Bootstrapping_(statistics)">bootstrapping</a> iterations (see p. 10 and p. 25 of the <a href="/wp-content/uploads/2012/08/emond-2012-exome-sequencing-of-extreme-phenotypes-supplement.pdf">supplement</a>).  MZ stands for Morris and Zeggini, but I&#8217;m not clear how MZ bootstrapping is different from regular old bootstrapping.  This is the one thing Emond doesn&#8217;t completely explain, even in the supplement.</li>
<li>Screened DCTN4 by Sanger sequencing (<a href="http://www.bio.davidson.edu/courses/molbio/molstudents/spring2003/obenrader/sanger_method_page.htm">what it is</a>, <a href="http://www.wired.com/wiredscience/2009/01/sanger-sequencing-is-not-dead/">why it&#8217;s still used</a>) in 696 additional individuals with CF.  (Though this appears here as a bulletpoint alongside the additional statistical tests, it is clearly a LOT more important and was also surely the most time-consuming aspect of the validation).</li>
<li>Performed <a href="http://www.statsdirect.com/help/survival_analysis/cox_regression.htm">Cox regression</a> and created <a href="http://www.statsdirect.com/help/survival_analysis/kaplan.htm">Kaplan Meier survival curves</a> (see Fig1b-d and Online Methods in the <a href="/wp-content/uploads/2012/08/emond-2012-exome-sequencing-of-extreme-phenotypes.pdf">main paper</a>) stratified on the age that the patients were each enrolled in the study, to rule out as a source of error the fact that someone&#8217;s P. aeruginosa infection status might affect their enrollment status.</li>
<li>Re-ran RVT1 on only the patients of European ancestry (whom they identified using principal component analysis rather than self-identified ethnic group; see p. 16-17 of the <a href="/wp-content/uploads/2012/08/emond-2012-exome-sequencing-of-extreme-phenotypes-supplement.pdf">supplement</a>) in order to rule out bias from different allele frequencies in different ethnic groups.</li>
<li>Created Kaplan Meier survival curves separately for each genotype (the two variants, hetero- or homozygous) to determine the relative severity of phenotype associated with each genotype as well as for mucoid infection (as opposed to age of infection) as the phenotype.</li>
</ul>
<p>Emond also quality controlled on the front end, with the usual list of measures&#8211; throwing out variants with low call quality (coverage &lt;5) , far out of Hardy-Weinberg equilibrium, etc. (it&#8217;s all in the online methods). And to top it all off, the paper concludes with an excellent discussion of the biological relevance of Dynactin 4, the protein encoded by DCTN4: evidently it&#8217;s got a well-established role in autophagy, and autophagy has got a well-established role in clearing P. aeruoginosa infections.</p>
<p>In the conclusion, Emond writes:</p>
<blockquote><p>To our knowledge, this is the first study to discover a gene for a complex trait, or at minimum a genetic modifier of a Mendelian trait, using exome sequencing and an extreme phenotype study design.</p></blockquote>
<p>Let&#8217;s hope it&#8217;s the first of many.</p>
