---
layout: post
title: "Getting started with analyzing sequence data"
author: ericminikel
date: 2012-08-13 15:28:04
---
<p>If you order <a href="/2012/08/02/exome-sequencing-and-analysis/">exome sequencing</a> from Illumina or the like, they&#8217;ll provide you with both the raw <a href="http://en.wikipedia.org/wiki/Shotgun_sequencing#Example">reads</a> and the finished, aligned sequence.  But in the words of one investigator at my institution, &#8220;no one who has the capability to do their own alignment trusts someone else to do it.&#8221;  The perception is that the company that does your sequencing may not have as high of standards for alignment quality as you do, may not tell you the details of the algorithm they use, and/or may not provide the particular descriptive statistics you need regarding the quality of  your data.</p>
<p>Thus many researchers find themselves in the position of doing alignments themselves.  This is a task I learned about, and implemented parts of, in a bioinformatics class but had never had to actually do for anything that mattered.  I&#8217;m getting oriented on it today and sharing what I learn here.</p>
<p>The most popular algorithm today for doing alignments is the <a href="http://bio-bwa.sourceforge.net/">Burrows-Wheeler Alignment</a>.  (Among other things, this approach takes advantage of a bit of wizardry called the <a href="http://en.wikipedia.org/wiki/Burrows%E2%80%93Wheeler_transform">Burrows-Wheeler Transform</a> which is an entertaining wiki read).  It takes in a reference sequence (such as the <a href="http://hgdownload.cse.ucsc.edu/downloads.html#human">human genome</a>*) in <a href="http://en.wikipedia.org/wiki/FASTA_format">FASTA</a> format and raw reads in <a href="http://en.wikipedia.org/wiki/FASTQ_format">FASTQ</a> format (basically FASTA + call quality scores) and outputs an alignment in <a href="http://samtools.sourceforge.net/SAM1.pdf">SAM</a> format which you can then manipulate using <a href="http://samtools.sourceforge.net/">SAMtools</a>.  BWA is open source, <a href="http://bio-bwa.sourceforge.net/bwa.shtml">well documented</a> and built for the *nix command line.  SEQwiki provides <a href="http://seqanswers.com/wiki/How-to/exome_analysis">an excellent how-to</a> for using these tools for exome sequencing analysis.  UMich also provides a useful <a href="http://genome.sph.umich.edu/wiki/Tutorial:_Low_Pass_Sequence_Analysis">tutorial on low pass sequence analysis</a>.</p>
<p>*UCSC provides the reference genome version 19, <a href="http://hgdownload.cse.ucsc.edu/downloads.html#human">hg19</a>, in a compressed file chromFa.tar.gz with the chromosomes (e.g. chr1.fa) as well as several  other files called things like chr4_gl000194_random.fa.  What are those?  Turns out they are contigs that couldn&#8217;t be confidently aligned to a particular location in the chromosome (or genome).  Including them tends to reduce mapping quality so people often leave them out unless they&#8217;re particularly interested in them for some reason.  And by the way, for BWA you need all the chromosomes in a single FASTA file.  <a href="http://seqanswers.com/wiki/How-to/exome_analysis">SeqAnswers</a> gives a bash command to do that (in the correct order so it matches <a href="http://www.broadinstitute.org/gatk/">GATK</a>):</p>
<pre>cat chr1.fa chr2.fa chr3.fa chr4.fa chr5.fa chr6.fa chr7.fa chr8.fa chr9.fa
chr10.fa chr11.fa chr12.fa chr13.fa chr14.fa chr15.fa chr16.fa chr17.fa chr18.fa
chr19.fa chr20.fa chr21.fa chr22.fa chrX.fa chrY.fa chrM.fa &gt; hg19.fa</pre>
<p>Once you&#8217;ve done your alignment and gotten the quality scores you want, odds are you&#8217;ll want to call variants.  &#8221;Call&#8221; here just means to recognize the variants present in your sequence data, so that you go from just having a sequence to having a tabular list of variants (relative to the reference sequence) found in it.   More on that later.</p>
