---
layout: post
title: "Population Substructure, Part II"
author: ericminikel
date: 2012-10-25 21:29:03
---
<p>This post is a sequel to <a href="/2012/10/15/population-covariates-using-1000-genomes/">How (and why) to create population covariates using 1000 Genomes data</a>.  Here are four more things you can and probably should do in the population substructure analysis started in that post.</p>
<p><strong>1. Update on MDS plotting: use only common variants to avoid bias from which rare variants are included</strong></p>
<p>In the previous post I described a method for combining your data with 1000 Genomes and then computing IBD on the whole set in order to generate an MDS plot. One problem I&#8217;ve discovered with this approach is as follows. The method I described will calculate IBD based on all variants that were called in 1000 Genomes and in your samples. My samples are much less numerous&#8211; I have 50 of my own samples and 629 1000 Genomes samples&#8211; and this introduces bias regarding rare variants. In particular, any variant will be included if it shows up as at least a singleton in both datasets. This means it must have allele frequency of 2% in my data and 0.15% in 1000 Genomes. This filter enriches the resulting list of SNPs for my own samples&#8217; rare variation, comparatively depleting it for 1000 Genomes rare variation. (A hypothetical variant with 1% frequency in 1000 Genomes but not seen in my data would not be included).</p>
<p>When I originally plotted my data with 1000 Genomes, my samples clustered with each other but did not overlap with the 1000 Genomes clusters. At first I thought this might just indicate my samples came from populations not represented in 1000 Genomes (which seems plausible&#8211; 1000 Genomes still has no Southern or Central Africans, Pacific Islanders, Arabs, Central Asians, the list goes on) but when I plotted SNP array data for some of the same individuals with 1000 Genomes (see step 2 below) I found that they clustered tightly with 1000 Genomes populations. Hmmm. After thinking about it more, I realized that the enrichment of my samples&#8217; rare variation makes my samples <em>look more different from 1000 Genomes than they actually are</em>. This causes my samples to cluster just a little ways away from the 1000 Genomes population that they really ought to belong to.  So I applied a MAF filter:</p>
<pre>plink --file variants.subset --freq --out variants # calculate allele frequency based on my variants
plink --file all --read-freq variants.frq --maf .10 --all --genome --out all_maf10 # calculate genome file for MAF &gt; 10% in my samples
plink --file all --read-genome all_maf10.genome --cluster --mds-plot 2 --out all_mds_2_maf10 # calculate MDS plot based on only the MAF &gt; 10% variants</pre>
<p>And immediately, my samples clustered tightly with various 1000 Genomes populations and matched almost exactly the pattern seen in the SNP array data. Problem solved.</p>
<p><strong>2. Validation with data from other studies</strong></p>
<p>Some of my samples had been previously genotyped in a SNP array study, which provided the chance to validate that the variants I called with the <a href="/2012/09/19/exome-sequencing-pipeline-using-gatk/">GATK pipeline</a> on exome data. Because the two studies used different sample IDs, I needed a match table. We had a match table lying around in some master Excel file of all samples we&#8217;ve genotyped, but just in case that file were to have any errors in it, I deemed it safest to create my own match table based on the actual genetic similarity between samples.</p>
<p>First, I extracted the relevant SNPs:</p>
<pre>grep -o 'rs[0-9]*' original.snparray.data.map | sort | uniq &gt; snparray.snplist.dedup
grep -o 'rs[0-9]*' exome.data.map | sort | uniq &gt; exome.snplist.dedup
cat exome.snplist.dedup snparray.snplist.dedup | sort | uniq -d &gt; shared.snplist</pre>
<p>And after trying once to combine the files I got a PLINK error as usual and a shared.missnp output file, so then I excluded those SNPs on the second try:</p>
<pre>plink --file original.snparray.data --extract shared.snplist --exclude shared.missnp --recode --out snparray.subset
plink --tfile exome.data --extract shared.snplist --exclude shared.missnp --recode --out exome.subset
plink --file snparray.subset --merge exome.subset.ped exome.subset.map --recode --out snparray-exome
plink --file snparray-exome --all --genome --out snparray-exome
sort snparray-exome.genome -k10 -r &gt; snparray-exome.genome.sorted # sort on k10, the pi_hat field</pre>
<p>This file then shows the pairs with highest pi_hat, a measure of relatedness.  I opened it up and sure enough, the top hits were SNP array samples matching to the same individual&#8217;s exome data, with pi_hat of .98 or better.  I created a match table based on this information.  At first I tried to just write out a .TFAM file directly from PostgreSQL, but for yet-undiagnosed reasons that didn&#8217;t work with PLINK, so finally I wrote out the match table named <code>recoded.txt</code> and followed the PLINK documentation for <a href="http://pngu.mgh.harvard.edu/~purcell/plink/dataman.shtml#updatefam">updating individual information</a> with the <code>--update-ids</code> command:</p>
<pre>plink --file original.snparray.data --update-ids recoded.txt --extract shared.snplist --exclude shared.missnp --recode --out snparray.subset.with.exome.ids</pre>
<p>And then compared the two datasets in <a href="http://pngu.mgh.harvard.edu/~purcell/plink/dataman.shtml#merge">merge mode 7</a>, so that PLINK would report the discrepancies between the two datasets only where the sample was genotyped in both datasets:</p>
<pre>plink --file snparray.subset.with.exome.ids --merge exome.subset.ped exome.subset.map --merge-mode 7 --out snparray.exome</pre>
<p>This writes out a file, <code>snparray.exome.diff</code>, listing all the sample-site combinations which were genotyped in both datasets and where the genotypes do not agree.  The log file also tells you the overall concordance rate:</p>
<pre>Of 174260 overlapping SNPs, 145580 were both genotyped
and 145259 were concordant
Concordance rate is 0.997795</pre>
<p>This exome data matched the SNP array data about 99.8%.  What&#8217;s nice is, I was able to do this with two different exome sequencing outputs (at this point I&#8217;ve run the GATK pipeline several times, with different combinations of settings, reference genomes, pre-alignment trimming of reads or not) and see which accorded most closely with the SNP array data.  Because this particular SNP array was genome-wide, there is only a small overlap (about 8500 SNPs) with the exonic variants I called in my exome sequencing, but still, arriving at the same answer through two completely different genotyping technologies definitely helps to build confidence in the data.</p>
<p><strong>3. Detecting familial relationships &#8211; continued</strong></p>
<p>In the <a href="/2012/10/15/population-covariates-using-1000-genomes/">earlier post</a> I mentioned the need to calculate IBD on a homogenous sample in order to reliably detect familial relationships among your samples. All of my samples clustered pretty clearly with either the European, Asian or African cluster from 1000 Genomes, so I coarsely sectioned them into those three groups in order to re-calculate the <code>.genome</code> files within these groups:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;"><span style="color: #008000; font-weight: bold;">copy</span> (
<span style="color: #008000; font-weight: bold;">select</span>   m.fid, m.iid
<span style="color: #008000; font-weight: bold;">from</span>     all_mds_2_maf10 m <span style="color: #008000; font-weight: bold;">left</span> <span style="color: #008000; font-weight: bold;">outer</span> <span style="color: #008000; font-weight: bold;">join</span> kg_metadata k
<span style="color: #008000; font-weight: bold;">on</span>       m.iid <span style="color: #666666;">=</span> k.coriell_sample_id
<span style="color: #008000; font-weight: bold;">where</span>    k.population <span style="color: #008000; font-weight: bold;">in</span> (<span style="color: #ba2121;">'GBR'</span>,<span style="color: #ba2121;">'CEU'</span>,<span style="color: #ba2121;">'FIN'</span>,<span style="color: #ba2121;">'TSI'</span>)
<span style="color: #008000; font-weight: bold;">or</span>       (m.iid <span style="color: #666666;">~</span> <span style="color: #ba2121;">'[0-9]*'</span> <span style="color: #008000; font-weight: bold;">and</span> c1::<span style="color: #008000;">numeric</span> <span style="color: #666666;">&lt;</span> <span style="color: #666666;">0</span> <span style="color: #008000; font-weight: bold;">and</span> c2::<span style="color: #008000;">numeric</span> <span style="color: #666666;">&gt;</span> <span style="color: #666666;">0</span>)
) <span style="color: #008000; font-weight: bold;">to</span> <span style="color: #ba2121;">'/your/path/here/euro_list.txt'</span>;

<span style="color: #008000; font-weight: bold;">copy</span> (
<span style="color: #008000; font-weight: bold;">select</span>   m.fid, m.iid
<span style="color: #008000; font-weight: bold;">from</span>     all_mds_2_maf10 m <span style="color: #008000; font-weight: bold;">left</span> <span style="color: #008000; font-weight: bold;">outer</span> <span style="color: #008000; font-weight: bold;">join</span> kg_metadata k
<span style="color: #008000; font-weight: bold;">on</span>       m.iid <span style="color: #666666;">=</span> k.coriell_sample_id
<span style="color: #008000; font-weight: bold;">where</span>    k.population <span style="color: #008000; font-weight: bold;">in</span> (<span style="color: #ba2121;">'CHS'</span>,<span style="color: #ba2121;">'CHB'</span>,<span style="color: #ba2121;">'JPT'</span>)
<span style="color: #008000; font-weight: bold;">or</span>       (m.iid <span style="color: #666666;">~</span> <span style="color: #ba2121;">'[0-9]*'</span> <span style="color: #008000; font-weight: bold;">and</span> c1::<span style="color: #008000;">numeric</span> <span style="color: #666666;">&lt;</span> <span style="color: #666666;">0</span> <span style="color: #008000; font-weight: bold;">and</span> c2::<span style="color: #008000;">numeric</span> <span style="color: #666666;">&lt;</span> <span style="color: #666666;">0</span>)
) <span style="color: #008000; font-weight: bold;">to</span> <span style="color: #ba2121;">'your/path/here/asian_list.txt'</span>;

<span style="color: #008000; font-weight: bold;">copy</span> (
<span style="color: #008000; font-weight: bold;">select</span>   m.fid, m.iid
<span style="color: #008000; font-weight: bold;">from</span>     all_mds_2_maf10 m <span style="color: #008000; font-weight: bold;">left</span> <span style="color: #008000; font-weight: bold;">outer</span> <span style="color: #008000; font-weight: bold;">join</span> kg_metadata k
<span style="color: #008000; font-weight: bold;">on</span>       m.iid <span style="color: #666666;">=</span> k.coriell_sample_id
<span style="color: #008000; font-weight: bold;">where</span>    k.population <span style="color: #008000; font-weight: bold;">in</span> (<span style="color: #ba2121;">'ASW'</span>,<span style="color: #ba2121;">'LWK'</span>,<span style="color: #ba2121;">'YRI'</span>)
<span style="color: #008000; font-weight: bold;">or</span>       (m.iid <span style="color: #666666;">~</span> <span style="color: #ba2121;">'[0-9]*'</span> <span style="color: #008000; font-weight: bold;">and</span> c1::<span style="color: #008000;">numeric</span> <span style="color: #666666;">&gt;</span> <span style="color: #666666;">0</span>)
) <span style="color: #008000; font-weight: bold;">to</span> <span style="color: #ba2121;">'your/path/here/african_list.txt'</span>;</pre>
</div>
<p>And, conscious of the bias introduced by rare variation as described in point 1 above, I filtered to only use variants of MAF 10% or higher in the genome calculations:</p>
<pre>plink --file all --maf .10 --genome --keep african_list.txt --out africans
plink --file all --maf .10 --genome --keep asian_list.txt --out asians
plink --file all --maf .10 --genome --keep euro_list.txt --out europeans</pre>
<p>Then I read the resulting <code>.genome</code> files into PostgreSQL and compiled them into one table:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;"><span style="color: #008000; font-weight: bold;">drop</span> <span style="color: #008000; font-weight: bold;">table</span> if <span style="color: #008000; font-weight: bold;">exists</span> genome;
<span style="color: #008000; font-weight: bold;">create</span> <span style="color: #008000; font-weight: bold;">table</span> genome <span style="color: #008000; font-weight: bold;">as</span> <span style="color: #008000; font-weight: bold;">select</span> <span style="color: #666666;">*</span>, <span style="color: #ba2121;">'africans'</span>::<span style="color: #008000;">varchar</span> <span style="color: #008000; font-weight: bold;">as</span> population <span style="color: #008000; font-weight: bold;">from</span> africans_genome;
<span style="color: #008000; font-weight: bold;">insert</span> <span style="color: #008000; font-weight: bold;">into</span> genome <span style="color: #008000; font-weight: bold;">select</span> <span style="color: #666666;">*</span>, <span style="color: #ba2121;">'asians'</span>::<span style="color: #008000;">varchar</span> <span style="color: #008000; font-weight: bold;">as</span> population <span style="color: #008000; font-weight: bold;">from</span> asians_genome;
<span style="color: #008000; font-weight: bold;">insert</span> <span style="color: #008000; font-weight: bold;">into</span> genome <span style="color: #008000; font-weight: bold;">select</span> <span style="color: #666666;">*</span>, <span style="color: #ba2121;">'europeans'</span>::<span style="color: #008000;">varchar</span> <span style="color: #008000; font-weight: bold;">as</span> population <span style="color: #008000; font-weight: bold;">from</span> europeans_genome;</pre>
</div>
<p>And then looked to see if my genome calculations had validated the known relationships noted in the 1000G metadata:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;"><span style="color: #008000; font-weight: bold;">select</span>   <span style="color: #008000; font-weight: bold;">g</span>.iid1, <span style="color: #008000; font-weight: bold;">g</span>.iid2, <span style="color: #008000; font-weight: bold;">g</span>.z0, <span style="color: #008000; font-weight: bold;">g</span>.z1, <span style="color: #008000; font-weight: bold;">g</span>.z2, <span style="color: #008000; font-weight: bold;">g</span>.pi_hat, k1.population, k2.population, k1.family, k1.type__unrel_duo_trio_, k1.relationship_, k2.family, k2.type__unrel_duo_trio_, k2.relationship_
<span style="color: #008000; font-weight: bold;">from</span>     genome <span style="color: #008000; font-weight: bold;">g</span> <span style="color: #008000; font-weight: bold;">left</span> <span style="color: #008000; font-weight: bold;">outer</span> <span style="color: #008000; font-weight: bold;">join</span> kg_metadata k1 <span style="color: #008000; font-weight: bold;">on</span> <span style="color: #008000; font-weight: bold;">g</span>.iid1 <span style="color: #666666;">=</span> k1.coriell_sample_id <span style="color: #008000; font-weight: bold;">left</span> <span style="color: #008000; font-weight: bold;">outer</span> <span style="color: #008000; font-weight: bold;">join</span> kg_metadata k2 <span style="color: #008000; font-weight: bold;">on</span> <span style="color: #008000; font-weight: bold;">g</span>.iid2 <span style="color: #666666;">=</span> k2.coriell_sample_id
<span style="color: #008000; font-weight: bold;">where</span>    pi_hat <span style="color: #666666;">&gt;</span> .<span style="color: #666666;">125</span> <span style="color: #408080; font-style: italic;">-- arbitrary cutoff to find high pi_hat</span>
<span style="color: #008000; font-weight: bold;">order</span> <span style="color: #008000; font-weight: bold;">by</span> pi_hat <span style="color: #008000; font-weight: bold;">desc</span>;</pre>
</div>
<p>And indeed, two known relationships validated quite well (I&#8217;ve bolded the z0, z1 and z2 columns):</p>
<pre>"HG00153";"HG00158";<strong>0.0989;0.8867;0.0144</strong>;0.4578;"GBR";"GBR";"";"duo";"Child";"";"duo";"Mother"
"HG00146";"HG00147";<strong>0.3051;0.4758;0.2191</strong>;0.4570;"GBR";"GBR";"";"duo";"Sibling";"";"duo";"Sibling"</pre>
<p>And I also discovered strong evidence of other relationships not annotated in the 1000G metadata:</p>
<pre>"HG00578";"HG00635";<strong>0.2471;0.5186;0.2344</strong>;0.4937;"CHS";"CHS";"SH052";"trio";"mother";"SH071";"trio";"mother"
"NA20526";"NA20792";<strong>0.3217;0.4235;0.2547</strong>;0.4665;"TSI";"TSI";"";"unrel";"";"";"unrel";""
"NA19381";"NA19382";<strong>0.1211;0.8457;0.0332</strong>;0.4561;"LWK";"LWK";"";"unrel";"";"";"unrel";""
"NA19445";"NA19453";<strong>0.1183;0.8817;0.0000</strong>;0.4409;"LWK";"LWK";"";"unrel";"";"";"unrel";""
"NA19434";"NA19444";<strong>0.3957;0.4234;0.1809</strong>;0.3926;"LWK";"LWK";"";"unrel";"";"";"unrel";""</pre>
<p>HG00578 and HG00635, two mothers from different CHS (Southern Chinese) trios, appear to be sisters. There&#8217;s almost a perfect 25-50-25 split on IBD 0/1/2. Two Tuscans (NA20526 and NA20792) and two Luhya (NA19434 and NA19444) also looked pretty strongly to be siblings, and two other pairs of Luhya people (NA19381 &amp; NA19382 and NA19445 &amp; NA19453) looked to be parent-child.</p>
<p>A Google search confirmed that I&#8217;m not the only one who has noticed these pairs. Jim Nemesh did a study of cryptic relatedness in the 1000G data summarized in <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/supporting/cryptic_relation_analysis/">documents now hosted at EBI</a>. His <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/supporting/cryptic_relation_analysis/20120220_cryptic_relations_summary.txt">cryptic relatedness summary</a> reports the Tuscan pair, NA20526 and NA20792 as siblings, and his <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/supporting/cryptic_relation_analysis/20120220_cryptic_relations_clusters.txt">cryptic relations clusters report</a> reports the other four relationships I listed above. The analysis is described further in the <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/supporting/cryptic_relation_analysis/README_cryptic_relations_20120220">readme</a> and <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/supporting/cryptic_relation_analysis/Nemesh_crypticrelatedness_20120213.pptx">powerpoint</a>.</p>
<p>That&#8217;s about all I was able to validate, because no other duos or trios are complete in the August 2010 1000 Genomes release. It took a while to figure this out because the metadata is not totally consistent yet&#8211; you&#8217;ll note that the mother/child and sibling/sibling duos validated above do not have the <code>family</code> field filled in, and the <code>relationship_</code> field is sometimes capitalized and sometimes not. But if you check:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;"><span style="color: #008000; font-weight: bold;">select</span>   <span style="color: #666666;">*</span>
<span style="color: #008000; font-weight: bold;">from</span>     kg_metadata
<span style="color: #008000; font-weight: bold;">where</span>    relationship_ <span style="color: #008000; font-weight: bold;">in</span> (<span style="color: #ba2121;">'child'</span>,<span style="color: #ba2121;">'Child'</span>)
<span style="color: #008000; font-weight: bold;">and</span>      coriell_sample_id <span style="color: #008000; font-weight: bold;">in</span> (<span style="color: #008000; font-weight: bold;">select</span> iid1 <span style="color: #008000; font-weight: bold;">from</span> genome);</pre>
</div>
<p>You&#8217;ll find only three &#8216;child&#8217; individuals, and on further investigation, two do not have parents in this data release. And if you look for <code>where relationship_ in ('sibling','Sibling')</code>, the sibling due validated above are the only one. So indeed, my analysis successfully validated all of the known immediate family relationships that were present in the August 2010 1000 Genomes release. This gives me confidence that my genome calculations will be accurate with respect to any relationships among my own samples.</p>
<p>Important note: because the <code>family</code> field is not yet consistently used and most trios are not complete, if you throw in a <code>where k1.family = k2.family</code> condition, you&#8217;ll find just a list of people&#8217;s grandparents and parents i.e. people who are not immediately blood-related to each other, which, not surprisingly when you think about it, often appear distantly related to one another, with z1+z2 of ~4%. Leave this condition out in order to find the parent/child and sibling/sibling relations mentioned above.</p>
<p>Troublingly, I found a lot of cryptic relatedness&#8211; tens of pairs of 1000 Genomes people who aren&#8217;t labeled as related to each other have about 40% IBD-1, and a couple of individuals in particular had 60% IBD-1 with many people.  Some of this may be an indicator that my European/Asian/African groupings were a bit on the coarse side, causing people from the same subpopulation to look excessively related to each other. But Jim Nemesh&#8217;s <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/supporting/cryptic_relation_analysis/">powerpoint on cryptic relatedness in 1000G</a> finds plenty of such combinations as well, supporting the idea that this is really just a feature of the data. The plot on slide 4 shows plenty of pairs of people in the z1 20%-50% range. Nemesh also notes that some particular samples appear to be contaminated with other people&#8217;s DNA and thus exhibit elevated heterozygosity and cryptic relatedness to a large number of individuals.  I&#8217;ll return to this point later.</p>
<p>Although it&#8217;s not perfect, for now I&#8217;m satisfied that my analysis validated the annotated (and unannotated but noticed-by-others) relationships in 1000 Genomes, and therefore is likely to discover any relationships among my own samples.</p>
<p><strong>4. Sex validation</strong></p>
<p>While I was comparing my data to the SNP array data in Step 2, I noticed that the pedigree files disagreed about the sex of one person. I decided it would be worthwhile to validate the sex of all individuals in my study based on the genetic data itself. I wrote a simple query to get the X het/hom ratio (an indicator of femaleness) and number of Y variants called (an indicator of maleness) for all of my samples. You might expect X heterozygosity to be 0 for men, and Y variants to be 0 for women, but in practice the <a href="http://en.wikipedia.org/wiki/Pseudoautosomal_region">pseudoautosomal region</a> muddles this a bit. This region is homologous on X and Y, and so reads for women will often align to Y and reads from men&#8217;s Y chromosomes will often align to X. Still, these measures create pretty clear clusters of men and women.</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;"><span style="color: #008000; font-weight: bold;">select</span>   s.sid,
         s.sex, <span style="color: #408080; font-style: italic;">-- nominal sex as annotated in my metadata</span>
<span style="color: #008000; font-weight: bold;"> sum</span>(<span style="color: #008000; font-weight: bold;">case</span> <span style="color: #008000; font-weight: bold;">when</span> chrom <span style="color: #666666;">=</span> <span style="color: #ba2121;">'X'</span> <span style="color: #008000; font-weight: bold;">and</span> variant_allele_count<span style="color: #666666;">=1</span> <span style="color: #008000; font-weight: bold;">then</span> <span style="color: #666666;">1</span>.<span style="color: #666666;">0</span> <span style="color: #008000; font-weight: bold;">else</span> <span style="color: #666666;">0</span>.<span style="color: #666666;">0</span> <span style="color: #008000; font-weight: bold;">end</span>)<span style="color: #666666;">/</span><span style="color: #008000; font-weight: bold;">sum</span>(<span style="color: #008000; font-weight: bold;">case</span> <span style="color: #008000; font-weight: bold;">when</span> chrom <span style="color: #666666;">=</span> <span style="color: #ba2121;">'X'</span> <span style="color: #008000; font-weight: bold;">and</span> variant_allele_count <span style="color: #666666;">=2</span> <span style="color: #008000; font-weight: bold;">then</span> <span style="color: #666666;">1</span>.<span style="color: #666666;">0</span> <span style="color: #008000; font-weight: bold;">else</span> <span style="color: #666666;">0</span>.<span style="color: #666666;">00001</span> <span style="color: #008000; font-weight: bold;">end</span>) x_het_hom_ratio,
<span style="color: #008000; font-weight: bold;"> sum</span>(<span style="color: #008000; font-weight: bold;">case</span> <span style="color: #008000; font-weight: bold;">when</span> chrom<span style="color: #666666;">=</span><span style="color: #ba2121;">'Y'</span> <span style="color: #008000; font-weight: bold;">and</span> sv.dp <span style="color: #666666;">&lt;&gt;</span> <span style="color: #ba2121;">'NA'</span> <span style="color: #008000; font-weight: bold;">then</span> <span style="color: #666666;">1</span>.<span style="color: #666666;">0</span> <span style="color: #008000; font-weight: bold;">else</span> <span style="color: #666666;">0</span>.<span style="color: #666666;">0</span> <span style="color: #008000; font-weight: bold;">end</span>) y_vars
<span style="color: #008000; font-weight: bold;">from</span>     sample_variants sv, variants v, samples s
<span style="color: #008000; font-weight: bold;">where</span>    sv.vid <span style="color: #666666;">=</span> v.vid <span style="color: #008000; font-weight: bold;">and</span> s.sid <span style="color: #666666;">=</span> sv.sid <span style="color: #008000; font-weight: bold;">and</span> chrom <span style="color: #008000; font-weight: bold;">in</span> (<span style="color: #ba2121;">'X'</span>,<span style="color: #ba2121;">'Y'</span>)
<span style="color: #008000; font-weight: bold;">and</span>      sv.dp <span style="color: #666666;">&lt;&gt;</span> <span style="color: #ba2121;">'NA'</span>
<span style="color: #008000; font-weight: bold;">group</span> <span style="color: #008000; font-weight: bold;">by</span> s.sid, s.sex
<span style="color: #008000; font-weight: bold;">order</span> <span style="color: #008000; font-weight: bold;">by</span> <span style="color: #666666;">3</span>,<span style="color: #666666;">4</span>
;</pre>
</div>
<p>When I plotted these data, I found two really clear clusters: men with ~35 Y variants and an X het/hom ratio of about 0.5, and women with ~9 Y variants called and X het/hom ratio in the range of 1.3 to 2.2. But there was one outlier, a sample with 35 Y variants and an X het/hom ratio of 1.7.  Here is some data I generated in R that looks about like my real data did (for various reasons it would be uncouth for me to post any real data on the blog):</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">y_vars <span style="color: #666666;">=</span> c(rnorm(m<span style="color: #666666;">=35.32</span>,sd<span style="color: #666666;">=.5</span>,<span style="color: #666666;">25</span>),rnorm(m<span style="color: #666666;">=10</span>,sd<span style="color: #666666;">=2</span>,<span style="color: #666666;">25</span>))
x_het_hom_ratio <span style="color: #666666;">=</span> c(rnorm(m<span style="color: #666666;">=.34</span>,sd<span style="color: #666666;">=.2</span>,<span style="color: #666666;">25</span>),rnorm(m<span style="color: #666666;">=1.64</span>,sd<span style="color: #666666;">=.4</span>,<span style="color: #666666;">25</span>))
y_vars[<span style="color: #666666;">1</span>] <span style="color: #666666;">=</span> <span style="color: #666666;">35</span>
x_het_hom_ratio[<span style="color: #666666;">1</span>] <span style="color: #666666;">=</span> <span style="color: #666666;">2.1</span>
plot(x_het_hom_ratio,y_vars)</pre>
</div>
<p><a href="/wp-content/uploads/2012/10/xy_chrom_plot1.png"><img class="alignnone size-full wp-image-937" title="xy_chrom_plot" src="/wp-content/uploads/2012/10/xy_chrom_plot1.png"/></a></p>
<p>At first I was certain this must be a <a href="http://en.wikipedia.org/wiki/Klinefelter's_syndrome">Kleinfelter&#8217;s</a> (XXY) individual. But after spending some quality time with the IBD figures from step 3, I started to get suspicious&#8211; this same individual also exhibited cryptic relatedness to a number of different 1000 Genomes people, including people from different subpopulations. When I read slide 8 of <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/supporting/cryptic_relation_analysis/Nemesh_crypticrelatedness_20120213.pptx">Jim Nemesh&#8217;s powerpoint</a>, I started to understand why:</p>
<blockquote>
<ul>
<li>Five samples have low-level reported &#8220;IBD&#8221; to dozens of other samples</li>
<li>This can arise when a DNA sample or cell line has some contamination from another genome</li>
<li>Three of these individuals also have excess heterozygosity</li>
</ul>
</blockquote>
<p>So then I calculated the exome-wide het/hom ratio for all of my samples:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;"><span style="color: #008000; font-weight: bold;">select</span>   sid, <span style="color: #008000; font-weight: bold;">sum</span>(<span style="color: #008000; font-weight: bold;">case</span> <span style="color: #008000; font-weight: bold;">when</span> variant_allele_count <span style="color: #666666;">=</span> <span style="color: #666666;">1</span> <span style="color: #008000; font-weight: bold;">then</span> <span style="color: #666666;">1</span>.<span style="color: #666666;">0</span> <span style="color: #008000; font-weight: bold;">else</span> <span style="color: #666666;">0</span>.<span style="color: #666666;">0</span> <span style="color: #008000; font-weight: bold;">end</span>)<span style="color: #666666;">/</span><span style="color: #008000; font-weight: bold;">sum</span>(<span style="color: #008000; font-weight: bold;">case</span> <span style="color: #008000; font-weight: bold;">when</span> variant_allele_count <span style="color: #666666;">=</span> <span style="color: #666666;">2</span> <span style="color: #008000; font-weight: bold;">then</span> <span style="color: #666666;">1</span>.<span style="color: #666666;">0</span> <span style="color: #008000; font-weight: bold;">else</span> <span style="color: #666666;">0</span>.<span style="color: #666666;">0</span> <span style="color: #008000; font-weight: bold;">end</span>) <span style="color: #008000; font-weight: bold;">as</span> het_hom_ratio
<span style="color: #008000; font-weight: bold;">from</span>     sample_variants sv
<span style="color: #008000; font-weight: bold;">group</span> <span style="color: #008000; font-weight: bold;">by</span> sid
<span style="color: #008000; font-weight: bold;">order</span> <span style="color: #008000; font-weight: bold;">by</span> sid
;</pre>
</div>
<p>And that clinched it. Virtually everyone had heterozygosity of about 1.7, but the one sample I had thought was XXY had a ratio of 3.4. Could it be an outlandish coincidence, that this one person had an XXY karyotype <em>and</em> was the only mixed-ancestry person in the sample? Another look at the IBD figures pretty much ruled this out: all the 1000 Genomes people to whom this individual was cryptically related were of the same race, which wouldn&#8217;t support the idea that this elevated heterozygosity was the result of mixed ancestry. So the parsimonious explanation was clear: this sample contains DNA from at least two different people, including at least one female and one male. Scanning the list of IBD pairs, it&#8217;s not clear which (if any) of my other samples was the source of contamination&#8211; several of them share high IBD with this sample. It looks like I have to simply exclude this sample from my study.</p>
<p>Having gone through this process for this one contaminated sample helps me understand why heterozygosity is among the <a href="/2012/10/17/descriptive-statistics-and-quality-control-on-variants/">quality metrics usually computed for variant data</a>, and why cryptic relatedness can be a sign of more than just substructure you need to control for.</p>
<p>In fact, I went back to my original IBD table and tried excluding my one contaminated sample as well as HG00098 (which Nemesh notes as being possibly contaminated) and HG00124 (which I had noticed exhibited cryptic relatedness to tens of the 1000G Europeans in my dataset).  Once these three samples were filtered out, the IBD data looked vastly more rational&#8211; after the familial relationships mentioned above and in my own data, the IBD figures dropped off pretty quickly to IBD-1 of 40% or less and a <code>pi_hat</code> of no more than about .25.</p>
<p><strong>update 2012-11-12: </strong>when you calculate exome-wide heterozygosity, you&#8217;ll probably want to color-code the population each person belongs to, since <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0013996">homozygosity varies by population</a>. Here&#8217;s some R code to make a colored bar chart:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">library(RPostgreSQL) <span style="color: #408080; font-style: italic;"># first load package RPostgresql</span>
drv <span style="color: #666666;">&lt;-</span> dbDriver(<span style="color: #ba2121;">"PostgreSQL"</span>)
readcon <span style="color: #666666;">&lt;-</span> dbConnect(drv, dbname<span style="color: #666666;">=</span><span style="color: #ba2121;">"mydb"</span>, user<span style="color: #666666;">=</span><span style="color: #ba2121;">"postgres"</span>, password<span style="color: #666666;">=</span><span style="color: #ba2121;">"password"</span>)
sql <span style="color: #666666;">&lt;-</span> <span style="color: #ba2121;">"</span>
<span style="color: #ba2121;">select sv.sid, c.cluster, sum(case when sv.variant_allele_count = 1 then 1 else 0 end)::numeric/sum(case when sv.variant_allele_count = 2 then 1 else 0 end)::numeric as het_hom_ratio</span>
<span style="color: #ba2121;">from clusters c, sample_variants sv</span>
<span style="color: #ba2121;">where c.iid::integer = sv.sid</span>
<span style="color: #ba2121;">group by sv.sid, c.cluster</span>
<span style="color: #ba2121;">order by sv.sid</span>
<span style="color: #ba2121;">;</span>
<span style="color: #ba2121;">"</span>
rs <span style="color: #666666;">&lt;-</span> dbSendQuery(readcon,sql)
tbl <span style="color: #666666;">&lt;-</span> fetch(rs,n<span style="color: #666666;">=-1</span>)
library(hash)
pops <span style="color: #666666;">=</span> c(<span style="color: #ba2121;">"African"</span>,<span style="color: #ba2121;">"Asian"</span>,<span style="color: #ba2121;">"European"</span>)
colors <span style="color: #666666;">=</span> c(<span style="color: #ba2121;">"forestgreen"</span>,<span style="color: #ba2121;">"blue"</span>,<span style="color: #ba2121;">"orange"</span>)
k <span style="color: #666666;">=</span> hash(keys<span style="color: #666666;">=</span>pops,values<span style="color: #666666;">=</span>colors)
kvec <span style="color: #666666;">=</span> tbl$cluster
<span style="color: #008000; font-weight: bold;">for</span> (i in <span style="color: #666666;">1</span>:length(tbl$cluster)) { kvec[i] <span style="color: #666666;">&lt;-</span> k[[tbl$cluster[i]]] }
barplot(tbl$het_hom_ratio,col<span style="color: #666666;">=</span>kvec,ylim<span style="color: #666666;">=</span>c(<span style="color: #666666;">0</span>,<span style="color: #666666;">4</span>),ylab<span style="color: #666666;">=</span><span style="color: #ba2121;">"Exome-wide het/hom ratio"</span>,xlab<span style="color: #666666;">=</span><span style="color: #ba2121;">"Samples"</span>,border<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">NA</span>)
legend(<span style="color: #ba2121;">"topleft"</span>,pops,col<span style="color: #666666;">=</span>colors,cex<span style="color: #666666;">=1.0</span>,bty<span style="color: #666666;">=</span><span style="color: #ba2121;">"n"</span>,pch<span style="color: #666666;">=15</span>)  
dbDisconnect(readcon)
dbUnloadDriver(drv)</pre>
</div>
<p>And the results (this is fake data that looks approximately like my real data):</p>
<p><img class="alignnone size-full wp-image-984" title="het-hom-plot-example" src="/wp-content/uploads/2012/10/het-hom-plot-example.png"/></p>
<p>This makes it extremely clear which sample is elevated above baseline.  That&#8217;s not a guarantee that sample is contaminated&mdash;you should still look at your MDS plots and IBD tables before you rule out the possibility the person is of mixed ancestry and is genuinely that heterozygous.</p>
