---
layout: post
title: "The decoy genome"
author: ericminikel
date: 2013-02-01 17:56:11
---
<p>At the <a title="Center for Human Genetic Research 2013 Retreat" href="/2013/01/30/center-for-human-genetic-research-2013-retreat/">CHGR retreat</a> earlier this week I bumped into <a href="http://chgr.org/index-faculty_neale.html">Ben Neale</a> and mentioned I&#8217;d been <a title="Aligning unmapped reads to viral genomes" href="/2013/01/25/aligning-unmapped-reads-to-viral-genomes/">aligning unmapped reads to virus genomes</a>.  He told me I could do much better: <a href="http://en.wikipedia.org/wiki/Heng_Li">Heng Li</a>, inventor of <a href="http://bio-bwa.sourceforge.net/bwa.shtml">BWA</a>, has created an entire &#8216;decoy genome&#8217; to capture reads from human exome or whole genome sequencing that fail to align to the human reference genome.</p>
<p>The Broad Institute hosts a copy of this decoy genome.  <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/README_human_reference_20110707">This readme</a> explains what&#8217;s in the decoy and how it was used in the 1000 Genomes analysis pipeline, and all the files you need are in <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/">this folder</a>.</p>
<p>What&#8217;s in this decoy genome, you ask?  For one, it contains the genome for <a href="http://www.ncbi.nlm.nih.gov/nuccore/82503188">human herpesvirus 4, type 1</a>, somewhat better known as <a href="http://en.wikipedia.org/wiki/Epstein-Barr_virus">Epstein-Barr virus</a> (EBV) and even better known as the cause of <a href="http://en.wikipedia.org/wiki/Mononucleosis">mono</a>.  Don&#8217;t be confused by &#8216;herpes&#8217; in the name: this isn&#8217;t <a href="http://en.wikipedia.org/wiki/Herpes_simplex">herpes simplex virus 1 or 2</a>, which cause oral and genital herpes.  <a href="http://en.wikipedia.org/wiki/Herpesviridae">Herpesviridae</a> is a large and pretty successful family of viruses that also includes <a href="http://en.wikipedia.org/wiki/Varicella_zoster_virus">Varicella zoster</a>, the mastermind behind chicken pox / shingles.  Herpesviridae are all <a href="http://en.wikipedia.org/wiki/DNA_virus">DNA viruses</a>, which means you can pick them up in DNA sequencing &#8211; as opposed to <a href="http://en.wikipedia.org/wiki/RNA_virus">most viruses, which have only RNA</a>.  (You can pick those up in <a href="http://en.wikipedia.org/wiki/RNA-Seq">RNAseq</a>).</p>
<p>Whether or not you&#8217;ve had mono, you&#8217;ve probably been exposed to EBV (and many other members of the Herpesviridae family) at some point and are likely to have latent copies of the viral genome lying around in your cells even if you&#8217;re not now (or never have been) symptomatic.  For this reason EBV very often turns up in human DNA sequencing.</p>
<p>EBV is also used in research laboratories to &#8216;<a href="http://en.wikipedia.org/wiki/Epstein%E2%80%93Barr_virus#Transformation_of_B-lymphocytes">immortalize</a>&#8216; <a title="white blood cell precursors" href="http://en.wikipedia.org/wiki/Lymphoblast">lymphoblasts</a> (white blood cell precursors): the virus makes them replicate and grow indefinitely.  This property of EBV is bad when it causes <a href="http://en.wikipedia.org/wiki/Hodgkin%27s_lymphoma">Hodgkin&#8217;s</a> or <a href="http://en.wikipedia.org/wiki/Burkitt%27s_lymphoma">Burkitt&#8217;s lymphoma</a> but good when you&#8217;re a researcher and you want an endless supply of human cells for experiments.  So if you&#8217;re sequencing DNA from cultured cell lines, you might find EBV that was intentionally introduced in the lab as well.</p>
<p>But the <a href="http://www.ncbi.nlm.nih.gov/nuccore/82503188">EBV genome</a> is only ~170kb, while the full decoy genome is ~36Mb.  The rest of Broad Institute&#8217;s decoy genome was constructed from sequences that, though absent from the current reference sequence (hg19 / GRCh37), <em>are</em> found either in <a href="http://www.ncbi.nlm.nih.gov/assembly/2568/">HuRef</a> (<a href="http://en.wikipedia.org/wiki/Craig_Venter">Craig Venter</a>&#8216;s genome) or Broad&#8217;s de novo assembly of <a href="http://ccr.coriell.org/Sections/Search/Sample_Detail.aspx?Ref=GM12878">NA12878</a> (an anonymous 1000 Genomes participant and one of the most thoroughly sequenced people in the world) or various other pieces of human DNA that have been cloned and studied in bacteria over the years.  What is all this stuff, then? Much of it really is human genomic material that most or all people have, but just slipped through the cracks when the reference genome was put together.  For instance, the first entry in the decoy is <a href="http://www.ncbi.nlm.nih.gov/nuccore/GL582971.1">GL582971</a>, a 1.3Mb piece of human chromosome 7 that just didn&#8217;t quite make it into build 37 but is slated for inclusion in build 38 whenever such is released.  It&#8217;s currently considered as a patch to b37.</p>
<p>If these are genuine pieces of the human genome, then re-aligning your unmapped reads to them will give you extra information, which can only be a good thing.  But actually from what I hear, the major motivation for people to use the decoy genome is <em>speed</em>.  If you include the decoy in your reference genome when you do the original alignment, many reads will quickly find a very confident alignment in the decoy, thus avoiding countless compute cycles spent trying to <a href="http://en.wikipedia.org/wiki/Smith%E2%80%93Waterman_algorithm">Smith-Waterman align</a> it to someplace it doesn&#8217;t belong.  This purpose &#8211; siphoning off these reads to keep them from slowing down the whole alignment &#8211; is why this is called the &#8216;decoy genome.&#8217;</p>
<p>With that in mind, let&#8217;s return to Broad&#8217;s <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/">repository of files</a>.  If you&#8217;re just starting alignment now, you can use <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/hs37d5.fa.gz">hs37d5.fa.gz</a>, which is all of human genome b37 <em>plus</em> the decoy, and if you&#8217;ve already done alignment and you just want to realign the unmapped reads you can use <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/hs37d5ss.fa.gz">hs37d5ss.fa.gz</a>, which is just the decoy.</p>
<p>Here I&#8217;ll consider the case of realigning unmapped reads to the decoy.</p>
<p><strong>step 1. prep files</strong></p>
<p><a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/hs37d5ss.fa.gz">hs37d5ss.fa.gz</a> is the decoy reference genome; you still need to <code>gunzip</code> it and <code>bwa index</code> it which takes &lt; 1 minute because it&#8217;s pretty small.</p>
<pre>wget ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/hs37d5ss.fa.gz
gunzip hs37d5ss.fa.gz
bwa index hs37d5ss.fa</pre>
<p>You&#8217;ll also want to pull the unmapped reads out of your original BAM files:</p>
<pre>samtools view -f 0x04 -h -b original.bam -o unmapped.bam</pre>
<p>For me, starting with BAMs of 30-50M reads, about 10% of which are unmapped, this takes ~10 minutes per BAM.</p>
<p><strong>step 2. re-align</strong></p>
<p>These are basically the same instructions as for <a href="/2013/01/25/aligning-unmapped-reads-to-viral-genomes/">aligning to viral genomes</a>:</p>
<pre>for i in {1820..1869}
do
 bsub.py short 00:50 "bwa aln -b decoygenome/hs37d5ss.fa hdex2.${i}.unmapped.bam &gt; ${i}.decoy.sai; bwa samse decoygenome/hs37d5ss.fa ${i}.decoy.sai hdex2.${i}.unmapped.bam | samtools view - -Sb -o ${i}.decoy.all.bam; samtools view ${i}.decoy.all.bam -b -F 0x04 -o ${i}.decoy.mapped.bam; samtools view ${i}.decoy.all.bam -b -f 0x04 -o ${i}.decoy.unmapped.bam;"
done</pre>
<p>For me this took anywhere from 5 to 50 minutes per sample.</p>
<p><strong>step 3. analyze</strong></p>
<p>You might ask, what fraction of the unmapped reads aligned to the decoy?  samtools can count reads from each:</p>
<pre>samtools view -c ${i}.decoy.mapped.bam
samtools view -c ${i}.original.bam</pre>
<p>You can also compute read depth (either with <a href="http://code.google.com/p/bedtools/">bedtools</a> or crudely at the command line) or call variants on these BAMs just like any other BAMs &#8211; for specifics see the <a href="/2012/09/19/exome-sequencing-pipeline-using-gatk/">GATK pipieline</a> and the <a title="Aligning unmapped reads to viral genomes" href="/2013/01/25/aligning-unmapped-reads-to-viral-genomes/">viral genomes pipeline</a>.</p>
<p><strong>discussion</strong></p>
<p>Odds are that re-aligning to the decoy genome will capture some of your originally unmapped reads, but far from all of them.  What <em>is </em>the rest of that stuff?</p>
<p>To peruse a random subset of my still-unmapped reads I used <a href="http://stackoverflow.com/a/2153897"><code>shuf</code></a>:</p>
<pre>samtools view 1820.decoy.unmapped.bam | shuf | less</pre>
<p>Glancing over them, I see that many are low information entropy reads: lots of homopolymer runs e.g. AAAAAAAAAAA and lots of low-quality reads full of unknown bases e.g. NNNNNNNNNNN.  These may well be human genomic DNA but just cannot map uniquely to any place in the reference genome nor the decoy.  If you <a href="http://blast.ncbi.nlm.nih.gov/">blast</a> them you&#8217;ll get no matches, because blast doesn&#8217;t bother trying to match information-poor sequences.</p>
<p>Other reads do look, at first glance, like they&#8217;ve got some real information. I blasted a handful of them to see what they are.  All of the ones I checked proved to be human DNA, but mapped equally well in several different places.  There are two issues here.  First, since the read aligned equally well in multiple places, it might be a repeated element, part of some larger <a href="http://en.wikipedia.org/wiki/Satellite_DNA">satellite</a> or <a href="http://en.wikipedia.org/wiki/Retrotransposon">retrotransposon</a> even if it doesn&#8217;t look repetitive in and of itself.  Second, the blast matches for these reads were especially enriched for <a href="http://en.wikipedia.org/wiki/Bacterial_artificial_chromosome">BAC</a> / <a href="http://en.wikipedia.org/wiki/Fosmid">Fosmid</a> clones rather than legit human reference genome matches.  BAC and Fosmid clones are little pieces of human DNA that someone has isolated and cloned into a bacterium.  I suppose, since blast didn&#8217;t find a better alignment in the human reference genome itself for these reads, that some of the matching BACs/Fosmids do represent human DNA but have not yet been incorporated into the reference genome nor the decoy genome.</p>
