---
layout: post
title: "How to do a GWAS for heterozygote advantage"
author: ericminikel
date: 2013-06-18 19:44:26
---
<p><a href="http://en.wikipedia.org/wiki/Heterozygote_advantage">Heterozygote advantage</a> is when the heterozygous genotype has a better phenotype than <em>either</em> homozygous genotype.  Perhaps the most famous example is <a href="http://www.ncbi.nlm.nih.gov/SNP/snp_ref.cgi?rs=334">rs334</a>, the E7V mutation in <a href="http://en.wikipedia.org/wiki/HBB">HBB</a> that causes <a href="http://en.wikipedia.org/wiki/Heterozygote_advantage#Sickle-cell_anemia">sickle-cell anemia</a>.  If you&#8217;re homozygous VV then you have sickle cell anemia (that&#8217;s bad), if you&#8217;re homozygous EE then you&#8217;re vulnerable to malaria (also bad), but if you&#8217;re heterozygous EV you&#8217;re off the hook for both.</p>
<p>Most genome-wide association studies I read don&#8217;t even bother to check for heterozygote advantage &#8211; they only run additive (allelic) models and dominant models.  This issue is dear to my heart, though, since some coding SNPs in the prion protein gene PRNP exhibit heterozygote advantage, having a dominant negative effect that slows or prevents the autocatalytic spread of prions.  The most common of these is <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1799990">rs1799990</a>, the codon 129 M/V polymorphism [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3226040/">Lukic &amp; Mead 2011</a>].  The other accepted examples in PRNP are <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=1800014">rs1800014</a> E219K [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9629853">Shibuya 1998</a>] and G127V, which is not in dbSNP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19923577">Mead 2009</a> (<a href="http://www.nejm.org/doi/full/10.1056/NEJMoa0809716#t=article">ft</a>)]).</p>
<p><a href="http://pngu.mgh.harvard.edu/~purcell/plink/">PLINK</a> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838/">Purcell 2007</a>] has options to run additive (allelic) and dominant models of genotype/phenotype association.  However it does not have an option to compare the heterozygous genotype against <em>both</em> homozygous genotypes. The closest thing PLINK offers is <a href="http://pngu.mgh.harvard.edu/~purcell/plink/anal.shtml#model"><code>--model</code></a>, which compares all three possible genotypes for each biallelic marker: DD vs. Dd vs. dd.  This option will pick up heterozygote advantage, but has the following limitations:</p>
<ul>
<li>Not compatible with quantitative phenotypes</li>
<li>Not compatible with quantitative covariates e.g. <a title="How (and why) to create population covariates using 1000 Genomes data" href="/2012/10/15/population-covariates-using-1000-genomes/">population principal components</a> (<code>ERROR: Cannot specify --covar with --model</code>)</li>
<li>No trivial way to scan results to see which associations were driven by Dd &gt; DD,Dd.</li>
</ul>
<p>So instead, I figured out a way to trick PLINK into doing what I want.  My idea was to change both homozygous genotypes to be the same, so that when you ask PLINK to run an allelic model (say, with <a href="http://pngu.mgh.harvard.edu/~purcell/plink/anal.shtml#glm"><code>--linear</code></a>) it will unwittingly do a heterozygous vs. homozygous comparison.  Here&#8217;s what I came up with:</p>
<pre>cat gwas.ped | awk '{printf $1" "$2" "$3" "$4" "$5" "$6; for(i=7;i&lt;6+142780*2;i=i+2) { j=i+1; if($i==$j) {printf " C C";} else {printf " C T";}} print""}' &gt; het.ped</pre>
<p><em>*Thanks to <a href="http://stackoverflow.com/questions/2626274/awk-print-all-other-columns-but-not-1-2-and-3">this Stackoverflow discussion</a> which was crucial in helping me figure out how to do this with <code>awk</code>.</em></p>
<p>Here&#8217;s what that code does in plain language: for each line, print the first 6 columns (the mandatory <code>.ped</code> columns) and then starting with column 7 and going up to column 6+142780*2 (where 142780 = number of SNPs in your dataset, <code>wc -l gwas.map</code> if you&#8217;re not sure), look at every pair of columns, and if they&#8217;re the same (e.g. A A or G G) recode them as C C, and if they&#8217;re different (e.g. A G) recode them as C T.  <code>printf</code> makes sure the output stays on the same line for the whole loop, then once the loop is over, <code>print""</code> ends the line. You don&#8217;t need to change the <code>.map </code>file at all.  You&#8217;re ready to roll, just specify different ped and map filenames for PLINK:</p>
<pre>plink --ped het.ped --map gwas.map --pheno quantitative.phe --covar quantitative.covars --linear --adjust --out heterozygote.advantage.analysis</pre>
<p>Needless to say, you lose the option to do any filtering based on <a href="http://en.wikipedia.org/wiki/Hardy%E2%80%93Weinberg_principle">Hardy-Weinberg equilibrium</a> with <a href="http://pngu.mgh.harvard.edu/~purcell/plink/thresh.shtml#hwd"><code>--hwe</code></a>.  But if you need to, you can always filter your SNPs before you do this little trick.</p>
<p>And that&#8217;s that.  If you&#8217;re lucky maybe you&#8217;ll find a really strong association, like this:</p>
<p><a href="/wp-content/uploads/2013/06/made.up_.manhattan.plot_.png"><img class="alignnone size-full wp-image-2443" alt="If only real data were always as interesting as this made-up data." src="/wp-content/uploads/2013/06/made.up_.manhattan.plot_.png"/></a></p>
<p>&nbsp;</p>
<p><em>Graphic created using a lightly modified version of <a href="http://gettinggeneticsdone.blogspot.com/2011/04/annotated-manhattan-plots-and-qq-plots.html">Stephen Turner&#8217;s Manhattan plot code</a>.</em></p>
