---
layout: post
title: "How prion protein was first purified"
author: ericminikel
date: 2013-10-17 04:04:05
---
<p>Months ago, I ran across the original series of papers in which Stanley Prusiner achieved a homogeneous purification of prion protein and began to decipher its amino acid sequence [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6818988">Prusiner 1982a</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/6418385">Prusiner 1983</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/6432339">Prusiner 1984</a>].  This purification process seemed like quite a feat, but I couldn&#8217;t begin to understand the methods used.  After attending <a title="Biochemistry 04: proteins and nucleic acids" href="/2013/10/11/biochemistry-04-proteins-and-nucleic-acids/">a biochem lecture</a> a couple of weeks ago covering methods for protein purification, I thought I was finally equipped to understand this, so I went back and took a deeper look at these old papers.</p>
<p>The process of purifying PrP was amazingly involved, and the road to get there even moreso.  Prusiner spent much of the 1970s <a href="http://en.wikipedia.org/wiki/Fractionation">fractioning</a> hamster brain homogenate on every imaginable dimension and assaying for which fraction contained infectivity, a journey partly chronicled in the seminal &#8220;prion&#8221; paper [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6801762">Prusiner 1982b</a>].  The purification procedure that finally emerged from this effort is presented in [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6418385">Prusiner 1983</a>] and is nicely summarized in [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6432339">Prusiner 1984</a>] as well.</p>
<p>In Step 1, the starting material &#8211; scrapie-infected brain homogenate &#8211; is simply &#8216;clarified&#8217;, which I understand to mean that large particles and impurities are removed. This is accomplished with continuous flow centrifugation, which <a href="http://newbrunswick.eppendorf.com/en/products/centrifuges/">Eppendorf describes</a> as being for &#8220;solids removal&#8221;.  A liquid is pushed up into the bottom of a rapidly rotating tube, and solids are retained on the walls of the tube while liquid is forced out the top.  The centrifugation speed is moderate (compared to later steps) at just 10,000 rpm.  As in any centrifugation process, the solids are referred to as the <em>pellet</em> (P1 since this is Step 1) and the liquid is the <em>supernatant</em> (S1).</p>
<p>In Step 2, PrP<sup>Sc</sup> is precipitated out based on its property of being detergent-insoluble.  The liquid from the previous step, S1, is mixed with two detergents: <a href="http://en.wikipedia.org/wiki/Triton_X-100">Triton X-100</a> and sodium <a href="http://en.wikipedia.org/wiki/Deoxycholic_acid">deoxycholate</a>, and then precipitated with <a href="http://en.wikipedia.org/wiki/Polyethylene_glycol">PEG</a>-8000 as a <a href="http://en.wikipedia.org/wiki/Precipitant">precipitant</a>.  Precipitation means causing a solute (in this case PrP<sup>Sc</sup>) to form particles in suspension, which can then be centrifuged into a pellet, as shown in these three steps:</p>
<p><a href="http://en.wikipedia.org/wiki/File:Chemical_precipitation_diagram.svg"><img class="alignnone size-full wp-image-3125" alt="200px-Chemical_precipitation_diagram" src="/wp-content/uploads/2013/10/200px-Chemical_precipitation_diagram.png"/></a></p>
<p>&nbsp;</p>
<p>(After reading about <a href="http://en.wikipedia.org/wiki/Polyethylene_glycol">polyethylene glycol</a>, of which PEG-8000 is simply the term for polymers of ~8,000 kDa molecular weight, I still don&#8217;t understand what makes it act as a precipitant.)  The precipitated material was pelleted out, again using continuous flow centrifugation but this time at a higher speed, 16,000 rpm.</p>
<p>In Step 3, the pellet from Step 2 (P2) is re-suspended in a solution with a buffer (Tris OAc), a detergent (Triton X-100), a reducing agent (<a href="http://en.wikipedia.org/wiki/Dithiothreitol">DTT</a>), and a salt (CaCl<sub>2</sub>), none of whose functions in this context I understand.  The suspension is then digested with <a href="http://en.wikipedia.org/wiki/Micrococcal_nuclease">micrococcal nuclease</a>, which breaks down RNA and DNA &#8211; an important step for removing contaminating nucleic acids and, more importantly, for proving that the &#8220;scrapie agent&#8221; didn&#8217;t require nucleic acid.</p>
<p>The suspension was then also subjected to a light proteinase  K (PK) digestion.  PK, mind you, can abolish scrapie infectivity &#8211; one early piece of evidence for the prion hypothesis &#8211; but its activity is dependent on enzyme concentration (more is more), time (more is more), and temperature (most active at 37°C).  The digestion in this protocol was 8h of 100 ug/mL of enzyme at 4°C.  That&#8217;s not much &#8211; the protocol required to denature 99.9% of infectivity is more like 30h of 500 ug/mL at 37°C [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6273882">Prusiner 1981</a>].  So I think my understanding is that this light PK digestion is here mostly to degrade other proteins, thus comparatively enriching for the at least partially PK-resistant desired protein: PrP<sup>Sc</sup>.</p>
<p>After the two digestion steps, the suspension was then again treated with two detergents (<a href="http://en.wikipedia.org/wiki/Sodium_lauroyl_sarcosinate">Sarkosyl</a> and sodium <a href="http://en.wikipedia.org/wiki/Deoxycholic_acid">deoxycholate</a>) and precipitated (with <a href="http://en.wikipedia.org/wiki/Ammonium_sulfate">ammonium sulfate</a>) and then centrifuged again (now at 19,000 rpm) to yield P3.</p>
<p>In Step 4, P3 is again re-suspended and then subjected to zonal rotor centrifugation to separate sucrose gradients.  &#8221;Zonal rotor&#8221; just means that instead of tubes full of liquid inside a centrifuge, the whole centrifuge is directly full of a liquid.  The principle of <a href="http://en.wikipedia.org/wiki/Differential_centrifugation#Sucrose_gradient_centrifugation">sucrose gradients</a> is quite interesting: the centrifuge is filled with distinct layers or a continuous gradient representing different concentrations of sucrose &#8211; in this case, a continuous gradient from 60% (at the outside) to 0% (on the inside).  Molecules will &#8220;sediment&#8221; according to density &#8211; a molecule with the same density as a 40% sucrose solution will settle in the 40% fraction.  Sucrose gradient layers are often referred to in &#8220;<a href="http://en.wikipedia.org/wiki/Svedberg">Svedbergs</a>&#8221; (e.g. &#8220;the 26<span style="text-decoration: underline;">S</span> proteasome&#8221;) though this is technically a measure of time rather than of density or sucrose concentration.</p>
<p>Prusiner assayed every different fraction of this gradient for scrapie infectivity, and found that infectivity peaked somewhere in the range of about 30-40% sucrose [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6418385">Prusiner 1983</a>, Fig 1].</p>
<p>This was enough purification to begin to study prions at the molecular level in ways that hadn&#8217;t been possible.  Prusiner performed electron microscopy on the most infectious sucrose gradients and saw 100-200nm long rods of aggregated PrP<sup>Sc</sup>, which resembled amyloid fibrils from Alzheimer&#8217;s brains [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6418385">Prusiner 1983</a>, Fig 4].  He also boiled the fractions in another detergent (<a href="http://en.wikipedia.org/wiki/Sodium_dodecyl_sulfate">sodium dodecyl sulfate</a>, SDS), which inactivated infectivity but also disaggregated the rods, thus making it possible to study pure PrP monomers for the first time.  He ran the resulting monomers through SDS-PAGE, a process whose principles I only <a title="Biochemistry 04: proteins and nucleic acids" href="/2013/10/11/biochemistry-04-proteins-and-nucleic-acids/">recently</a> came to understand.  If you just put proteins on a <span style="text-decoration: underline;">p</span>olyacryl<span style="text-decoration: underline;">a</span>mide gel and run an <span style="text-decoration: underline;">e</span>lectro<span style="text-decoration: underline;">p</span>horesis (PAGE), they&#8217;ll separate according to charge, with cationic proteins headed towards the anode and anionic proteins bound for the cathode.  By first treating proteins with SDS, you uniformly coat them in negative charge so that they all migrate towards the cathode, distinguished only by their molecular mass &#8211; extra mass hinders their progress. So this is a method for determining the molecular mass of an isolated protein.  Mind you, this <em>looks</em> just like a Western blot [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6418385">Prusiner 1983</a>, Fig 2] but a Western blot is an SDS-PAGEd protein transferred to nitrocelluose and then blotted with antibodies.  In 1983 the world had never seen no antibodies to PrP yet because prion diseases don&#8217;t cause an immune response and the protein had never before been purified.  Instead, the protein had to be labeled using radioactive iodine and <a href="http://www.piercenet.com/instructions/2160476.pdf">Bolton-Hunter reagent</a> and then developed as an autoradiogram.</p>
<p>The next year, Prusiner was back with another huge breakthrough: the beginnings of a characterization of the amino acid sequence of PrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6432339">Prusiner 1984</a>].  This study used the same purification protocol as before, plus an additional step of a proteinase K digestion (4 ug/mL at 37°C for 2h), apparently necessary to disaggregate the rods enough to make it possible to hydrolyze (break down) the peptide bonds between every amino acid.  Subjecting PrP to protease digestion also meant that only the protease-resistant fragment was being studied, a fragment dubbed PrP 27-30.  Even after the protein was run through <a href="http://en.wikipedia.org/wiki/High-performance_liquid_chromatography">high-performance liquid chromatography</a> (HPLC) to isolate it based on molecular weight, its mass still ranged from 27 to 30 kDa, probably reflecting differences in <a title="Prion protein N-linked glycosylation: review and assessment of therapeutic potential" href="/2013/05/05/prion-protein-n-linked-glycosylation-review-and-assessment-of-therapeutic-potential/">glycosylation</a>, <a title="Proteolysis of prion protein in the central region: alpha and beta cleavage" href="/2013/06/26/proteolysis-of-prion-protein-in-the-central-region-alpha-and-beta-cleavage/">beta cleavage</a> site and/or PK-resistant fragment size.</p>
<p>To figure out the amino acid <em>composition</em> (not sequence) of PrP, Prusiner digested all of the peptide bonds and then ran the free amino acids through <a href="http://en.wikipedia.org/wiki/High-performance_liquid_chromatography">high-performance liquid chromatography</a> (HPLC), separating the amino acids by mass.  At least at that time, this technique was unable to distinguish D from N and E from Q, but it did show that PrP contained at least 17 of the 20 amino acids (in fact, it contains all 20).  This disproved what had apparently been one hypothesis about prions up to that time: that perhaps they were able to replicate without nucleic acid because they were just simple repetitive sequences of a couple of amino acids that happened to code for a protein that produces itself.</p>
<p>Prusiner also determined the amino acid <em>sequence</em> of the N-terminal edge of PrP 27-30: GQGGGTHNQWNKPSK, corresponding to codons 81-95 of SHaPrP.  Interestingly, weaker signals were observed for sequences starting at other codons, revealing heterogeneity in where the PK-resistant fragment ended.</p>
<p>When I first saw this paper, I couldn’t begin to understand how amino acid sequencing was possible in 1983 considering that, 30 years later, we still don’t have any really high-throughput methods for sequencing novel proteins today.  Mass spec proteomics is still horrifyingly expensive and even then, the quantification of proteins involves essentially a lookup table (UniProt) of known proteins &#8211; a far cry from the ability of next-gen DNA sequencing to read an arbitrary sequence.</p>
<p>But as I learned <a title="Biochemistry 04: proteins and nucleic acids" href="/2013/10/11/biochemistry-04-proteins-and-nucleic-acids/">recently</a>, amino acid sequencing has actually been around <em>since 1949</em> , just at very, very low throughput [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18134557">Edman 1949</a> (<a href="http://actachemscand.org/pdf/acta_vol_04_p0283-0293.pdf">ft</a>)].  Though Prusiner does not go into the technique in detail in the paper, I am assuming it&#8217;s basically the same technique, known as <a href="http://en.wikipedia.org/wiki/Edman_degradation">Edman degradation</a>.  The purified protein is treated with <a href="http://en.wikipedia.org/wiki/Phenyl_isothiocyanate">phenylisothiocyanate</a> (&#8220;Edman&#8217;s reagent&#8221;), which (if I understand correctly) reacts with only the exposed N-terminal amino acid of a protein, and makes that amino acid&#8217;s peptide bond more vulnerable to cleavage by acids.  So: react with phenylisothiocyanate, react with acid, run the one cleaved amino acid through HPLC, wash, repeat.  In this manner Prusiner managed to get through 15 amino acids down the N-terminus of the PK-digested protein.</p>
<p>It&#8217;s also possible to get a protein&#8217;s entire sequence via Edman degradation, by cleaving the protein with different enzymes (e.g. <a href="http://en.wikipedia.org/wiki/Trypsin">trypsin</a> and <a href="http://en.wikipedia.org/wiki/Chymotrypsin">chymotrypsin</a>) so that you get different overlapping peptides, separating those by mass, and then sequencing each of those and reassembling the sequences.  Prusiner didn&#8217;t do this, presumably because the purified protein was too precious &#8211; it took 10 liters of a diluted brain homogenate as the starting material to get just a few nanomoles of purified PrP (each aliquot used for a different experiment was 300 &#8211; 1000 pmoles).  At a molecular mass of ~27-30,000 g/mol, the total quantity probably amounted to something on the order of a few hundred μg of PrP.</p>
<p>It was a battle hard won.  The purification and characterization of PrP set the stage for the discovery of PrP mRNA and the revelation that it was encoded in the host genome [<a href="http://www.ncbi.nlm.nih.gov/pubmed/2859120">Oesch 1985</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/3923361">Chesebro 1985</a>], and for the development of the first anti-PrP antibodies [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6431296">Bendheim 1984</a>].  All of the tools that make studying prions so easy today &#8211; transgenic mouse models, monoclonal antibodies, recombinant PrP &#8211; required first being able to study the protein in its pure state.</p>
