---
layout: post
title: "Cell Biology 04: The Secretory Pathway"
tag: bios-e-16
author: ericminikel
date: 2013-02-24 02:48:06
---
<p>These are notes from lecture 4 of <a href="http://www.extension.harvard.edu/courses/22958">Harvard Extension’s Cell Biology course</a>.</p>
<p><strong><a href="http://en.wikipedia.org/wiki/Secretory_pathway">The secretory pathway</a> </strong>refers to the <a href="http://en.wikipedia.org/wiki/Endoplasmic_reticulum">endoplasmic reticulum</a>, <a href="http://en.wikipedia.org/wiki/Golgi_apparatus">Golgi apparatus</a> and the vesicles that travel in between them as well as the <a title="Cell Biology 02: The Plasma Membrane" href="/2013/02/10/cell-biology-02-the-plasma-membrane/">cell membrane</a> and <a href="http://en.wikipedia.org/wiki/Lysosome">lysosomes</a>.  It&#8217;s named &#8216;secretory&#8217; for being the pathway by which the cell secretes proteins into the extracellular environment.  But as usual, etymology only tells a fraction of the story.  This pathway also processes proteins that will be membrane-bound (whether in the cellular membrane or in the ER or Golgi membranes themselves), as well as lysosomal enzymes, and also any proteins that will live their lives in the secretory pathway itself.  It also does some things other than process proteins.</p>
<p>The cytosol and the &#8216;lumen&#8217; (the liquid that fills the secretory pathway) are different chemical environments, and they normally never mix.  The cytosol is reductive (when you&#8217;re in the cytosol, you keep meeting molecules that want to offer you electrons), and the ER, Golgi and extracellular environment are oxidative (molecules keep coming up to you asking for electrons).  See <a title="oxidation is losing [electrons], reduction is gaining [electrons]" href="http://en.wikipedia.org/wiki/Redox">redox</a> if still confused.  This makes for different protein-folding conditions: for instance, <a title="bonds between the sulfurs of two cysteines" href="http://en.wikipedia.org/wiki/Disulfide_bond#Occurrence_in_proteins">disulfide bonds</a> usually only form in oxidative conditions.  Moreover, different proteins may live only in the secretory pathway or only in the cytosol.  The secretory pathway provides a route for the cell to handle things that might not be good to have in the cytoplasm, and/or are most useful when kept concentrated in a specialized compartment with their desired interacting partners.  <a title="the cells that make up most of the liver" href="http://en.wikipedia.org/wiki/Hepatocyte">Hepatocytes</a> (in the liver) sequester drugs and toxins in the smooth ER and break them down for excretion from the body there.  The secretory pathway is not contiguous, but every movement between its components is in little bubbled-off microcosms of its own chemical world, called <a href="http://en.wikipedia.org/wiki/Vesicle_(biology_and_chemistry)">vesicles</a>.</p>
<p>Many proteins that go through the secretory pathway never touch the cytosol &#8211; except the parts of membrane proteins that stick out on the cytosolic side.  Many of them need <a href="http://en.wikipedia.org/wiki/Chaperone_(protein)">chaperones</a> to help with folding, and/or a whole series of <a href="http://en.wikipedia.org/wiki/Posttranslational_modification">post-translational modifications</a> in order to be ready for their native function, and the secretory pathway specializes in providing them all of that.</p>
<p>Today&#8217;s lecture will focus on how proteins get translated into the ER and how they travel (in vesicles) between the ER, Golgi and other destinations.  This is beautifully depicted in the Life of the Cell video:</p>
<p><iframe src="http://www.youtube.com/embed/wJyUtbn0O5Y?feature=player_embedded&amp;start=119&amp;end=250" frameborder="0" width="640" height="360"></iframe></p>
<p>The <strong><a href="http://en.wikipedia.org/wiki/Endoplasmic_reticulum">endoplasmic reticulum</a> </strong>is the first step in the secretory pathway.  Its membrane is continuous with the outer nuclear membrane, though it&#8217;s not clear why that matters, since it&#8217;s not like proteins begin their life in the nucleus.  Rather, mRNAs drift around in the cytoplasm until they get picked up by a ribosome interested in translating them.  In &#8216;posttranslational translocation&#8217; the new protein is moved into the ER after it&#8217;s translated.  In the more interesting phenomenon called &#8216;cotranslational translocation&#8217; the ribosome starts translation just like any other protein, but somewhere in the first 16 to 30 amino acids it hits a <a href="http://en.wikipedia.org/wiki/Signal_peptide">signal peptide</a> (aka signal sequence).  That signal&#8217;s motif is often 1 positively charged amino acid followed by 6-12 hydrophobic amino acids.  This motif gets recognized by <a href="http://en.wikipedia.org/wiki/Signal-recognition_particle">signal recognition particle</a> (SRP, a &#8216;ribonucleoprotein&#8217; or hybrid RNA/protein molecule) which binds to it and prevents the ribosome from continuing translation.  Translation is stopped until the ribosome/SRP complex encounters an SRP receptor on the ER membrane.  When they meet, SRP and its receptor each bind one <a title="guanosine triphosphate, just another cellular energy source" href="http://en.wikipedia.org/wiki/Guanosine_triphosphate">GTP</a> molecule in the ER membrane, which apparently strengthens their interaction.  Fortuitously, this all happens adjacent to a <a href="http://en.wikipedia.org/wiki/Sec61">Sec61 translocon</a> &#8211; a protein complex that forms a channel crossing the ER membrane.  The translocon is actually a complex of three different proteins (genes: SEC61A1 or SEC61A2, SEC61B, SEC61G), of which the Sec61a subunit has 10 membrane-spanning a-helices which form the channel.  Once the ribosome is docked at the membrane it continues translation, pushing the signal peptide and eventually the whole protein through the channel into the ER lumen.  When translation stops, SRP and SRP receptor both hydrolzye their GTP to release each other and the ribosome cargo (this has to require the energy of GTP, since the original binding was downhill), a <a title="an enzyme that cleaves (hence &quot;-ase&quot;) off a signal peptide" href="http://en.wikipedia.org/wiki/Signal_peptidase">signal peptidase</a> cleaves the signal peptide off of the <a title="people use this word all the time in biology.  it just means newly formed." href="http://en.wiktionary.org/wiki/nascent">nascent</a> protein, and the protein is free to start folding in the ER.</p>
<p>A couple of other players are involved for some ER proteins.   <a title="or oligosaccharyltransferase if you prefer" href="http://en.wikipedia.org/wiki/Oligosaccharyltransferase">Oligosaccharide transferase</a>, which adds glycosyl groups to asparagines in the nascent protein, is part of the translocon complex and it actually performs glycosylation <em>while </em>the new protein is still being translated.  So although we call glycosylation a &#8216;post-translational modification&#8217; it is actually made during translation in this case.  Also, to achieve their proper structure, some proteins need to be fully translated before they are allowed to start folding &#8211; if the N-terminal portion was allowed to start folding as soon as it entered the lumen, it would end up with the wrong overall structure.  To prevent this, sometimes <a href="http://en.wikipedia.org/wiki/Binding_immunoglobulin_protein">BiP the chaperone</a> binds the protein to keep it unfolded for a while.  Imagine BiP as another Pac-Man that bites down on the protein to keep it linear, like Hsc70 in the mitochondrial targeting process (see <a title="Cell Biology 03: Targeting proteins to organelles" href="/2013/02/17/cell-biology-03-targeting-proteins-to-organelles/">last week</a>).</p>
<p>Here&#8217;s a video of it:</p>
<p><iframe src="http://www.youtube.com/embed/PUy_Em5dXmc" frameborder="0" width="420" height="315"></iframe></p>
<p>The first couple of minutes show the basic scenario described above.  Then it moves on to a more complex scenario I&#8217;ll introduce in a minute.  FYI, the video depicts two &#8216;controversial&#8217; things not included in the above description: (1) the signal peptide being degraded in the membrane, and (2) a &#8216;plug protein&#8217; that stops up the channel before/after translation.  Not all scientists agree on these two things yet.</p>
<p>All of the proteins that we know go through the secretory pathway were pinpointed there by people doing localization experiments to see where in the cell a protein lies.  A weird fact about the ER is that you can put the cell in a blender and afterwards the ER will just start reconnecting to itself, forming little &#8216;microsomes&#8217; that are not attached to the nucleus but form contiguous bubbles of ER.  You can then start to play games with <a title="proteins that chop up other proteins" href="http://en.wikipedia.org/wiki/Protease">proteases</a> &#8211; which break down proteins &#8211; and <a title="amphipathic molecules that break up hydrophobic clumps and make them soluble in water" href="http://en.wikipedia.org/wiki/Detergent">detergents</a> &#8211; which solubilize the ER membrane.  Assuming your protein of interest is translated, you can check if it (1) survives protease treatment but (2) <em>doesn&#8217;t</em> survive protease + detergent treatment, then  it&#8217;s a secretory pathway protein.  The logic is that in case (1) it was protected inside the ER, but in case (2) you dissolved the ER, so it got eaten by the protease.  All this assumes you have an antibody or some other way of detecting whether the protein of interest is there after these treatments.</p>
<p>People also used such techniques to figure out that only 70 amino acids of a new protein can be translated before it becomes too late for that protein to end up in the ER.  Remember, the signal peptide is in the first 16-30 amino acids, and translocation to the ER depends on SRP being present.  Ribosomes translate at a predictable rate, so people got ribosomes started on translating some mRNA and then waited set amounts of time before adding SRP, to see how much translation could occur before SRP could no longer do its job.</p>
<p>The SRP receptor and the Sec61 proteins are ER membrane proteins &#8211; and there many other ER membrane, Golgi membrane and lysosome membrane proteins as well. In fact, even the <a href="http://en.wikipedia.org/wiki/Membrane_protein">membrane proteins</a> (see <a title="Cell Biology 02: The Plasma Membrane" href="/2013/02/10/cell-biology-02-the-plasma-membrane/">class 02</a>) of the cell membrane get processed in the secretory pathway.  Many of these have several or tens of transmembrane domains (20-25 hydrophobic amino acids each) that have to be inserted in the correct order and orientation (for example, you really want your ion channels and transporters pointed in the right direction, into vs. out of the cell).  Accordingly there are a bunch of fancy biological mechanisms for getting these proteins inserted into the membrane correctly.  This is what the latter half of the above video depicts.</p>
<p>So here&#8217;s a tautology: some proteins have a <a title="a sequence which determines a protein's orientation in the membrane" href="http://en.wikipedia.org/wiki/Topogenic_sequence">topogenic sequence</a> which determines their orientation in the membrane.  This sequence is made of two types of signal sequences:</p>
<ul>
<li>a<strong> stop-transfer sequence</strong> (abbreviated STA for some reason) is a 22-25 hydrophobic amino acid sequence somewhere in the middle of the protein that forms an alpha helix.  When encountered it gets shoved into the membrane, and then translation of the rest of the protein continues in the cytosol.  So this kind of &#8216;undoes&#8217; the translocation to the ER that was started by the signal peptide at the beginning (N terminus) of the protein.</li>
<li>a <strong>signal anchor sequence</strong> (abbreviated SA) is also a 22-25aa hydrophobic alpha helix, but with a series of ~3 positively charged amino acids on its left or right.  Like the signal peptide, this is recognized by SRP, which brings the ribosome to the ER.  But unlike the signal peptide, this alpha helical sequence will be inserted into the ER membrane.  The orientation of insertion is determined by the 3 positively charged amino acids. The positive charges have to always end up on the cytosolic side, so if they come after (i.e. C-terminal of) the hydrophobic sequence, the protein ends up with its C terminal end pointed into the cytosol, but if they come before (i.e. N-terminal of) the hydrophobic sequence, the protein ends up with its N terminus pointed into the cytosol.</li>
</ul>
<p>With those two signals as building blocks, you can imagine a protein with a series of stop transfer and signal anchor sequences to create a whole series of back and forth transmembrane domains stitched into the membrane as if by a sewing machine.  People have classified the membrane proteins into five categories:</p>
<ol>
<li>Type I has just a signal peptide and then one stop transfer in the middle.  Therefore it ends up with its (hydrophilic) N terminus in the lumen, its (hydrophobic) middle in the membrane and its (hydrophilic) C terminus in the cytosol.</li>
<li>Type II does not start with a signal peptide.  It starts out like any other protein, but in the middle it has a signal anchor sequence with the +++ amino acids coming first and the hydrophobic series after.  This makes the protein get translocated midway through translation, with the already-translated N-terminal part sticking out into the cytosol (since the +++ have to stay cytosolic) and the now-beginning-to-be-translated C-terminal part getting translated directly into the ER.  So it ends up transmembrane with its C terminus in the ER and N terminus in the cytosol &#8211; opposite of Type I.</li>
<li>Type III is like Type II &#8211; no signal peptide, just a signal anchor in the middle, but in this case the +++ come after the hydrophobic sequence, which reverses the orientation.  So this ends up with its N terminus in the ER and its C terminus in the cytosol.  Opposite of Type II and, in the end, the same as Type I, though it got there in a different way &#8211; it does not have a signal peptide that gets cleaved off in the ER.</li>
<li>Type IV or &#8217;multipass&#8217; proteins have an alternating series of signal sequences and stop transfer sequences.  These are clearly more than one &#8216;type&#8217;, yet are not nearly as diverse as your combinatoric imagination might allow.  The orientation of the first signal sequence determines whether the N terminus will end up in the cytosol or ER, and total number of stop transfer + signal anchor sequences determines where the C terminus will end up: an even number = same side as N terminus, odd number = opposite side as N terminus.  The STA and SA sequences have to strictly alternate, with the exception that you can start with two signal anchor sequences if the first one is oriented with the N terminus into the cytosol.  Just to make a mockery of this categorization scheme, people have defined some incompletely-defined subtypes of Type IV, where Type IVa is N-terminal in cytosol (thus it starts like a Type II protein) and Type IVb is N-terminal in the lumen (it starts like a Type III protein but then has another SA sequence that puts it back into the ER).   <a href="http://en.wikipedia.org/wiki/GLUT1">GLUT1</a> from <a title="Cell Biology 02: The Plasma Membrane" href="/2013/02/10/cell-biology-02-the-plasma-membrane/">Class 02</a> is a Type IVa.</li>
<li><a href="http://en.wikipedia.org/wiki/Glycophosphatidylinositol">GPI</a>-anchored proteins, which are the fifth type but aren&#8217;t called Type V, start with a signal peptide and end with a hydrophobic C-terminus which stays embedded in the membrane.  That hydrophobic end gets cleaved off and replaced with GPI, which also stays embedded in the membrane.  PrP is one of these &#8211; more on that later.</li>
</ol>
<p>By now we&#8217;ve discussed how proteins can end up in the ER lumen or spanning the ER membrane.  Most proteins leave the ER within minutes, transported in vesicles bound for the Golgi and then later for excretion, lysosomes or the cell membrane.  That forward direction of travel is called anterograde; going backwards from Golgi to ER is retrograde transport.</p>
<p>Both types of transport take place in membrane-bound <a href="http://en.wikipedia.org/wiki/Vesicle_(biology_and_chemistry)">vesicles</a>.  These bud off of the membrane of wherever they&#8217;re coming from, and later fuse to the membrane of wherever they&#8217;re headed &#8211; beautifully depicted at ~2:25 in the Life of the Cell video above.  The body from which the vesicles form is the &#8216;donor compartment&#8217;, and the destination they later fuse to is the &#8216;acceptor compartment&#8217;.</p>
<p>The budding process requires that <a title="proteins that bind GTP to be 'on' and GDP to be 'off'" href="http://en.wikipedia.org/wiki/G_protein">G proteins</a> in the membrane recruit Coat proteins.  Specifically, for anterograde transport, G protein <a href="http://en.wikipedia.org/wiki/SAR1A">Sar1</a> (gene: SAR1A) recruits <a href="http://en.wikipedia.org/wiki/COPII">COPII</a> (&#8216;cop two&#8217;); for retrograde transport, an <a title="ADP ribolysation factor" href="http://en.wikipedia.org/wiki/ADP_ribosylation_factor">ARF</a> G protein recruits <a href="http://en.wikipedia.org/wiki/COPI">COPI</a> (pronounced &#8216;cop one&#8217;).  These G proteins are activated to do this job when <a href="http://en.wikipedia.org/wiki/Guanine_nucleotide_exchange_factor">GEF</a> loads them with GTP, swapping out GDP.</p>
<p>So the steps in anterograde transport, for example, are as follows:</p>
<ol>
<li>Sec12-GEF (Sec stands for secretory) loads Sar1 with GTP.  When bound to GDP, Sar1 just floats around the donor compartment, but when bound to GTP, it undergoes conformational change that causes its otherwise-buried N-terminal hydrophobic tail to protrude, making it stick into the membrane, where COPII proteins then start to accumulate because they really like that tail.</li>
<li>The COPIIs start to polymerize and, due to its conformation, have an intrinsic preference for curvature, so their accumulation starts to make budding happen.  At the same time, membrane bound proteins that need to be transported &#8211;  identified by a DXE (i.e. aspartate-anything-glutamate) amino acid sequence that forms a binding site in their cytosolic part &#8211; get recruited to the newly forming vesicle.  Membrane-bound proteins act as receptors, recruiting lumenal proteins that are bound for the Golgi to hang out in the concave space where they&#8217;ll end up in the vesicle once it forms.</li>
<li>Once enough COPII have arrived, the vesicle buds off, at which point Sar1 hydrolyzes its GTP, providing the energy for it to suck its hydrophobic tail back into itself, cutting the COPIIs loose.  The vesicle is now disconnected from the donor compartment.</li>
<li>Now, for poorly explained (or poorly understood?) reasons, the coat of COPIIs just disassembles, exposing receptors under the coat which direct the targeting of the vesicle.  Once the vesicle arrives at its destination, <a href="http://en.wikipedia.org/wiki/Rab_(G-protein)">Rab</a>-GTP embedded in the vesicle membrane interacts with a Rab effector embedded in the acceptor compartment membrane.  A sideways glance is exchanged, interest is kindled.  Soon the vesicle will fuse to the membrane.</li>
<li><a href="http://en.wikipedia.org/wiki/SNARE_(protein)">SNARE</a> proteins present on both the <span style="text-decoration: underline;">v</span>esicle and <span style="text-decoration: underline;">t</span>arget membrane (V-SNARE and T-SNARE respectively) interact to bring the membranes even closer.  In this example we&#8217;ll consider <a title="vesicle-associated membrane protein" href="http://en.wikipedia.org/wiki/Vesicle-associated_membrane_protein">VAMP</a> (the VAMP_ genes) as the V-SNARE and <a href="http://en.wikipedia.org/wiki/Syntaxin">Syntaxin</a> (the STX__ genes) and <a href="http://en.wikipedia.org/wiki/SNAP25">SNAP25</a> (SNAP25 gene) as the T-SNAREs.  Syntaxin and SNAP25 are both membrane proteins; Syntaxin has 1 alpha helix and SNAP25 has 2, all on the cytosolic side.  The alpha helices drive the interaction with VAMP.  The opposing sides&#8217; alpha helices have extremely strong affinity for one another, bringing the membranes close enough to fuse.  Once this has happened, prying the V-SNAREs and T-SNAREs apart again requires two proteins: <a href="http://en.wikipedia.org/wiki/N-ethylmaleimide_sensitive_fusion_protein">NSF</a> (gene: NSF; stands for NEM sensitive factor) and <a href="http://en.wikipedia.org/wiki/NAPA_(gene)">alpha-SNAP</a> (gene: NAPA), a soluble NSF attachment protein.  NSF is an ATPase, and burns ATP to drive the energetically uphill disassembly of the complex.</li>
</ol>
<p>Now for retrograde transport.  Why is there retrograde transport at all?  Here is a non-exhaustive list of some reasons:</p>
<ul>
<li>Some membrane proteins start their life in the ER, need to get modified in the Golgi, but then need to get back to the ER.  They do this with a KKXX amino acid sequence.</li>
<li>There&#8217;s also a <a href="http://en.wikipedia.org/wiki/KDEL_(amino_acid_sequence)">KDEL</a> amino acid sequence at the C terminus of some lumenal proteins which is suppsoed to keep them in the ER, but it&#8217;s not perfect &#8211; sometimes they end up in the Golgi, in which case they&#8217;re targeted back to the ER via retrograde transport dependent on that KDEL sequence for recognition.  The mechanism is kind of neat &#8211; the proteins that recognize and bind to KDEL do so only at low pH, and the pH of the Golgi is lower than the ER, so they bind KDEL in the Golgi, then release it when they&#8217;re back in the more neutral pH of the ER.</li>
<li>Also, think about it, all the proteins that participate in anterograde transport &#8211; the V-SNARES, Rab, etc. &#8211; have to get back to the ER so they can do it all over again, like how the bus has to get back to the bus depot at the end of the day.</li>
<li>As we&#8217;ll see shortly, the Golgi come in multiple stages which depend on the addition of enzymes from further downstream.</li>
</ul>
<p>The process of retrograde transport is not so different from anterograde.  It uses ARF instead of Sar1, COPI instead of COPII, but it works the same: ARF loaded with GTP lets its hydrophobic tail stick into the membrane, attracting the attention of COPIs.  COPI has two components, COPIalpha and COPIbeta, both of which interact with that KKXXX sequence to recruit membrane-bound proteins destined for retrograde transport.  Some proteins also have an RR sequence (anywhere in the protein) which can flag them for retrograde transport.</p>
<p>The <a href="http://en.wikipedia.org/wiki/Golgi_apparatus">Golgi apparatus</a> is not contiguous.  It is a stacked set of separate subcompartments called sacs or cisternae.  Different compartments have different properties and proteins visit them in a particular order.  In order from ER to cell membrane, the Golgi compartments are called cis, medial, trans and trans-Golgi network.  Each compartment has different enzymes that modify proteins, and the modifications have to happen in a certain order, hence the need for a stacked set of compartments.</p>
<p>But as proteins mature in the Golgi, it&#8217;s not as though they bud off in vesicles from one compartment and move to the next.  Rather, the compartment they are <em>already in</em> moves outward and &#8216;matures&#8217; as new enzymes are added to it (from further down the Golgi chain) via retrograde transport.  Weird, right?  It&#8217;s kind of like if instead of moving from an elementary school to a middle school to a high school you just stayed in one school building for your whole childhood and adolescence, and they just brought in new textbooks and teachers every year to keep it appropriate to the grade that you and your classmates had now reached. Here&#8217;s what the Golgi look like as they move and evolve:<br/>
<iframe src="http://www.youtube.com/embed/9nd1AWVd26w" frameborder="0" width="420" height="315"></iframe></p>
<p>So there&#8217;s (little or) no anterograde transport within the Golgi, but plenty of retrograde transport to bring each new round of enzymes in.  When proteins have finally completed the full K-12 curriculum of the Golgi network, they do undergo transport to move on to their final destinaton.  They bud off in a vesicle which will go one of three places:</p>
<ul>
<li><a href="http://en.wikipedia.org/wiki/Exocytosis">Exocytosis</a> &#8211; fusion with the cell membrane.  Thus the lumenal proteins will be secreted extracellularly, and the membrane proteins will become cell membrane proteins.</li>
<li><a href="http://en.wikipedia.org/wiki/Vesicle_(biology_and_chemistry)#Types_of_secretory_vesicles">Secretory vesicles</a> &#8211; these just stick around as vesicles in the cell until needed &#8211; where &#8216;needed&#8217; means they do eventually undergo exocytosis.  In neurons, this is where neurotransmitters are stored until an action potential demands their secretion into the synapse.  In the stomach, the cells that produce gastric enzymes keep those enzymes in secretory vesicles until food intake triggers their release into the stomach.</li>
<li><a href="http://en.wikipedia.org/wiki/Lysosome">Lysosomes</a> - where misfolded proteins go to get degraded.</li>
</ul>
<p>The transport from the trans-Golgi network on to these destinations is different from the other transport discussed above and often involves <a href="http://en.wikipedia.org/wiki/Clathrin">clathrin</a> (CLT__ genes).  Vesicles budding off have a two-layer coat, with <span style="text-decoration: underline;">a</span>dapter <span style="text-decoration: underline;">p</span>rotein (AP) complexes as the inner layer and clathrin as the outer layer.  The adapter proteins have a target signal with a YXXh motif (h = Φ = any hydrophobic amino acid).  Clathrin forms the so-called &#8216;clathrin-triskelion&#8217; formation shown here:</p>
<p><a href="http://en.wikipedia.org/wiki/File:Clathrin.png"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/d/d5/Clathrin.png/320px-Clathrin.png" alt=""/></a><br/>
(Image thanks to Wikimedia Commons user Phoebus87)</p>
<p>Clathrin is also responsible for <a href="http://en.wikipedia.org/wiki/Endocytosis">endocytosis</a> &#8211; budding off of vesicles of extracellular stuff (and cell membrane proteins) to come <em>into </em>the cell.  This is called <a href="http://en.wikipedia.org/wiki/Receptor-mediated_endocytosis">clathrin-mediated endocytosis</a>.  Receptors in the cell membrane get endocytosed very frequently: the whole population of hormone receptors turns over about every hour, especially when hormones are being received.  Taking up the receptor into a vesicle is one way for the cell to cut off the incoming signal until it can be processed.<br/>
The <a title="Cell Biology 02: The Plasma Membrane" href="/2013/02/10/cell-biology-02-the-plasma-membrane/">plasma membrane notes</a> discuss cystic fibrosis briefly: <a href="http://en.wikipedia.org/wiki/Cystic_fibrosis_transmembrane_conductance_regulator">CFTR</a> is an ABC transporter responsible for pumping Cl<sup>-</sup> out of the cell (it also lets Na<sup>+</sup> in).  Loss-of-function mutants don&#8217;t pump Cl<sup>-</sup>, which removes the driving force for osmosis, thickening the mucus and causing breathing problems.  There are at least 127 different loss-of-function CFTR mutants (at least, <a href="http://www.natera.com/cf-127-testing.html">that&#8217;s how many Natera tests for</a>) that (if both alleles are disabled) cause cystic fibrosis.  The most common mutation is <a href="http://en.wikipedia.org/wiki/%CE%94F508">ΔF508</a>, which is ~3% of all European CFTR alleles and about 70% of mutant ones.  The loss of that one phenylalanine changes CFTR&#8217;s conformation so that the di-acidic exit code (amino acids D565 and D567) that targets CFTR for exocytotic vesicles is no longer correctly exposed and the protein never makes it to the cell membrane [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2172508/">Wang 2004</a>].</p>
<p><strong>discussion section</strong></p>
<p>In section we read <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2746359/">Hu 2009</a>, who showed that <a href="http://en.wikipedia.org/wiki/Atlastin">atlastin</a> proteins are involved in creating the tubular ER network. The evidence came almost entirely from protein-protein interactions.  I was surprised this paper was a big deal, because there have been a million papers showing protein-protein interactions for <a href="http://en.wikipedia.org/wiki/Huntingtin">huntingtin</a>, and no one really believes all of them and it hasn&#8217;t necessarily gotten us any closer to knowing what huntingtin does or what goes wrong in Huntington&#8217;s Disease.  But apparently Hu was able to make a pretty clean case for the atlastins&#8217; interactions with <a href="http://en.wikipedia.org/wiki/Reticulon">reticulons</a> as implying a role in ER formation.  It helps that Hu was able to show a &#8216;genetic interaction&#8217; in addition to a physical (binding) interaction.  A &#8216;genetic interaction&#8217; (I had to look it up) means when &#8220;Sometimes mutations in two genes produce a phenotype that is surprising in light of each mutation&#8217;s individual effects. This phenomenon, which defines genetic interaction, can reveal functional relationships between genes and pathways.&#8221; [<a href="http://www.pnas.org/content/105/9/3461.abstract">Mani 2007</a>].</p>
<p><strong>PrP</strong></p>
<p>This is a decade old, so some stuff may be outdated, but I found <a href="http://www.ncbi.nlm.nih.gov/pubmed/14522850">Harris 2003</a> (<a href="http://bmb.oxfordjournals.org/content/66/1/71.full">ft</a>)&#8217;s review of PrP cell biology extremely clear and helpful.  <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC133591/">Kim &amp; Hegde 2002</a> was also helpful.  <a href="http://en.wikipedia.org/wiki/PRNP">PrP</a> is a secretory pathway protein.  Its first 22 amino acids (MANLGCWMLVLFVATWSDLGLC) are a signal peptide that causes cotranslational translocation to the ER.  Normally, PrP just gets GPI-linked at its C terminus and is anchored to the exoplasmic side of the membrane.  But amino acids 111-134 (HMAGAAAAGAVVGGLGGYMLGSAM) are a sort of weak<em> </em>signal anchor sequence (Type II, with the +++ amino acids coming before the signal anchor) that <em>sometimes but not always</em> becomes a transmembrane domain, inverting the C terminus into the lumen.  Even more confusingly, that sequence can sometimes just end up as a transmembrane domain <em>without</em> the inversion, so that the N terminus is in the lumen.  So there are three membrane topologies of PrP: regular old GPI-anchored, and two transmembrane orientations, as depicted in <a href="http://bmb.oxfordjournals.org/content/66/1/71/F4.expansion.html">Harris 2003 Fig 3</a>:</p>
<p><a href="http://bmb.oxfordjournals.org/content/66/1/71/F4.expansion.html"><img class="alignnone size-full wp-image-1677" title="harris-2003-fig3" src="/wp-content/uploads/2013/02/harris-2003-fig3.jpg"/></a></p>
<p>Note how weird <sup>Ctm</sup>PrP is.  It&#8217;s transmembrane yet also GPI-anchored, and the N-terminal signal peptide is never cleaved off.  Normally, the transmembrane forms are &lt; 10% of total PrP.  In some laboratory conditions the percentage is higher, and two of the <a href="http://en.wikipedia.org/wiki/Gerstmann%E2%80%93Str%C3%A4ussler%E2%80%93Scheinker_syndrome">GSS</a>-causing mutations (A117V and P105L) also increase the fraction of <sup>Ctm</sup>PrP to 20-30% of all PrP.  Of these three forms, there is a good amount of evidence that <sup>Ctm</sup>PrP is toxic, and that it might play a role in prion formation, though most genetic prion disease mutations (including FFI D178N) do not appear to affect the membrane topology of PrP or the fraction of  <sup>Ctm</sup>PrP.</p>
<p>After PrP goes through the Golgi, it is targeted for the cell membrane.  But according to Harris, it doesn&#8217;t just sit there &#8211; it frequently through <a href="http://en.wikipedia.org/wiki/Receptor-mediated_endocytosis">clathrin-mediated endocytosis</a> and cycles through the cell every ~60 minutes, with some molecules being cleaved on each cycle.  Copper stimulates this endocytosis of PrP.  Most genetic prion disease mutations change the localization of PrP &#8211; usually when a mutation is present, less PrP is found on the cell surface, with more accumulating in the ER.</p>
