---
layout: post
title: "Proteolysis of prion protein in the central region: alpha and beta cleavage"
author: ericminikel
date: 2013-06-26 20:59:04
---
<p>Proteo- means protein, and -lysis means clevage. This post is about where and how prion protein (PrP) is cleaved in its central, highly conserved region.  There are two types of PrP cleavage in this region, known as alpha and beta cleavage.</p>
<p><strong>the central region of PrP</strong></p>
<p>If you zoom out from Mammalia and look at other vertebrates, there is only one large part of PrP that is really conserved even between us, chickens and zebrafish.  You can see it in <a href="/wp-content/uploads/2012/11/t-coffee-human-mouse-chicken-zebrafish.html">this multiple alignment</a> from <a title="Multiple alignment of mammalian PrP amino acid sequences" href="/2012/11/14/multiple-alignment-of-mammalian-prp-amino-acid-sequences/">this post</a>:</p>
<p><a href="/wp-content/uploads/2013/06/prp-conservation-104-160.png"><img class="alignnone size-full wp-image-2486" alt="prp-conservation-104-160" src="/wp-content/uploads/2013/06/prp-conservation-104-160.png"/></a></p>
<p>This stretch runs from human amino acids 104 &#8211; 160, with particularly strong conservation at 112 &#8211; 150 or so.</p>
<p>The first half of this tract, approximately 112 &#8211; 134, is the so-called &#8220;hydrophobic core&#8221;, rich in A, M, L and V that gets embedded in the membrane in <sup>Ctm</sup>PrP and <sup>Ntm</sup>PrP [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pubmed/14522850">Harris 2003</a> (<a href="http://bmb.oxfordjournals.org/content/66/1/71.full">ft</a>)].  Mutations that increase the hydrophobicity of this stretch cause GSS and tend to increase the fraction of PrP that ends up with a <sup>Ctm</sup>PrP transmembrane topology [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9452375">Hegde 1998</a> (<a href="http://cbmp.nichd.nih.gov/upb/pdfs/827.pdf">ft</a>)].</p>
<p>Overlapping with the &#8220;hydrophobic core&#8221; are codons 106-126, believed by some to be the toxic region of PrP.  When it is synthesized as a standalone peptide, PrP 106-126 forms amyloid fibrils [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8105481/">Tagliavini 1993</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC47633/pdf/pnas01527-0453.pdf">ft</a>)], and is neurotoxic to cultured hippocampal cells [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8464494">Forloni 1993</a>], but only if those cells express PrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7865744">Brown 1994</a>].  Conversely, ΔCR PrP, which precisely <em>lacks</em> this &#8220;<span style="text-decoration: underline;">C</span>entral <span style="text-decoration: underline;">R</span>egion&#8221;, is highly toxic and neonate lethal, though its toxicity can be reduced by co-expression of wild-type PrP [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1783448/">Li 2007</a>].  All this is quite interesting and certainly, together with the phylogenetic conservation of this region, says something important is going on there.</p>
<p>This central region of PrP &#8211; between the octapeptide repeats (codons 60-92) and the more structured C-terminal region (beginning with beta sheet 1 at codon ???) &#8211; also turns out to be where most of the action is in terms of proteolysis.  PrP can undergo so-called &#8220;alpha cleavage&#8221; before or after codon 111 to yield two fragments dubbed N1 and C1, or it can undergo &#8220;beta cleavage&#8221; which occurs before codon 91 or earlier, yielding fragments called N2 and C2.  Here&#8217;s a diagram:</p>
<p><a href="/wp-content/uploads/2013/06/prp-cleavage-sites-core-numbered-hires.png"><img class="alignnone size-large wp-image-2498" alt="prp-cleavage-sites-core-numbered-hires" src="/wp-content/uploads/2013/06/prp-cleavage-sites-core-numbered-hires-1024x317.png"/></a></p>
<p><strong>alpha cleavage</strong></p>
<p>Cleavage before or after H111 to generate the N1 and C1 fragments, now called alpha cleavage, was first reported in chicken PrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8093841">Harris 1993</a> (<a href="http://lab.rockefeller.edu/chait/pdf/93/93_harris_biochem.pdf">ft</a>)] and later confirmed in human samples [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7642585/">Chen 1995</a> (<a href="http://www.jbc.org/content/270/32/19173.long">ft</a>)].  Some of the proteolytic enzymes involved in this cleavage were identified (in human cells) several years later [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11477090">Vincent 2001</a> (<a href="http://www.jbc.org/content/276/41/37743.long">ft</a>)].  The best introduction I found to the subject is a recent review [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3510859/">Liang &amp; Kong 2012</a>].</p>
<p>The <a href="http://en.wikipedia.org/wiki/A_disintegrin_and_metalloproteinase">ADAM</a> family of proteolytic enzymes &#8211; originally from one enzyme dubbed &#8220;<b><span style="text-decoration: underline;">a</span> <span style="text-decoration: underline;">d</span>isintegrin <span style="text-decoration: underline;">a</span>nd <span style="text-decoration: underline;">m</span>etalloproteinase</b>&#8221; &#8211; are involved in alpha cleavage of PrP.  The original report [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11477090">Vincent 2001</a> (<a href="http://www.jbc.org/content/276/41/37743.long">ft</a>)] implicated <a href="http://en.wikipedia.org/wiki/ADAM10">ADAM10</a> and <a href="http://en.wikipedia.org/wiki/ADAM17">ADAM17</a> (latter formerly known as TACE).  <a href="http://en.wikipedia.org/wiki/ADAM9">ADAM9</a> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16236709/">Cisse 2005</a> (<a href="http://www.jbc.org/content/280/49/40624.long">ft</a>)] and, in muscle at least, <a href="http://en.wikipedia.org/wiki/ADAM8">ADAM8</a> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3351330/">Liang 2012</a>] have also been added to the list.  There&#8217;s still a fair bit of controversy over which of these is directly involved and how they are regulated [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3510859/">Liang &amp; Kong 2012</a>].  The cleavage event probably does not occur on the cell surface.  Instead it is thought to occur either in endocytic vesicles [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8101844/">Shyng 1993</a> (<a href="http://www.jbc.org/content/268/21/15922.full.pdf">ft</a>)] or the late secretory pathway [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19056496/">Walmsley 2009</a>].</p>
<p>Alpha cleavage appears to occur quite often in uninfected brains and there is about half as much C1 present as there is full-length PrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7642585/">Chen 1995</a> (<a href="http://www.jbc.org/content/270/32/19173.long">ft</a>)].</p>
<p><strong>beta cleavage</strong></p>
<p>As shown in the diagram, beta cleavage can occur at a variety of sites.  Wherever it occurs, it is said to generate fragments called N2 and C2.  A bit of history is in order to clarify the distinction between C2 and PrP 27-30.</p>
<p>The first protocol for homogeneous purification of PrP used proteinase K digestion as one step [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6418385">Prusiner 1983</a>].  This protocol applied to infected hamster brains yielded a 27 &#8211; 30 k<a href="http://en.wikipedia.org/wiki/Atomic_mass_unit">Da</a> protein dubbed PrP 27-30, so pure that it was possible to read the N-terminus of the PK-resistant protein fragment, which started with GQGGGTH [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6432339">Prusiner 1984</a>], i.e. starting with the equivalent of human codon 90.  But when polyclonal antibodies were finally raised against PrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6431296">Bendheim 1984</a>], it was discovered that they not only pulled down PrP 27-30 from infected hamster brains, but also pulled down a 33-35 kDa version of PrP in both infected and uninfected hamster brains [<a href="http://www.ncbi.nlm.nih.gov/pubmed/2859120">Oesch 1985</a>].  Proteinase K was capable of completely digesting PrP 33-35 from uninfected brains, but only digested part of PrP 33-35 from infected brains, resulting in PrP 27-30.</p>
<p>The PrP 27-30 pulled down by the antibodies without PK digestion probably represented C2 fragments that had already been cleaved in the brain, while PrP 33-35 was full-length PrP, which in infected brains contained a protease-resistant fragment (approximately? exactly?) coincident with C2.</p>
<p>We now know that the protease-resistant fragment varies in size in different prion strains [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9069279">Parchi 1997</a>]: so-called &#8220;type 1&#8243; PrP<sup>Sc</sup> contains a 21 kDa fragment starting at G82, and &#8220;type 2&#8243; PrPSc contains a 19kDa fragment starting at S97 [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pubmed/21107851">Parchi 2011</a>].  (Those are the molecular weights after de<a title="Prion protein N-linked glycosylation: review and assessment of therapeutic potential" href="/2013/05/05/prion-protein-n-linked-glycosylation-review-and-assessment-of-therapeutic-potential/">glycosylation</a>.)</p>
<p>We also know now that C2 is created even in uninfected brains, just at levels too low for Oesch to have detected.  In retrospect, the earliest report of beta cleavage of PrP<sup>C</sup> may have been [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1363897/">Pan 1992</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2142104/pdf/1363897.pdf">ft</a>)].  Pan reported on purification of PrP<sup>C</sup> from the hamster brain and found &#8220;PrPC-I&#8221; (full-length PrPC), &#8220;PrPC-II&#8221; (the C1 fragment), and also reported that:</p>
<blockquote><p>Interestingly, an immunoreactive 30-kDa band, located just below PrPC-I and often obscured by its intense staining, copurified with PrPC throughout the purification&#8230; αP1 raised against a synthetic peptide homologous to the N-terminus of PrP 27-30, also reacted with this 30-kDa protein&#8230;</p></blockquote>
<p>This 30 kDa protein must have been the (diglycosylated) C2 fragment.  It contained more of the N-terminus of PrP than the C1 fragment did, so it reacted with antibodies raised against the protease resistant PrP 27-30.  It was present in uninfected brains, meaning it was the result of a normal biological cleavage process and suggesting it didn&#8217;t necessarily result from protease resistance.</p>
<p>Soon it was confirmed that this C2 fragment is naturally present in uninfected human brains too [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7642585/">Chen 1995</a> (<a href="http://www.jbc.org/content/270/32/19173.long">ft</a>), <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1853409/">Jimenez-Huete 1998</a>], though it wasn&#8217;t abundant.  Chen found there was only about 5% as much C2 as there was full-length PrP<sup>C</sup>, or about 10% as much C2 as C1, indicating there isn&#8217;t a ton of beta cleavage going on under normal healthy circumstances.  By contrast, C2 is enriched in prion disease: the brains of CJD patients contain large amounts of C2 &#8211; i.e. without proteinase K digestion, one finds both full-length PrP<sup>Sc</sup> as well as PrP<sup>Sc</sup> corresponding to only the C2 portion of the protein, just as in Oesch&#8217;s hamsters.</p>
<p>Later on, detection of the N2 fragment confirmed that C2 resulted from <em>cleavage</em> rather than <em>digestion</em> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/15050367/">Mange 2004</a>].   It was also discovered that beta cleavage can be stimulated by reactive oxygen species (ROS) [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11060296">McMahon 2001</a> (<a href="http://www.jbc.org/content/276/3/2286.long">ft</a>)].  McMahon added hydrogen peroxide (<a href="http://en.wikipedia.org/wiki/Hydrogen_peroxide">H<sub>2</sub>0<sub>2</sub></a> - a source of ROS) and superoxide (<a href="http://en.wikipedia.org/wiki/Superoxide">O<sub>2</sub><sup>-</sup></a> - a ROS itself), to cell cultures (<a href="http://en.wikipedia.org/wiki/Chinese_hamster_ovary_cell">CHO</a> cells) and found that incubation of cells with these oxidizing agents caused a loss of full-length PrP<sup>C</sup> in favor of the C2 fragment.  The effect was deemed &#8220;probably a non-enzymatic event&#8221; &#8211; i.e. the ROS don&#8217;t recruit a proteolytic enzyme to cleave PrP, but rather, the ROS directly react with PrP to cleave a peptide bond, something which apparently is possible (McMahon cites <a href="http://www.ncbi.nlm.nih.gov/pubmed/2866184">Kim 1985</a> (<a href="http://www.jbc.org/content/260/29/15394.full.pdf">ft</a>)).</p>
<p>It has since been reported that beta cleavage can also be accomplished enzymatically, by calpains [<a href="http://www.ncbi.nlm.nih.gov/pubmed/15026410/">Yadavelli 2004</a> (<a href="http://www.jbc.org/content/279/21/21948.long">ft</a>)] and cathepsin [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2856230/">Dron 2010</a>].  There&#8217;s slightly more coverage of this issue in another review [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3560451/">Altmeppen 2012</a>].  The enzymatic beta cleavage has only been reported for PrP<sup>Sc</sup>, not PrP<sup>C</sup>.</p>
<p><em>In vivo</em>, PrP<sup>Sc</sup> beta cleavage can occur at G74, G78, G82, G86, G90, G92, S97, W99, or S103 [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27779/">Parchi 2000</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27779/table/T1/">Table 1</a>], with the distribution for type 1 PrP<sup>Sc</sup> centered on G82 (in the third octarepeat) and for type 2 PrP<sup>Sc</sup> centered on S97.  These sites are shown in the diagram of the central region above.  <em>In vitro</em>, at least, beta cleavage of PrP<sup>C</sup> has also been reported as early as the second octarepeat [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11060296">McMahon 2001</a> (<a href="http://www.jbc.org/content/276/3/2286.long">ft</a>)].</p>
<p><strong>disease implications of alpha and beta cleavage</strong></p>
<p>Alpha cleavage cleaves through the core region of amino acids 106-126.  This is thought to render the remaining C1 fragment incapable of converting to PrP<sup>Sc</sup>.  The best line of evidence for this is that mice constitutively expressing only C1 cannot be infected with prions [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3243553/">Westergard 2011</a>]. Indeed C1 appears to be dominant negative, since Westergard found that its co-expression with full-length PrP lengthens incubation times compared to when only full-length PrP is expressed. It has been reported that N1 is also neuroprotective [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2791025/">Guillot-Sestier 2009</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3281657/">Guillot-Sestier 2012</a>].  While a few other studies have suggested that C1 makes cells or animals more sensitive to pro-apoptotic stimuli (which is bad), Westergard observed no neurological symptoms in C1 mice even with ~7x expression levels.   Overall, evidence seems to point to alpha cleavage being a good thing from the standpoint of prion disease.</p>
<p>Beta cleavage, by contrast, leaves PrP 106-126 intact, and when a C2 fragment starting at amino acid 73 is expressed alone, it is still capable of supporting prion disease [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8635458/">Fischer 1996</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC450028/pdf/emboj00006-0055.pdf">ft</a>)].  This, together with the observation that C2 is more abundant in prion-diseased brains than healthy ones [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7642585/">Chen 1995</a> (<a href="http://www.jbc.org/content/270/32/19173.long">ft</a>)], casts beta cleavage as a bad thing from the standpoint of prion disease.</p>
<p>All this is too simplistic &#8211; it&#8217;s likely that both types of cleavage have a native function, and there is not yet quite enough evidence to say that beta cleavage has any <em>causal</em> relationship with prion pathology or disease progression.  But it does appear that the two are mutually exclusive, and that alpha cleavage precludes a PrP molecule from becoming PrP<sup>Sc</sup>.</p>
<p>Interesting aside: all of this, including the nomenclature, closely parallels the cleavage of APP and its relation to Alzheimer&#8217;s disease.  <a href="http://en.wikipedia.org/wiki/Amyloid_precursor_protein">Amyloid precursor protein</a> (APP) can be cleaved by <a href="http://en.wikipedia.org/wiki/Alpha_secretase">alpha secretase</a> or <a href="http://en.wikipedia.org/wiki/Beta_secretase">beta secretase</a> (BACE1).  These are mutually exclusive, and the latter generates <a href="http://en.wikipedia.org/wiki/Beta_amyloid">amyloid beta</a> (Aβ), so alpha secretase cleavage of APP may be seen as benevolent and preventive.  <a href="http://en.wikipedia.org/wiki/Beta-secretase_1">BACE1 inhibitors</a> are under investigation as potential anti-Alzheimer&#8217;s drugs.  <a href="http://en.wikipedia.org/wiki/A_disintegrin_and_metalloproteinase">ADAM</a> peptidases are involved in the alpha cleavage of both APP and PrP (probably the reason for the common &#8220;alpha&#8221; label), and we now know <a title="Review of prion protein and amyloid beta interactions" href="/2013/05/02/review-of-prion-protein-and-amyloid-beta-interactions/">PrP is a receptor for Aβ</a>.</p>
<p><strong>preview</strong></p>
<p>Besides alpha and beta cleavage, PrP can also be cleaved at the extreme C terminus, &#8220;shedding&#8221; the almost-full-length protein from the cell surface.  Shedding will be the subject of <a href="/2013/06/29/proteolytic-shedding-of-prion-protein/">my next post</a>.</p>
