---
layout: post
title: "Introduction to conformation-dependent immunoassay"
author: ericminikel
date: 2013-07-29 07:45:22
---
<p><strong>background</strong></p>
<p>PrP was first discovered and purified using a protocol that included proteinase K digestion and yielded a 27-30 kDa protein fragment dubbed PrP 27-30, which was found in the brains only of infected animals [<a href="http://www.ncbi.nlm.nih.gov/pubmed/6418385">Prusiner 1983</a>].  But soon it became clear that PrP was not a strictly foreign pathogen but in fact a host-encoded protein, a gene present in the genome of infected and uninfected animals [<a href="http://www.ncbi.nlm.nih.gov/pubmed/3923361">Chesebro 1985</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/2859120">Oesch 1985</a>].  However the PrP in infected animals appeared to differ from that in uninfected animals by some post-translational event.   PrP<sup>Sc</sup> therefore came to be operationally defined by two biochemical properties that PrP<sup>C</sup> lacked: protease resistance and detergent insolubility.</p>
<p>That paradigm stood pretty solid for over a decade.  By the late 1990s, PrP’s identity as the “scrapie agent” was pretty well-accepted &#8211; like, Nobel Prize well-accepted.  Still, occasionally results came up that didn&#8217;t fit with the paradigm of infectivity being inseparable from PrP-res &#8211; for instance, BSE on first passage in mice was found to cause disease without any PrP-res at all [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8994041">Lasmezas 1997</a>], and a few skeptics continued to question whether PrP was really the disease agent.  But the world would soon learn that the cracks beginning to show weren&#8217;t cracks in the prion hypothesis, just in the particular operational definition of PrP<sup>Sc</sup> in use at the time.</p>
<p>Luckily by this time a third property of PrP<sup>Sc</sup> had attracted notice.  Besides its PK resistance and detergent insolubility, its abnormal conformation also buried several epitopes that are normally exposed on the surface of the protein in PrPC.  This meant that some antibodies, including 3F4, were in practice specific to PrPC and would not bind to PrPSc [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1967489">Serban 1990</a>].</p>
<p>The conformation-dependent immunoassay or CDI [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9771749">Safar 1998</a>] turned this conformational property into an assay.  In developing this assay, Safar accomplished several things.  He clearly demonstrated the existence of proteinase K-sensitive PrP<sup>Sc</sup>, and created a tool for measuring it.  He also created a tool for quantifying certain dimensions of conformational diversity among prion strains.</p>
<p>Here’s how it works.  Guanidium hydrochloride (GdnHCl) denatures proteins &#8211; including even PrP<sup>Sc</sup> at high enough concentrations.  Denature here means causing the proteins to unfold, to lose their secondary, tertiary and quaternary structure &#8211; the proteins maintain primary structure and are not degraded.  Safar found that if you treat PrP<sup>Sc</sup> with GdnHCl, 3F4 (and other antibodies) will bind once again, just as they do to PrP<sup>C</sup>.  So: measure the binding of 3F4 <i>before</i> GdnHCl treatment (PrP in its “native state” or N) and <i>after </i>GdnHCl treatment (“denatured state” or D).   In principle, the “N” quantity is PrP<sup>C</sup> and “D” is total PrP, so D &#8211; N = PrP<sup>Sc</sup>, including both PK-resistant and PK-sensitive fractions. He refers to these as rPrP<sup>Sc</sup> and sPrP<sup>Sc</sup> respectively.</p>
<p><a href="/wp-content/uploads/2013/07/cdi-fractions.png"><img class="alignnone size-full wp-image-2604" alt="cdi-fractions" src="/wp-content/uploads/2013/07/cdi-fractions.png"/></a></p>
<p>The principle of it is that simple.  In reality, this diagram is a great oversimplification.  For one, even PrP<sup>C</sup> actually has slightly increased affinity for 3F4 after denaturation with a small (1M) concentration of GdnHCl [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9771749">Safar 1998</a> Fig 2a], though it returns to its native affinity level at even higher concentrations [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9771749">Safar 1998</a> Fig 4a, “C” control curve].  For another, in practice this assay is conducted in an ELISA format with fluorescence as a readout, which means that the ability to detect antibody binding at all, and especially to detect a difference in binding in denatured vs. native protein, depends on the total concentration of PrP, level of background from other proteins, etc.  Much of the paper is devoted to establishing the sensitivity and dynamic range of the assay.</p>
<p>But amidst these annoying real-life implementation complexities, CDI also has another complexity that makes it extra useful.  Above I’ve just said “GdnHCl treatment”, but I didn’t say <i>how much</i> GdnHCl.  It turns out that by the time you get up to 6.5 M (quite a high concentration indeed), pretty much 100% of PrP<sup>Sc</sup> is denatured, in every strain where this has been tested.  So if you just wanted a way to quantify total PrP<sup>Sc</sup>, then D &#8211; N at 6.5M GdnHCl is your number.  On the other hand, the fact that GdnHCl causes only incomplete denaturation at lower concentrations is informative about strain properties.  Fig 4 presents curves for how the D/N ratio changes for three different hamster prion strains at different GdnHCl concentrations &#8211; each strain is unique, indicating a different degree of susceptibility to denaturation, a property prion researchers often refer to as “conformational stability.”  To sum up such a curve in just one number, many people refer to the [GdnHCl]<sub>½</sub> value of a given strain, which is the concentration of GdnHCl at which the strain is half denatured, i.e. has half the D/N ratio that it has at 6.5M GdnHCl.</p>
<p>CDI also offers other ways of typing strains.  Since it is capable of measuring total PrP<sup>Sc</sup>, it allows one to quantify the concentration (μg/mL) of PrP<sup>Sc</sup> in brain samples taken at the time of terminal illness.  This concentration, too, turns out to vary by strain.  Finally, because CDI can be combined with PK digestion, it is possible to determine the fraction of PrP<sup>Sc</sup> that is PK-resistant, a property which, again, differs by strain.</p>
<p>Used directly on unpurified brain homogenate, CDI is great for strain typing but not amazing at detecting small amounts of PrP<sup>Sc</sup>.  Safar offered an improvement on the assay’s sensitivity by also showing that sodium phosphotungstate (NaPTA) precipitates PrP<sup>Sc</sup> (regardless of PK resistance) from brain homogenate, making it possible to run CDI on a purer sample and detect as few as 10<sup>3</sup> infectious doses (ID<sub>50</sub>s).  Later authors have created other variations on CDI that offer yet more sensitivity [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12810888">Bellon 2003</a> (<a href="http://vir.sgmjournals.org/content/84/7/1921.long">ft</a>)].</p>
<p>CDI is sufficiently labor-intensive that plenty of researchers don’t bother with it.  At the same time, it doesn’t seem an overstatement to say that CDI revolutionized the way we define PrP<sup>Sc</sup>.  Other protocols have since been developed for purifying particular fractions of PK-sensitive PrP<sup>Sc</sup> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/17176093">Pastrana 2006</a>] but no other method offers the ability to quantify the entirety of PK-sensitive PrP.</p>
<p>This quantification has led to some surprises.  Of the eight hamster prion strains that Safar typed in his original paper, PrP<sup>Sc</sup> turned out to comprise about 75 &#8211; 90% of total PrP in the brain at the time of terminal illness.  (This is phrased as a 3-10 fold excess of PrP<sup>Sc</sup> over PrP<sup>C</sup>).  The total concentration of PrP<sup>Sc</sup> also differed from about 10 to 40 μg/mL.  The proportion of PrP<sup>Sc</sup> that was PK resistant also varied widely &#8211; percentages are never given, but can be eyeballed from Fig 3b-c, for instance, DY has about 12 μg/mL PrP<sup>Sc</sup> and only &lt; 1 μg/mL rPrP<sup>Sc</sup>, meaning probably &lt; 8% is PK-resistant.  ME7 is more like 40 vs. 8 μg/mL, so ~20% PK-resistant.  HY is about 26 vs. 18, so ~70% PK-resistant.</p>
<p>A few years later, Safar applied this assay to typing human sCJD cases, with equally surprising results [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC552933/">Safar 2005</a>].  PrP<sup>Sc</sup> in sCJD turns out to be at minimum 80% PK-sensitive, and on average more like 90%.  (He combines data for 8 patients with different 129 genotypes and Type 1 vs. 2).  CDI also detected PrP<sup>Sc</sup> in every single brain region in every single patient &#8211; even in regions where MRI and histological examination did not turn up anything.   This placed CDI in the running as a possible diagnostic tool, though that has never really come to be, and instead RT-QuIC now looks poised to become the first biochemical PrP<sup>Sc</sup> diagnostic test [<a href="http://www.ncbi.nlm.nih.gov/pubmed/21278748">Atarashi 2011</a> (<a href="http://naosite.lb.nagasaki-u.ac.jp/dspace/bitstream/10069/25642/1/NatMEd17_175.pdf">ft</a>) and see overview by <a href="http://www.cjdfoundation.org/shared/cjdfoundation/files/webfm/admin/2012-conference/Byron-Caughey.pdf">Caughey 2012</a>].</p>
<p>Safar’s CDI data have also served as a basis for a good deal of hypothesizing about some of the fundamental mysteries of prion disease.  In Fig 3d, Safar plots strain incubation time (y axis) vs. the concentration of PK-sensitive PrPSc found in the brain at terminal illness (x axis).  He finds a linear correlation, and argues that this represents something fundamental about prion pathogenesis:</p>
<blockquote><p>the different incubation times of various prion strains should arise mostly from distinct rates of PrPSc clearance rather than from different rates of PrPSc formation24. Prion strains that are readily cleared, therefore, should have prolonged incubation times, whereas those that are poorly cleared should have abbreviated incubation periods. We investigated this by relying upon the difference in brain PrPSc concentrations before and after proteinase K treatment as a surrogate for <i>in vivo </i>clearance of each prion strain. When clearance, as approximated by [PrPSc]–[PrP 27-30], was plotted as a function of the incubation time for eight strains, a linear relationship was found</p></blockquote>
<p>I find myself unable to agree with this interpretation.  I don’t have the raw data but it sure looks like the linear correlation in Fig 3d is almost entirely driven by ME7, which is an outlier on both dimensions, with a 300+ day incubation time and a huge concentration of sPrP<sup>Sc</sup>.  I also notice that the error bars on many of the dots (strains) are large enough to overlap.  To me, it’s not clear there’s even a real correlation here.</p>
<p>Supposing there was a correlation, though, I don’t see the basis for equating PK-sensitive PrP<sup>Sc</sup> concentration with “clearance”.  If PK-sensitive ME7 prions are so easy to clear, then why are there so many of them by the time of terminal illness?  Far from representing clearance, the level of sPrP<sup>Sc</sup> would seem to reflect the cell’s <i>inability</i> to clear it.  PrP<sup>Sc</sup>, whether sensitive or resistant, seems to accumulate over the course of prion disease -certainly the data in Fig 3a support that, as does Safar’s later work [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16186247">Safar 2005b</a> (<a href="http://vir.sgmjournals.org/content/86/10/2913.long">ft</a>)].</p>
<p>Here’s one alternate interpretation of the correlation: PrP<sup>Sc</sup> accumulates over time in any prion strain, with fairly similar rates but quite different degrees of toxicity, so that in some strains (ME7) a large amount must accumulate over a long time in order to cause terminal disease, while  in other strains (HY) it takes only a small amount of PrP<sup>Sc</sup> to kill neurons, and so incubation times are short.</p>
<p>This gets into the question of PrP<sup>L</sup> &#8211; is there a specific lethal species of PrP? &#8211; which I’ll address in detail in a forthcoming post.  Based on Safar’s CDI data on the concentrations of PrP<sup>Sc</sup> in Fig 3a, though, it seems hard to deny that different amounts of PrP<sup>Sc</sup> are present at death in different strains.  The right interpretation isn’t clear, but at least it is probably not sufficient to suppose that all PrP<sup>Sc</sup> is equally toxic.</p>
<p>A more recent paper by Safar’s group seems to abandon the earlier interpretation of the CDI data, no longer arguing for a correlation between sPrP<sup>Sc</sup> levels and incubation time.  Instead, a connection is drawn between incubation time and the conformational stability of sPrP<sup>Sc</sup> oligomers as measured by CDI [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3169556/">Kim 2011</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3410855/">Kim 2012</a>].</p>
