---
layout: post
title: "Huntington's Disease phenotypes in cell culture"
author: ericminikel
date: 2013-01-23 20:55:57
---
<p>Many neurodegenerative diseases are difficult to study in cell culture because, in human patients, the diseases only cause pathology in a particular cell or tissue type, and only after decades of life.  For example, fatal familial insomnia specifically targets <a href="http://en.wikipedia.org/wiki/Thalamus">thalamic</a> neurons, with lesser effects on neurons elsewhere in the brain and no known phenotype in non-brain tissues.  And the disease takes ~50 years to onset, which makes it challenging to see a disease phenotype in cells cultured for a few days or a few months.  Recent advances in <a title="Advances in cell reprogramming" href="/2013/01/21/advances-in-cell-reprogramming/">reprogramming adult cells</a> and creating isogenic disease models with <a title="TALENs and ZFNs" href="/2013/01/19/talens-and-zfns/">ZFNs/TALENs</a> show promise in this regard, but we still don&#8217;t know how to differentiate cells into many of the very specific neuronal subtypes affected by neurodegenerative disease, nor how to induce many late-onset phenotypes in cell culture.  For instance, cells with prion disease-causing mutations don&#8217;t necessarily spontaneously produce PrP<sup>Sc</sup> in culture (though there have been reports of them doing so &#8211; ex. <a href="http://www.horizonpress.com/cimb/v/v1/07.pdf">Harris 1999</a>).</p>
<p>I was surprised, therefore, to learn recently that <a href="http://www.buckinstitute.org/ellerbyLab">Lisa Ellerby&#8217;s group</a> at the <a href="http://www.buckinstitute.org/">Buck Institute</a> has successfully shown Huntington&#8217;s Disease phenotypes in stem cells, and has even had some success differentiating the stem cells into striatal neurons, the cell type affected in HD.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2633781/">Park 2008</a> created <a title="induced pluripotent stem cells" href="http://en.wikipedia.org/wiki/Induced_pluripotent_stem_cell">iPSC</a> from an HD patient with 72 CAG repeats.  <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2966296/">Zhang 2010</a> then differentiated and characterized these cells and compared them to control cells from a different person who didn&#8217;t have HD.  Once the iPSC were partially differentiated into neural stem cells (NSCs) &#8211; but not all the way to neurons &#8211; they exhibited one quantifiable phenotype that is known to occur in vivo in HD patients: increased activity of <a href="http://en.wikipedia.org/wiki/Caspase_3">caspase 3</a> and <a href="http://en.wikipedia.org/wiki/Caspase_7">7</a>.  These are proteins that cleave other proteins, including that they cleave huntingtin into toxic fragments.  In the NSCs, this phenotype occurred when growth factors were taken away (apparently not while the cells were still growing). And this phenotype in turn appeared to lead to higher toxicity (meaning, perhaps, cell death?  The authors never say.)</p>
<p>Although the phenotypes were only measured (or only reported?) for neural stem cells, Zhang also was able to differentiate the neural stem cells into neurons &#8211; not just that, but supposedly <a href="http://en.wikipedia.org/wiki/Striatum">striatal</a> neurons.  The striatum is the brain region most heavily affected in HD, so being able to study striatal neurons in culture would be a big deal.  But iPSC (sometimes disparagingly called &#8216;frankencells&#8217;) don&#8217;t always become <em>exactly</em> the cell type you want &#8211; rather, they&#8217;ll show some characteristics of the desired cell type and not other characteristics thereof.  In this case, Zhang followed a protocol by <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2575484/">Aubry 2008</a>, originally designed to differentiate human embryonic stem cells into striatal neurons.  Zhang first differentiated the iPSC into neural stem cells (NSCs), then (stage 1) exposed the NSCs to three proteins: <a href="http://en.wikipedia.org/wiki/Sonic_hedgehog">SHH</a>, <a href="http://en.wikipedia.org/wiki/DKK1">DKK1</a> and <a href="http://en.wikipedia.org/wiki/Brain-derived_neurotrophic_factor">BDNF</a> as well as the small molecule <a href="http://en.wikipedia.org/wiki/Y-27632">Y27632</a>, a <a href="http://en.wikipedia.org/wiki/Rho-associated_protein_kinase">ROCK inhibitor</a>, to keep cells from dying, then (stage 2) exposed the cells to BDNF, Y27632, and the small molecules <a href="http://en.wikipedia.org/wiki/Cyclic_adenosine_monophosphate">cAMP</a> and <a href="http://en.wikipedia.org/wiki/Valproic_acid">valproic acid</a>.  All of this supposedly mimics the exposure that soon-to-be-striatal neurons get from their neighbors during embryonic development, that determines their striatal fate. And indeed, after the iPSCs underwent this treatment, the resulting neurons stained positive for <a href="http://en.wikipedia.org/wiki/Calbindin">calbindin</a> and <a href="http://en.wikipedia.org/wiki/Gamma-Aminobutyric_acid">GABA</a> (both markers of striatal neurons). However only about 10% of them stained positive for <a href="http://en.wikipedia.org/wiki/PPP1R1B">DARPP-32</a>, a marker of medium spiny neurons, which comprise 96% of striatal neurons in vivo.  So were these neurons really striatal neurons?  Well, they were probably an imperfect model, but at least a better model of striatal neurons than just any-old-neurons-in-a-dish would be.</p>
<p>So that experiment looks promising: some indication of disease-relevant phenotype in a dish, and some indication of creating disease-relevant cell types from iPSC.  Still, not a super clean experiment &#8211; the HD NSCs were compared to control NSCs from a different person with a whole different genome.  And although neurons with some striatal characteristics were created, the phenotype was not tested (or not reported) in those.</p>
<p>The same group was back two years later with an improved experiment [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22748967">An 2012</a>] taking advantage of <a title="TALENs and ZFNs" href="/2013/01/19/talens-and-zfns/">TALENs</a>.  This time, to avoid the comparing-cells-from-different-people issue, they started with HD cells and used TALENs to replace a whole chunk of the HTT: exon 1 (including the CAG repeat region) and a big piece of the 5&#8242;UTR.  Starting from cells with 73 CAG repeats, they successfully obtained two clones (i.e. two sets of cells) that had the new, shorter, allele knocked in.  (One of the clones had 21 CAGs and one had 20 CAGs &#8211; perhaps the CAG repeats contracted somewhere in the process.)  Now they had cell lines that were basically isogenic except for CAG length &#8211; a much better control than in the earlier experiment.</p>
<p>Comparing phenotypes in the neural stem cells from each line, they found that the original HD cells had HD phenotypes and the &#8216;corrected&#8217; cells did not.  Specifically, at the neural stem cell stage, the original HD cells had elevated caspase 3/7 activity and more cell death when growth factors were removed (cell death measured by <a href="http://en.wikipedia.org/wiki/TUNEL_assay">TUNEL staining</a>).  They also reported other phenotypes that correspond to ones observed in HD patients &#8211; changes in <a href="http://en.wikipedia.org/wiki/Brain-derived_neurotrophic_factor">BDNF</a> levels and mitochondrial metabolism.  And again, they repeated the differentiation into striatal neurons successfully but did not look at disease phenotypes in those.</p>
<p>Now, to be very rigorous, this still isn&#8217;t a perfectly controlled experiment.  The piece of DNA knocked in with the TALENs was a whole different HTT haplotype than the original cells had, including a different promoter region.  So the cells differ in more than just the CAG length. Still, it seems very likely that the phenotypic differences observed here really were due to the CAG length.</p>
<p>Isogenic human disease models using TALENs are a hot new thing &#8211; the most famous recent example is <a href="http://www.ncbi.nlm.nih.gov/pubmed/23246482">Ding 2013</a> - and now many people are racing to do them.  But <a href="http://www.ncbi.nlm.nih.gov/pubmed/22748967">An 2012</a> is the first paper I&#8217;ve seen that (1) applies this approach to neurodegenerative disease, (2) uses iPSC, (3) observes a relevant phenotype, and (4) also shows some success differentiating iPSC into the right cell type.  These are all good things that should really help our ability to understand disease biology and test therapeutics.</p>
