---
layout: post
title: "Association of repeat length polymorphisms with complex traits and common diseases"
author: ericminikel
date: 2013-04-03 20:55:21
---
<p>Huntington&#8217;s Disease is just one of many Mendelian diseases caused by a pathological repeat expansion.  A recent review, <a href="http://www.ncbi.nlm.nih.gov/pubmed/20177426">La Spada and Taylor 2010</a>, lists 22 Mendelian repeat expansion disorders, and more continue to be discovered &#8211; most recently a non-coding GGGGCC hexanucleotide expansion in <a href="http://en.wikipedia.org/wiki/C9orf72">C9ORF72</a> has been found to segregate with FTD-ALS [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3202986/">DeJesus-Hernandez 2011</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3200438/">Renton 2011</a>].</p>
<p>It is likely that the phenotypic influence of repeat length polymorphisms is not limited to Mendelian diseases.  There is also some evidence of repeat length polymorphisms contributing to common traits, complex diseases and acting as secondary modifiers of other diseases.</p>
<p>In reading the linked papers, note the divergent terminology: repeats with a period of 2-6 nucleotides are called <em>short tandem repeats </em>(STRs) by some authors and <em>microsatellites</em> by others.  (Wikipedia calls them <a href="http://en.wikipedia.org/wiki/Microsatellite">microsatellites</a>).  Some authors also include repeats with a period of 1, such as CCCCCCCCCCCC, in the definition of these terms, though these simple runs of a single base are also called <em>homopolymers</em>.</p>
<p><strong>associations from the literature</strong></p>
<p>As mentioned in <a href="/2013/03/05/heritability-and-genetic-modifiers-in-mendelian-diseases/">a recent post</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16000334">Pulst 2005</a> (<a href="http://brain.oxfordjournals.org/content/128/10/2297.long">ft</a>) discovered that sub-pathological repeat length polymorphisms in <a href="http://en.wikipedia.org/wiki/Cav2.1">CACNA1A</a> contribute to an age of onset phenotype in spinocerebellar ataxia 2, a Mendelian disorder caused by a repeat expansion in <a href="http://en.wikipedia.org/wiki/ATXN2">ATXN2</a>.  So repeat length polymorphisms in one gene can modify the phenotype of a disease-causing repeat expansion in another gene.</p>
<p>Certain repeat length polymorphisms have also been suggested to cause, or modify, disorders that (to our knowledge) aren&#8217;t otherwise repeat-related.  <a href="http://en.wikipedia.org/wiki/Microsatellite">Wikipedia</a> pointed me to a few examples: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19111531">Jemaa 2009</a> found a  intronic 27bp repeat in NOS3 associated with hypertension among Tunisians, and <a href="http://www.ncbi.nlm.nih.gov/pubmed/18464244">Kersting 2008</a> found an intronic dinucleotide repeat in EGFR to be associated with osteosarcoma.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/19054556">Akagi 2009</a> examined an intronic 14bp repeat in the <a href="http://en.wikipedia.org/wiki/Asparagine_synthetase">asparagine synthetase</a> (ASNS) gene in relation to acute lymphoblastic leukemia and found biochemical evidence of differing levels of regulatory activity based on repeat length, though there was not a large enough sample size to propose a genotype-phenotype association.  And as noted in <a title="Fatal familial insomnia as a sleep disorder" href="/2013/02/05/fatal-familial-insomnia-as-a-sleep-disorder/">the sleep post</a>, a protein-coding 54bp repeat in PER3 may be associated with certain sleep phenotypes [<a href="http://www.ncbi.nlm.nih.gov/pubmed/17346965/">Viola 2007</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2542962/">Groeger 2007</a>].</p>
<p>Perhaps the most studied example concerns an (AC)<sub>n</sub> dinucleotide repeat length polymorphism near (2.1kb upstream of) ALR2.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/7789640">Ko 1995</a> found this polymorphism to be associated with diabetic retinopathy among Chinese <a href="http://en.wikipedia.org/wiki/Diabetes_mellitus_type_2">type II diabetes</a> (T2D) patients.  Following that finding, <a href="http://www.ncbi.nlm.nih.gov/pubmed/9000706">Heesom 1997</a> found evidence that the same polymorphism was also associated with kidney problems among <a href="http://en.wikipedia.org/wiki/Diabetes_mellitus_type_1">type I diabetes</a> (T1D) patients.  To be clear, T1D and T2D are both complex disorders with some genetic component, and the ALR2 repeat length polymorphism was not proposed as a cause of either disease, but rather as a phenotypic modifier.</p>
<p>It&#8217;s not clear if either of those associations is real.  Both findings have been replicated by some studies and not by others; one meta-analysis [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18434430">Xu 2008</a> (<a href="http://intl-jme.endocrinology-journals.org/content/40/5/243.full">ft</a>)] suggested that the T1D association was real and the T2D association was not.  Assuming a correlation really exists, it is also not yet totally clear that the repeat is the directly causal variation and not just tagging some other variation.  For instance <a href="http://www.ncbi.nlm.nih.gov/pubmed/10342825">Kao 1999</a> (<a href="http://diabetes.diabetesjournals.org/content/48/6/1338.long">ft</a>) found a separate SNP in the ALR1 promoter region in nearly perfect linkage disequilibrium with the repeat.  Interestingly, <a href="http://www.ncbi.nlm.nih.gov/pubmed/9709964">Shah 1998</a> found that the repeat length polymorphism strongly correlated with ALR1 gene expression level <em>only in </em>diabetics and not in controls.</p>
<p>Repeat length polymorphisms elsewhere in the genome have also been associated with other diabetic phenotypes, but it also seems as yet unclear whether these associations are real [<a href="http://www.ncbi.nlm.nih.gov/pubmed/21067489">Uthra 2010</a>].</p>
<p>Most of the above studies used relatively small sample sizes &#8211; for instance, the T1D and T2D studies had tens or a few hundreds of patients each (see <a href="http://www.ncbi.nlm.nih.gov/pubmed/18434430">Xu 2008</a>&#8216;s <a href="http://intl-jme.endocrinology-journals.org/content/40/5/243/T1.expansion.html">Table 1</a> and <a href="http://intl-jme.endocrinology-journals.org/content/40/5/243/T2.expansion.html">Table 2</a>).  No wonder the results were (are?) still debated even more than a decade after the original discovery.</p>
<p>Probably the most compelling association of a repeat length polymorphism with a complex trait comes from <a href="http://www.ncbi.nlm.nih.gov/pubmed/16415884">Grant 2006</a>.  A previous study had reported a linkage peak at 10q for type II diabetes risk.  The linkage region was 10.5Mb long and contained 228 microsatellites.  Grant genotyped all of these microsatellites in almost 2000 cases and 2000 controls (split across three cohorts), and found that one microsatellite, named DG10S478, was strongly associated with T2D.</p>
<p>This microsatellite is a bit mysterious &#8211; the <a href="http://www.nature.com/ng/journal/v38/n3/extref/ng1732-S8.pdf">supplementary methods</a> give the genomic location as <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgHubConnect.destUrl=..%2Fcgi-bin%2FhgTracks&amp;clade=mammal&amp;org=Human&amp;db=hg16&amp;position=chr10%3A114460845-114461228&amp;hgt.positionInput=chr10%3A114460845-114461228&amp;hgt.suggestTrack=knownGene&amp;Submit=submit&amp;hgsid=331828055">hg16/b34 chr10:114,460,845-114,461,228</a> but it appears that the reference genome actually represents the 0-repeat allele, i.e. you can&#8217;t see the repeat anywhere in the reference sequence. Grant says the repeat is tetranucleotide, i.e. a period of 4, but never actually says what the repeated sequence is.</p>
<p>In any event, Grant reports that the association of this repeat with T2D is slightly stronger (relative risk 1.56, p = 4.7E-18) than the most strongly associated nearby SNP (<a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=7903146">rs7903146</a>, relative risk 1.54, p = 2.1E-17), which to my eye would suggest that the repeat is truly causal and the SNP is just a marker.  Grant even sequences the nearest exon to look for possible coding variation tagged by the repeat, and finds none.  Yet still, Grant never asserts that the repeat is causal, instead offering that DG10S478 <em>might</em> be causal, or might &#8220;more likely&#8221; be a marker for something else intronic.</p>
<p><strong>what&#8217;s the mechanism?</strong></p>
<p>In contrast to the Mendelian repeat disorders, where the repeats are very well-established to be causal, in the end it is not clear that there is really solid evidence for any of the above-mentioned repeats being directly causal for these complex traits/diseases.</p>
<p>But there is no shortage of mechanisms by which a repeat length polymorphism <em>could</em> directly influence a phenotype.  Consider how repeat expansions cause Mendelian disorders: sometimes they result in a loss of a gene&#8217;s expression (ex. <a href="http://en.wikipedia.org/wiki/Fragile_X_syndrome">Fragile X</a>) or of just one transcript thereof (FTD-ALS, see <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3202986/">DeJesus-Hernandez 2011</a>).  Alternately, protein-coding repeats can cause a gain of function at the protein level &#8211; we don&#8217;t know exactly how that works but one theory is that longer repeat tracts make for a pathological exaggeration of the protein&#8217;s native function [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22284692">Orr 2012</a>].  Any of these mechanisms could apply to less extreme repeat length polymorphisms as well.  Coding repeats might control the activity level of a protein; intergenic, UTR or intronic repeats might control dosage of a gene or of a specific transcript.  One specific possibility is that repeat lengths in promoter regions &#8211; especially (AC)<sub>n</sub> repeats &#8211; control the level of DNA binding by transcription factors [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12411608">Rockman &amp; Wray 2002</a> (<a href="http://mbe.oxfordjournals.org/content/19/11/1991.full">ft</a>)].</p>
<p><strong>conclusion</strong></p>
<p>Repeat lengths mutate much more quickly than SNPs do [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3459271/">Sun 2012</a>] and are often highly polymorphic.  It seems logical that they might well contribute to human phenotypic diversity in complex traits.  So far, several repeat length polymorphisms have been proposed to associate with phenotypes, but I couldn&#8217;t find an example where the repeat has very confidently been identified as the causal variant.</p>
