---
layout: post
title: "Center for Human Genetic Research 2013 Retreat"
author: ericminikel
date: 2013-01-30 20:53:44
---
<p>Today I&#8217;m blogging from the <a href="http://chgr.org/">Center for Human Genetic Research</a> 2013 annual retreat.  These notes will be pretty unpolished and there are more notes on some speakers than others.</p>
<p><strong>Speaker 1: <a href="http://chgr.org/index-faculty_rosand.html">John Rosand</a> &#8211; Shared risk loci between different brain phenotypes</strong></p>
<p>Dr. Rosand&#8217;s area of study is cerebrovascular disease &#8211; when blood clots block circulation to part of the brain (i.e. stroke) or when bleeding occurs in the brain (i.e. intracerebral hemorrhage).</p>
<p>The Trial of Org-10172 in Acute Stroke Treatment (TOAST) tried to create six phenotypic categories for cerebrovascular disease, with the hopes that which phenotypic category you are (along with genetics) would help predict drug response.  When they did their genotype-phenotype association study, they found that the strongest risk locus had already been identified as associated with white matter hyperintensity on MRI.  Another risk locus which validated on replication had already been associated with cardiovascular disease.  There&#8217;s also a connection with Alzheimer&#8217;s.  Hemorrhagic stroke can result from amyloid buildup in arteries &#8211; does this phenotype share risk factors with amyloid buildup in Alzheimer&#8217;s Disease?  Indeed, <a href="http://en.wikipedia.org/wiki/Apolipoprotein_E">APOE</a> E2 and E4 showed up as very significant risk factors for hemorrhagic stroke &#8211; but with opposite effect directions (E4 is protective and E2 is harmful, whereas for Alzheimer&#8217;s E4 is harmful and E2 is protective).</p>
<p>After finding how many shared loci there are, they decided to do a grand cross-disorder analysis of genetic risk factors in a variety of neurological and psychiatric disorders in the hopes of identifying shared mechanisms, cleverly titled &#8220;BRAINSTORM&#8221;.</p>
<p>Q&amp;A: Small <a href="http://en.wikipedia.org/wiki/Infarction">infarcts</a> (normally associated with stroke) are also very common in Alzheimer&#8217;s disease.  We are still wondering why the <a href="http://en.wikipedia.org/wiki/Apolipoprotein_E">APOE</a> E2 and E4 effect directions are opposite.  May have to do with Aβ drainage: E2 may facilitate Aβ drainage, clearing it from the neurons but clogging the vessels.</p>
<p><strong>Speaker 2: <a href="http://chgr.org/index-faculty_doyle.html">Alysa Doyle</a> &#8211; Translating emerging schizophrenia genetics to a child clinical cohort</strong></p>
<p>Dr. Doyle focuses on schizophrenia because it&#8217;s common: there&#8217;s a large public health impact and it is easy to gather large cohorts for studies.  So far what we know about schizophrenia genetics is there are lots of tiny effect size SNPs identified through GWAS, as well as (often de novo) microdeletions and microduplications.  What we know about the clinical course is that there are subtle motor, cognition, language and social impacts long before the first &#8216;episode&#8217;.</p>
<p>We&#8217;d like to be able to intervene early.  On the other hand, treating kids for psychiatric stuff is really controversial so we&#8217;ll need to have very strong evidence in order to proceed.  In the psychiatry field there is a move towards &#8216;dimensional&#8217; nature of disorders and having a &#8216;dimensional&#8217; diagnostic schema (i.e. multiple quantitative traits) rather than discrete diagnostic categories.  Therefore our clinical translational efforts must be able to deal with quantitative traits.</p>
<p>Cognition is a valuable quantitative phenotype here &#8211; cuts across bipolar, ADHD, autism, OCD &#8211; and is often seen in &#8216;asymptomatic&#8217; relatives.  It is hypothesized that cognition may underlie the liability for a lot of different disorders.  Therefore: the newly proposed longitudinal study of genetic influences on cognition (LOGIC) will follow follow a deeply phenotyped youth cohort over time.</p>
<p><strong>Speaker 3: <a href="http://chgr.org/index-faculty_newtoncheh.html">Chris Newton-Cheh</a> &#8211; Blood pressure genetics</strong></p>
<p>High blood pressure is a risk factor for stroke, heart failure, heart attacks, and end-stage kidney disease.  <a href="http://en.wikipedia.org/wiki/Natriuretic_peptide">Natriuretic peptides</a> are peptides (very short proteins) that cause sodium to be excreted through the urine.  Chris studies two major ones: <a href="http://en.wikipedia.org/wiki/Atrial_natriuretic_peptide">atrial natriuretic peptide</a> (ANP, released in the upper heart chambers which are called &#8216;atria&#8217;) and <a href="http://en.wikipedia.org/wiki/Brain_natriuretic_peptide">brain/basic natriuretic peptide</a> (BNP, named for its discovery in pig brains but mostly released in the heart in humans).  SNPs in the <a href="http://en.wikipedia.org/wiki/Untranslated_region">UTR</a>s of ANP and BNP are associated with the extent of ANP/BNP response to salt intake, and thus with blood pressure. How can an untranslated SNP have this phenotypic effect?</p>
<p>The answer is differential binding of <a href="http://en.wikipedia.org/wiki/MicroRNA">microRNAs</a>.  Pankaj Arora identified three microRNAs that are predicted to bind differentially to the NPPA (gene which codes for ANP) 3&#8242;UTR depending on these SNPs.  Two of these microRNAs are expressed in cardiac tissue.  A luciferase assay identified one miR that suppresses the major allele but not minor allele of ANP.  Another assay showed that an antisense RNA to suppress that miR increases major allele but not minor allele expression.  And finally, an experiment on human cardiomyocytes showed that exogenous introduction of that miR suppresses ANP release.  This confirms miR binding as a mechanism for this SNP&#8217;s effect.</p>
<p>They have not yet found mechanisms for the effect of the other SNPs yet and they think they are almost certainly just tagging other things via linkage disequilibrium.</p>
<p><strong>Speaker 4 &#8211; <a href="http://chgr.org/index-faculty_scharf.html">Jeremiah Scharf</a> &#8211; Tourette syndrome</strong></p>
<p><a href="http://en.wikipedia.org/wiki/Tourette_syndrome">Tourette syndrome</a> (TS) is characterized by <a href="http://en.wikipedia.org/wiki/Tic">tics</a> such as in this video:</p>
<p><iframe src="http://www.youtube.com/embed/aDipNAuZuZI" frameborder="0" width="420" height="315"></iframe>.</p>
<p>Tourette&#8217;s is likely to co-occur with <a href="http://en.wikipedia.org/wiki/Obsessive%E2%80%93compulsive_disorder">OCD</a> and <a href="http://en.wikipedia.org/wiki/Attention_deficit_hyperactivity_disorder">ADHD</a>.  It has estimated 60-80% heritability based on identical twin studies and fraternal twin studies, but at present we know almost nothing about the genetic architecture of this heritability, and there are no convincing <a title="controlled by a single genetic variation" href="http://en.wikipedia.org/wiki/Mendelian_inheritance#Mendelian_trait">Mendelian</a> forms of the syndrome.  GWAS have identified some hits that are close to genome-wide significant, but not quite.  One is <a href="http://en.wikipedia.org/wiki/SLITRK5">SLITRK5</a>, and Slitrk5 knockout mice have OCD-like behaviors [perhaps he refers to <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2907076/">Shmelkov 2010</a>?] so that looks like a strong candidate.</p>
<p>They are currently embarking on GWAS as well as rare variant studies combining TS and OCD to maximize power.</p>
<p><strong>Speaker 5 &#8211; <a href="http://chgr.org/index-faculty_cotman.html">Susan Cotman</a> &#8211; Neuronal ceroid lipofuscinosis (NCL)</strong></p>
<p><a href="http://en.wikipedia.org/wiki/Neuronal_ceroid_lipofuscinosis">Neuronal ceroid lipofuscinosis</a> includes a bunch of different diseases including <a href="http://en.wikipedia.org/wiki/Batten_disease">Batten disease</a>, which is basically the juvenile form of NCL. (Ran out of battery during this talk).</p>
<p><strong>Speaker 6 (Keynote) - <a href="http://genetics.med.harvard.edu/reich/Reich_Lab/Welcome.html">David Reich</a> &#8211; Evidence for human interbreeding with neanderthals</strong></p>
<p>Three challenges in getting neanderthal DNA from ancient bones:</p>
<ol>
<li>Getting enough DNA at all</li>
<li>Of the DNA you extract from bone, only 0.1 &#8211; 3.0% of it is from the neanderthal &#8211; the rest is from microbes, fungi etc that colonized their bodies after death.</li>
<li>It is very easy to contaminate the neanderthal DNA with modern human DNA during handling (by archaeologists, lab techs, etc.) and since the genomes are almost identical (~1 variant in 600b, so 100b sequencing reads will be mostly identical to human genome) it is hard to tell when contamination has happened.</li>
</ol>
<p>Today&#8217;s solutions: extract DNA from center of bone (less chance for human contamination), prepare libraries in a &#8216;clean room&#8217; and ligate barcodes in the clean room that will rule out later contamination.</p>
<p>Early studies used mitochondrial DNA because there are ~1000 mitochondria per cell, so it&#8217;s easier to get more mtDNA than nuclear DNA.  Based on mtDNA it could be shown that neanderthals were always more different from humans than humans were from each other.  This suggested that neanderthals and humans never interbred.</p>
<p>However, non-Africans match neanderthal SNPs more often than Africans do.  Africans and non-Africans split a long time ago and Africans are more diverse.  So we looked for anomalous genomic sites where non-Africans exhibited more diversity than Africans do.  Of those, the non-African variant matched neanderthals in 80% of cases.  This provides really strong evidence for &#8216;gene flow&#8217; due to neanderthals interbreeding with non-Africans after the out of Africa migration.</p>
<p>We now think 1-4% of DNA in non-Africans is from interbreeding with neanderthals.  Based on linkage disequilibrium we think the interbreeding occurred within the last 85,000 years, which establishes the interbreeding as having occurred outside of Africa.</p>
<p>Neanderthal ancestry is especially strong in human immune genes esp. HLA; other human genomic regions have almost no neanderthal ancestry left.  Perhaps neanderthals were better adapted to the geographic regions (Europe, Asia) that humans migrated into and so variants helpful for surviving in those regions were selected-for.</p>
<p>Topic 2: admixture in Indian history.  [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/pmc2842210/">Reich 2009</a>, and a forthcoming paper by Priya Moorjani].  Indians are less studied in genomics because the Indian government makes it very hard to get genetic material out of the country.  Only now are South Asians from Pakistan, Bangladesh and Indians in the U.S., U.K., etc finally making it into 1000 Genomes, etc.</p>
<p>When you MDS plot Indians, you find &#8220;the Indian Cline&#8221; &#8211; Indians are on a spectrum between peoples west (Persians, Europeans) and east (Andamanese, East &amp; Southeast Asian).   Or if you MDS plot Europeans vs. Chinese you find PC1 separates them; if you then add Indians, they are spread along PC1.  Whereas if you plot Indians vs. Chinese, then add Europeans, you don&#8217;t see any cline.</p>
<p>Explanation consistent with history is that Indians are admixed from ancestral north Indians (&#8216;ANI&#8217;) and ancestral south Indians (&#8216;ASI&#8217;).  Different groups in India vary from 20% &#8216;northern&#8217; and 80% &#8216;southern&#8217; to 70%/30%.  Dravidian speakers and lower castes are associated with more ASI ancestry.  Some disease traits &#8211; esp. cardiac risk &#8211; associated with Indians come from the ASI ancestry.  Linkage disequilibrium size suggests that the ANI-ASI admixture occurred less than 3500 years ago with the rise of Vedic religion (Hinduism).  This represents a major demographic transformation long after the advent of agriculture and is different from more recent Indian history where endogamy (marrying people of same region, caste etc) has been expected.</p>
<p>Different Indian groups have different levels of endogamy which can be detected by linkage disequilibrium.  In Vysya, huge LD size suggests a recent (~2000 years) founder event and less than 1 in 100 marriages to outsiders per generation.  Medical implication: the close genetic relationships within whole regions / caste groups are what is responsible for most recessive genetic diseases in Indians, <em>not </em>due to recent consanguinity (marriage between second cousins, etc.)</p>
<p>An ancient sample from Luxembourg is &#8216;more European than Europeans&#8217;, suggesting that Europeans are also an admixture of ancient Europeans with Middle Easterners.</p>
<p>Q&amp;A: Hybrid incompatibility issues that lead to speciation tend to arise first on the X chromosome. We do see plenty of neanderthal SNPs on modern human X chromosomes [perhaps referring to <a href="http://mbe.oxfordjournals.org/content/early/2011/01/25/molbev.msr024.full.pdf+html">Yotova 2011</a>??]</p>
<p>Q&amp;A: The ANI admixture into India cannot be entirely, or maybe even at all, explained by Indo-European speakers&#8217; migration into India as told by the Veda.  There are at least two different admixture events among upper caste North Indians and we don&#8217;t know if either corresponds to the Indo-European migration.</p>
<p><strong>Speaker 7 &#8211; <a href="http://chgr.org/index-faculty_karmacharya.html">Rakesh Karmacharya</a> &#8211; Psychiatric genetics</strong></p>
<p>Schizophrenia is a psychosis disorder and has both &#8216;positive symptoms&#8217; = things they have that you don&#8217;t (e.g. hallucinations) and &#8216;negative symptoms&#8217; = things they don&#8217;t have that you do, as well as cognitive symptoms.  Bipolar is a mood disorder: manic and depressive.  But there is actually a spectrum between these two where patients have different amounts of mood and psychosis symptoms.  Both disorders have high heritability (identical twin concordance = 50%).</p>
<p>Treatment has not changed much.  Even today, first line of treatment in bipolar is lithium (discovered as a drug 1949); for schizophrenia the first line is still dopamine antagonists based on chlorpromazine (disccovered 1952), though chlorpromazine itself is not used as much.  All of these drugs have low tolerability and efficacy.  Our study of them is hampered by lack of access to relevant tissue (i.e. the brain).</p>
<p>Goal: identify molecular &#8216;signatures&#8217; of bipolar and schizophrenia, so that we can then screen for small molecules that modify those signatures.  Open question: will cellular deficits / molecular signatures be present at baseline or only under specific environmental conditions, and if the latter, can we simulate relevant environmental factors in vitro?</p>
<p>Currently looking for molecular or morphological signatures in patient iPSC differentiated into neurons in culture.  Looking at different differentiation conditions to find relevant cell types where we find differences due to genetic background and not just artifacts of differentiation process.</p>
<p>Overall vision is to perform &#8216;an unbiased image-based and gene expression-based profiling of disease and control cells in the presence of small molecule perturbations.&#8217;  Procedure:</p>
<ul>
<li>Obtain iPSC from schizophrenia/bipolar patients and healthy controls</li>
<li>Grow and differentiate cells in 384-well plates</li>
<li>Treat with a battery of 320 different small molecules</li>
<li>Stain with 6 stains for different parts of cell (nuclei, lysosomes, etc.)</li>
<li>Use CellProfiler to segment images and generate about 500 features on each object in the image</li>
<li>Search these 500 features for any that correlate with disease vs. healthy genotype</li>
<li>Also do gene expression profiling on the wells and look for gene expression signature of the disease(s).</li>
</ul>
<div>Currently in data analysis stage.  The goal is to find a morphological or gene expression &#8216;signature&#8217; of schizophrenia / bipolar which can then be used in high throughput screens for therapeutics.</div>
<p><strong>Speaker 8 &#8211; <a href="http://chgr.org/index-faculty_talkowski.html">Mike Talkowski</a> &#8211; Translating genomics into the clinic</strong></p>
<p>The challenge: for clinical purposes (as opposed to research purposes), n = 1.  You need to interpret <em>one</em> person&#8217;s genome and advise accordingly.  Our &#8216;annotation of the morbid genome&#8217; &#8211; i.e. our knowledge of what variations cause disease &#8211; is still very poor.</p>
<p>Many of the same genes that cause developmental abnormalities if disrupted through chromosomal translocations turn out to also be highly enriched for SNPs that are risk factors for ADHD, schizophrenia, etc.  But even that is just looking at phenotypes relatively early at life.  Next step is we need to look at whether they are also enriched for late-onset things like dementia e.g. Alzheimer&#8217;s.</p>
<p>Open question is how to educate clinicians and patients on how to interpret high-resolution genetic information when even researchers don&#8217;t even know enough.  However remember: these medical decisions are currently being made without any high-resolution information.  The standard in the field is that if a fetus has an abnormal <a href="http://en.wikipedia.org/wiki/Karyotype">karyotype</a>, the parents are told that there is a 6.1% risk of an &#8216;untoward outcome.&#8217;  This is an extremely crude estimate based on a study from decades ago.</p>
<p><strong>Speaker 9 &#8211; <a href="http://chgr.org/index-faculty_macarthur.html">Daniel MacArthur</a> &#8211; Genomic approaches to finding causal mutations for Mendelian diseases</strong></p>
<p>5-8% of U.S. population has a rare disease.  Many of these cannot be definitively diagnosed yet.  Many are genetically and phenotypically heterogeneous.</p>
<p>Exome sequencing identifies loads of variants &#8211; how do we filter for the causal variant?</p>
<ul>
<li>Functional annotation &#8211; missense, nonsense, splice, etc.</li>
<li>Transmission &#8211; variant must be shared between all affected individuals in a family</li>
<li>Frequency &#8211; look only at rarer variants</li>
<li>Expression &#8211; gene must be expressed in affected tissue type</li>
<li>Protein-protein interaction</li>
<li>Replication &#8211; variant or disease gene turns up in multiple families</li>
</ul>
<p>Politics of research make the &#8216;replication&#8217; difficult for very rare diseases.  If one researcher has a possible causal gene in one family but has no other families, they can&#8217;t reach out to other researchers to share data without fear of being scooped.  Two possible solutions:</p>
<p>1. Reach out directly to the public rather than to other researchers.  This is what <a href="http://www.raregenomics.org/">Rare Genomics Institute</a> does &#8211; it encourages families to come in and use crowdsourcing to support their own exome sequencing.</p>
<p>2. Need a forum for &#8216;micropublications&#8217; where researchers can publish uncertain works in progress so that their name is associated with it at an early stage and then researchers can find each other.</p>
