---
layout: post
title: "Highlights of ASHG2013"
author: ericminikel
date: 2013-10-26 13:02:02
---
<p>This week we attended the American Society of Human Genetics (ASHG) <a href="http://www.ashg.org/2013meeting/">2013 annual meeting</a> in Boston. Here are a few highlights.</p>
<p><strong>gene therapy</strong></p>
<p><em>For some background on what gene therapy is and its relevance to prion disease, see <a title="Prospects for gene therapy of prion diseases" href="/2013/03/03/prospects-for-gene-therapy-of-prion-diseases/">this post</a>, and for a bit more detail on some relevant approaches and literature see <a title="Updates on gene therapy for prion diseases" href="/2013/05/18/updates-on-gene-therapy-for-prion-diseases/">this followup</a>.</em></p>
<p>Our first move at ASHG was to attend <a href="http://www.ashg.org/2013meeting/pages/sessionlisting.shtml#sess3">a 2-hour session</a> devoted to the latest advances in gene therapy. Interestingly, not one of the gene therapy talks (nor any of the dozens of gene therapy posters) at this conference was devoted to RNAi &#8211; rather, they were all focused on the viral delivery of crucial genes lost in recessive loss-of-function diseases. Still, much of what we learned is relevant to the idea of knocking down PrP with virally delivered RNAi.</p>
<p><a href="http://www.schepens.harvard.edu/profile-luk/luk-vandenberghe.html">Dr. Luk Vandenberghe</a> of the Schepens Eye Research Institute presented on <a href="http://en.wikipedia.org/wiki/Adeno-associated_virus">adeno-associated virus</a> (AAV) serotype-specific gene therapy. AAV is a widely used gene therapy vector, in part because it&#8217;s not very immunogenic. Still, according to Dr. Vandenberghe, about 70% of people have antibodies to some form or another of AAV. A person&#8217;s &#8220;AAV serotype&#8221; is the set of AAV types they have antibodies to. Dr. Vandenberghe has 8 different AAV strains and is exploring personalized therapy where each patient&#8217;s serotype is tested and then they are treated with an AAV to which they do not have antibodies. This is one potential solution to the immune response challenge in gene therapy. Still, Dr. Vandenberghe&#8217;s work focuses on gene therapy for the eyes, an &#8220;immune privileged&#8221; organ. Gene therapy for the brain will be more difficult. Accordingly, the next talks dealt with gene therapy of neurological disorders.</p>
<p><a href="http://weillcornell.org/ronaldcrystal/">Dr. Ron Crystal</a> of Cornell presented on gene therapy for <a href="http://en.wikipedia.org/wiki/Batten_disease">Batten disease</a> (NCL), specifically forms of Batten caused by loss of function mutations in the CLN2 gene. Crystal is in charge of three ongoing clinical trials [<a href="http://clinicaltrials.gov/ct2/show/NCT00151216">1</a>, <a href="http://clinicaltrials.gov/ct2/show/NCT01414985">2</a>, <a href="http://clinicaltrials.gov/ct2/show/NCT01161576">3</a>] in which AAV vectors encoding a functional copy of CLN2 are being injected directly into patients&#8217; brains. He showed data from the non-human primate trials that led up to the current clinical trials and stated that the vector is able to get about 85% of the central nervous system to express the desired gene at &gt; 10% above background levels. The therapy has already been used on 8 children, and Crystal showed one image of the surgical process, which involves drilling 6 burr holes in the skull and making two injections (at two different depths) in each hole, for a total of 12 injection sites. The main constraint limiting the amount of the brain that they can reach with injections is the conflict between (1) the maximum time under anesthesia, and (2) the flow rate of the viral vector &#8211; currently each injection is a three-hour flow. The image of the surgery was a bit scary, but amazingly, the main take-home of Crystal&#8217;s talk was that so far both the surgery and the viral vector appear to be completely safe: there have been no adverse reactions. On the question of efficacy, he was more tight-lipped, and said they wouldn&#8217;t be releasing any efficacy data until the trial&#8217;s conclusion.</p>
<p>But the talks weren&#8217;t strictly about safety: <a href="http://www.hsr.it/research/organization/divisions-centers/division-of-regenerative-medicine-stem-cells-and-gene-therapy/luigi-naldini/">Dr. Luigi Naldini</a> also presented on a very small-scale trial that already has some fabulous efficacy data to show. Naldini&#8217;s disease of study presents a fairly unique therapeutic opportunity: <a href="http://en.wikipedia.org/wiki/Metachromatic_leukodystrophy">metachromatic leukodystrophy</a> (MLD) is a neurological disorder but is caused by toxic metabolites from blood. MLD is caused by loss of function mutations in the enzyme arylsulfatase A (<a href="http://en.wikipedia.org/wiki/Arylsulfatase_A">ARSA</a>), leading to an accumulation of cerebroside 3-sulfate, which eventually destroys <a href="http://en.wikipedia.org/wiki/Myelin">myelin</a> in the brain. Naldini&#8217;s trial is actually a form of <a href="http://en.wikipedia.org/wiki/Autologous_stem_cell_transplantation">autologous transplantation</a> (where the tissue donor and recipient are the same person): <a href="http://en.wikipedia.org/wiki/Hematopoietic_stem_cell">hematopoietic stem cells</a> are taken from the person&#8217;s bone marrow, transformed with a lentivirus (read: modified HIV) encoding a working copy of ARSA, and transplanted back in. The procedure has only been performed on three children so far, but the results are astounding. MLD is normally fatal by age 5, and all three of the children in Naldini&#8217;s trial were able to be ascertained for very early intervention only because they all had older siblings who had been diagnosed years earlier before the trial was started. According to Naldini, all three treated children are not only alive and well but actually have <em>no disease phenotype at all </em>(exact words: &#8221;No Disease Manifestation or Progression&#8221; &#8211; see <a href="http://mldfoundation.org/research-SanRaffaele.html">press releases</a> and the publication: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23845948">Biffi 2013</a>). He showed a video of a healthy, completely normal girl running around.</p>
<p>Finally, a talk by <a href="http://irp.nih.gov/pi/stephen-kaler">Dr. Stephen Kaler</a> of NIH covered a project at a much more preliminary stage (preclinical mouse trials) but with some intriguing results. Kaler is developing viral vectors encoding <a href="http://en.wikipedia.org/wiki/ATP7A">ATP7A</a>, a copper pump, as a treatment for the X-linked, ATP7A loss-of-function <a href="http://en.wikipedia.org/wiki/Menkes_disease">Menkes disease</a>. This was interesting to me because of <a title="The complicated relationship between copper, PrP and prion disease" href="/2013/06/24/the-complicated-relationship-between-copper-prp-and-prion-disease/">copper&#8217;s relationship to prion disease</a> and <a title="Regulation of prion protein transcription" href="/2013/05/15/regulation-of-prion-protein-transcription/">PrP transcription</a> - it has been reported that ATP7A overexpression abolishes PrP transcription altogether [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18990686">Bellingham 2009</a> (<a href="http://www.jbc.org/content/284/2/1291.long">ft</a>)]. But Kaler also had a surprising result regarding gene delivery: achieving ATP7A expression in only the <a href="http://en.wikipedia.org/wiki/Choroid_plexus">choroid plexus</a> (where cerebrospinal fluid is produced) appeared to be sufficient to restore the balance of copper across the brain and largely revert the phenotype. In practice, such targeted therapy probably won&#8217;t ever be feasible for prion disease &#8211; odds are every single PrP-expressing neuron would need to be treated in order to be rescued [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8552188">Brandner 1996</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/14593181">Mallucci 2003</a>] &#8211; but it has been argued that targeting just the earliest and most severely affected brain region could have outsize beneficial effects [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2802775/">White &amp; Mallucci 2009</a>].</p>
<p><strong>big exome data</strong></p>
<p>When <a href="http://www.prionalliance.org/our-story/">we first learned</a> about fatal familial insomnia in our family in December 2011, a few questions immediately surfaced. How prevalent is this disease really? How sure are we that it&#8217;s 100% penetrant? Could there be anything else in a person&#8217;s genome that might help protect them from it? As we&#8217;ve become scientists over the years, we&#8217;ve realized these questions aren&#8217;t actually the most relevant to our quest for therapeutics (though finding protective variants can pave the way for drug development &#8211; see <a title="How Tafamidis was born" href="/2012/08/03/how-tafamidis-was-born/">the story of Tafamidis</a>). Yet still, our curiosity returns to these questions from time to time.</p>
<p>These questions are surprisingly difficult to answer with the data we see in the literature. If you just go out and find people who have a disease and then look at their genotypes, it is relatively difficult to ask whether there is anyone who has the genotype and <em>doesn&#8217;t </em>have the disease.</p>
<p>But now, we&#8217;re just at the very beginning of having data on enough people to start asking these sorts of questions. A number of talks at ASHG2013 centered on <a href="https://twitter.com/dgmacarthur">Daniel MacArthur</a>&#8216;s effort to aggregate patient exomes from dozens of different studies, an effort which has now accumulated 62,000 exomes. <a href="http://www.linkedin.com/pub/monkol-lek/30/26b/439">Monkol Lek</a> spoke on the technical challenges of processing this ~1 petabyte of data, and MacArthur spoke on how they&#8217;re using this data to interpret disease variants.</p>
<p>Most likely, even 62,000 exomes is still too few to obtain any estimates on the prevalence of FFI or other genetic prion diseases.  The overall incidence of prion disease is something like 1 in 1 million per year [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2797136/">Holman 2010</a>]; most cases aren&#8217;t genetic in nature, and FFI isn&#8217;t even the most common genetic form [<a href="https://cshmonographs.org/index.php/monographs/article/view/4035">Kong 2003</a>], so it&#8217;s plausible that the prevalence of the FFI D178N 129M haplotype really is something like 1 in 1 million chromosomes, and it will be years before the world has enough sequencing data to start seeing people with FFI just by chance.</p>
<p>Still, the accumulation of genetic data on loads of controls does seem to offer some interesting opportunities. Kaitlin Samocha of <a href="http://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/mark-daly">Mark Daly</a>&#8216;s lab gave a remarkable talk on her calculation of <em>gene constraint</em> (currently in review at Science). Whereas gene <em>conservation</em> refers to a gene&#8217;s resistance to change <em>between</em> species, gene <em>constraint</em> refers to a gene&#8217;s resistance to change <em>within</em> species. Using 25,000 human exomes (the size of the dataset as of last year), she computed a metric of gene constraint for every human gene. The concept is similar to <a href="http://en.wikipedia.org/wiki/Ka/Ks_ratio">dN/dS</a>. Based on genetic variation in intergenic regions in the 1000 Genomes project, she computed <em>a priori</em> (i.e. absent selective pressure) probabilities of every possible DNA mutation in every possible trimer context, for instance the probability of ACG → ATG mutations, and then based on the trimers in each gene&#8217;s exons she computed the expected number of mutations in each gene. For synonymous mutations, the actual number of mutations in the 25,000 exomes was normally distributed around the expected number, suggesting little selective pressure. For missense mutations, the actual number of mutations was skewed small, indicating most genes are under purifying selection. She labeled the genes significantly depleted for missense mutations at p &lt; 1e-3 as &#8220;constrained&#8221; and showed that dominant (but not recessive) disease genes were highly enriched in this set.</p>
<p>This represents one approach to asking how constrained PRNP is &#8211; similar (but not identical) to <a title="Objective measures of gene conservation applied to PrP" href="/2013/09/01/objective-measures-of-gene-conservation-applied-to-prp/">asking how conserved it is</a>. It also provides an unbiased way to look at the distribution of genetic variation along the PRNP coding sequence. The ~43 reported disease-causing mutations in PRNP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/20583301">Beck 2010</a>] are overwhemingly concentrated in the C-terminal half of the protein. Is that because mutations simply don&#8217;t happen in the N-terminal half? Or because they&#8217;re so deleterious that they&#8217;re embryonic lethal? Or because they&#8217;re benign and so we don&#8217;t see them when we ascertain individuals with a disease?</p>
<p>Right now the 62,000 exomes are still under lock and key due (I think) mostly to patient privacy reasons &#8211; the largest public database of exomes is still the <a href="http://evs.gs.washington.edu/EVS/">Exome Variant Server</a>, with ~6,500 exomes. But the ASHG2013 talks were an exciting window into some of the analyses that will be possible soon as datasets get bigger and bigger.</p>
<p><strong>a moment of gratitude</strong></p>
<p>ASHG2013 was also a nice opportunity to reflect on just how advanced prion research is today. A large fraction of the talks and posters at the conference were on diseases whose genetic architecture we are only now beginning to unlock, from the multitude of genes that can be disrupted in autism to the question of how to pinpoint the causal mutation in ultra-rare Mendelian diseases with just one or two families.</p>
<p>There wasn&#8217;t a single poster or talk on prion disease. That&#8217;s probably because although I&#8217;ve highlighted above some interesting questions and approaches that remain at the intersection of prions and genetics, the most important questions were answered decades ago. Fatal familial insomnia is a dominant, autosomal, highly penetrant, monogenic, gain-of-function disorder caused by the D178N-129M haplotype in the PRNP gene on chromosome 20p [<a href="http://www.ncbi.nlm.nih.gov/pubmed/3762620">Lugaresi 1986</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/1346338/">Medori 1992</a> (<a href="http://www.nejm.org/doi/full/10.1056/NEJM199202133260704#t=article">ft</a>), <a href="http://www.ncbi.nlm.nih.gov/pubmed/1439789">Goldfarb 1992</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/7908444">Monari 1994</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC43466/pdf/pnas01129-0471.pdf">ft</a>)].</p>
<p>When Sonia and I first found out she had the FFI mutation, it seemed like incalculable bad luck. I&#8217;ve now come to appreciate the ways in which we have amazingly good luck. The basics of this disease were worked out 20 years ago, and now we are far, far beyond that: we have tens of top-notch monoclonal antibodies against PrP, several <a title="Comparison of high-throughput screening approaches used for anti-prion small molecule discovery" href="/2013/07/30/comparison-of-high-throughput-screening-approaches-used-for-anti-prion-small-molecule-discovery/">high-throughput assays</a>, a mouse model of FFI [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2775465/">Jackson 2009</a>] and tens of excellent mouse models for prion disease generally. We have <a href="http://www.prionalliance.org/2013/06/02/highlights-of-prion2013/">an annual conference</a>, a fabulous community of researchers, and we really know an astonishing amount about such a tiny protein. Who could possibly ever ask for all this?</p>
