---
layout: post
title: "Objective measures of gene conservation applied to PrP"
author: ericminikel
date: 2013-09-01 20:05:25
---
<p><strong>motivation</strong></p>
<p>When I go to scientific talks, I very often hear an introduction that includes some statement about how &#8220;this gene is highly conserved and therefore important.&#8221;  My boss once pointed out to me, you never hear someone say &#8220;the gene I study is not very highly conserved.&#8221;  Yet I also have never heard someone offer a single objective or quantitative metric as evidence that the gene they study is any more highly conserved than any other gene.</p>
<p>This matters because the notion that a gene is &#8220;highly conserved&#8221; seems to influence people&#8217;s thinking about whether a gene is important and whether it could be knocked down as part of a therapeutic strategy.  This comes up often in the prion literature.  Consider this quote from a classic paper [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8035877">Collinge 1994</a>]:</p>
<blockquote><p><b>Although PrP is highly conserved among mammals and widely expressed in early embryogenesis, mice homozygous for disrupted PrP genes appear developmentally and behaviourally normal.</b></p></blockquote>
<p>The authors cite [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1373228">Bueler 1992</a>] regarding the PrP knockout mice but as for the claim of &#8220;highly conserved,&#8221; one might want to add this note:</p>
<p><a href="http://store-xkcd-com.myshopify.com/products/citation-needed-sticker-pack"><img class="alignnone size-full wp-image-2816" alt="citation-needed" src="/wp-content/uploads/2013/09/citation-needed.png"/></a></p>
<p><strong>a comparison to platypus</strong></p>
<p>Yes, of course, orthologs of our <em>PRNP</em> gene are found in every mammal species, even <a href="http://en.wikipedia.org/wiki/Platypus">platypus</a> - a member of the <a href="http://en.wikipedia.org/wiki/Monotreme">monotremes</a>, the mammalian clade most distant from humans - and they all have strong sequence similarity, as seen in <a title="Expanded multiple alignments of PrP" href="/2013/06/29/expanded-multiple-alignments-of-prp/">multiple alignments</a>.</p>
<p>But is that so special?  Isn&#8217;t that true of <em>most</em> human genes?  86% of human genes have at least one ortholog in platypus [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18464734">Warren 2008</a> (<a href="http://www.nature.com/nature/journal/v453/n7192/pdf/nature06936.pdf">ft</a>), see <a href="http://www.nature.com/nature/journal/v453/n7192/extref/nature06936-s1.pdf">Supplement</a> Table 5].</p>
<p><a href="http://commons.wikimedia.org/wiki/File:Duck_billed_platypus_schnabeltier.jpg"><img class="alignnone size-full wp-image-2818" title="The platypus has PrP, but it has most of our other genes too." alt="Duck_billed_platypus_schnabeltier-600px" src="/wp-content/uploads/2013/09/Duck_billed_platypus_schnabeltier-600px.jpg"/></a></p>
<p><em>Above: The platypus has PrP, but it also has its own version of most of our genes. </em></p>
<p>Those authors also noted that 41% of human genes have simple 1:1 orthology across all mammals included in the analysis, matching <em>exactly one</em> gene in each of dog, mouse, <a href="http://en.wikipedia.org/wiki/Monodelphis_domestica">gray short-tailed opossum</a> (a model <a href="http://en.wikipedia.org/wiki/Marsupial">marsupial</a>) and platypus  [<a href="http://www.nature.com.ezp-prod1.hul.harvard.edu/nature/journal/v453/n7192/extref/nature06936-s2.pdf">Supplement-to-the-supplement</a>, Table A11].  But when I looked up PRNP in their database by its Ensembl gene symbol, <a href="http://genserv.anat.ox.ac.uk/clades/amniota/queryForGene?schema=mm_hsapiens1&amp;gene_id=ENSG00000171867&amp;SUBMIT=GO">ENSG00000171867</a>, I discovered it&#8217;s not even one of the elite 41%.  By the definitions those authors used, <a href="http://genserv.anat.ox.ac.uk/clades/amniota/displayOrthologGroup?group_id=13202">the orthology group for PRNP</a> includes PRND (Doppel, <a href="http://useast.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171864;r=20:4702556-4709106;t=ENST00000305817">ENSG00000171864</a>), both in humans and some of the other mammals included in the analysis.  Also, when only human and platypus are considered, there are <a href="http://genserv.anat.ox.ac.uk/clades/amniota/statsStrictOrthologSets">11,287 1:1 orthology relationships</a>, i.e. accounting for about half of human genes.</p>
<p>Taken together, all of this says that the sole fact that PrP is found across all mammals, including platypus, does not necessarily place it in even the top 50% of most conserved human genes.  The fact that PrP also has orthologs in chicken and frog is slightly stronger, especially (in my mind) since, although there is less sequence identity between our PrP and these more distant species, <a title="Proteolytic shedding of prion protein" href="/2013/06/29/proteolytic-shedding-of-prion-protein/">shedding</a> and <a title="Proteolysis of prion protein in the central region: alpha and beta cleavage" href="/2013/06/26/proteolysis-of-prion-protein-in-the-central-region-alpha-and-beta-cleavage/">alpha cleavage</a> of PrP are still conserved all the way out to chicken [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8093841">Harris 1993</a> (<a href="http://lab.rockefeller.edu/chait/pdf/93/93_harris_biochem.pdf">ft</a>)], and alpha cleavage may be essential for myelin maintenance [<a href="http://www.ncbi.nlm.nih.gov/pubmed/20098419">Bremer 2010</a>].  And of course, there are even orthologs in zebrafish, though the conservation is much weaker.</p>
<p>How special do these facts make PrP?  Still not very.  I turned to the <a href="http://www.informatics.jax.org/homology.shtml">JAX homology database</a>, downloaded the &#8220;<a href="ftp://ftp.informatics.jax.org/pub/reports/HOM_AllOrganism.rpt">Complete List of Vertebrate Homology Classes</a>&#8220;, and ran some SQL queries in R to determine how many human gene symbols have nominal orthologs in various species.</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">library(sqldf)
setwd(<span style="color: #ba2121;">'c:/sci/003prigen/blog/coding_conservation'</span>)
tbl <span style="color: #666666;">=</span> read.table(<span style="color: #ba2121;">'HOM_AllOrganism.rpt'</span>,header<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">TRUE</span>,quote<span style="color: #666666;">=</span><span style="color: #ba2121;">''</span>,comment.char<span style="color: #666666;">=</span><span style="color: #ba2121;">''</span>,sep<span style="color: #666666;">=</span><span style="color: #ba2121;">'\t'</span>) <span style="color: #408080; font-style: italic;"># http://www.informatics.jax.org/homology.shtml</span>
colnames(tbl)[<span style="color: #666666;">1</span>] <span style="color: #666666;">=</span> <span style="color: #ba2121;">"homolog_id"</span> <span style="color: #408080; font-style: italic;"># use SQL-compatible table names</span>
colnames(tbl)[<span style="color: #666666;">2</span>] <span style="color: #666666;">=</span> <span style="color: #ba2121;">"species"</span>

<span style="color: #408080; font-style: italic;"># count the human gene symbols in this database</span>
sql_query <span style="color: #666666;">=</span> <span style="color: #ba2121;">"</span>
<span style="color: #ba2121;">select   count(distinct t1.Symbol)</span>
<span style="color: #ba2121;">from     tbl t1</span>
<span style="color: #ba2121;">where    t1.species = 'human'</span>
<span style="color: #ba2121;">;"</span>
sqldf(sql_query) <span style="color: #408080; font-style: italic;"># 18843.  </span>

<span style="color: #408080; font-style: italic;"># how many human gene symbols have mouse orthologs?</span>
sql_query <span style="color: #666666;">=</span> <span style="color: #ba2121;">"</span>
<span style="color: #ba2121;">select   count(distinct t1.Symbol)</span>
<span style="color: #ba2121;">from     tbl t1, tbl t2</span>
<span style="color: #ba2121;">where    t1.homolog_id = t2.homolog_id</span>
<span style="color: #ba2121;">and      t1.species = 'human'</span>
<span style="color: #ba2121;">and      t2.species = 'mouse, laboratory'</span>
<span style="color: #ba2121;">;"</span>
sqldf(sql_query) <span style="color: #408080; font-style: italic;">#16887</span></pre>
</div>
<p>And so on.  JAX only lists 18,843 human gene symbols, not <a href="/wp-content/uploads/2013/09/hg19.refseq.gene_.symbols.txt">the full 23,705 in hg19 RefSeq</a>, but I&#8217;m assuming that&#8217;s just because some human genes didn&#8217;t have orthologs in any of the species they included.  Mouse orthologs exist, then, for 16887/23705 = 71% of human genes, and if you modify the last SQL query to say &#8216;chicken&#8217; or &#8216;zebrafish&#8217; you&#8217;ll get figures of 12,379 and 12,251, so in either case ~52% of human genes have orthologs in those species.</p>
<p>So, again, the presence of orthologs by itself &#8211; even in zebrafish &#8211; does not even necessarily place PrP in the top half of most conserved human genes.  Given that orthology does exist, we can also ask <em>how</em> similar PrP is between humans and other mammals.</p>
<p><strong>sequence evolution rate</strong></p>
<p>When a gene does have orthologs in other species, you can align them and make judgments about the conservation of each nucleotide or each amino acid.  The bioinformatics community is awash in tools for quantifying either how highly conserved a given base or amino acid is, or how flagrantly a missense mutation violates that conservation: <a href="http://genetics.bwh.harvard.edu/pph2/">PolyPhen2</a>, <a href="http://mendel.stanford.edu/SidowLab/downloads/gerp/">GERP</a>, <a href="http://compgen.bscb.cornell.edu/phast/background.php">PhastCons &amp; PhyloP</a>, <a href="http://sift.bii.a-star.edu.sg/">SIFT</a>, and so on.  These use phylogenomics and in some cases also physical traits of the original and substituted amino acid.</p>
<p>But those tools are all for conservation of a particular amino acid or nucleotide.  How do you make judgments about the conservation of a whole <em>gene</em>?</p>
<p>The most common metric used is the ratio of non-synonymous to synonymous amino acid substitutions, dN/dS.  dN is the count of non-synonymous substitutions between species being compared, and dS is the count of synonymous substitutions.  This is based on an assumption that evolution doesn&#8217;t act on synonymous substitutions, though obviously <a href="/2013/01/24/how-synonymous-are-synonymous-snps/">that&#8217;s an oversimplification</a>.</p>
<p>dN/dS is too simple in another way as well: if you glance at <a href="http://en.wikipedia.org/wiki/Genetic_code#RNA_codon_table">the genetic code</a>, some sites in a gene are constrained &#8211; ATG is the only methionine (M) codon, so any substitution is necessarily non-synonymous; and ACN is always threonine (T), no matter what N is, so substitutions at that third base are necessarily synonymous.  To normalize for the number of <em>opportunities</em> for synonymous or nonsynonymous substitutions, an alternate formulation is <a href="http://en.wikipedia.org/wiki/Ka/Ks_ratio">Ka/Ks</a>, where Ka is the ratio of actual to potential non-synonymous substitutions and Ks is the ratio of actual to potential synonymous substitutions.</p>
<p>If you accept for a moment the simplifying assumption that synonymous SNPs are neutral, then Ka/Ks &gt; 1 implies that evolution <em>favors</em> change &#8211; this is called <a href="http://en.wikipedia.org/wiki/Directional_selection">positive, Darwinian or directional selection</a>.  Ka/Ks = 1 implies no selection &#8211; evolution doesn&#8217;t care what happens to this gene.  Ka/Ks &lt; 1 implies evolution <em>resists</em> change &#8211; this is called <a href="http://en.wikipedia.org/wiki/Negative_selection_(natural_selection)">negative or purifying selection</a>.  (There are also such things as <a href="http://en.wikipedia.org/wiki/Balancing_selection">balancing selection</a>, which favors heterozygosity, <a href="http://en.wikipedia.org/wiki/Disruptive_selection">disruptive selection</a>, which favors phenotypic extremes, and <a href="http://en.wikipedia.org/wiki/Stabilizing_selection">stabilizing selection</a>, which favors phenotypic moderation, none of which are directly captured by Ka/Ks.)  See also <a href="http://en.wikipedia.org/wiki/McDonald%E2%80%93Kreitman_test">McDonald-Kreitman Test</a> and <a href="http://en.wikipedia.org/wiki/Tajima's_D">Tajima&#8217;s D statistic</a>.</p>
<p>The Ka/Ks value for PRNP within humans is 0.20 and between humans and chimpanzees is 0.50 (2 nonsynonymous / 4 synonymous) [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12690204">Mead 2003</a>], compared to an average of 0.23 for all human-chimpanzee orthologs [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2288694/">de Maghalaes 2007</a>].  In other words, PRNP appears to be under purifying selection, though not necessarily moreso than most of our genes.</p>
<p>On the other hand, the nonsynonymous polymorphisms that do exist in PRNP, especially M129V, appear to be more common than they should be compared to how rare new polymorphisms are.  Using <a href="http://en.wikipedia.org/wiki/Tajima's_D">Tajima&#8217;s D statistic</a>, it has therefore been argued that evolution has exerted balancing selection on PRNP, favoring the maintenance of polymorphisms that are dominant negative against prion disease [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12690204">Mead 2003</a>].</p>
<p><strong>does conservation imply deleterious knockout?</strong></p>
<p>The notion that PrP is highly conserved, yet its knockout phenotypes are relatively mild, was part of the original basis for positing the existence of <a title="pi: PrP’s hypothetical backup protein" href="/2013/09/01/pi-prps-hypothetical-backup-protein/">pi</a>, a hypothesized protein that has function similar or identical to that of PrP and carries out PrP&#8217;s job when PrP is absent, thus preventing PrP knockout mice from having severe phenotypes.</p>
<p>But all that is premised on an expectation that highly conserved genes should produce dramatic knockout phenotypes.  It seems intuitively obvious that this should be the case, and it was proposed long ago [<a href="http://www.ncbi.nlm.nih.gov/pubmed/409339/">Wilson 1977</a>], yet people have struggled to demonstrate it, and there appears to be only a slight correlation between gene conservation and knockout severity.</p>
<p>Some of the earliest work was done on a couple hundred mouse and rat genes before we had full genome sequences for any vertebrate [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10421576/">Hurst &amp; Smith 1999</a>].  68 genes classified as &#8220;essential&#8221; (knockout causes death or infertility) were compared to 107 non-essential genes.  Nominally, the essential genes had lower sequence evolution rates, but this was in part because some of the non-essential genes were immune genes deemed to be under directional selection, i.e. selection <em>favoring</em> change rather than <em>preventing</em> change.  Once this was corrected for, Hurst &amp; Smith found no difference.</p>
<p>But a subsequent, more quantitative analysis of yeast knockouts using growth rate as the quantitative phenotype did find a negative correlation [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11429604/">Hirsh &amp; Fraser 2001</a>]: genes that were more similar between <em>S. cerevisiae</em> and <em>C. elegans</em> slowed down S. cerevisiae&#8217;s growth more dramatically when they were knocked out.  Interestingly, though this study found a significant slope in a linear model, it did <em>not</em> find a significant difference in a binary comparison of &#8220;essential vs. non-essential&#8221;.  That&#8217;s because the linear correlation was driven by the <em>most</em> non-essential genes, those which had very little impact on growth rate and very strong divergence from <em>C. elegans</em>.  Combining these with the merely <em>somewhat</em> dispensible genes in the middle of the distribution sapped all the power out of a <a href="http://en.wikipedia.org/wiki/Welch's_t_test">Welch&#8217;s t test</a>.  In contrast, a study of bacterial genomes found both a significant correlation and a significant dichotomous difference between essential and non-essential genes [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1383730/">Jordan 2002</a>].</p>
<p>A re-analysis of Hirsh and Fraser&#8217;s yeast data using a much larger dataset concluded that the correlation was far smaller than originally reported, with a gene&#8217;s &#8220;dispensability&#8221; explaining only ~2% of variance in its evolution rate, compared to ~20% in Hirsh&#8217;s study [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12556881">Pal 2003</a>].  As time went on, the datasets got larger and the math got fancier.  It began to appear that dispensability &#8211; as well as gene expression level &#8211; are indeed correlated with sequence evolution rate, though it is hard to tell how strongly [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC555735/">Wall 2005</a>], and the effect is far from black-and-white &#8211; for a visual see <a href="http://mbe.oxfordjournals.org/content/22/4/1147/F2.expansion.html">Fig 2</a> in [<a href="http://www.ncbi.nlm.nih.gov/pubmed/15689524/">Zhang &amp; He 2005</a> (<a href="http://mbe.oxfordjournals.org/content/22/4/1147.long">ft</a>)].</p>
<p>The most comprehensive analysis I&#8217;ve found is in [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1560323/">Wolf 2006</a>].  Wolf considers several different (but correlated) variables: sequence evolution rate, expression level, number of physical interactors, number of genetic interactors, number of paralogs per species and knockout effect.  Wolf&#8217;s narrative interpretation of the principal components of these variables, though speculative, is a highlight: PC1 is said to reflect a gene&#8217;s &#8220;status&#8221; or prominence and ranks highly those genes that are both essential (strong knockout phenotype) but also well backed-up (many paralogs), whereas PC2 reflects &#8220;adaptability&#8221; &#8211; genes that are well backed-up but can also be done without.</p>
<p>A problem (or confusion?) I have about all of these analyses is that they seem to only account for genes that actually <em>have </em>orthologs among all species studied.  You can&#8217;t calculate a sequence evolution rate for a gene in species A if there is no equivalent gene in species B to which you can align the sequence.  Therefore it would seem that these analyses all leave out the least highly conserved genes of all &#8211; those that have only recently emerged, or have been lost in other species. (Or am I missing something? If so please leave me a comment).  An exception is one study which does consider gene loss events [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC403683/">Krylov 2003</a>].</p>
<p><strong>discussion</strong></p>
<p>I&#8217;m just beginning to get a handle on the different ways that people measure gene conservation and the different tools and formulas available for quantifying this.</p>
<p>I got interested in this for a few reasons.  First, people often say that the gene they study is conserved, but rarely back this up with evidence, and I like to quantify claims like this whenever possible.  Second, I often hear people be surprised at how mild PrP knockout phenotypes are given how &#8220;highly conserved&#8221; it is, and I want to know whether that surprise is really justified.  Third, I&#8217;ve heard people argue against <a title="The case for depleting PrP as a therapeutic strategy against prion diseases" href="/2013/04/08/the-case-for-depleting-prp-as-a-therapeutic-strategy-against-prion-diseases/">PrP depletion as a therapeutic strategy</a> on the basis that PrP is conserved and therefore must be too important to knock down.</p>
<p>In fact, probably no grand pronouncements are possible &#8211; if there is only a modest correlation between knockout phenotype severity and sequence evolution rate, then neither should people be surprised that PrP knockouts are viable, nor would it make much of a difference in our expectations if we realized that PrP is, quantitatively, not as highly conserved as we thought.</p>
<p>Still, this notion of conservation keeps coming up, and I&#8217;d like to have the conversation based on hard data.  In the literature I only found Simon Mead&#8217;s estimates of Ka/Ks for chimpanzee vs. human and it would be interesting to do this for several other species as well &#8211; it appears non-trivial based on <a href="http://www.biostars.org/p/5817/">this Biostars discussion</a> and the number of possible settings in <a href="http://abacus.gene.ucl.ac.uk/software/paml.html">PAML</a>, so I&#8217;ve got a bit to learn yet.  I also suspect there may be other, entirely different, quantitative metrics of gene conservation which I haven&#8217;t run across yet (please leave a comment if you know!).  And finally, it will be important to tease out whether PrP&#8217;s short length biases any of the potential metrics &#8211; it is among the smaller proteins in our proteome.</p>
