---
layout: post
title: "Calling repeat length polymorphisms with lobSTR"
author: ericminikel
date: 2013-01-08 00:32:41
---
<p>It is generally held to be quite tough to call repeat length polymorphisms from sequence data for several reasons: it&#8217;s hard to uniquely align repeat-rich elements to the reference genome, repeats may be longer than the read length, and pileup depth over GC-rich repeats will be depleted due to poor PCR amplification.</p>
<p>Enter <a href="http://jura.wi.mit.edu/erlich/lobSTR/">lobSTR</a>, created by <a href="https://twitter.com/mgymrek">Melissa Gymrek</a> at the <a href="http://wi.mit.edu/">Whitehead Institute</a>.  The &#8216;STR&#8217; stands for <a href="http://en.wikipedia.org/wiki/Microsatellite">Short Tandem Repeat</a>, which simply means 2-6 nucleotide repeat elements. This term is almost never used in the Huntington&#8217;s / trinucleotide repeat disease literature, but is the term of choice in law enforcement and forensics circles.  The FBI has identified 13 STRs which are highly polymorphic between individuals but highly stable within individuals, meaning they&#8217;re ideal for &#8216;DNA fingerprinting&#8217; &#8211; using these repeat lengths to create a DNA profile, &#8220;the probability that two unrelated individuals will have the same DNA profile purely by chance is less than 1 in 200 billion&#8221; [<a href="http://www.justice.gov/oig/special/0405/final.pdf">Dept of Justice 2004</a>].  And lobSTR got <a href="http://esciencenews.com/articles/2012/04/27/lobstr.algorithm.rolls.dna.fingerprinting.21st.century">some secondary press</a> for its potential forensics applications.</p>
<p>But short tandem repeats also seem to be a good candidate for where some of the &#8216;missing heritability&#8217; of human disease might lie.  As Gymrek points out, repeat lengths mutate much more quickly than SNPs do, and reapet expansions are already known to be responsible for a large number of diseases [see <a href="http://www.br.inter.edu/dirlist/Ciencia_Tecnologia/juan_negron/BIOL%206905/ARTICULOS/DNA%20repeats%20diseases.pdf">Mirkin 2007</a> for a review of molecular mechanisms of repeat expansion and its consequences].</p>
<p>lobSTR&#8217;s ability to detect repeat length polymorphisms is largely limited by read length.  In the original lobSTR paper [<a href="http://genome.cshlp.org/content/early/2012/04/19/gr.135780.111.full.pdf+html">Gymrek 2012</a>], Gymrek states that lobSTR only calls lengths for repeats that are completely spanned by a read.  So if you&#8217;ve got 100b reads and you need 10b flanking each side of the repeat in order to uniquely align, you&#8217;d be limited to calling repeats that total less than 80bp. That&#8217;s ~26 trinucleotide repeats, less than the pathogenic threshold for HD and most other trinucleotide diseases (see <a href="/wp-content/uploads/2012/12/orr-zoghbi-2007-trinucleotide-repeat-disorders.pdf">Orr &amp; Zoghbi 2007</a>).  However, in v2.0, she added support for using paired-end reads to uniquely align one repeat-rich read without much (or any?) flanking sequence as long as its mate aligns uniquely, and to call <em>minimum</em> repeat lengths based on reads that do not completely span the repeat tract (though this is still in beta).  So I figured it was worth a try to see if lobSTR could detect the HTT CAG expansion in my HD exomes.</p>
<p>As background, I have 50 paired end 100bp exomes (49, really, since one turned out to be a duplicate) of HD patients who have been separately genotyped for repeat length in the HTT CAG tract (<a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgHubConnect.destUrl=..%2Fcgi-bin%2FhgTracks&amp;clade=mammal&amp;org=Human&amp;db=hg19&amp;position=chr4%3A3076603-3076666&amp;hgt.positionInput=chr4%3A3076603-3076666&amp;hgt.suggestTrack=knownGene&amp;Submit=submit&amp;hgsid=320376463">chr4:3076603-3076666</a>).  The hg19 reference genome has 21 CAG repeats (we think the population average is actually around 17), including the CAACAG that usually appears at the end of the repeat region.  Each patient in my dataset has an expanded allele between 41 and 55 CAGs in length (i.e. 123 &#8211; 165bp of repeats), and a normal allele of between 14 and 32 repeats.</p>
<p>lobSTR was pretty easy to get started with using the binaries and indices provided on the <a href="http://jura.wi.mit.edu/erlich/lobSTR/download.html">download</a> page.  One caveat: after you <code>wget</code>, <code>gunzip</code> and <code>tar -xvf</code> the hg19 index, you&#8217;ll notice it has indices for a huge number of different 2-6nt repeats, mostly starting with A and a few starting with C.  None starting with G or T, and even CAG, for example, is not in there.  On further investigation it became clear that each repeat listed here represents all six different frame/strand combinations, so AGC is standing in for CAG=AGC=GCA=CTG=TGC=GCT.</p>
<p>For all the different sample ids in my dataset, denoted by <code>{sid}</code>, I ran the following code:</p>
<pre>lobSTR --p1 /data/HD/dataset/human_hd_extreme_exome/fastq_unzipped/{sid}_1.fq --p2 /data/HD/dataset/human_hd_extreme_exome/fastq_unzipped/{sid}_2.fq -q --index-prefix /data/HD/dataset/lobSTR-index/index_trf_hg19_extend1000/lobSTR_ --out {sid} --max-diff-ref 500
python ~/bin/lobstr_v2.0.2_1.linux64/scripts/lobSTR_alignment_checks.py -f {sid}.aligned.tab --plot
allelotype --command classify --bam {sid}.aligned.bam --haploid chrX,chrY --noise_model ~/bin/lobstr_v2.0.2_1.linux64/models/illumina2 --out {sid} --strinfo ~/bin/lobstr_v2.0.2_1.linux64/data/strinfo.hg19.tab</pre>
<p>Running time ranged from 0.5 to 2.5 hours (these are ~41x exomes with 15-27M read pairs total).</p>
<p>Next, to see what calls lobSTR had made for the HTT CAG tract, I grepped out the relevant line from each output file:</p>
<pre>grep $'^chr4\t3076603' *.genotypes.tab</pre>
<p>Results: for 41 out of the 50 exomes, lobSTR called an allelotype of NA/NA meaning it did not have enough information to make a call.  For 2 exomes, lobSTR called the site homozygous reference (allelotype 0/0) and for 7 exomes it called a homozygous variant shorter than the reference (e.g. allelotype -15/-15).  Of these 9 cases, the repeat length called was equal to the length of the normal allele that we have on file for that patient in only two cases, and within a few bases plus or minus for the other patients. The expanded allele was not called for any exomes, and all of the minimums given in column 16 (see page 15-16 of the <a href="http://jura.wi.mit.edu/erlich/lobSTR/lobSTR_manual_v2.0.pdf">v2.0 manual</a> for table schema) were less than 0, meaning lobSTR had not detected the presence of an expanded allele of unknown length.</p>
<p>None of this is too surprising &#8211; lobSTR doesn&#8217;t claim to be able to call expansions longer than read length, and the use of partially overlapping reads is still in beta.  Moreover the HTT repeat is particularly hard to call because reads are depleted there.  In my exomes we have average 41x coverage of exons, but coverage at the edges of the CAG tract ranges from 2 &#8211; 10x across the different samples.  We&#8217;re not totally sure why that is, though we have some ideas.  First, the GC-rich nature of this repeat region makes it hard to amplify, and our exomes are pretty low quality &#8211; ~27% <a title="How PCR duplicates arise in next-generation sequencing" href="/2012/12/11/how-pcr-duplicates-arise-in-next-generation-sequencing/">PCR duplicates</a> on average, which suggests that they were amplified more than is ideal before sequencing.  Second, it may just be that the exome capture probes for exon 1 of HTT are of poor quality and aren&#8217;t capturing enough molecules from this region.  If we had unbiased whole genome data, which we don&#8217;t yet, we&#8217;d be able to examine whether that&#8217;s the case.  Third, at the alignment level you have the difficulty of uniquely aligning reads from such a repetitive region, particularly since the CAG repeat tract is flanked on its immediate right by a polyproline CCG repeat.</p>
<p>In short, it&#8217;s no fault of lobSTR that it couldn&#8217;t make an accurate call of the allelotype at this locus.  Still, it is tempting to think there might be more ways to glean repeat length information from sequence data.</p>
<p>First, most of my HD samples have one or two <em>pure</em> CAG reads in the HTT CAG tract, demonstrating that one allele has a repeat of at least 33 trinucleotide units.  These reads have no flanking sequence at all but have a mate that aligned uniquely to some nearby stretch of sequence.  As far as I can tell from the documentation and the <a href="http://sourceforge.net/p/lobstr/code/ci/23598ee9d634aad9e287f7fbf11c6871308f192a/tree/src/">C++ source on SourceForge</a>, lobSTR is currently not taking advantage of these reads &#8211; it only uses the mate&#8217;s information to gain confidence in an alignment, not to entirely substitute for flanking sequence on the individual read itself.  To test this, I tried aligning with <code>--minflank 0</code> and lobSTR gave me an error: <code>lobSTR: Error: invalid min flank length</code>.</p>
<p>Second, I wonder whether combining lobSTR&#8217;s approach with insert size information might be fruitful.  Insert sizes from paired end reads are used in some more generic indel callers (see <a title="UAB day 1: intro to sequencing" href="/2012/12/11/uab-day-1-intro-to-sequencing/">UAB day 1 notes on SVs</a>).  While the HTT expanded allele is not long enough to cause insert sizes that are <em>individually </em>anomalous, the <em>distribution</em> of insert sizes could still be informative.</p>
<p>Third, some things may be telling by their absence.  Given sufficient read depth around the repeat region, the <em>absence</em> of any read spanning the entire repeat region may suggest a length greater than 100bp, and the absence of a read pair spanning the repeat region may suggest a repeat length greater than the typical range of insert sizes.  HTT is a bad example for this, but consider <a title="the gene where repeat expansions cause myotonic dystrophy" href="http://en.wikipedia.org/wiki/Myotonic_dystrophy_protein_kinase">DMPK</a>.  In that gene, healthy alleles have 5-37 CTG repeats.  That&#8217;s up to 111bp of repeat, so it might not be spanned by a single read but is very likely to be spanned by a pair.  By contrast, pathogenic repeats in DMPK can be 1000 repeat units (3000bp) in length, where in conventional paired-end sequencing you&#8217;d be very unlikely to have a pair that spans the repeat tract (for instance, in my exomes the average insert size is 200bp and 99% of pairs are less than 500bp).  Provided that you had enough read depth, the absence of such a spanning pair would be conspicuous.</p>
<p>Overall, my review of lobSTR is very positive: it&#8217;s easy to set up, runs fast, is well-documented and is a clever approach to solving a problem no one else has managed to do well (or at all).</p>
<p>Calling repeats longer than read length, though, is still a tough problem.</p>
