---
layout: post
title: "Heritability and genetic modifiers in Mendelian diseases"
author: ericminikel
date: 2013-03-05 21:55:08
---
<p>A Mendelian disease is one where a single genetic mutation decides whether you get the disease or not.  For instance, PRNP mutations are wholly responsible for fatal familial insomnia, and CFTR mutations are wholly responsible for cystic fibrosis.</p>
<p>By contrast, complex traits and complex diseases are determined by a whole bunch of different genetic factors.  Height, for instance: if your parents are tall, you&#8217;re likely to be tall too, but it&#8217;s not as simple as &#8216;you got the tall gene.&#8217;  Though we don&#8217;t know the <em>full</em> genetic architecture of any complex diseases, it is believed that hundreds of genomic loci contribute to many complex diseases as well &#8211; such as schizophrenia (I couldn&#8217;t find any good recent reviews; one relevant paper is <a href="http://www.ncbi.nlm.nih.gov/pubmed/19571811">International Schizophrenia Consortium 2009</a>).</p>
<p>The concept of &#8216;heritability&#8217; is pretty simple for Mendelian diseases themselves because they follow Mendelian inheritance.  If the disease is dominant, then an affected person&#8217;s kids are 50/50 to have the disease too, and if the disease is recessive then each child of two heterozygous carriers has a 25% chance of having the disease.  For complex traits and complex diseases, a great deal of effort goes into trying to figure out how &#8216;heritable&#8217; they are &#8211; how much of the phenotypic variance in the population can be explained by genetics, as opposed to environmental factors or stochastic variation (see <a title="How to calculate heritability" href="/2013/02/04/how-to-calculate-heritability/">heritability post</a>).</p>
<p>However, even Mendelian diseases can have genetic <em>modifiers</em>, and those modifiers can be heritable.  <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3040597/">Cutting 2011</a> reviews what we know about genetic modifiers in cystic fibrosis.  Even though everyone with both CFTR alleles gets cystic fibrosis, some people get very sick very young and others stay healthy much longer.  Because cystic fibrosis is a loss-of-function disease, it can be caused by a wide variety of loss-of-function mutations in CFTR: Cutting says over 1800 mutations have been identified so far.  Not surprisingly the biggest genetic modifier of many cystic fibrosis phenotypes (especially in the pancreas and intestines) &#8211; is simply which CFTR mutation(s) you have.  Some are complete loss-of-function while others are able to preserve a little bit of functionality in some tissues.</p>
<p>However it has long been suspected that there are other genetic modifiers in cystic fibrosis as well. Cutting reviews a number of studies that have tried to estimate the heritability of some of the disease phenotypes (quantitative measurements of lung function, body mass index, etc.).  The methods used are largely the same as <a title="How to calculate heritability" href="/2013/02/04/how-to-calculate-heritability/">the methods for estimating heritability in complex traits</a> &#8211; MZ vs DZ twin studies, sibling correlation, etc.  I didn&#8217;t get any sense of a consensus from Cutting&#8217;s article, though: the heritability estimates for cystic fibrosis phenotypes vary dramatically, all the way from 0.0 (the first MZ/DZ twin study found MZ twins no more similar than DZ twins for individual phenotypes, though there was heritability of one composite measure [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11463149/">Mekus 2000</a>]) to 1.0 [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1899267/">Vanscoy 2007</a>].  The only methodology new to me here was a study of phenotypic concordance between twins before and after they moved out of the home [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2948620/">Collaco 2010</a>], a methodology purported to separate out the effects of shared vs. individual environment.</p>
<p>Cystic fibrosis is a bit of a special case: it is one of the most common Mendelian diseases.  For more rare diseases you aren&#8217;t going to find enough twins to do twin studies.  So I also spent some time looking at how heritability has been estimated in very rare diseases.</p>
<p>For prion disease, one study has suggested that age of onset has at least some heritability [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18729123">Webb 2009</a>].  Webb looked at three large pedigrees, each with a different mutation (6-OPRI CJD, P102L GSS and A117V GSS), and used parent-offspring regression to assess heritability of age of onset and age of death.  Each of these 3 mutations has a different average age of onset, and for these particular mutations PRNP codon 129 genotype also matters: MV heterozygotes have later onset than MM or VV homozygotes.  (Aside: as far as we know this is not true for FFI [<a href="https://cshmonographs.org/index.php/monographs/article/view/4035">Kong 2003</a>]).  Webb tried models with and without codon 129 genotype accounted for, because this information wasn&#8217;t available for some patients.  Failure to include codon 129 genotype in models does add some <em>noise</em>, but it does not actually introduce any <em>bias</em> because as Webb points out, each mutation segregates with a particular allele such that the codon 129 genotype is entirely determined by the unaffected parent and therefore uncorrelated between affected parents and affected offspring.  In any event, peoples&#8217; ages of onset and death were Z-scored for each mutation, and then parents were regressed against offspring.  If you look at the data, they don&#8217;t exactly scream correlation &#8211; here is Fig 4:</p>
<p><img class="alignnone size-full wp-image-1724" title="webb-2009-fig4" src="/wp-content/uploads/2013/03/webb-2009-fig4.jpg"/></p>
<p>But apparently there was a global correlation with <a title="Pearson's correlation coefficient" href="http://en.wikipedia.org/wiki/Pearson_product-moment_correlation_coefficient">r</a> = .25 &#8211; which, as I interpret it, would mean an upper limit on heritability of 2r = 50%.  This correlation seemed to be driven mostly by the P102L cases (black dots in above), which had a significant r = .47, though the other mutations also trended towards correlation.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/11186892">Squitieri 2000</a>, studying Huntington&#8217;s Disease, has pointed out the risk of ascertainment bias in heritability studies examining age of onset.  Imagine you&#8217;re recruiting patients for a study during a particular window in time, and you&#8217;re only considering patients who already have an age of onset, i.e. already have the disease.  Only relatively <em>concordant</em> sibling pairs are going to be both available for the study at the same time &#8211; if two siblings are discordant, one will have already passed away or one will not yet have the disease at the time when the other one is enrolling in the study.  The opposite is true intergenerationally: if you try to enroll parent-child pairs or trios, only parents with late onset and children with early onset will be available to enroll at the same time, and thus you&#8217;re selecting for <em>discordant</em> pairs.  There are other issues too (early onset people are less likely to ever become parents, early onset children are likely to attract more notice and get enrolled in studies more often, etc.)</p>
<p>Because Webb&#8217;s study used parent-offspring pairs, it&#8217;s biased towards underestimating heritability if anything.  Webb does not go into great detail about how the clinical data were processed and how/whether not-yet-affected carrier offspring were considered in the analysis, so it is hard to know how big an issue this might be.</p>
<p>Moving on, for prion disease in particular another source of evidence towards the existence of genetic modifiers is that scrapie incubation times vary about twofold between different inbred mouse lines [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pubmed/21528440">Lloyd 2011</a>].  This is no guarantee that there are similar modifiers in humans, but it&#8217;s fairly suggestive.  Lloyd reviews several studies which have used mice for <a title="quantitative trait locus" href="http://en.wikipedia.org/wiki/Quantitative_trait_locus">QTL</a> mapping &#8211; a concept reviewed by <a href="http://www.ncbi.nlm.nih.gov/pubmed/17173058/">Peters 2007</a>.  In brief, two inbred mouse lines are crossed to get highly heterozygous F1s, then the F1s are crossed again to get mixed F2s in which you can look for which genomic locations correlate best with the phenotype.  For prion diseases these studies have revealed some candidate loci, but no conclusive modifier genes yet.</p>
<p>There is also some interesting literature on heritability of age of onset in <a href="http://en.wikipedia.org/wiki/Spinocerebellar_ataxia">spinocerebellar ataxia</a> 2 (SCA2), which is caused by polyQ repeat expansion in the <a href="http://en.wikipedia.org/wiki/ATXN2">ATXN2</a> gene.  Just like in Huntington&#8217;s Disease, the Q repeat length explains much of the variance in age of onset (more repeats is worse).  So just like in HD, people who study SCA2 consider a residual age of onset &#8211; the difference between actual onset and predicted onset based on polyQ length.  There are a number of other SCAs, each caused by polyQ expansion in a different gene, and <a href="http://www.ncbi.nlm.nih.gov/pubmed/16000334">Pulst 2005</a> (<a href="http://brain.oxfordjournals.org/content/128/10/2297.long">ft</a>) hypothesized that some of the <em></em>residual age of onset in SCA2 might be explained by polyQ length in the <em>other</em> SCA-causing genes.  Pulst genotyped polyQ length in those genes in ~400 unrelated people and sure enough, it turned out that polyQ length polymorphisms in <a href="http://en.wikipedia.org/wiki/Cav2.1">CACNA1A</a>, the gene that causes SCA6, explained about 6% of the residual (not total) variance in age of onset in SCA2.  None of the SCA2 patients had a CACNA1A allele long enough to cause SCA6 &#8211; but the early onset people had slightly longer alleles than late onset.</p>
<p>That&#8217;s quite an interesting finding and the results were significant even after Bonferroni correction.  However, the heritability analysis in that paper was somewhat less awesome.  The heritability calculations used a separate set of patients from the above analyses, and simply doubled the sibling correlation to obtain a figure of 55%.  Pulst correctly describes this as an <em>upper bound</em> and not an <em>estimate </em>of heritability (no control to rule out shared environment contribution).  But still, there is no control here for the ascertainment bias that <a href="http://www.ncbi.nlm.nih.gov/pubmed/11186892">Squitieri 2000</a> discusses &#8211; age onset studies are simply more likely to include concordant sib pairs than discordant sib pairs.  If Pulst did anything to control for this problem, it&#8217;s not discussed in the paper &#8211; all it says about the sib pair data is that &#8220;For the analysis of residual variance and heritability, information from 148 SCA2 patients from 57 sibships was used.&#8221;  This ascertainment bias will tend to inflate Pulst&#8217;s figure, so I would take 55% as a rather loose upper bound.</p>
<p>In <a href="http://en.wikipedia.org/wiki/Myotonic_dystrophy">myotonic dystrophy</a>, the CTG repeat length explains most of the age onset variance.  The repeat length also explains most of the level of somatic instability, but the <em>residual </em>instability explains part of the age of onset [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22595968">Morales 2012</a>].  Morales also suggests that the level of somatic instability itself is itself ~40% heritable, but here the analysis is not transparent.  Morales feeds the data into QTDT, a software tool introduced by <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1288332/">Abecasis 2000</a>.  Abecasis&#8217; formula for partitioning variance into genetic, shared environment and individual environment components depends upon &#8220;the proportion of alleles shared identical by descent (IBD) between siblings <em>j</em> and <em>k</em> in family <em>i</em>&#8221; &#8211; to my reading, this is sibling IBD regression, similar to that used by <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1413498/">Visscher 2006</a> in calculating height heritability.  Visscher actually <em>had</em> genome-wide SNP markers to determine sibling IBD (which ranges from ~40% to ~60% &#8211; not all sibling pairs are equally alike!)  Yet people have for years been using QTDT to do heritability estimates (both Morales above and <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC373491/">U.S.–Venezuela Collaborative Research Project 2004</a>) <em>without </em>any empirical sibling IBD estimates.  Perhaps this involves an assumption that all sib pairs share 50% IBD, but if so, neither Morales nor the USVCRP paper mention this assumption, and it is not clear to me from Abecasis&#8217; paper how the shared environment and genetic components can possibly be partitioned if IBD doesn&#8217;t vary at all.  I also checked the <a href="http://www.sph.umich.edu/csg/abecasis/QTDT/tour/tour5.html">QTDT online tour</a> and, indeed, the variance partitioning relies on IBD from sibling pedigree files.</p>
<p>Maybe there is something I&#8217;m missing here (if so, please leave a comment to let me know) but, even after reading <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1288332/">Abecasis 2000</a> and looking at the <a href="http://www.sph.umich.edu/csg/abecasis/QTDT/tour/tour5.html">QTDT online tour</a>, I am still at a loss for understanding how authors are using QTDT to create heritability estimates without any genome-wide SNP genotyping to determine IBD.</p>
<p>Heritability estimation is difficult to begin with [see <a href="http://www.ncbi.nlm.nih.gov/pubmed/18319743">Visscher 2008</a> for a review].  Doubly so in diseases that are too rare for twin studies to be possible, and doubly so again for age of onset phenotypes, where family studies suffer from often unexamined ascertainment bias.    It is clear that some Mendelian diseases do have heritable modifiers of age of onset.   <a href="http://www.ncbi.nlm.nih.gov/pubmed/16000334">Pulst 2005</a>&#8216;s discovery of a modifier for SCA2 is convincing and (to my knowledge) has never been contested in the literature; and <a href="http://www.ncbi.nlm.nih.gov/pubmed/18729123">Webb 2009</a>&#8216;s finding of heritability in prion diseases, because it uses parent-offspring regression, is more likely to underestimate than overestimate heritability.  But point estimates of heritability in these diseases should probably be taken with a large dose of salt &#8211; even more than for heritability studies generally.</p>
