---
layout: post
title: "TALENs and ZFNs"
author: ericminikel
date: 2013-01-19 23:39:31
---
<p>The world now has two major technologies for <a href="http://en.wikipedia.org/wiki/Genome_editing_with_engineered_nucleases">targeted editing of the genome</a>: zinc finger nucleases (ZFNs) and TAL effector nucleases (TALENs).  In principle these technologies are capable of targeting virtually any site in the genome and editing the DNA that is there.  These have already begun to revolutionize research and medicine, and their potential over the coming years seems enormous, so I sat down to wrap my head around how they work.</p>
<p><iframe src="http://www.youtube.com/embed/zDkUFzZoQAs" frameborder="0" width="560" height="315"></iframe></p>
<p>Gene targeting, while fairly new, isn&#8217;t <em>as</em> new as ZFNs or TALENs.  In the late 1980s, Cappechi, Evans and Smithies figured out that the cellular process of homologous recombination could be hijacked to swap out genomic DNA for a new sequence of interest.  <a href="http://en.wikipedia.org/wiki/Homologous_recombination">Homologous recombination</a> - where enzymes swap two similar pieces of DNA for each other &#8211; is a mechanism native to all living organisms.  Bacteria use it to exchange useful genes with one another; we animals use it to criss-cross our pairs of chromosomes during <a href="http://en.wikipedia.org/wiki/Meiosis">meiosis</a>, which is important because it allows natural selection to act on individual genes rather than entire chromosomes. Otherwise a really beneficial gene could never enjoy positive selection if it happened to be just downstream from a really bad gene.  Anyway, Cappechi, Evans and Smithie figured out that if you could just create DNA homologous (i.e. similar, not identical) to an existing gene in an organism and get that DNA into the nucleus of a cell &#8211; which requires <a href="http://en.wikipedia.org/wiki/Electroporation">electroporation</a> to permeabilize the cell membrane &#8211; there was some chance it would recombine and be integrated into the cell&#8217;s genome.  By applying this to mouse embryonic stem cells and recombining mutant, non-functional versions of genes into the mouse genome to replace working genes, they invented the <a href="http://en.wikipedia.org/wiki/Knockout_mouse">knockout mouse</a>, an achievement that won them a <a href="http://www.nobelprize.org/nobel_prizes/medicine/laureates/2007/press.html">2007 Nobel Prize</a>.</p>
<p>That achievement has been the foundation of pretty much all <a href="http://en.wikipedia.org/wiki/Reverse_genetics">reverse genetics</a> (i.e. create a genotype, study the resulting phenotype) since then.  But just crossing your fingers and hoping for the homologous DNA to recombine is a fairly inefficient process &#8211; only a very small fraction of cells get successful recombination.</p>
<p>The process is inefficient in part because homologous recombination requires that first the DNA has to undergo a double-stranded break a relevant location, and then the repair mechanisms have to repair that break by fusing the new (rather than old) DNA to the broken end.  So if you were to sit down and brainstorm ways to try to speed this process up, you might hit upon the idea of cutting the genomic DNA first, to create the double-strand break.</p>
<p>Biology has just such a mechanism for creating double-stranded breaks: <a href="http://en.wikipedia.org/wiki/Restriction_enzyme">restriction enzymes</a>. Some species of bacteria evolved these enzymes to chop up the DNA of invading bacteriophages (viruses that infect bacteria).  The bacteria <a href="http://en.wikipedia.org/wiki/DNA_methylation">methylate</a> those sequences of their <em>own </em>DNA to prevent friendly fire. Brilliant.  For decades now, we humans have hijacked this mechanism for all manner of biology protocols.  If you think back, you might remember that in the days before sequencing, we used them for DNA fingerprinting in forensics and paternity tests.  Restriction enzymes have single nucleotide resolution in their binding specificity, so one SNP can be the difference between having a restriction site and not having a restriction site.  Therefore if you expose two different people&#8217;s DNA to the same restriction enzyme, it will cut them in different places, resulting in fragments of different lengths.  These differences in lengths, called <a href="http://en.wikipedia.org/wiki/Restriction_fragment_length_polymorphism">restriction fragment length polymorphisms</a> or RFLPs, featured prominently in the <a href="http://en.wikipedia.org/wiki/O._J._Simpson_murder_case">OJ trial</a>.</p>
<p>The trouble with restriction enzymes is, their recognition sequences are really short, and are therefore any given restriction enzyme will cut in hundreds or thousands of places in the genome.  For instance the EcoRI and SamI restriction enzymes&#8217; recognition sites:</p>
<p><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/7/78/EcoRI_restriction_enzyme_recognition_site.svg/175px-EcoRI_restriction_enzyme_recognition_site.svg.png" alt="File:EcoRI restriction enzyme recognition site.svg"/> &#8230; <img src="http://upload.wikimedia.org/wikipedia/commons/thumb/9/9f/SmaI_restriction_enzyme_recognition_site.svg/165px-SmaI_restriction_enzyme_recognition_site.svg.png" alt="File:SmaI restriction enzyme recognition site.svg"/></p>
<p>Just six bases each.  What was needed, instead, was a restriction enzyme that would recognize a much longer sequence &#8211; long enough to be unique to one position in the whole genome.</p>
<p><a href="http://en.wikipedia.org/wiki/Meganuclease">Meganucleases</a> are just that.  Despite the prefix, they don&#8217;t recognize a 1 million base pair sequence, just a 12-40 base pair sequence, but in many cases that is already enough to have one, and only one, recognition site in an entire organism&#8217;s genome.  So if the genomic site you want to target just <em>happens</em> to bear the sequence recognized by a known meganuclease, then you&#8217;re in business.  But there are 4<sup>18</sup>/2 possible 18 base pair sequences you might want to target, and you&#8217;re not just going to stumble upon 4<sup>18</sup>/2 different meganucleases to do the job.  People have tried using random mutagenesis screens to generate new meganucleases, with some success, but we&#8217;re still far from being able to target just <em>any</em> possible sequence.</p>
<p>Meganucleases have proven useful and you can buy them from <a href="http://www.cellectis-bioresearch.com/genome-customization/genomic-scissors/meganucleases">Cellectis</a> today &#8211; they&#8217;re by no means out of the picture.  But they never revolutionized science because they just weren&#8217;t programmable enough.  You couldn&#8217;t design them from the ground up to do your bidding.</p>
<p>Enter the <a href="http://en.wikipedia.org/wiki/Zinc_finger#Cys2His2">Cys<sub>2</sub>His<sub>2</sub> zinc finger protein</a>.  These are sequence-specific DNA-binding proteins that act as <a href="http://en.wikipedia.org/wiki/Transcription_factor">transcription factors</a>, i.e. regulating gene expression by binding selectively to gene promoters.  Lots of species have them, including us humans (the <a href="http://en.wikipedia.org/wiki/Zif268">EGR1 gene</a> codes a zinc finger transcription factor).  But while we have thousands of transcription factors that are no more &#8216;reprogrammable&#8217; than a meganuclease, the beauty of the zinc finger is that it&#8217;s modular.  Each zinc finger has two to four domains, each with one <a href="http://en.wikipedia.org/wiki/Alpha_helix">α-helix</a> which fits into the major groove of a DNA double helix and binds selectively to a 3 base pair sequence.</p>
<p>The most commonly used zinc fingers have three &#8216;fingers&#8217; (DNA-binding domains), meaning you&#8217;ve got 9-base specificity.  The zinc finger proteins are then used in pairs, giving you 18-base specificity. This is enough to uniquely target almost any sequence you want to.</p>
<p>Zinc fingers, by themselves, just bind DNA.  So if all you needed to do was design a protein to selectively bind a promoter to activate a gene, you&#8217;d be done right there.   But if you want to use them to create targeted breaks in DNA, you need a pair of scissors to go with them.  Naturally you&#8217;d look around for another DNA-cleaving protein or domain of a protein that you could fuse to the zinc fingers.  All restriction enzymes have DNA-cleaving capability, but you know how complicated protein folding is: most of them have the DNA-cleaving capability all bound up with the DNA-binding capability and you can&#8217;t cleanly separate the two.  But the <a href="http://en.wikipedia.org/wiki/FokI">FokI</a> restriction enzyme, isolated by <a href="http://www.ncbi.nlm.nih.gov/pubmed/6282705">Sugisaki &amp; Kanazawa 1981</a>,  has just what you need: a DNA-cleaving domain that has no sequence-specificity at all and can function completely separate from the enzyme&#8217;s DNA-binding domain.</p>
<p>Now picture two zinc finger proteins, each with three fingers for a total of 18 base pair specificity, bound up and downstream of a site of interest, and each fused to one FokI DNA-cleaving domain in between them.  That&#8217;s a <a href="http://en.wikipedia.org/wiki/Zinc_finger_nuclease">zinc finger nuclease</a>.  People have even been able to make the FokI cleavage domain on each protein be slightly different and be able to cleave DNA only as <em>heterodimers</em> (combination of the two different FokI domains) rather than <em>homodimers</em>, to reduce off-target effects.  You can also go longer for more specificity: <a href="http://www.sigmaaldrich.com/life-science/zinc-finger-nuclease-technology/custom-zfn.html">CompoZr custom ZFNs</a>, sold by Sigma-Aldrich under sole commercial license to sell ZFNs, recognize 24 &#8211; 36 bases.  (By the way, although ZFNs are patented you can still legally roll your own, and the <a href="http://www.zincfingers.org/default2.htm">Zinc Finger Consortium</a> provides open source <a href="http://www.zincfingers.org/protocols.htm">protocols</a> for synthesizing ZFNs as well as <a href="http://www.zincfingers.org/software-tools.htm">software</a> to design them.)</p>
<p>The design of zinc fingers to target a specific DNA sequence is not quite as perfect or elegant as you might hope: <a href="https://www.google.com/search?q=zinc+finger+code">try Googling &#8216;zinc finger code&#8217;</a> and see if you have any better luck than I did.  I was hoping for a beautiful table showing which amino acid sequences in the alpha helix correspond to which DNA base triplets.  It seems it&#8217;s not quite that simple because the fingers interact with each other and change each other&#8217;s folding properties: &#8220;In both natural and designed multifinger ZFPs&#8230; individual fingers do not always function as completely modular units&#8221; [<a href="http://www.pnas.org/content/100/21/12271.full">Hurt 2003</a>].  So there is still a bit of trial and error and you can&#8217;t always target the exact sequence you want to and you might need to settle for another site a little ways away from your ideal site.</p>
<p>In spite of that, zinc fingers have proven to be pretty revolutionary.  Most famously, they have played a role in one of the awesomest biological advances in our lifetimes to date: stem cell gene therapy for HIV.  By the mid-1990s, people had some idea that HIV required the <a href="http://www.ncbi.nlm.nih.gov/gene/920">CD4</a> protein (found on <a href="http://en.wikipedia.org/wiki/CD4">CD4+ T cells</a>) as a receptor in order to infect cells. <a href="http://www.ncbi.nlm.nih.gov/pubmed/8639485">Samson 1996a</a> characterized the CCR5 gene, which soon turned out to be a <em>co-receptor</em> for HIV &#8211; the virus needs to bind first to CD4 and then to CCR5 in order to enter the T cells.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/8751444">Samson 1996b</a> discovered a CCR5 allele where the gene product is completely disabled by a 32bp frameshift deletion (now called the CCR5Δ32 allele).  This allele has about 10% frequency in Europeans (unseen in other populations) and Samson concluded that it must confer resistance to HIV infection since no HIV positive people were found to be homozygous for this allele in 723 HIV cases and even heterozygotes were depleted by about 35%.  Therefore deleting CCR5 from the genome seemed to be a potential therapeutic route for HIV.  By 2010 several groups had been able to demonstrate in mice the feasibility of using ZFNs to disrupt CCR5 and produce HIV resistance.  <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3422503/">Perez 2008</a> used ZFNs to disrupt CCR5 in human <a href="http://en.wikipedia.org/wiki/CD4">CD4+ T cells</a>, achieving disruption of about 50% of CCR5 alleles, and then transplanted these cells into immunodeficient <a href="http://en.wikipedia.org/wiki/NOG_mouse">NOG mice</a> and showed that the mice with the CCR5-disrupted CD4+ cells produced less virus and maintained higher CD4+ cell counts (both indicators of HIV infection severity) than control mice that received unmodified CD4+ cells.   This approach moved quickly into <a href="http://clinicaltrials.gov/show/NCT01252641">a clinical trial</a> which is ongoing  (See also <a href="http://www.sangamo.com/pipeline/sb-728.html">this self-promoting but well-written intro to the clinical trial</a> by Sangamo, the company that holds the patent on ZFNs) .  Another approach is, instead of transplanting modified CD4+ cells, to modify the stem cells that create the CD4+ cells, thus creating a permanent supply of HIV-resistant cells.  <a href="http://lightning.chem.wwu.edu/dept/facstaff/prody/nbt.1663.pdf">Holt 2010</a> did exactly this using hematopoietic (blood-forming) stem cells from human <a href="http://en.wikipedia.org/wiki/Cord_blood">cord blood</a>, transfected with ZFNs targeted to disrupt CCR5 and transplanted into mice.  HIV quickly killed off most of the undisrupted cells leaving a population of homozygous CCR5 knockout cells incapable of being infected.  Holt&#8217;s approach &#8211; using stem cells &#8211; is similar to the case of the one person who has been (we think) completely cured of HIV.  <a href="http://bloodjournal.hematologylibrary.org/content/117/10/2791.long">Allers 2011</a> reports on a HIV-positive leukemia patient who received a transplantation of bone marrow from a matched donor who was a CCR5Δ32 homozygote; the patient now appears not to be infected with HIV any longer.  No one has yet done that sort of transplantation with a patient&#8217;s <em></em>own ZFN-modified cells, but that could be next.  (This sort of procedure is limited to patients who already have life-threatening leukemia in addition to HIV, though, since <a href="http://en.wikipedia.org/wiki/Hematopoietic_stem_cell_transplantation">bone marrow transplants</a> are incredibly risky).</p>
<p>Zinc finger nucleases are all grown up at this point, having already made it to a clinical trial.    But all the buzz right now is about a new, comparatively untested but very promising alternative: <a href="http://en.wikipedia.org/wiki/Transcription_Activator-Like_Effector_Nuclease">transcription activator-like effector nucleases</a>, or TALENs.</p>
<p>Like ZFNs and most other cool things in biology, humans didn&#8217;t invent TALENs from scratch &#8211; instead we hijacked biological tools from a highly specialized organism that had evolved to do what we want to do. TALENs, make use of DNA binding proteins from the <a href="http://en.wikipedia.org/wiki/Xanthomonas"><em>Xanthomonas</em></a> genus of bacteria. <em>Xanthomonas</em> are plant pathogens that produce proteins to specifically bind to and up- or down-regulate promoter sequences in host genes important to the disease process &#8211; much like a transcription activator would. Hence the &#8220;transcription activator-like&#8221; part of the name. The central region of the TAL protein is made up of 17 to 18 repeats of the same 34 amino acid unit, with amino acids 12 and 13 of each unit determining what single DNA nucleotide it binds to.  And unlike ZFNs, these units are pretty much completely modular and spell out an almost-unambiguous code for the targeted DNA sequence.  Here&#8217;s the beautiful elegant table you&#8217;ve been waiting for:</p>
<p><table style="width:100%; " class="easy-table easy-table-default " border="0">
<thead>
<tr><th>Amino acids 12 &amp; 13</th>
<th>DNA letter targeted</th>
</tr>
</thead>
<tbody>
<tr><td>NI</td>
<td>A</td>
</tr>

<tr><td>HD</td>
<td>C</td>
</tr>

<tr><td>NN</td>
<td>G or A</td>
</tr>

<tr><td>NK</td>
<td>G</td>
</tr>

<tr><td>NG</td>
<td>T</td>
</tr>
</tbody></table></p>
<p>(Sources: <a href="http://www.cellectis-bioresearch.com/genome-customization/genomic-scissors/talen/what-are-tal-effector-nucleases">Cellectis</a> &amp; <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3084533/">Zhang 2011</a>)</p>
<p>So with, say, 17 repeats = 17*34 = 578 amino acids plus the non-repeating C- and N-terminal sequences of a TAL, you can uniquely target a 17 base pair DNA sequence.  Use two TALs and you can recognize 34 bases; put FokI DNA-cleaving domains in between them, and you&#8217;ve got a TALEN.</p>
<p>The process for synthesizing TALENs &#8211; creating your custom amino acid sequence to bind the DNA sequence of interest, adding FokI, and so on &#8211; is not exactly trivial. For a 20-page protocol on how to make your own, see <a href="http://www.ncbi.nlm.nih.gov/pubmed/22222791">Sanjana 2012</a>, and for some open source resources to help you design it, see <a href="https://tale-nt.cac.cornell.edu/">Cornell University&#8217;s TALE-NT tools</a>.   Or you can buy a custom-designed one from <a href="http://www.cellectis-bioresearch.com/genome-customization/genomic-scissors/talen">Cellectis</a>, which holds the exclusive commercial license on TALENs, for $5000.</p>
<p>TALENs are quite new at this point but have already been used in some cool applications. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23246482">Ding 2013</a> reports on using them to make isogenic models of human disease &#8211; pairs of cell lines that are (theoretically) identical except for the single site of a disease-causing genetic mutation.  (This is what we at Prion Alliance have proposed to do for FFI using ZFNs in <a href="http://www.prionalliance.org/2012/12/17/prion-alliance-partners-with-all-star-team-for-rare-disease-challenge/">our Rare Disease Challenge proposal</a>).  In practice, the cell lines didn&#8217;t turn out quite identical: the TALENs exhibited minimal off-target effects, but simply growing cells in culture leads to the gradual accumulation of mutations here and there (just as aging leads to gradual accumulation of somatic mutations in cells in your body).</p>
<p><a href="http://nar.oxfordjournals.org/content/33/18/5978.full">Durai 2005</a>, writing about ZFNs, says that making a double-stranded DNA break at an appropriate location can increase the efficiency of homologous recombination by 50,000-fold.  But even with highly site-specific ZFNs or TALENs there is still no one &#8216;at the wheel&#8217; making sure the ZFNs/TALENs and the new homologous DNA all end up at the right place at the right time and that DNA repair mechanisms stitch the chromosome back together in the way you want.  So after you&#8217;ve used your ZFNs/TALENs you have to do sequencing to see what fraction of the cells got successful integration of the edits you were trying to make.   In practice it will be some small-ish fraction of cells (say, anywhere from 1.6% to 34% in a recent report by <a href="http://www.ncbi.nlm.nih.gov/pubmed/23246482">Ding 2013</a>) and so you&#8217;ll select those for your further experiments and cull the rest.</p>
<p>That&#8217;s one reason why these technologies are still a long ways from allowing us to perform gene therapy in every cell of an adult human.  But in the immediate term, these will be amazing tools for research, letting us create new models of human diseases.  In the slightly longer term, there is also the possibility that they&#8217;ll play a role in making it possible to transplant modified versions of your own cells back into your body, like they did in HIV.  While that&#8217;s harder to do with brain than blood, <a href="http://www.stemcellsinc.com/">Stem Cells Inc</a> has already brought <a href="http://www.stemcellsinc.com/Therapeutic-Programs/Overview.htm">human neural stem cell transplants</a> to the clinical development stage, so don&#8217;t rule it out.</p>
<p><strong>update 2013-01-22: </strong>I just learned that the folks behind the <a href="http://www.ncbi.nlm.nih.gov/pubmed/23246482">Ding 2013</a> paper are sharing their plasmids through <a href="http://www.addgene.org/TALEN/">addgene.org</a> as a public resource to help other researchers build their own TALENs. Awesome!  Also another quick update: if you watched the Nature Biotechnology video on Youtube at the top of this post, you should be aware that this graphic at <a href="http://www.youtube.com/watch?feature=player_embedded&amp;v=zDkUFzZoQAs#t=84s">1:24</a> is not really accurate:</p>
<p><a href="http://www.youtube.com/watch?feature=player_embedded&amp;v=zDkUFzZoQAs#t=84s"><img class="alignnone size-full wp-image-1436" title="homology-directed-repair-nature-biotech-mistake" src="/wp-content/uploads/2013/01/homology-directed-repair-nature-biotech-mistake.jpg"/></a></p>
<p>The above graphic depicts the template for <a href="http://en.wikipedia.org/wiki/Homology_directed_repair">homology-directed repair</a> (HDR) as being of the same length as the piece of DNA that was cut out of the original genome.  In fact, HDR depends upon <em>homology</em> between the new piece of DNA and the existing, uncut parts of the genome, so the introduced piece of DNA on top that you are trying to knock into the genome would need to extend <em>beyond</em> the points of the cut.  So it should look something more like this:</p>
<p><img class="alignnone size-full wp-image-1437" title="homology-directed-repair-nature-biotech-fixed" src="/wp-content/uploads/2013/01/homology-directed-repair-nature-biotech-fixed.jpg"/></p>
<p>Although, while qualitatively more correct, even this graphic does not depict properly the <em>length </em>of homology required for HDR.  For ZFNs, apparently the new template DNA can be just ~100b longer than the cut site on each side.  That&#8217;s a big improvement over conventional knock-in without ZFNs/TALENs, which requires several kb of homology on each side of the desired mutation.</p>
<p><strong>update 2013-01-22 #2: </strong>The question arose today as to whether TALENs, like ZFNs, could be designed to only function as heterodimers so as to reduce off-target effects.  The answer is yes: <a href="http://nar.oxfordjournals.org/content/early/2012/06/07/nar.gks518.full">Cade 2012</a> uses &#8216;obligate heterodimer&#8217; TALENs in zebrafish and compares the results to homodimeric TALENs.  Predictably, the &#8216;obligate heterodimer&#8217; TALENs have fewer off-target effects.</p>
<p><strong>update 2013-01-22 #3: </strong>We also had a discussion about costs for TALENs.  As mentioned above, Cellectis sells custom-designed TALENs for <a href="http://www.cellectis-bioresearch.com/products-and-services/talen%E2%84%A2-access%E2%84%A2">$5000 a pair</a>.  Buying the requisite kit from addgene will run you just <a href="http://www.addgene.org/talengineering/TALENkit/">$300</a>, plus several days of labor (assuming you know what you&#8217;re doing!)  And although <a href="http://www.google.com/patents/EP2510096A2">U.S. patent application EP2510096A2</a> looks to preclude anyone but Cellectis from selling TALENs commercially, research institutions do produce them internally as a service to research groups within the institution.  I&#8217;m told that the &#8216;all-in&#8217; price for getting TALEN-modified cell lines runs about $14,000.  That includes design and assembly of TALENs, screening for cells that take up the TALEN plasmids, then screening for successfully modified cells and unmodified wild-type cells.</p>
<p><strong>update 2013-06-07: </strong>Sigh &#8211; outdated no sooner than it was posted!  <em>Now</em> the hot new genome-editing technology is CRISPR, see e.g. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23287718">Cong 2013</a>.  Also a colleague pointed out that the screenshot from that YouTube video above is inaccurate in another way &#8211; most of the time you don&#8217;t need to make <em>two </em>double-stranded breaks, just one, in order to get homology-directed repair to kick in.</p>
