---
layout: post
title: "Can humans live without PrP?"
author: ericminikel
date: 2013-03-20 00:29:11
---
<p>The purpose of this post is to collect in one place the available evidence as to whether we humans need prion protein and what the consequences of PRNP knockout or knockdown might be.  This is of interest because <a title="The case for depleting PrP as a therapeutic strategy against prion diseases" href="/2013/04/08/the-case-for-depleting-prp-as-a-therapeutic-strategy-against-prion-diseases/">suppressing prion protein</a> is one possible therapeutic strategy for prion diseases,<a title="Prospects for gene therapy of prion diseases" href="/2013/03/03/prospects-for-gene-therapy-of-prion-diseases/"> for instance in gene therapy</a>.</p>
<p><a name="animals"></a><strong>evidence from other animals</strong></p>
<p><a title="Multiple alignment of mammalian PrP amino acid sequences" href="/2012/11/14/multiple-alignment-of-mammalian-prp-amino-acid-sequences/">Prion protein is highly conserved amongst mammals</a>, and people tend to think conserved things must be important.  But PrP knockout mice survive [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8100741">Bueler 1993</a>] and are reported to look, breed and act pretty normal.  They have some sleep abnormalities [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8602267/">Tobler 1996</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/9030645/">Tobler 1997</a> (<a href="http://www.jneurosci.org/content/17/5/1869.long">ft</a>)] but most of the other reported phenotypes are observable only under stress [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2634447/">Steele 2007</a>]. The most prominent and well-established PrP knockout mouse phenotype is one of <a href="http://www.mayoclinic.com/health/peripheral-neuropathy/DS00131">peripheral neuropathy</a> resulting from myelin loss, not entirely unlike <a href="http://en.wikipedia.org/wiki/Charcot%E2%80%93Marie%E2%80%93Tooth_disease">Charcot Marie Tooth disease</a> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/20098419">Bremer 2010</a>].</p>
<p>PrP knockout cattle are reported to be quite normal too [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2813193/">Richt 2007</a>].  Here they are (from <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2813193/figure/F1/">Richt 2007 Fig 1a</a>):</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2813193/figure/F1/"><img class="alignnone size-full wp-image-1851" title="" alt="" src="/wp-content/uploads/2013/03/richt-2007-fig1a-prp-knockout-cattle.jpg"/></a></p>
<p>Richt did a battery of tests on them including brain histology, blood work and immune system tests as well as:</p>
<blockquote><p>a general examination of all organic systems and a detailed examination of the nervous system. Examination of the nervous system was focused on the following aspects: (i) evaluation of mental status, studied by observation of animal behavior and reactions to stimulation (approaching, menace, sounds and light); (ii) evaluation of sensory function in limbs and trunk, including study of superficial sensitivity, medular reflexes and conscious proprioception; (iii) evaluation of motor function in limbs and trunk by studying muscular tone, motor irritability (presence of muscle fasciculation and tremor) and gait abnormalities and, finally, (iv) evaluation of cranial nerves by observation of disorders in the corresponding innervated regions.</p></blockquote>
<p>Across all of these tests, the sole thing that appeared perhaps slightly abnormal was a slightly lower <a href="http://en.wikipedia.org/wiki/Mean_corpuscular_volume">red blood cell size</a> and hemoglobin levels in the knockout cows, though even the wild-type controls were at the low end of the reference range [see <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2813193/bin/NIHMS165952-supplement-Supplementary_information.pdf">Supplementary Table 2</a>].  Nothing else turned up abnormal.</p>
<p>More recently a team in Norway discovered a nonsense allele of PRNP in goats [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3542104/">Benestad 2012</a>].  This is interesting because whereas the prion protein gene was knocked out of mice and cows by researchers, no one has laid a finger on these goats&#8217; genomes.  Benestad discovered a naturally occurring null allele of PRNP with the 32nd codon mutated to a stop codon, which leaves only 7 amino acids in the mature protein because the first 24 amino acids (in goats; 22 in humans) are a signal peptide.  The allele frequency of this nonsense allele is estimated at 11% among Norwegian Dairy Goats, and the researchers found two homozygous <sub>32</sub>Stop goats.  These goats were shown to produce no PrP, but other than that, no abnormalities of any sort were noted.</p>
<p><a name="humanko"></a><strong>natural human knockouts?</strong></p>
<p>What about humans?  <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2650004/">Kamath 2009</a> provides the most thorough analysis to date of chromosomal deletions on 20p (short arm of chromosome 20).  Of Kamath&#8217;s 21 cases, only one, patient 17, had PRNP deleted.  It was a <em>de novo</em> hemizygous 7.4Mb deletion which also spanned <a href="http://en.wikipedia.org/wiki/JAG1">JAG1</a>. JAG1 is a known haploinsufficient gene, hemizygous deletions of which cause <a href="http://en.wikipedia.org/wiki/Alagille_syndrome">Alagille syndrome</a> (AGS).<em>   </em>Patient 17 died of AGS complications at age 18 months, so there is no way of knowing whether the PRNP deletion would have contributed any pathology above and beyond the AGS.  The major contribution of Kamath&#8217;s study was to show that <em>only </em>JAG1 is required for AGS and that several nearby genes are haplosufficient as their deletion does not contribute any additional pathology &#8211; but that analysis did not extend to PRNP because no other patients had deletions there.  In any event, hemizygous deletions could only tell us about the phenotype (if any) of reducing PrP gene dosage by ~50%, not of total knockout or more extreme knockdown.</p>
<p>Kamath cited the <a href="http://projects.tcag.ca/variation/">Database of Genomic Variants</a>, so I checked that as well.  No PRNP deletions there.  Then <a href="http://www.ncbi.nlm.nih.gov/dbvar?term=prnp">I searched in NCBI&#8217;s dbVar</a>, which is much larger.  As of today, dbVar has 32 reports for PRNP indels and copy number variations in humans.  All but one of the PRNP-only ones are indels of just a few base pairs reported by <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1557762/">Mills 2006</a> (a big famous indel-mapping study) rather than whole-gene deletions.  There is <a href="http://www.ncbi.nlm.nih.gov/dbvar/variants/esv141127/">one hemizygous 4kb deletion</a> reported, in sample NA19238 from 1000 Genomes at <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgHubConnect.destUrl=..%2Fcgi-bin%2FhgTracks&amp;clade=mammal&amp;org=Human&amp;db=hg19&amp;position=chr20%3A4%2C665%2C551-4%2C669%2C450&amp;hgt.positionInput=chr20%3A4%2C665%2C551-4%2C669%2C450&amp;hgt.suggestTrack=knownGene&amp;Submit=submit&amp;hgsid=330167849">chr20:4,665,551-4,669,450</a>, spanning the entire <del>protein-coding sequence</del> <strong>transcription start site and 5&#8242;UTR</strong> of PRNP.  It&#8217;s never been validated, though &#8211; the deletion was just called from next-gen sequencing data using some copy number variant detection algorithm.  <strong>update: I examined the raw sequencing reads and <a title="PRNP deletions in 1000 Genomes are not real" href="/2013/03/31/prnp-deletions-in-1000-genomes-are-not-real/">determined the deletion is not real</a>.</strong> Then the rest of the reports are larger deletions which span PRNP as well as other genes, which means it would be difficult to know whether the patient&#8217;s phenotype (if any) were due to PRNP or other genes being deleted.</p>
<p>That&#8217;s all I could find.  As the ranks of people undergoing genotyping or sequencing swell, we&#8217;ll just have to wait and see whether any more hemizygous deletions &#8211; or homozygous knockouts &#8211; turn up.</p>
<p><a name="knockoutneknockdown"></a><strong>knockout ≠ knockdown</strong></p>
<p>By the way, it&#8217;s not safe to assume that even extreme knockdown (by siRNA for example) of PRNP would be the same as knockout.  I learned an interesting story recently that illustrates this in relation to <a title="Cell Biology 05: The Cytoskeleton Part I: Actin" href="/2013/03/02/cell-biology-05-the-cytoskeleton-part-i-actin/">actin</a>.  <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1283463/">Di Nardo 2005</a> knocked down the Arp2/3 complex by 90% and found no phenotype, therefore concluding that &#8220;The Arp2/3 complex can be dispensable for leading-edge actin remodeling&#8221;.  Wrong.  <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3328382/">Suraneni 2012</a> knocked ARPC3 <em>out</em>, not <em>down</em>, and this resulted in E3.5 embryonic lethality in mice and severe lamellipodia formation deficits in isolated fibroblasts.  Even ~10% of Arp2/3 was enough to maintain native function, while total knockout was lethal.  Not safe to assume knockout = knockdown.</p>
<p>So even if we thought PRNP knock<em>out</em> would be highly detrimental (a view which the animal studies mentioned above certainly don&#8217;t support), it&#8217;s not a foregone conclusion that knock<em>down</em> (by siRNA or other means) would be just as bad.</p>
