---
layout: post
title: "Mouse models for studying human prion strains"
author: ericminikel
date: 2013-09-22 13:16:36
---
<p><strong>motivation</strong></p>
<p>Over a hundred <a title="Prion Therapeutic Review" href="/prion-therapeutic-review/">treatments</a> have been tested <em>in vivo</em> as therapeutic possibilities for prion disease.  Virtually all of these experiments have been done in rodent models expressing rodent PrP and infected with rodent prion strains.  To public knowledge, only two treatments have ever been tested against human prion strains <em>in vivo</em>: <a title="2-aminothiazoles: exciting progress and lessons for the field" href="/2013/06/01/2-aminothiazoles-exciting-progress-and-lessons-for-the-field/">2-aminothiazoles</a> (not yet published) and cpd-B [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23965382">Lu &amp; Giles 2013</a> (<a href="http://jpet.aspetjournals.org/content/early/2013/08/21/jpet.113.205799.long">ft</a>)].  Both were entirely ineffective despite showing considerable efficacy against RML, the most popular rodent prion strain.</p>
<p>Because genetic prion disease mutations like A117V, D178N and E200K produce prions in animals that are biochemically and phenotypically similar to the corresponding diseases in humans [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2749997/">Yang 2009</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2775465/">Jackson 2009</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/23959875">Jackson 2013</a>], it is our hope that rodent models of these diseases will be predictive of efficacy in humans, as argued <a title="Why we want to test anle138b in GSS mice" href="/2013/08/19/why-we-want-to-test-anle138b-in-gss-mice/">here</a>.  But sporadic Creutzfeldt-Jakob disease patients comprise the majority of prion disease patients who would stand to benefit from a new treatment. Moreover, genetic prion disease patients are too rare to be the basis for a clinical trial, so before any drug could reach them, it will probably need to show efficacy against sCJD in order to gain regulatory approval. For both of these reasons it&#8217;s important to show efficacy against sporadic CJD prions <em>in vivo</em> during a drug candidate&#8217;s preclinical phase.</p>
<p>Finding a drug effective against sCJD prions, after all the screening and lead optimization and early <em>in vivo</em> work has been done with RML prions, looks to be pretty difficult.  But surprisingly, even <em>doing the experiment at all</em> proves to be pretty difficult.  <a title="Biosafety levels for dealing with prions" href="/2012/12/27/biosafety-levels-for-dealing-with-prions/">Biosafety regulations</a>, in the U.S. at least, only require BSL2 facilities &#8211; sometimes with &#8220;BSL3 practices&#8221; &#8211; for prion work, including human prions.  Yet many institutions choose to restrict human prions to BSL3 facilities, which are considerably more expensive and difficult to set up.  As a result there are shockingly few laboratories capable of testing a drug&#8217;s efficacy against sCJD prions <em>in vivo</em>.</p>
<p>The goal of this post is to collect information on mouse strains expressing human PrP or chimeric human/mouse PrP and successfully infected with human prions.</p>
<p><strong>what is meant by human prions?</strong></p>
<p>Strain and transmission properties appear to be dictated by a combination of PrP conformation and primary structure (amino acid sequence) [<a href="http://www.ncbi.nlm.nih.gov/pubmed/17991853">Collinge 2007</a>].  We don&#8217;t yet know what properties matter for a potential drug&#8217;s therapeutic efficacy, and the answer seems likely to vary by drug.  Compounds that bind to PrP<sup>C</sup> might only bind to a particular amino acid sequence and therefore be species-specific; compounds that interfere with oligomerization might interfere only with a particular conformation and therefore be strain-specific; some molecules might be specific to both or neither.</p>
<p>Human prions can be transmitted to wild-type mice expressing ordinary MoPrP, but with very long incubation times and low attack rates [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC44932/">Telling 1994</a>].  In such cases, strains adapt to the new host&#8217;s PrP primary structure and so we shouldn&#8217;t take it for granted that the passaged strains still represent a faithful model of the original strain.  For instance, Fukuoka-1 is mouse-passaged human GSS [<a href="http://www.ncbi.nlm.nih.gov/pubmed/382976/">Tateishi 1979</a>], but we don&#8217;t know whether that means that cpd-B, which has some limited efficacy against Fukuoka-1 [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2169081/">Kawasaki 2007</a>], would be effective against GSS.</p>
<p>I&#8217;ll restrict this discussion mostly to animals expressing HuPrP or human/mouse chimeras, and later in the post I&#8217;ll return to the question of whether bank vole BvPrP might also be a model system for studying human prion strains.</p>
<p><strong>facilities</strong></p>
<p>To try to figure out what facilities worldwide can handle human prions in animals, I looked to the literature, searching Google Scholar for terms like &#8220;transgenic mouse cjd&#8221; and &#8220;huprp mice&#8221; and clicking through the first several pages of hits.  In all I identified only five labs that appear to have ever done work with animals expressing human PrP or mouse/human PrP chimeras and infected with human prion strains.  If I&#8217;ve missed any, please leave me a comment to let me know.</p>
<p><table style="width:100%; " class="easy-table easy-table-default " border="0">
<thead>
<tr><th>institution</th>
<th>location</th>
<th>responsible investigator</th>
<th>example publications</th>
</tr>
</thead>
<tbody>
<tr><td>Case Western</td>
<td>Cleveland, OH</td>
<td><a href="http://www.case.edu/med/pathology/faculty/kong.html">Qingzhong Kong</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16135751">Kong 2005</a> (<a href="http://digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1128&amp;context=zoonoticspub">ft</a>)</td>
</tr>

<tr><td>MRC Prion Unit</td>
<td>London, U.K.</td>
<td><a href="http://www.prion.ucl.ac.uk/welcome/the-director/">John Collinge</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/7658786">Collinge 1995a</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/8524411">Collinge 1995b</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/8878476">Collinge 1996</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/9333232">Hill 1997</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC136957/">Asante 2002</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/15539564">Wadsworth 2004</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2885063/">Asante 2009</a></td>
</tr>

<tr><td>INRA</td>
<td>Jouy-en-Josas, France</td>
<td>Hubert Laude</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2171367/">Beringue 2008</a></td>
</tr>

<tr><td>Roslin Institute</td>
<td>Edinburgh, U.K.</td>
<td><a href="http://www.roslin.ed.ac.uk/jean-manson/">Jean Manson</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16632309/">Bishop 2006</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2900653/">Bishop 2010</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3020518/">Plinston 2011</a></td>
</tr>

<tr><td>Tohoku</td>
<td>Sendai, Japan</td>
<td>Tetsuyuki Kitamoto</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1892390/">Taguchi 2003</a></td>
</tr>

<tr><td>UCSF</td>
<td>San Francisco, CA</td>
<td><a href="http://profiles.ucsf.edu/kurt.giles">Kurt Giles</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC44932/">Telling 1994</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/7553876/">Telling 1995</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/8953038">Telling 1996</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC153633/">Korth 2003</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2935907/">Giles 2010</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3372217/">Giles 2012</a></td>
</tr>
</tbody></table></p>
<p><iframe src="https://www.google.com/maps/ms?msa=0&amp;msid=207699150473782566242.0004e65db6261546bdf59&amp;ie=UTF8&amp;t=m&amp;ll=47.989922,6.328125&amp;spn=141.035805,298.828125&amp;z=1&amp;output=embed" height="350" width="425" frameborder="0" marginwidth="0" marginheight="0" scrolling="no"></iframe><br/>
<small>View <a style="color: #0000ff; text-align: left;" href="https://www.google.com/maps/ms?msa=0&amp;msid=207699150473782566242.0004e65db6261546bdf59&amp;ie=UTF8&amp;t=m&amp;ll=47.989922,6.328125&amp;spn=141.035805,298.828125&amp;z=1&amp;source=embed">HuPrP labs</a> in a larger map</small></p>
<p><strong>HuPrP vs. chimeric mouse models</strong></p>
<p>Although HuPrP mice infected with CJD prions have 100% attack rates and much shorter incubation times than wild-type mice infected with CJD prions, their incubation times are still longer than wild-type mice infected with RML prions, like about 263 days [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7553876/">Telling 1995</a>, reviewed in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC33918/">Prusiner 1998</a>].  The lowest I&#8217;ve seen published for pure HuPrP mice is 226 days for MM1 sCJD prions [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16135751">Kong 2005</a> (<a href="http://digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1128&amp;context=zoonoticspub">ft</a>)] and some HuPrP mouse lines are much higher, like &gt; 400 days [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2900653/">Bishop 2010</a>].  These long incubation times make it harder to study these mice.</p>
<p>MoPrP and HuPrP differ at 28 amino acids out of the 254 (mouse) or 253 (human) that comprise the full protein before post-translational modification.  7 of these 28 differences are in the signal peptide and 1 is in the GPI signal sequence, leaving just 20 differences in the post-translationally modified protein [<a href="/wp-content/uploads/2013/09/hu-mo-sha-prp-alignment.html">alignment</a>].  Chimeric PrP genes are those with some human and some mouse amino acids.  The first chimeric PrP gene was MHu2M, which has 19 mouse and 9 human amino acids.  The 9 human ones are codons 97, 109, 112, 138, 143, 145, 155, 166 and 168 (human numbering; subtract 1 for mouse numbering) [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC44932/">Telling 1994</a>].  MHu2M has slightly shorter incubation times than HuPrP, 238 days instead of 263 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7553876/">Telling 1995</a>, see <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC33918/">Prusiner 1998</a> - <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC33918/table/T4/">Table 4</a>].  That MHu2M has a shorter incubation time than HuPrP was one piece of the original evidence for <a title="Is Protein X the basis of dominant negative effects in prion disease?" href="/2013/08/16/is-protein-x-the-basis-of-dominant-negative-effects-in-prion-disease/">Protein X</a>.</p>
<p>Strangely, overexpressing MHu2M by 8-fold or even 32-fold did not significantly reduce this incubation time [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC153633/">Korth 2003</a>].  However, reverting codons 166 and 168 to mouse sequence reduced incubation times to just 110 days for MM1 sCJD [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC153633/">Korth 2003</a>].  This new chimeric transgene with just 7 human amino acids (97, 109, 112, 138, 143, 145, 155) has since been dubbed MHu#2, and the mice that carry this transgene are called Tg22372.</p>
<p>Kurt Giles subsequently led a systematic search for the exact combination of human and mouse amino acids that would give the minimum incubation time for human prions [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2935907/">Giles 2010</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3372217/">Giles 2012</a>].  The winner had just one fewer humanized amino acid: codon 112 was reverted to mouse.  The new transgene with just six humanized amino acids (97, 109, 138, 143, 145, 155) doesn&#8217;t have a short name; it is called MHu#2,M129,M111V.  The mice expressing this transgene are called Tg1014, and they have an incubation time of &lt; 80 days for MM1 sCJD prions.  This is the mouse line that has been used in UCSF&#8217;s therapeutic studies on <a title="2-aminothiazoles: exciting progress and lessons for the field" href="/2013/06/01/2-aminothiazoles-exciting-progress-and-lessons-for-the-field/">2-aminothiazoles</a> and cpd-B [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23965382">Lu &amp; Giles 2013</a> (<a href="http://jpet.aspetjournals.org/content/early/2013/08/21/jpet.113.205799.long">ft</a>)].</p>
<p>A different chimeric gene, dubbed ChM, is MoPrP with 129M and just 6 humanized amino acids near the C terminus (215, 219, 220, 227, 228, 230 &#8211; human numbering) [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1892390/">Taguchi 2003</a>].  Mice with this gene knocked-in at a 1.0x expression level have MM1 sCJD incubation times of about 150 days [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1892390/">Taguchi 2003</a>].</p>
<p>To the extent that primary structure matters for an antiprion compound&#8217;s efficacy, chimeric PrP genes might not be as faithful a model for human prions as HuPrP itself. But we don&#8217;t yet have any evidence that primary structure does matter, and in any case, the economies inherent in the much shorter incubation times would make it sensible to do most preclinical testing in chimeric PrP mice even if one &#8220;gold standard&#8221; experiment in HuPrP mice also had to be done.</p>
<p>Another question is whether some chimeric PrP sequences are inherently unstable and prone to misfold, and whether this would affect the outcome of therapeutic studies.  Giles&#8217; studies have largely addressed the question of instability by monitoring uninoculated mice for more than 600 days to look for spontaneous prion disease.  A handful of lines do have spontaneous disease [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3372217/">Giles 2012</a>] but most, including Tg1014, do not.</p>
<p>The fact that <em>some</em> mouse/human chimeric PrP genes bring about spontaneous disease [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3372217/">Giles 2012</a>] is evidence that the particular <em>combination</em> of amino acids in a PrP molecule can affect the propensity to misfold, even if no <em>single</em> amino acid change is pathogenic.  This is relevant to the study of bank vole PrP: it&#8217;s still an open research question whether the instability of BvPrP is due to the single unique glutamate (E) near the GPI anchor (see <a title="Proteolytic shedding of prion protein" href="/2013/06/29/proteolytic-shedding-of-prion-protein/">shedding</a> post) or due to the particular combination of 8 amino acids (in the post-translationally modified protein) that differ from mouse PrP.  In addition, I109 BvPrP is much less stable than M109 [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3295307/">Watts 2012</a> - <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3295307/table/t01/">Table 1</a>]; I109 is not found in <a href="/wp-content/uploads/2013/06/prp-multalign-mammals.html">any other mammals</a> I&#8217;m aware of and [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3295307/">Watts 2012</a>] does not cite any either.  Further experiments could address whether I109 is pathogenic on other PrP sequence backgrounds or whether it is only in combination with other BvPrP amino acids that it creates such propensity for prion formation.</p>
<p><strong>mouse models</strong></p>
<p>I originally set out to collect information on all transmissions of human prions to HuPrP or chimeric PrP mice.  This proved to be a daunting task: there are a lot of such experiments, differing in mouse model, prion strain and source of prions, and fitting a meaningful tabular comparison into the 600 pixel width of this blog was a challenge.</p>
<p>The table below summarizes the HuPrP and chimeric mouse strains that appear to currently be in use.  For brevity I&#8217;ve omitted the huge battery of genotypes screened by <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3372217/">Giles 2012</a>.</p>
<p><table style="width:100%; " class="easy-table easy-table-default " border="0">
<thead>
<tr><th>mouse name</th>
<th>prnp genotype</th>
<th>expression level in Tg homozygotes</th>
<th>announcing citation</th>
</tr>
</thead>
<tbody>
<tr><td>Tg110</td>
<td>HuPrP 129V</td>
<td>?</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC44932/">Telling 1994</a></td>
</tr>

<tr><td>Tg152</td>
<td>HuPrP 129V</td>
<td>4-8</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC44932/">Telling 1994</a></td>
</tr>

<tr><td>Tg5378</td>
<td>MHu2M 129M</td>
<td>1.0</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/7553876/">Telling 1995</a></td>
</tr>

<tr><td>Tg35</td>
<td>HuPrP 129M</td>
<td>2.0</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC136957/">Asante 2002</a></td>
</tr>

<tr><td>Tg45</td>
<td>HuPrP 129M</td>
<td>4.0</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC136957/">Asante 2002</td>
</tr>

<tr><td></a>Tg22372</td>
<td>MHu#2</td>
<td>1.6</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC153633/">Korth 2003</a></td>
</tr>

<tr><td>ki-ChM</td>
<td>ChM</td>
<td>1.0</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1892390/">Taguchi 2003</a></td>
</tr>

<tr><td>Tg1</td>
<td>HuPrP 129M, mouse signal peptide</td>
<td>2.0</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16135751">Kong 2005</a> (<a href="http://digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1128&amp;context=zoonoticspub">ft</a>)</td>
</tr>

<tr><td>Tg40</td>
<td>HuPrP 129M, mouse signal peptide</td>
<td>1.0</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16135751">Kong 2005</a> (<a href="http://digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1128&amp;context=zoonoticspub">ft</a>)</td>
</tr>

<tr><td>ki-HuPr P-129M*</td>
<td>HuPrP 129M, mouse signal peptide</td>
<td>1.0</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16632309/">Bishop 2006</a></td>
</tr>

<tr><td>ki-HuPrP-129V*</td>
<td>HuPrP 129V, mouse signal peptide</td>
<td>1.0</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16632309/">Bishop 2006</a></td>
</tr>

<tr><td>Tg650</td>
<td>HuPrP 129M</td>
<td>6.0</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2171367/">Beringue 2008</a></td>
</tr>

<tr><td>Tg1014</td>
<td>MHu#2 129M M111V</td>
<td>2.8</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2935907/">Giles 2010</a></td>
</tr>
</tbody></table><br/>
<em>*Jean Manson doesn&#8217;t refer to this mouse line by any particular name. In Bishop 2006 they are referred to as &#8220;gene-targeted transgenic lines&#8221; [details in <a href="http://www.sciencedirect.com/science/MiamiMultiMediaURL/1-s2.0-S1474442206704136/1-s2.0-S1474442206704136-mmc1.pdf/272647/FULL/S1474442206704136/b2bbfd1841a1cec8934100e4f132a27d/mmc1.pdf">supplement</a>] which I understand to mean <a title="The difference between knock-in and transgenic mice" href="/2012/11/13/the-difference-between-knock-in-and-transgenic-mice/">knock-in</a>.</em></p>
<p>I have included the expression levels for reference even though surprisingly there doesn&#8217;t appear to be a clear correlation between expression level and incubation time for chimeric transgenes [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3372217/">Giles 2012</a>].</p>
<p>None of these mouse lines are available from Jackson Labs.  However, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2900653/">Bishop 2010</a> states that the knock-in mice are being shared with other investigators, and in addition all of MRC Prion Unit&#8217;s HuPrP experiments prior to the development of the Tg35 and Tg45 mice used the Tg110 and Tg152 lines developed in San Francisco [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7658786">Collinge 1995a</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/8524411">Collinge 1995b</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/8878476">Collinge 1996</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/9333232">Hill 1997</a>].  So at least some of these mouse lines are being shared between different groups.</p>
<p><strong>bank vole PrP</strong></p>
<p>Human prions are also readily transmissible to bank voles [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1383487/">Nonno 2006</a>] and mice expressing BvPrP [Joel Watts presented at Prion2013].  Such animals could also therefore be a model system for studying human prions.  It&#8217;s not yet clear if these animals offer any biosafety advantage for studying human prions, and therefore any cost savings, though the very short incubation times, particularly in I109 BvPrP mice, would help make experiments economical.  To the extent that amino acid sequence matters for drug efficacy, they might not be the ideal system, but could still be a starting point.  And the fact that BvPrP appears to be able to adopt all known prion conformations does make it a perfect substrate for experiments specifically to tease out whether prion conformation is the sole determinant of a particular drug&#8217;s strain specificity.</p>
<p><strong>good laboratory practices</strong></p>
<p>An additional consideration in studying humanized mice is that in the U.S., the FDA wants any preclinical data submitted as part of an <a href="http://www.fda.gov/drugs/developmentapprovalprocess/howdrugsaredevelopedandapproved/approvalapplications/investigationalnewdrugindapplication/default.htm">Investigational New Drug application</a> to have been created in a laboratory certified as following <a href="http://en.wikipedia.org/wiki/Good_Laboratory_Practice">Good Laboratory Practices</a>.  (Other countries have similar regulations).  FDA&#8217;s <a href="http://www.fda.gov/ICECI/EnforcementActions/BioresearchMonitoring/NonclinicalLaboratoriesInspectedunderGoodLaboratoryPractices/ActiveLabsInspectionList/default.htm">list of certified labs</a> does not include any of the above-listed institutions; thus at present it&#8217;s impossible to do preclinical HuPrP therapeutic studies in a manner acceptable to the FDA.  I&#8217;ve heard there is a way to apply for a waiver, but I couldn&#8217;t find a relevant FDA link.</p>
