---
layout: post
title: "How to explain the discrepancy in PrPC half-life estimates in vivo"
author: ericminikel
date: 2013-09-25 09:12:52
---
<p>To date, only two studies have sought to estimate the half-life of PrP<sup>C</sup> in the brain <em>in vivo</em> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16186247">Safar 2005</a> (<a href="http://vir.sgmjournals.org/content/86/10/2913.long">ft</a>), <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2922600/">Price 2010</a>].  They came up with very different estimates: 18 hours and 4.95 days respectively.  The purpose of this post is to discuss possible explanations for this difference.</p>
<p>If you don&#8217;t read the whole post, the take-home message is: alpha and beta cleavage, conformational change and PrP binding to other ligands could all potentially contribute, but some of these things are pretty hypothetical and I&#8217;m not sure they could add up to a full explanation for this large difference in measurements.</p>
<p>I&#8217;ll begin with a comparison of the methods.</p>
<p><strong>Safar&#8217;s methods</strong></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/16186247">Safar 2005</a> (<a href="http://vir.sgmjournals.org/content/86/10/2913.long">ft</a>) created mice expressing PrP under the control of a Tet-off switch and treated them with doxycycline to suppress PrP expression, then serially sacrified groups of 3-5 of them to see how the concentration of PrP<sup>C</sup> decreased over time after doxycycline administration [<a href="http://vir.sgmjournals.org/content/86/10/2913/F2.expansion.html">Fig 2</a>]. Although these mice were not infected with prions, Safar used <a title="Introduction to conformation-dependent immunoassay" href="/2013/07/29/introduction-to-conformation-dependent-immunoassay/">CDI</a> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9771749">Safar 1998</a>] (normally a tool for looking at misfolded conformations of PrP) to assess the concentration of PrP<sup>C</sup>.  According to Safar&#8217;s <a href="http://vir.sgmjournals.org/content/86/10/2913.long#sec-2">methods</a>, CDI was performed using recFab D13 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9356250">Peretz 1997</a>] as the &#8220;antibody&#8221; (<a href="http://en.wikipedia.org/wiki/Fragment_antigen-binding">Fabs</a> are recombinant antibody fragments).  recFab D13 is used based on the observation that it binds little to PrP 27-30 (PK-digested PrP<sup>Sc</sup>) but binds more strongly to GdnHCl-denatured PrP 27-30, thus making it an indicator of PrP&#8217;s conformation [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9356250">Peretz 1997</a>].  Peretz states that the D13 Fab &#8220;showed a pattern of immunoreactivity with native and denatured PrP 27-30 similar to that found for the 3F4 Fab, which binds to PrP residues 109 to 112&#8243; and that it &#8220;reacted with peptides corresponding to residues 95 to 104&#8243;.  Together these statements indicate that D13&#8242;s binding site lies within PrP codons 95-104 and that it is not (very) accessible in a PrP<sup>Sc</sup> conformation.  Safar&#8217;s data show an exponential decay of PrP<sup>C</sup> as quantified by D13 binding in CDI [<a href="http://vir.sgmjournals.org/content/86/10/2913/F2.expansion.html">Fig 2</a>], with an estimated half-life of 18h.</p>
<p><strong>Price&#8217;s methods</strong></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2922600/">Price 2010</a> fed mice a diet consisting exclusively of <sup>15</sup>N radiolabeled algae, sacrificed them at intervals after the diet began and performed mass spectrometry-based proteomics to quantify the accumulation of radiolabeled proteins in the brain.  Based on nine timepoints (0 days,0.38 days, 1 day, 2 days, 4 days, 8 days, 16 days, 24 days and 32 days after the diet began) Price calculated a turnover rate for each protein, k<sub>turnover</sub>, which can be interpreted as the rate (λ) of an exponential growth curve, thus (assuming the total protein level is at equilibrium) the half-life of the protein is ln(2)/k<sub>turnover</sub>.  According to [<a href="http://www.pnas.org/content/suppl/2010/08/09/107.32.14508.DCSupplemental/sd01.xls">Supplementary Dataset</a>], just one PrP peptide was successfully quantified in the proteomics analysis, which was &#8220;VVEQMC(Carbamidomethyl)VTQYQK&#8221;.  This corresponds to codons 209-220 (inclusive) in human PrP, very close to the C terminus (the post-translationally modified protein ends at codon 231).  This peptide gave a k<sub>turnover</sub> of .14, thus a half-life of ln(2)/.14 = 4.95 days.</p>
<p><strong>discussion of differences</strong></p>
<p>One explanation that might contribute to the discrepancy is <a title="Proteolysis of prion protein in the central region: alpha and beta cleavage" href="/2013/06/26/proteolysis-of-prion-protein-in-the-central-region-alpha-and-beta-cleavage/">alpha and beta cleavage</a>.  The diagram below compares the alpha and beta cleavage sites in mouse PrP to the D13 epitope:</p>
<p><a href="/wp-content/uploads/2013/09/cleavage.vs_.recfabd13.png"><img class="alignnone size-full wp-image-3055" alt="cleavage.vs.recfabd13.600px" src="/wp-content/uploads/2013/09/cleavage.vs_.recfabd13.600px.png"/></a></p>
<p>Alpha cleavage would leave the D13 epitope intact but situated in the N1 fragment shed from the cell surface.  As far as I can tell, not much is known about the subsequent degradation of the N1 fragment. I looked at a few relevant papers [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8093841">Harris 1993</a> (<a href="http://lab.rockefeller.edu/chait/pdf/93/93_harris_biochem.pdf">ft</a>), <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2791025/">Guillot-Sester 2009</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3597823/">Fluharty 2013</a>] and didn&#8217;t see any information.  Alpha cleavage occurs in a considerable fraction of PrP<sup>C</sup> molecules &#8211; in a quick roundup of literature, [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3243553/">Westergard 2011</a>] cites a range of 10-50%. So <em>if</em> N1 is degraded quickly but C1 is more durable, this could go <em>some</em> of the way towards explaining the difference between Safar&#8217;s figure and Price&#8217;s figure.  But I don&#8217;t have any citations to actually claim a difference in degradation rates between these two fragments.</p>
<p>A second observation is that 3 of the 9 beta cleavage sites depicted above would abolish the D13 epitope.  (Note: The cleavage sites depicted above are actually mouse equivalents of known HuPrP cleavage sites from [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27779/">Parchi 2000</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27779/table/T1/">Table 1</a>], see <a href="/2013/06/26/proteolysis-of-prion-protein-in-the-central-region-alpha-and-beta-cleavage/">this post</a>; I didn&#8217;t find any data on beta cleavage sites specifically in MoPrP).  This means that beta cleavage of PrP could also potentially have contributed to the difference between Safar&#8217;s and Price&#8217;s figures.  However, this is unlikely to be a very large contributor as beta-cleaved PrP is said to account for only a few percent of total PrP in uninfected brains [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7642585/">Chen 1995</a> (<a href="http://www.jbc.org/content/270/32/19173.long">ft</a>)].</p>
<p>Together, both of these phenomena could potentially point in the right direction.  If you suppose that maybe C1 is more durable than N1, then this would tend to explain why Safar&#8217;s number is artificially low and Price&#8217;s artificially high.  But neither alpha nor beta cleavage seems large enough in magnitude to fully account for the large discrepancy.</p>
<p>Another potential explanation is that PrP<sup>C</sup> exists in a variety of conformations, with 5-25% of PrP molecules being what <a href="http://www.linkedin.com/pub/wen-quan-zou/13/b79/602">Wenquan Zou</a> calls &#8221;silent prions&#8221; [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16987816">Yuan 2006</a> (<a href="http://www.jbc.org/content/281/46/34848.long">ft</a>)].  These conformations have some biochemical properties of PrP<sup>Sc</sup>, such as detergent insolubility and NaPTA precipitation.  Whether they might also exhibit reduced binding to D13 has not, to my knowledge, been reported.  Perhaps these more aggregated species of PrP are the ones that stick around longer, thus contributing to Price&#8217;s number, yet don&#8217;t contribute much to Safar&#8217;s due to reduced D13 affinity.</p>
<p>Now, as mentioned above, I am assuming that the overall protein level is at equilibrium and so the rate of production of PrP<sup>C</sup> equals the rate of degradation.  That&#8217;s why in the previous paragraph I referred to aggregated species that take a long time to degrade still &#8220;contributing to Price&#8217;s number&#8221; when in fact Price&#8217;s number is based on PrP<sup>C</sup> translation (radioisotope incorporation) rather than degradation.</p>
<p>Which is an interesting point when you think about it.  Any and all varieties of PrP, regardless of conformation, cleavage state, etc., will contribute to the PrP quantified by Price, provided only that they include the intact VVEQMCVTQYQK peptide, which as far as I know has never been reported to be destroyed in any cleavage event and which will still be detected by mass spec regardless of PrP&#8217;s conformation.</p>
<p>In contrast, Safar&#8217;s measurement requires that PrP be intact (or, at a minimum, at least the N1 fragment be intact), and that the D13 epitope be exposed.  Conformational change can bury this epitope.  Could PrP binding events also occlude D13 from binding PrP?</p>
<p>For instance, <a title="Review of prion protein and amyloid beta interactions" href="/2013/05/02/review-of-prion-protein-and-amyloid-beta-interactions/">amyloid beta (Aβ) binds PrP</a> at PrP amino acids 95-110, as discovered through experiments with PrP deletion mutants [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2748841/">Lauren 2009</a>] and confirmed through experiments with 6D11, another antibody to this site [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3431439/">Um 2012</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3568961/">Larson 2012</a>, <a href="http://www.cell.com/neuron/abstract/S0896-6273(13)00552-7">Um 2013</a>]. Aβ binds PrP with a <a title="dissociation constant, a measure of affinity. lower number = higher affinity" href="http://en.wikipedia.org/wiki/Dissociation_constant">Kd</a> of 0.4 nM [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2748841/">Lauren 2009</a>] while full-length D13 antibody binds with a Kd of 3.3 nM [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11507642">Peretz 2001</a> - <a href="http://www.nature.com/nature/journal/v412/n6848/extref/412739a0_S1.htm">Supp Table 1</a>].  So apparently amyloid beta actually has <em>stronger </em>affinity for PrP than does D13, and it binds at the same site.  Could some of the decrease in PrP<sup>C</sup> in Safar&#8217;s measurement be due to occlusion of a higher and higher fraction of the remaining PrP molecules by Aβ?  Or, for that matter, by other ligands?</p>
<p><strong>summary</strong></p>
<p>In sum, I can think of three possible reasons to explain the difference between these two estimates:</p>
<ol>
<li><strong>Proteolytic cleavage.</strong>  Beta cleavage would abolish the D13 epitope measured by Safar. Alpha cleavage would send Safar&#8217;s D13 epitope floating away on the N1 fragment and Price&#8217;s PrP peptide still anchored in the C1 fragment.  If the N1 fragment is quickly degraded and the C1 fragment is durable, that would mean Safar&#8217;s measurement is deflated and Price&#8217;s measurement is inflated.  Which measurement is more &#8220;correct&#8221; would depend on what species of PrP you&#8217;re interested in measuring.  Because C1 appears incapable of becoming a prion [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3243553/">Westergard 2011</a>], for prion disease modeling purposes perhaps the half life of non-alpha-cleaved PrP is the more relevant figure.</li>
<li><strong>Conformational change.</strong>  Even in uninfected brains, some fraction of PrP<sup>C</sup> has some PrP<sup>Sc</sup>-like character; perhaps this PrP<sup>Sc</sup>-like PrP<sup>C</sup> is relatively longer-lived, yet it had the D13 epitope buried and so was not measured by Safar&#8217;s assay.</li>
<li><strong>Occlusion by other ligands.</strong>  At least one ligand &#8211; amyloid beta &#8211; can bind the D13 epitope with greater affinity than D13 itself.  Perhaps this or other ligands occluded a higher and higher fraction of the remaining PrP<sup>C</sup> after PrP transcription was shut off with doxycycline in Safar&#8217;s study, contributing to an appearance of faster PrP degradation than was really the case.</li>
</ol>
<p>Other than the (purely hypothetical) high half-life of the C1 fragment, I haven&#8217;t yet managed to come up with any other specific explanations for why Price&#8217;s figure might be inflated.  Of course, having worked with some mass spec proteomics data myself, I know it&#8217;s far from perfect.  But on the other hand, radioisotope incorporation has been around a long time and seems to be a tried-and-true method for determining protein turnover rates [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3570024/">Toyama &amp; Hetzer 2013</a>].</p>
<p>So that&#8217;s all I&#8217;ve got so far.  Are any of these ideas totally off base?  Are there any other good explanations for the vast discrepancy between 18 hours and 4.95 days?  I welcome any comments.</p>
