---
layout: post
title: "Is Protein X the basis of dominant negative effects in prion disease?"
author: ericminikel
date: 2013-08-16 00:01:48
---
<p><strong>definition</strong></p>
<p><strong>Protein X</strong> is a hypothesized ligand of PrP which is necessary, and rate-limiting, for PrP<sup>Sc</sup> to induce PrP<sup>C</sup> → PrP<sup>Sc</sup> conversion.  It is proposed to be species-specific such that, for instance, MoX binds to MoPrP with higher affinity than it does to HuPrP.</p>
<p><strong>why?</strong></p>
<p>Heterozygosity is protective in prion disease &#8211; for instance, in humans, codon 129MV heterozygosity is associated with reduced risk of sCJD, later onset and/or longer duration of genetic prion disease, and apparently complete resistance to vCJD [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3226040/">Lukic &amp; Mead 2011</a>].  Humans (almost always) only have two alleles of PRNP, which makes it difficult to tell if heterozygosity is protective simply because one allele just can&#8217;t participate in the prion action, or because it actively inhibits the other allele from participating.</p>
<p>However, studies of transgenic animals have made it clear that PrP molecules differing in their primary sequence are <strong>dominant negative</strong>, meaning they do indeed actively inhibit the spread of prions [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7553876/">Telling 1995</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC23307/">Kaneko 1997</a>].  Telling found that mice expressing HuPrP transgenes in addition to their endogenous MoPrP genes were actually <em>less</em> susceptible to infection by human prions than mice that expressed <em>only</em> HuPrP, on a PrP knockout mouse background.  That&#8217;s surprising because normally the more PrP you express, the more susceptible to prion infection you are [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8790598">Bueler 1994</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2229922/pdf/molmed00043-0025.pdf">ft</a>), <a href="http://www.ncbi.nlm.nih.gov/pubmed/8635458/">Fischer 1996</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC450028/pdf/emboj00006-0055.pdf">ft</a>)].  The expression of MoPrP actually reduced the vulnerability of HuPrP mice to human prions, despite increasing the total amount of PrP being expressed.</p>
<p>Then there&#8217;s the peculiar matter of MHu2M PrP.  Mouse and human PrP differ at 28 amino acids; MHu2M is a chimeric PrP gene that is mostly mouse but has 9 of the human amino acids in the central region, from codon 96 to 167 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7937921">Telling 1994</a> (<a href="http://www.pnas.org/content/91/21/9936.full.pdf">ft</a>)].  Mice expressing MHu2M PrP are actually more susceptible to human prions than pure HuPrP mice are.  Just as with HuPrP, the co-expression of MoPrP along with MHu2M actually reduces the susceptibility of these mice to human prions.  But the difference between MHu2M + MoPrP vs. MHu2M only is much smaller than the difference between HuPrP + MoPrP vs. HuPrP only.  The most salient comparison is summarized in [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC33918/">Prusiner 1998</a> - <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC33918/table/T4/">Table 4</a>].  Here are the key facts:</p>
<p><table style="width:100%; " class="easy-table easy-table-default " border="0">
<thead>
<tr><th>PrP genes</th>
<th>incubation time</th>
<th>attack rate</th>
</tr>
</thead>
<tbody>
<tr><td>HuPrP / MoPrP</td>
<td>721</td>
<td>1/10</td>
</tr>

<tr><td>HuPrP</td>
<td>263</td>
<td>6/6</td>
</tr>

<tr><td>MHu2M / MoPrP</td>
<td>238</td>
<td>8/8</td>
</tr>

<tr><td>MHu2M</td>
<td>191</td>
<td>10/10</td>
</tr>
</tbody></table></p>
<p>At a mechanistic level, why would MoPrP inhibit the conversion of HuPrPC to HuPrPSc?  And more weirdly, why would it inhibit HuPrP more than MHu2M?</p>
<p>To explain this weirdness, Prusiner and colleagues proposed the existence of Protein X. I&#8217;ll refer to mouse Protein X as MoX, even though no one else uses this notation.  The idea is that HuPrP<sup>Sc</sup> can bind to MoX well enough to do its job of converting more HuPrP<sup>C</sup>, as long as no one is competing for MoX&#8217;s attention.  But MoX is in short supply, binding is competitive, and MoPrP binds to MoX with higher affinity than HuPrP binds to MoX.  Therefore when MoPrP is expressed it sequesters all the MoX, making the propagation of human prions very slow and inefficient. By contrast, MHu2M, because it has 19 mouse amino acids that HuPrP lacks, binds to MoX with higher affinity than HuPrP does &#8211; high enough affinity to be able to compete with MoPrP on almost equal footing.</p>
<p>It was a complicated explanation for a complicated problem.  But it appealed in part because it would help to explain several other weird phenomena.  For instance, as seen in the table above, human prions transmit more efficiently to MHu2M mice than to pure HuPrP mice &#8211; maybe that&#8217;s because MHu2M has an easier time accessing all the MoX it needs.  Chimeric mouse/hamster PrP produces infectious material transmissible to both mice and hamsters [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8098995">Scott 1993</a>] &#8211; maybe the chimera is good at accessing both MoX and SHaX.  The GSS P102L mutation causes spontaneous disease in MoPrP mice and MHu2M mice but not HuPrP mice &#8211; again, maybe HuPrP just can&#8217;t access enough MoX to cause spontaneous prion disease within the mouse lifetime [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7553876/">Telling 1995</a>].  There are other surprising observations as well, some of which aren&#8217;t easily explainable even under the Protein X paradigm.  For instance, HuPrP / MoPrP mice are no more susceptible to human prions than just regular old MoPrP mice are &#8211; if anything, they are slightly less susceptible [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7937921">Telling 1994</a> (<a href="http://www.pnas.org/content/91/21/9936.full.pdf">ft</a>)].</p>
<p>A first step towards finding Protein X was to narrow down its binding site on PrP.  PrP with the N-terminal region deleted could still propagate prions [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8635458/">Fischer 1996</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC450028/pdf/emboj00006-0055.pdf">ft</a>)] and the central region was humanized in MHu2M, so it was reasoned that the binding site must be at the C terminus, past codon 167, where only 5 amino acids differ between human and mouse [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7553876/">Telling 1995</a>].</p>
<p>This was later refined to just a few amino acids which were sufficient to create or destroy a transmission barrier [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9294164">Kaneko 1997</a>].  You can see them in <a href="/wp-content/uploads/2013/08/hu-mo-sha-prp-alignment.html">this multiple alignment of HuPrP, MoPrP &amp; SHaPrP</a>:</p>
<p><a href="/wp-content/uploads/2013/08/huprp-215-219.png"><img class="alignnone size-full wp-image-2730" alt="huprp-215-219" src="/wp-content/uploads/2013/08/huprp-215-219.png"/></a></p>
<p>In N2a cell culture, these two amino acids seemed to control the transmission barrier.  Mutating amino acids 168, 172 or 219 was also sufficient to prevent transmission (though these codons are not different between humans, mouse and hamsters).  Those positions are far away in 1-D space but close to 215/219 in 3-D space when PrP is folded.  172 (171 in sheep) <a href="http://www.aphis.usda.gov/animal_health/nahms/sheep/downloads/sheep01/Sheep01_is_Genotypes.pdf">is an important scrapie susceptibility modifer</a> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7926780/">Westaway 1994</a>], as is 168 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12271119/">Perrier 2002</a>], and 219 is dominant negative against sCJD in humans [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9629853">Shibuya 1998</a>] &#8211; so all of these make sense.</p>
<p><strong>so what is Protein X?</strong></p>
<p>That&#8217;s the trouble.  If Protein X does exist, it&#8217;s now been 18 years and no one has found it.  Prusiner and colleagues had already done some searching for ligands of PrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1974464/">Oesch 1990</a>] and hadn&#8217;t found anything promising.  It&#8217;s also not known if the hypothetical Protein X would bind to PrP<sup>C</sup> or to PrP<sup>Sc</sup>, and under what conditions, which is one complication in trying to set up a massive screen for interacting partners, as has been done for amyloid beta [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2748841/">Lauren 2009</a>].</p>
<p>But interacting partners do come up from time to time &#8211; <a title="Review of prion protein and amyloid beta interactions" href="/2013/05/02/review-of-prion-protein-and-amyloid-beta-interactions/">amyloid beta is one</a> &#8211; and so a number of these proteins have been knocked out or overexpressed to see if that would alter the course of prion disease.  None had a large effect &#8211; APP knockout slowed prion disease down by 13%, but that&#8217;s not nearly enough for Aβ to be Protein X [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2828448/">Tamguney 2008</a>].</p>
<p>As time has gone on and Protein X hasn&#8217;t presented itself, people seem to talk less about it.  But it does still come up from time to time.</p>
<p><strong>Protein X = phosphatidylethanolamine?</strong></p>
<p>At Prion2013, Surachai Supattapone presented his group&#8217;s finding that <a href="http://en.wikipedia.org/wiki/Phosphatidylethanolamine">phosphatidylethanolamine</a> (PE) is necessary for <em>in vitro</em> prion conversion and that the availability of other cofactors helps to determine faithful propagation of strain identity [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3365173/">Deleault 2012a</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3396481/">Deleault 2012b</a>].  I heard someone ask, &#8220;Is phosphatidylethanolamine Protein X??&#8221;  Perhaps the reason we couldn&#8217;t find Protein X all these years is that we were looking for a protein and the true answer was a phospholipid.</p>
<p>Supattapone&#8217;s results are clear that PE is necessary for prion conversion in the cell-free assay PMCA.  Ideally we&#8217;d like to check if PE is necessary <em>in vivo</em>.  That looks like a tough proposition: PE is produced by four different pathways and no one has yet knocked out the production of PE altogether [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22960354">Vance &amp; Tasseva 2013</a>].  Even if they did it would probably be embryonic lethal &#8211; knockout of both of the two pathways that produce phosphatidylserine (PS) proved lethal [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22960354">Vance &amp; Tasseva 2013</a>].</p>
<p>But some aspects of PE make it unlikely to be &#8220;Protein X&#8221;.  Although the fatty acid chains on PE can vary, a single preparation of PE was sufficient to propagate prions from four different mammal species [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3365173/">Deleault 2012a</a>], which is not consistent with the idea of MoX ≠ HuX ≠ SHaX, etc.  Also PE seems too abundant to be a rate-limiting factor in prion conversion: it comprises 45% of the <a title="Cell Biology 02: The Plasma Membrane" href="/2013/02/10/cell-biology-02-the-plasma-membrane/">plasma membrane</a>&#8216;s phospholipid content in the CNS [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22960354">Vance &amp; Tasseva 2013</a>], though there&#8217;s somewhat less of it on the exoplasmic leaflet where PrP is [<a href="http://www.ncbi.nlm.nih.gov/books/NBK21583/">Lodish 2000 Ch4</a>].  (I couldn&#8217;t find any hard numbers on that).</p>
<p><strong>Protein X = PrP?</strong></p>
<p>If you&#8217;re looking for a protein that binds PrP, differs from species to species in a way that facilitates binding with host PrP, co-localizes with PrP, and is necessary for prion conversion, you could ask for no better candidate than PrP itself.  Could PrP be Protein X?</p>
<p>In such a model, you could argue that in HuPrP / MoPrP mice, HuPrP<sup>Sc</sup> wastes a lot of its time binding to MoPrP<sup>C</sup>, which it is not very able to convert. In mice without MoPrP, it doesn&#8217;t get so distracted.  In MHu2M / MoPrP mice, the two types of PrP do bind to each other, but the binding is more &#8220;productive&#8221; because MHu2M PrP<sup>Sc</sup> is able to convert some MoPrP.</p>
<p>The main trouble with positing that PrP = Protein X is one of <a href="http://en.wikipedia.org/wiki/Stoichiometry">stoichiometry</a>.  MoPrP potently inhibits HuPrP from spreading human prions even when its expression level is just 5 or 10% that of HuPrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7553876/">Telling 1995</a>].</p>
<p>A low percentage of MoPrP could still have a large inhibitory effect on HuPrP<sup>Sc</sup> if PrP<sup>Sc</sup> needs to form higher-order oligomers for maximum infectivity and the incorporation of PrP molecules into these oligomers were random.  For instance, say MoPrP is 5% of total PrP, the rest being HuPrP, and that the most infectious forms of PrP<sup>Sc</sup> are those composed of 14-28 PrP molecules, as has been proposed [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1513539/">Silveira 2005</a>], then any randomly formed 14-molecule oligomer has a 1-.95<sup>14</sup> = ~51% chance of having at least one MoPrP molecule.  But this is really a stretch here &#8211; if MoPrP binds HuPrP less efficiently than HuPrP does, why would it get swept up into these oligomers randomly, with equal probability as HuPrP?  And why would just one MoPrP molecule ruin the party for everyone? And if the other 49% of oligomers are still perfectly infectious, why is the disease slowed down quite so dramatically?</p>
<p><strong>not all PrP<sup>C</sup> is available for conversion</strong></p>
<p>An important related point is that not all PrP<sup>C</sup> is available to be converted into PrP<sup>Sc</sup>.  I have tentatively reached this conclusion based on a few lines of evidence.</p>
<p>First, not all PrP<sup>C</sup> becomes PrP<sup>Sc</sup> in infected cells.  Pulse-chase in RML-infected ScN2a cells showed that &lt; 3% of PrP<sup>C</sup> ever becomes protease-resistant [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1680859">Caughey &amp; Raymond 1991</a> (<a href="http://www.jbc.org/content/266/27/18217.full.pdf">ft</a>)].  That&#8217;s surely a very lower-end estimate of what proportion of PrP<sup>C</sup> becomes PrP<sup>Sc</sup> <em>in vivo</em>, for two reasons.  First, not all PrP<sup>Sc</sup> is protease-resistant &#8211; in fact, the only estimate I&#8217;ve found for RML prions is that just 15% is PK-resistant <em>in vivo</em> under standard conditions [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16186247">Safar 2005</a> (<a href="http://vir.sgmjournals.org/content/86/10/2913.long">ft</a>)].  Second, constant cell division in N2a cells dilutes PrP<sup>Sc </sup>[<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2084281/">Ghaemmaghami 2007</a>], meaning there is less infectious material on the cell surface to convert nascent PrP<sup>C</sup> molcules as might be found on neurons are at late stage <em>in vivo</em>.</p>
<p>OK, so not all PrP<sup>C</sup> becomes PrP<sup>Sc</sup> &#8211; but that could just be because prion conversion is too slow and inefficient to convert PrP<sup>C</sup> before it gets degraded given its <a title="Half life of PrP and its mRNA" href="/2013/04/08/half-life-of-prp-and-its-mrna/">short half-life</a>.</p>
<p>But then there&#8217;s the well-studied phenomenon of <a href="http://en.wikipedia.org/wiki/Superinfection">superinfection</a>.  This is when an animal is infected with two different prion strains in succession.  A slow or less neuroinvasive strain of prion can actually be prophylactic against faster strains introduced later into the same animal [two examples: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4624846/" target="_blank">Dickinson 1972</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/14718642" target="_blank">Bartz 2004</a>].  This usually requires the slow strain to be inoculated 60 or 90 days before the fast strain &#8211; it doesn&#8217;t work after the fact.  Still, it&#8217;s remarkable.  You can still find PrP<sup>C</sup> even in terminally diseased brains [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16186247">Safar 2005</a> (<a href="http://vir.sgmjournals.org/content/86/10/2913.long">ft</a>)], so it&#8217;s not as though PrP<sup>C</sup> is <em>used up</em> by the first strain.  And the latest evidence is that the fast strain in these superinfection experiments actually doesn&#8217;t die out, it just continues replicating at a low level and never manages to become the dominant strain [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1797475/" target="_blank">Bartz 2007</a>].  If the second strain is present and PrP<sup>C</sup> is present, then why would the first strain inhibit the second strain from replicating?</p>
<p>One possibility, I suppose, is that the first strain uses up most of the Protein X.  Another contributing factor may be if the first strain triggers <a href="/2013/06/01/prpc-downregulation-in-prion-infection/">downregulation of PrP<sup>C</sup></a>, which makes life harder for the second strain &#8211; but this downregulation doesn&#8217;t happen until about halfway through CNS infection.  I&#8217;ve also heard it posited is that PrP<sup>C</sup> needs to adopt an intermediate conformation dubbed PrP* in order to convert to PrP<sup>Sc</sup>, and that the first strain uses up all the PrP*.  Another explanation, similar to the PrP* theory, is that only a subset of PrP<sup>C</sup> - perhaps based on subcellular localization, for example &#8211; is available for conversion, and that subset gets used up by the first strain.</p>
<p>OK, so suppose that there&#8217;s a limited pool of PrP<sup>C</sup> available for prion conversion. For the sake of example, maybe this pool is cell surface PrP<sup>C</sup> in lipid rafts at synaptic densities.  Or maybe it&#8217;s just the microenvironment of PrP near wherever the first PrP<sup>Sc</sup> molecule alights upon the a given neuron. (Has anyone done FRAP on PrP to determine its mobility? I Googled and only found yeast and C. elegans prion studies.)  Remember that neurons are long, scraggly things, of vast proportions compared to tiny PrP.  Prions are deadly but they can only bind what&#8217;s near them. They think globally, but act locally.</p>
<p>In pure HuPrP mice on a knockout background, 100% of this pool is converted to PrP<sup>Sc</sup>.  But in HuPrP / MoPrP mice, even at low stoichiometry, each pool &#8211; each lipid raft &#8211; will still have a few MoPrP<sup>C</sup> molecules hanging around and&#8230; performing their native function?</p>
<p><strong>maybe PrP<sup>C</sup> is protective</strong></p>
<p>Following the discovery that <a title="Review of prion protein and amyloid beta interactions" href="/2013/05/02/review-of-prion-protein-and-amyloid-beta-interactions/">PrP<sup>C</sup> binds Aβ oligomers</a> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2748841/">Lauren 2009</a>], it has been speculated that maybe PrP<sup>C</sup>&#8216;s native function is to act as an amyloid receptor, detecting dangerous misfolded proteins outside the cell [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22735515">Aguzzi &amp; Falsig 2012</a>] and presumably triggering an intracellular signaling cascade to coordinate some response.  I&#8217;m not sure how much I believe it, but at risk of embarrassing myself I&#8217;m going to run with this idea for a minute and see if it goes anywhere interesting.</p>
<p>Maybe in 95% HuPrP / 5% MoPrP mice, all of the HuPrP within any given limited conversion-ready pool on any given cell is converted to PrP<sup>Sc</sup>, but that remaining bit of inconvertible MoPrP<sup>C</sup> still manages to carry out its duty as a receptor &#8211; binding the amyloid and triggering endocytosis to ferry the cargo to a <a title="Prion protein degradation in lysosomes and the effect of autophagy-activating drugs" href="/2013/07/11/prion-protein-degradation-in-lysosomes-and-the-effect-of-autophagy-activating-drugs/">lysosome</a> for degradation, or recruiting extracellular proteases to shear up the mess, or <em>something</em> useful &#8211; whatever it is that earns PrP its reputation as being &#8220;neuroprotective&#8221;.</p>
<p>In this model, the low stoichiometry works.  It only takes one undercover federal agent per terrorist cell to scuttle the whole attack &#8211; as long as that agent is doing something actively useful (carrying out an anti-amyloid native function) and not merely standing around being in the way (binding to a single HuPrP<sup>Sc</sup> molecule).</p>
<p>This could maybe also explain the difference between HuPrP and MHu2M.  MoPrP<sup>C</sup>&#8216;s ability to carry out its protective function is more likely to be compromised by the presence of closer-to-identical PrP<sup>Sc</sup>.</p>
<p>And it could explain why the <a title="Proteolysis of prion protein in the central region: alpha and beta cleavage" href="/2013/06/26/proteolysis-of-prion-protein-in-the-central-region-alpha-and-beta-cleavage/">alpha-cleaved</a> C1 fragment, believed to be conversion-incompetent, is dominant negative against conversion of full-length PrP, even when the two come from the same species [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3243553/">Westergard 2011</a>].</p>
<p>The least believable thing about this model is as follows.  According to my recent review of studies of <a title="Allelic origin of PrPSc in heterozygotes" href="/2013/08/15/allelic-origin-of-prpsc-in-heterozygotes/">allelic origin of PrP<sup>Sc</sup></a>, in many genetic prion diseases, the wild-type allele does not convert to PrP<sup>Sc</sup> at all.  I don&#8217;t quite believe this since FFI, for instance, is transmissible to mice expressing wild-type HuPrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8953038">Telling 1996</a>].  But if 5% MoPrP can inhibit human prions, then having 50% of your PrP be conversion-incompetent ought to confer absolute resistance, and genetic prion diseases ought to be recessive, not dominant.  But in fact, homozygous E200K patients have an age of onset which is not even that much earlier than heterozygotes [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10665501">Simon 2000</a>].</p>
<p><strong>how to test it</strong></p>
<p>If this model is correct, then expressing 5% HuPrP and 95% MoPrP in mice should have just as large an inhibitory effect on Mo prions as the converse does against Hu prions. Has anyone tested this? I couldn&#8217;t find an example of it in the literature.  Alternately, 95% HuPrP should be just as easily inhibited by 5%, say, BoPrP as by 5% MoPrP &#8211; what matters is not any third molecule such as MoX, BoX, etc. but just the primary structure identity between the two types of PrP being expressed.</p>
<p>This model would also predict that more MoPrP gets converted to MoPrP<sup>Sc</sup> in MHu2M / MoPrP mice than it does in HuPrP / MoPrP mice.  Again, I could not find where anyone has asked this question (perhaps because <a title="Allelic origin of PrPSc in heterozygotes" href="/2013/08/15/allelic-origin-of-prpsc-in-heterozygotes/">allelic origin</a> is really labor-intensive to ascertain).</p>
<p>Finally, the model would also predict that in cell-free conversion, where there is no &#8220;microenvironment&#8221; and no cellular response to be mounted anyway, no dominant negativity would be observed.  This seems like something someone would have looked at, but I couldn&#8217;t find a reference.  Incompatible species of PrP<sup>Sc</sup> and PrP<sup>C</sup> have been tested in PMCA [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7732006/">Kocisko 1995</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC42074/pdf/pnas01493-0299.pdf">ft</a>)], but what I&#8217;m talking about is mixing two different species of PrP<sup>C</sup> and showing that, for example, the presence of SHaPrP<sup>C</sup> has no effect on MoPrP<sup>Sc</sup>&#8216;s ability to convert MoPrP<sup>C</sup>.</p>
<p><strong>summary</strong></p>
<p>In this post I&#8217;ve reviewed the evidence for Protein X and proposed an alternate model in which incompatible species of PrP<sup>C</sup> are dominant negative against one another simply because they resist conversion and are therefore able to carry out a protective native function.  This implies that in non-transgenic animals where no dominant negative allele is present, 100% of PrP<sup>C</sup> gets converted to PrP<sup>Sc</sup> <em>in the microenvironment where prion conversion is occurring</em>.</p>
<p>There is surely a lot of good literature on this that I missed in preparing this blog post, much of which may outright refute this model &#8211; indeed, I&#8217;ve already noted some things that are inconsistent about it.  Still, I put it out here in the hopes of generating some discussion.  Protein X may yet reveal itself to us, but since it hasn&#8217;t yet, it seems worth considering models in which PrP alone can explain the strange kinetics of dominant negativity.</p>
