---
layout: post
title: "Cell Biology 02: The Plasma Membrane"
tag: bios-e-16
author: ericminikel
date: 2013-02-10 02:13:48
---
<p>This semester I&#8217;m taking a <a href="http://www.extension.harvard.edu/courses/22958">Cell Biology class at Harvard Extension</a>.  My plan is to turn my notes from each lecture into a blog post, figuring out how everything we learned relates to PrP (and maybe huntingtin) along the way.  The first class was just review and syllabus, so I&#8217;m starting on lecture 2: the plasma membrane.</p>
<p><a href="http://en.wikipedia.org/wiki/Cell_membrane"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/d/da/Cell_membrane_detailed_diagram_en.svg/800px-Cell_membrane_detailed_diagram_en.svg.png" alt="File:Cell membrane detailed diagram en.svg"/></a></p>
<p>The <a href="http://www.ncbi.nlm.nih.gov/books/NBK26871/">plasma membrane</a> refers both to the membrane that surrounds cells and also to the membranes that surround <a href="http://en.wikipedia.org/wiki/Organelle">organelles</a> within the cell. The functions of the plasma membrane are held to be as follows:</p>
<ul>
<li>Compartmentalization (separate organelles from other stuff)</li>
<li>Scaffolding (give the cell shape)</li>
<li>Barrier (keep some things out and others in)</li>
<li>Gatekeeper (do let some things through but not others)</li>
<li>Monitoring outside signals (receptors on the membrane signal to other proteins inside the cell)</li>
<li>Energy transduction (transferring energy across the membrane?)</li>
</ul>
<p>The membrane is made of a lipid bilayer (the red/orange/yellow stuff in the above diagram).  It&#8217;s made predominantly of <a href="http://en.wikipedia.org/wiki/Phospholipid">phospholipid</a> molecules, which are <a title="same as &quot;amphiphilic&quot; - it loves (phile) both (amphi) fat and water" href="http://en.wikipedia.org/wiki/Amphiphile">amphipathic</a> &#8211; part hydrophobic and part hydrophilic:</p>
<p><a href="http://en.wikipedia.org/wiki/Phospholipid"><img src="http://upload.wikimedia.org/wikipedia/commons/3/39/Phospholipid_TvanBrussel.jpg" alt="File:Phospholipid TvanBrussel.jpg"/></a></p>
<p>They contain, from top to bottom:</p>
<ol>
<li>A polar head group. This can be charged: <a href="http://en.wikipedia.org/wiki/Serine">serine</a> (-), <a href="http://en.wikipedia.org/wiki/Inositol">inositol</a> (-), or it can be neutral, with both positively and negatively charged groups: <a href="http://en.wikipedia.org/wiki/Choline">choline</a>, <a href="http://en.wikipedia.org/wiki/Ethanolamine">ethanolamine</a>.</li>
<li>A <a href="http://en.wikipedia.org/wiki/Glycerol">glycerol</a> linker.</li>
<li>A <a href="http://en.wikipedia.org/wiki/Phosphate">phosphate</a> group.</li>
<li>Two fatty acid chains.</li>
</ol>
<p>The polar head group faces outward into the aqueous intracellular or extracellular space, while the hydrophobic interior of the membrane is the fatty acid chains.  The phospholipids are named after their head groups, thus <a href="http://en.wikipedia.org/wiki/Phosphatidylserine">phosphatidylserine</a> (PS), <a href="http://en.wikipedia.org/wiki/Phosphatidylcholine">phosphatidylcholine</a> (PC), <a href="http://en.wikipedia.org/wiki/Phosphatidylethanolamine">phosphatidylethanolamine</a> (PE), <a href="http://en.wikipedia.org/wiki/Phosphatidylinositol">phosphatidylinositol</a> (PI).  Then, randomly, there is also <a href="http://en.wikipedia.org/wiki/Sphingomyelin">sphingomyelin</a> which has an ethanolamine head group but differs from the others in that it has <a href="http://en.wikipedia.org/wiki/Sphingosine">sphingosine</a> (a long carbon chain amino alcohol) instead of one of the fatty acid chains, and an <a href="http://en.wikipedia.org/wiki/Amide">amide</a> linker instead of glycerol.</p>
<p>The fatty parts &#8211; the lipids &#8211; make up about 50% of the total mass of the membrane.  The fatty acid chains can be either saturated or unsaturated, which accounts for some variety in membrane properties.  The <a href="http://en.wikipedia.org/wiki/Fatty_acid#Saturated_fatty_acids">saturated chains</a> have no double bonds (all the carbons&#8217; bonding capacities are used up by hydrogens), so the chains lie straight, pack tightly and interact much with each other.  The <a href="http://en.wikipedia.org/wiki/Fatty_acid#Unsaturated_fatty_acids">unsaturated chains</a> form kinks due to double bonds &#8211; most fatty acids in nature are <em>cis </em>fatty acids, meaning all the hydrogens are on one side, so the chain kinks; <em>trans</em> fats, which are abundant in fast food but rare in nature, lie more straight like saturated fats.  The kinks in unsaturated <em>cis</em> fatty acids makes them pack less tightly, and the greater amount of space makes for a more &#8216;fluid&#8217; membrane.  Here, &#8216;fluid&#8217; means it is easier for other things (like transmembrane proteins) to move in two dimensions along the plane of the membrane.</p>
<p>Phospholipids are just most &#8211; not all &#8211; of the lipid bilayer.  There are also <a href="http://en.wikipedia.org/wiki/Glycolipid">glycolipids</a>, which have a glycerol (which is a sugar) instead of phosphate.  These make up 2-10% of membrane mass, and are more abundant in the nervous system than elsewhere.  <a href="http://en.wikipedia.org/wiki/Cholesterol">Cholesterol</a> is also an important part of the membrane.  Here&#8217;s what cholesterol looks like:</p>
<p><a href="http://en.wikipedia.org/wiki/Cholesterol"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/9/9a/Cholesterol.svg/440px-Cholesterol.svg.png" alt="File:Cholesterol.svg"/></a></p>
<p>The OH makes it polar on that end, while the rings and carbon chain on the other end is hydrophobic.  So this is also an amphipathic molecule, though it can&#8217;t form lipid bilayers on its own. Cholesterol is a <a href="http://en.wikipedia.org/wiki/Steroid">steroid</a>, which apparently is a chemical designation referring to that four-ring core, and not a functional designation.  Cholesterol is found mostly in eukaryotes and not in bacterial membranes.  Its bulky four-ring structure makes the membrane less flexible, the OH head group interacts strongly with other phospholipid head groups thus holding them in place, and the hydrocarbon chain interacts strongly with the phospholipids&#8217; fatty acid chains, making them pack more tightly.  For all of these reasons, cholesterol reduces membrane fluidity, though in a temperature-dependent matter (higher temperature means more fluidity).</p>
<p>The two halves of the lipid bilayer are called leaflets: the cytoplasmic leaflet faces (predictably) towards the cytoplasm while the exoplasmic leaflet faces outside the cell or into an organelle.  Here is a quick comparison of composition:</p>
<ul>
<li><a href="http://en.wikipedia.org/wiki/Phosphatidylserine">phosphatidylserine</a> (PS) (-): almost all cytoplasmic</li>
<li><a href="http://en.wikipedia.org/wiki/Phosphatidylcholine">phosphatidylcholine</a> (PC) (neutral) found on both sides, esp. exoplasmic</li>
<li><a href="http://en.wikipedia.org/wiki/Phosphatidylethanolamine">phosphatidylethanolamine</a> (PE) (neutral) found on both sides</li>
<li><a href="http://en.wikipedia.org/wiki/Phosphatidylinositol">posphatidylinositol</a> (PI) (-) almost all cytoplasmic</li>
<li><a href="http://en.wikipedia.org/wiki/Sphingomyelin">sphingomyelin</a> mostly exoplasmic</li>
<li><a href="http://en.wikipedia.org/wiki/Cholesterol">cholesterol</a> equal on both leaflets</li>
</ul>
<p>The exoplasmic side tends to be less fluid.  But both sides are pretty fluid, with the phospholipids constantly skating around in 2 dimensions (&#8216;lateral diffusion&#8217;), rotating, bending tails (&#8216;flection&#8217;), and very occasionally, switching from one leaflet to the other (&#8216;flip-flopping&#8217;).</p>
<p>A neat laboratory protocol for studying the fluidity of the membrane is <a href="http://en.wikipedia.org/wiki/Fluorescence_recovery_after_photobleaching">fluorescence recovery after photobleaching</a> (FRAP).  In this protocol, a molecule of interest is fluorescently tagged &#8211; for instance, a membrane protein is tagged with a fluorescently labeled antibody or fused with <a href="http://en.wikipedia.org/wiki/Green_fluorescent_protein">green fluorescent protein</a>, or an amphipathic molecule is tagged with a fluorophore.  Then you &#8216;bleach&#8217; part of the membrane with a laser which exhausts the fluorescent properties of the fluorescent molecule in that patch.  Then you watch as still-fluorescent molecules from elsewhere in the membrane diffuse into the bleached patch.  The main goal here is to see how &#8216;mobile&#8217; the protein you&#8217;re studying is.  Generally, membrane proteins are less mobile than the phospholipids because they are so much larger.  And if the protein is anchored to the <a href="http://en.wikipedia.org/wiki/Cytoskeleton">cytoskeleton</a>, it won&#8217;t move at all, so the bleached patch will recover very slowly if at all.  To control for the effects of membrane fluidity you can also tag the phospholipids themselves and compare the mobility of the protein of interest to that of the phospholipids.</p>
<p>There are two kinds of <a href="http://en.wikipedia.org/wiki/Membrane_protein">membrane proteins</a>: integral (top) and peripheral (below).  Images thanks to Wikimedia Commons user Foobar:</p>
<p><a href="http://en.wikipedia.org/wiki/Membrane_protein#Integral_membrane_proteins"><img src="http://upload.wikimedia.org/wikipedia/commons/d/db/Polytopic_membrane_protein.png" alt="File:Polytopic membrane protein.png"/></a></p>
<p>An integral membrane protein (above) has one or more hydrophobic domains anchored in the membrane by simple thermodynamics while the hydrophobic parts stick out into the cytoplasm or extracellular (or intraorganelle) environment).  The transmembrane parts might be alpha helices (1 and 2) or beta sheets (3).</p>
<p><a href="http://en.wikipedia.org/wiki/Membrane_protein#Peripheral_membrane_proteins"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/c/c3/Monotopic_membrane_protein.svg/614px-Monotopic_membrane_protein.svg.png" alt="File:Monotopic membrane protein.svg"/></a></p>
<p>The peripheral membrane proteins (above) may have just a segment that buries in the membrane but doesn&#8217;t cross it (1, amphipathic alpha helix; 2, hydrophobic loop; 4, hydrophobic interactions with polar head groups) or be post-translationally modified to add an anchor which buries in the membrane (3) or interact with a transmembrane protein (not pictured).  The post-translational modifications might attach a fatty acid chain, oligosaccharide, sugars or farnesyl alcohol group (more common on the cytoplasmic leaflet).</p>
<p>All the different molecules that make up the plasma membrane are not evenly distributed across the whole 2-dimensional surface.  Instead, there are patches that are more ordered and tightly packed and have 3-5x the concentration of cholesterol and about 1.5x the concentration of sphingomyelin. These patches are called <a href="http://en.wikipedia.org/wiki/Lipid_raft">lipid rafts</a>.  In the below image, section 1 is normal plasma membrane and section 2 is a lipid raft:</p>
<p><a href="http://en.wikipedia.org/wiki/Lipid_raft"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/c/c6/Lipid_raft_organisation_scheme.svg/800px-Lipid_raft_organisation_scheme.svg.png" alt="File:Lipid raft organisation scheme.svg"/></a></p>
<p>Lipid rafts have all sorts of highly specialized functions and some particular proteins tend to locate there.  People who study membrane proteins often add detergent to their solution to solubilize the glycolipids and bring the proteins into solution.  That works less well if the protein of interest is found in lipid rafts, which are harder to dissolve.</p>
<p>The membrane is permeable to gases and small uncharged molecules, which can simply diffuse right through according to their own driving force.  Naturally, water, ions, small polar molecules and large, uncharged-but-polar molecules are unable to pass through the membrane on their own.  Moving these sorts of things requires some form of transport mediated by membrane proteins: either active (using <a href="http://en.wikipedia.org/wiki/Adenosine_triphosphate">ATP</a> to fight the transported molecules&#8217; own thermodynamic desires) or passive (allowing molecules to follow their own electrochemical gradient).</p>
<p><a href="http://en.wikipedia.org/wiki/Passive_transport">Passive transport</a> can take a few different forms. First, it can involve channels that allow ions or molecules to diffuse through.  These channels might be highly selective or not at all selective, and might be electrically or chemically gated so they only open under certain conditions. Second, it can involve &#8216;passive transporters&#8217;  (aka &#8216;porters&#8217;), proteins which, upon encountering their substrate ion or molecule, undergo a change in conformation which propels that substrate through to the other side.  Predictably, they tend to be more specific than channels, but surprisingly (to me at least), the porters are often <em>faster </em>than just straight-up diffusion would be.  Porters are divided into three types:</p>
<p><a href="http://en.wikipedia.org/wiki/File:Porters.PNG"><img src="http://upload.wikimedia.org/wikipedia/commons/6/6c/Porters.PNG" alt="File:Porters.PNG"/></a></p>
<p><a href="http://en.wikipedia.org/wiki/Uniporter">Uniporters</a> just transport one type of molecule,  <a href="http://en.wikipedia.org/wiki/Symporter">symporters</a> move two molecules in the same direction and <a href="http://en.wikipedia.org/wiki/Antiporter">antiporters</a> move two molecules in opposite directions.  The trick of the symporters and antiporters is that they allow one of their two substrates to move <em>with</em> its electrochemical gradient, and they harness the energy of that molecule&#8217;s thermodynamically favorable movement in order to propel the other molecule <em>against </em>its electrochemical gradient.  The symporters and antiporters all require that the multiple substrates bind at the same time &#8211; they can&#8217;t store energy for later.</p>
<p><a href="http://en.wikipedia.org/wiki/GLUT1">GLUT1</a> is the classic example of a uniporter.  It imports glucose into cells when extracellular concentrations are higher then intracellular.  It is highly specific for <a href="http://en.wikipedia.org/wiki/Glucose">glucose</a>, having limited affinity even for <a href="http://en.wikipedia.org/wiki/Mannose">mannose</a> and <a href="http://en.wikipedia.org/wiki/Galactose">galactose</a> which differ from glucose in just one carbon atom.  Once inside the cell, glucose is immediately modified with a phosphate group so it can&#8217;t find its way back out.  <a title="some people call them erythrocytes to be fancy" href="http://en.wikipedia.org/wiki/Red_blood_cell">Red blood cells</a> have tons of GLUT1 because, lacking mitochondria, they rely on anaerobic <a href="http://en.wikipedia.org/wiki/Glycolysis">glycolysis</a> for energy, which is really inefficient (only 2 ATP per glucose as opposed to 36 ATP per glucose via aerobic metabolism).</p>
<p>People who study uniporters use antibodies to pull them down, genetically introduce them into cell types where they are not normally expressed (e.g. GLUT1 in skin cells) or mutate their substrate binding region to see how that changes their function.  In spite of this, much is unknown &#8211; <a href="http://en.wikipedia.org/wiki/GLUT1#Function">according to Wikipedia</a> we still don&#8217;t know the structure of GLUT1 or how it binds glucose.</p>
<p>A classic antiporter is the <a href="http://en.wikipedia.org/wiki/Sodium-calcium_exchanger">Na<sup>+</sup>/Ca<sup>2+</sup> exchanger</a> (genes: SLC8A1, SLC8A2, SLC8A3).  An alternate term for it, &#8216;sodium-linked calcium antiporter&#8217; tells you more about its energetics: it&#8217;s powered by sodium in order to move calcium against its gradient.  Specifically, it accepts 3 Na<sup>+</sup> into the cell in exchange for expelling 1 Ca<sup>2+</sup>.  Its affinity to calcium is low enough that it only works at really high calcium concentrations, so it&#8217;s useful for preventing <a href="http://en.wikipedia.org/wiki/Excitotoxicity">excitotoxicity</a> in neurons &#8211; getting rid of the excess calcium after repeated stimulus.  It&#8217;s also found in cardiac muscle cells.</p>
<p>Another classic antiporter is the <a href="http://en.wikipedia.org/wiki/Sodium%E2%80%93hydrogen_antiporter">sodium-hydrogen antiporter</a> (genes: SLC9A1,2,3,5,6,8), which accepts 1 Na<sup>+</sup> into the cell in order to export 1 H<sup>+</sup> ion.  This maintains the preferred pH of 7.2 inside cells.  It also is one step in exporting acid into the stomach, because H<sup>+</sup> is produced in cells but secreted into the stomach.  (But &#8211; I had to check &#8211; this antiporter is <em>not</em> the target of <a href="http://en.wikipedia.org/wiki/Proton-pump_inhibitor">proton pump inhibitors</a> like <a href="http://en.wikipedia.org/wiki/Omeprazole">omeprazole</a> - those target an active transporter, <a href="http://en.wikipedia.org/wiki/Hydrogen_potassium_ATPase">H<sup>+</sup>/K<sup>+</sup> ATPase</a>).</p>
<p>A classic symporter is the <a href="http://en.wikipedia.org/wiki/Sodium-glucose_transport_proteins">sodium glucose symporter</a> (genes: SLC5A1, SLC5A2 &amp; SLC5A4).  Remember how <a href="http://en.wikipedia.org/wiki/GLUT1">GLUT1</a> uniporter brings glucose into the cell when its extracellular concentration is higher?  When the intracellular glucose concentration is higher than extracellular and cells need to import glucose against its concentration gradient, this symporter does so by accepting 2 sodiums into the cell in order to bring one glucose in.</p>
<p><a href="http://en.wikipedia.org/wiki/Active_transport">Active transport</a> relies on pumps that couple <a title="releasing the energy of ATP" href="http://en.wikipedia.org/wiki/ATP_hydrolysis">ATP hydrolysis</a> with movement.  You may have noticed that sodium seems to be the common currency of the exampels of symporters and antiporters listed above.  They all allow sodium to move into the cell, as is thermodynamically favorable, in order to export something the cell doesn&#8217;t want.  Predictably, someone has to do the hard work of maintaining that sodium gradient while everyone else is mooching off of it.  That someone is <a href="http://en.wikipedia.org/wiki/Na%2B/K%2B-ATPase">Na<sup>+</sup>/K<sup>+</sup>-ATPase</a>, which expends 1 ATP in order to export 3 Na<sup>+</sup> and import 2 K<sup>+</sup>, both against their respective concentration gradients.  This thing is a workhorse, accounting for 20% of energy consumption in most animal cells and 67% in neurons.  You don&#8217;t get anything for free, so if it just doesn&#8217;t feel right to even call the symporters and antiporters &#8216;passive&#8217; transporters, you can call them by their other name, which is &#8216;<a href="http://en.wikipedia.org/wiki/Secondary_active_transport">secondary active transporters</a>&#8216;, in recognition of the fact that, while they don&#8217;t burn ATP directly, they do use energy by relying on actively maintained sodium gradients.</p>
<p>The active transporters are divided in to <a href="http://en.wikipedia.org/wiki/F-ATPase">F</a>, <a href="http://en.wikipedia.org/wiki/P-type_ATPase">P</a>, <a href="http://en.wikipedia.org/wiki/V-ATPase">V</a>, and <a href="http://en.wikipedia.org/wiki/ATP_binding_cassette">ABC</a> families.</p>
<p>P refers to <a href="http://en.wikipedia.org/wiki/P-type_ATPase">P-type ATPases</a> which have two catalytic alpha domains (one of which is phosphorylated hence the &#8216;P&#8217; name) that bind and hydrolyze ATP, and two regulatory beta domains.  These include the <a href="http://en.wikipedia.org/wiki/Na%2B/K%2B-ATPase">Na<sup>+</sup>/K<sup>+</sup>-ATPase</a> mentioned above as well as <a href="http://en.wikipedia.org/wiki/Serca">SERCA</a> calcium pumps, which stores calcium in the <a href="http://en.wikipedia.org/wiki/Endoplasmic_reticulum#Sarcoplasmic_reticulum">sarcoplasmic reticulum</a>, and the <a href="http://en.wikipedia.org/wiki/Hydrogen_potassium_ATPase">H<sup>+</sup>/K<sup>+</sup> ATPase</a> which acidifies your stomach, mentioned above.</p>
<p>ABC refers to the <a href="http://en.wikipedia.org/wiki/ATP-binding_cassette_family">ATP-binding cassette</a> family of hundreds of proteins.  They have two transmembrane domains and two ATP-binding domains.  Some of them move molecules and not just ions.  This class is said to include <a href="http://en.wikipedia.org/wiki/Cystic_fibrosis_transmembrane_conductance_regulator">CFTR</a>, the protein whose loss of function is responsible for <a href="http://en.wikipedia.org/wiki/Cystic_fibrosis">cystic fibrosis</a>.  But CFTR is a bit weird and, though it evolved from the active transporters, it does <em>not</em> actually pump anything uphill against a concentration gradient.  Rather, it allows passive diffusion of chloride and <a href="http://en.wikipedia.org/wiki/Thiocyanate">thiocyanate</a> ions along their gradients, but only when opened by an ATP-powered gate.  Loss of function mutations in CFTR make it impossible for these ions to diffuse, which in turn eliminates the thermodynamic incentive for water to osmose out of the cell.  Without that extra water from osmosis, the extracellular mucus becomes too thick, leading to coughing and chest infections, the hallmarks of cystic fibrosis.</p>
<p>Another important ABC family transporter is <a href="http://en.wikipedia.org/wiki/P-glycoprotein">MDR1</a> (gene: ABCB1).  MDR stands for multi-drug resistance.  This amazing protein found in animal cells is not very specific but somehow can recognize a broad range of &#8216;<a href="http://en.wikipedia.org/wiki/Xenobiotic">xenobiotic</a>&#8216; molecules &#8211; i.e. foreign compounds that we don&#8217;t produce and don&#8217;t normally have in our cells &#8211; and, using energy from ATP, kick them out of the cell.  Presumably this keeps the cells from accumulating toxins from the environment.  But it also makes treatment difficult because it exports some antiretroviral drugs and chemo drugs &#8211; the latter especially in tumor cells that acquire mutations that heighten the action of MDR1.  In this way, tumors can sort of evolve resistance to cancer drugs.  This is especially a big problem in liver cancers.  An old, boring theory about MDR1 was that it was just an ATP-powered channel, but <a href="http://en.wikipedia.org/wiki/Fluorescence_recovery_after_photobleaching">FRAP</a>-like experiments have now shown that it&#8217;s in fact something much more interesting: a <a href="http://en.wikipedia.org/wiki/Flippase">flipase</a>.  That means that rather than just shooting the molecule out of the cell, MDR1 binds the molecule and then flips a section of the membrane, swapping the exoplasmic and cytoplasmic leaflets.  It&#8217;s like punching a door in a fence in Super Mario World:</p>
<p><iframe src="http://www.youtube.com/embed/74Pb8itMaCI?start=72&amp;end=75" frameborder="0" width="560" height="315"></iframe></p>
<p>The <a href="http://en.wikipedia.org/wiki/F-ATPase">F</a> family are found in bacteria, mitochondria and chloroplasts, and based on their Wikipedia description, don&#8217;t sound like active transporters at all.</p>
<p>The <a href="http://en.wikipedia.org/wiki/V-ATPase">V</a> family are found in many other organelles and are responsible mostly for pH maintenance.  <a href="http://en.wikipedia.org/wiki/Lysosomes">Lysosomes</a> - one place where misfolded proteins and other cellular waste go to get broken down &#8211; have pH 4.8 (compared to 7.2 in the cytosol), which is a clever evolutionary trick.  The lysosomes are full of enzymes that could break down everything in the cytosol, which is dangerous, but the enzymes evolved to only function at low pH, so if the lysosome ever ruptures, the enzymes won&#8217;t destroy the cell because they won&#8217;t function in the cytosol.  This pH gradient between lysosome and cytosol is maintained by V ATPases.</p>
<p><strong>PrP and the plasma membrane</strong></p>
<p><a href="http://en.wikipedia.org/wiki/PRNP">PrP</a> in humans is 253 codons long but the final protein is only 208 amino acids.  22 N terminal and 23 C terminal amino acids are signal peptides which direct the protein into the endoplasmic reticulum, where it gets post-translationally modified.  That modification cleaves off the signal peptides, replacing the C-terminal segment with a <a href="http://en.wikipedia.org/wiki/Glycophosphatidylinositol">glycosylphosphatidylinositol</a> (GPI) anchor (attached to 231S) which makes PrP technically a <a href="http://en.wikipedia.org/wiki/Peripheral_membrane_protein">peripheral membrane protein</a> (though GPI-anchored proteins have some properties more like integral membrane proteins), with the GPI buried in the exoplasmic leaflet of the plasma membrane and the protein sticking out into the extracellular space.  GPI-anchored proteins tend to congregate in <a href="http://en.wikipedia.org/wiki/Lipid_raft">lipid rafts</a>.  PrP is also a <a href="http://en.wikipedia.org/wiki/Glycoprotein">glycoprotein</a> because during post-translational modification it gets <a href="http://en.wikipedia.org/wiki/Glycosylation">glycosylated</a> (has carbohydrate side chains added) at amino acids 181N and 197N.</p>
<p>There has been some debate over whether PrP&#8217;s anchoring to the membrane via the GPI anchor is important in prion disease.  Experiments seem to suggest that when PrP is not GPI-anchored, it can still misfold but tends not to do so on a large scale: GPI anchorless mice are tough to infect with prions [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3028874/">Klingeborn 2011</a>] and prions don&#8217;t propagate well in vitro when the PrP is not GPI-anchored [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2802777/">Priola 2009</a>].  PrP&#8217;s localization in lipid rafts and proximity to cholesterol are thought to be somehow important for conversion to PrPSc, which might be part of the explanation for the (albeit limited) therapeutic efficacy of <a title="Statins as prion therapeutics" href="/2012/09/28/statins-as-prion-therapeutics/">statins</a> and <a href="http://en.wikipedia.org/wiki/Amphotericin_B">amphotericin B</a> in prion-infected mice.  Some people think the N terminus of PrP interacts with another lipid raft protein (possibly <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2773931/">Glypican-1</a> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2773931/">Taylor 2009</a>]) and that this interaction is necessary (or at least pretty important) for conversion to PrPSc.</p>
