---
layout: post
title: "Accounting for cell division and PrPSc availability in prion kinetic models"
author: ericminikel
date: 2013-12-03 17:54:23
---
<p><strong>quick recap</strong></p>
<p>In <a href="/2013/12/02/analytical-solutions-for-a-model-of-prion-protein-degradation/" title="Analytical solutions for a model of prion protein degradation">my last post</a> I derived the formulas for a model of prion protein degradation which I had previously <a href="/2013/08/01/how-compound-incubation-time-affects-power-in-anti-prion-drug-discovery-screens/" title="How compound incubation time affects power in anti-prion drug discovery screens">simulated</a>. The analytical solution matches the simulation (whew!) but the question remains: is this a good model?</p>
<p>This post will discuss two possible issues with the original model:</p>
<ol>
<li>It didn&#8217;t account for cell division</li>
<li>It assumed that the PrP<sup>Sc</sup> formation rate was independent of the concentration of PrP<sup>Sc</sup>.</li>
</ol>
<p><strong>cell division</strong></p>
<p>It has been pointed out that PrP<sup>Sc</sup> accumulation in cell culture depends on three factors: the rates of conversion, degradation and cell division [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2084281/">Ghaemmaghami 2007</a>].  My model only considered conversion and degradation. Is that wrong?</p>
<p>Before we get to PrP<sup>Sc</sup>, let&#8217;s consider the simpler cases of PrP mRNA and PrP<sup>C</sup>. I&#8217;ll borrow Ghaemmaghami&#8217;s assumption that cells divide once every 24 hours, giving an exponential parameter ln(2)/24 = .029h<sup>-1</sup>, which I&#8217;ll call &xi; (xi).  If I consider the diluting effects of cell division, then my original expression for mRNA:</p>
<p><img src="//s0.wp.com/latex.php?latex=R%27%28t%29+%3D+r+-+%5Clambda+R%28t%29&#038;bg=ffffff&#038;fg=000&#038;s=0" alt="R&#039;(t) = r - &#92;lambda R(t)" title="R&#039;(t) = r - &#92;lambda R(t)" class="latex"/></p>
<p>Becomes:</p>
<p><img src="//s0.wp.com/latex.php?latex=R%27%28t%29+%3D+r+-+%28%5Clambda+%2B+%5Cxi%29R%28t%29&#038;bg=ffffff&#038;fg=000&#038;s=0" alt="R&#039;(t) = r - (&#92;lambda + &#92;xi)R(t)" title="R&#039;(t) = r - (&#92;lambda + &#92;xi)R(t)" class="latex"/></p>
<p>Hmm. Now all of a sudden, the steady state level of mRNA has gone from being R<sub>steady</sub> = r/&lambda; to being R<sub>steady</sub> = r/(&lambda; + &xi;). The new steady state is &lambda;/(&lambda;+&xi;) less than the old steady state. For the numbers considered here (mRNA half-life of 7h and cell division time of 24h), this works out to 77% of the original steady state value.  Or viewed the other way around, you would have to transcribe RNA (&lambda; + &xi;)/&lambda; &#8211; 1 = 29% faster in order to maintain the original steady-state level of mRNA.</p>
<p>And I don&#8217;t think that this is just a simple issue of having forgotten to account for there being more cells producing RNA as well. The above equation can be inferred to describe the steady state <em>per</em> cell (or per X number of cells), and even if daughter cells start producing mRNA as soon as cytokinesis is finished, they still don&#8217;t each have the same <em>accumulation</em> of previously-produced mRNA that the mother cell would if it hadn&#8217;t divided.  Or to think of it a different way, mRNA production (r) is per diploid genome, and while two daughter cells have 4N ploidy / 2 cells = avg. 2N at the present moment, if you integrate over their recent history thhey haven&#8217;t always had as much DNA to go around.</p>
<p>But this, you&#8217;ll note, is a general issue that applies to <em>all</em> RNA transcripts and all proteins, not just PrP. Indeed, most proteins have a half-life on the order of several days, not hours [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2922600/">Price 2010</a>], meaning they&#8217;d be much more depleted in dividing cells than PrP would be using the above numbers.  Yet it seems unreasonable to suppose that all proteins and RNA are simply depleted in dividing cells.</p>
<p>There are several possible explanations which could resolve this issue. A better biologist than I would know which one is correct: perhaps RNA and protein synthesis are generally upregulated in interphase by enough to make up for the shortfall; perhaps continuously dividing cells are slightly smaller so that RNA and protein are not (as) depleted on a per volume or per surface area basis; and/or perhaps the answer is obscured by the fact that a huge number of genes change expression level during the cell cycle, such that the whole proteome is different and dynamic.</p>
<p>In any case, the empirical observation in [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2084281/">Ghaemmaghami 2007</a>] was that the expression level of PrP<sup>C</sup> <em>per &mu;g of total protein</em> was not different in dividing vs. stationary cells. I&#8217;ll assume the same is probably true of PrP mRNA as well. </p>
<p>What does this mean, then?</p>
<p>As one possible model of what&#8217;s going on, consider the case where:</p>
<p><img src="//s0.wp.com/latex.php?latex=R%27%28t%29+%3D+%28r+%2B+%5Cxi%29+-+%28%5Clambda+%2B+%5Cxi%29R%28t%29&#038;bg=ffffff&#038;fg=000&#038;s=0" alt="R&#039;(t) = (r + &#92;xi) - (&#92;lambda + &#92;xi)R(t)" title="R&#039;(t) = (r + &#92;xi) - (&#92;lambda + &#92;xi)R(t)" class="latex"/></p>
<p>This could be held to represent an actual upregulation of PrP mRNA synthesis during cell division, or just a <em>proportional</em> upregulation due to other mRNAs being similarly depleted. Either way, the implication is that the same steady state is maintained as in stationary cells, but if you knock down transcription by half (0.5 * (r + &xi;)) then the system reaches its new, knocked-down steady state slightly faster than it would otherwise. The assumptions in my last post dictated that a 50% reduction in transcription led to 76% of maximal inhibition after 24h; with cell division in the mix, that figure rises to 83%.</p>
<p>So that&#8217;s one possible interpretation. However it is also fair to ask whether the half-life estimates from the literature already <em>include</em> the effects of cell division. The citations for the mRNA and PrP<sup>C</sup> half-lives are [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8095862">Pfeifer 1993</a>] and [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1968466/">Borcheldt 1990</a>] respectively. These studies involved radiolabeling and then extracting RNA or protein for slot blot or Western blot. Usually you standardize the mass of input material (e.g. protein) before doing a blot, in which case the measured half-lives probably already include effects of cell division; but if you just use the entire mass of RNA or protein from cell lysate, then you&#8217;re getting the rate of catabolism without the confounding effects of cell division. This is such a minute detail that I actually don&#8217;t see it noted anywhere in the materials and methods of these papers.</p>
<p>The standard error on these half-life measurements is probably huge anyway, to the point that the distinction doesn&#8217;t matter numerically. But it&#8217;s nice to have a model that&#8217;s conceptually correct. Here, I&#8217;m not sure which model that is.</p>
<p>Now, for PrP<sup>Sc</sup>, the situation is different, for two reasons.</p>
<p>First, the reported half-life of PrP<sup>Sc</sup> is longer than one cell division cycle (30h vs. 24h), so accounting for cell division makes a much bigger difference.  &nu;/(&nu; + &xi;) = .44, i.e. once cell division is accounted-for, the steady-state level of PrP<sup>Sc</sup> is predicted to be only 44% of that in stationary cells.  And sure enough, PrP<sup>Sc</sup> accumulation in dividing cells is less than in stationary cells &#8211; standard errors are large, but the estimates is plausibly around the 44% value [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2084281/">Ghaemmaghami 2007</a>].</p>
<p>The second reason is that, unlike mRNA and PrP<sup>C</sup>, whose production depends ultimately upon the number of copies of the genome (and therefore of the <em>Prnp</em> gene) available, PrP<sup>Sc</sup> production depends upon the availability of PrP<sup>Sc</sup> &#8211; which is diluted in cell divisions &#8211; to convert PrP<sup>C</sup>.</p>
<p>Or does it?</p>
<p><strong>PrP<sup>Sc</sup> availability</strong></p>
<p>In my last post, I assumed that the rate of PrP<sup>Sc</sup> production is proportional to the amount of PrP<sup>C</sup> available as substrate but does <em>not</em> depend on the amount of PrP<sup>Sc</sup> available to do the converting. </p>
<p>That assumption is, of course, inconsistent with the physical reality that PrP<sup>Sc</sup>-PrP<sup>C</sup> binding is required for prion conversion. Ghaemmaghami calls this the &#8220;heterodimer model&#8221;.  In short:</p>
<p><a href="/wp-content/uploads/2013/12/prpsc-plus-prpc-equals-2prpsc.png"><img src="/wp-content/uploads/2013/12/prpsc-plus-prpc-equals-2prpsc.png"/></a></p>
<p>The 1:1 stoichiometry here seems to indicate that the conversion rate should be proportional to both [PrP<sup>C</sup>] and [PrP<sup>Sc</sup>]. (Note to non-biologists: square brackets mean &#8220;concentration of&#8221;).</p>
<p>Ghaemmaghami started from this assumption, proposing that:</p>
<p><img src="//s0.wp.com/latex.php?latex=%5Cfrac%7BdPrP%5E%7BSc%7D%7D%7Bdt%7D+%3D+k_%7Bconversion%7D%5BPrP%5EC%5D%5BPrP%5E%7BSc%7D%5D+-+k_%7Bdegradation%7D%5BPrP%5E%7BSc%7D%5D+-+k_%7Bdivision%7D%5BPrP%5E%7BSc%7D%5D&#038;bg=ffffff&#038;fg=000&#038;s=0" alt="&#92;frac{dPrP^{Sc}}{dt} = k_{conversion}[PrP^C][PrP^{Sc}] - k_{degradation}[PrP^{Sc}] - k_{division}[PrP^{Sc}]" title="&#92;frac{dPrP^{Sc}}{dt} = k_{conversion}[PrP^C][PrP^{Sc}] - k_{degradation}[PrP^{Sc}] - k_{division}[PrP^{Sc}]" class="latex"/></p>
<p>But he then discovered that [PrP<sup>C</sup>] appeared never to change, so he absorbed it into the constant:</p>
<p><img src="//s0.wp.com/latex.php?latex=%5Cfrac%7BdPrP%5E%7BSc%7D%7D%7Bdt%7D+%3D+k%5E%2A_%7Bconversion%7D%5BPrP%5E%7BSc%7D%5D+-+k_%7Bdegradation%7D%5BPrP%5E%7BSc%7D%5D+-+k_%7Bdivision%7D%5BPrP%5E%7BSc%7D%5D&#038;bg=ffffff&#038;fg=000&#038;s=0" alt="&#92;frac{dPrP^{Sc}}{dt} = k^*_{conversion}[PrP^{Sc}] - k_{degradation}[PrP^{Sc}] - k_{division}[PrP^{Sc}]" title="&#92;frac{dPrP^{Sc}}{dt} = k^*_{conversion}[PrP^{Sc}] - k_{degradation}[PrP^{Sc}] - k_{division}[PrP^{Sc}]" class="latex"/></p>
<p>meaning that there ought to exist but one steady state, where:</p>
<p><img src="//s0.wp.com/latex.php?latex=k%5E%2A_%7Bconversion%7D+%3D+k_%7Bdegradation%7D+%2B+k_%7Bdivision%7D&#038;bg=ffffff&#038;fg=000&#038;s=0" alt="k^*_{conversion} = k_{degradation} + k_{division}" title="k^*_{conversion} = k_{degradation} + k_{division}" class="latex"/></p>
<p>And yet he observed two steady states, one in dividing cells and one in stationary cells. The evidence for a steady state in stationary cells was that [PrP<sup>Sc</sup>] appears to level off in the last few days of 10 days in culture without splitting, when the cells have pretty much grown to confluence, as shown in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2084281/figure/F1/">Fig 1C</a>:</p>
<p><a href="/wp-content/uploads/2013/12/ghaemmaghami-2007-fig1c.png"><img src="/wp-content/uploads/2013/12/ghaemmaghami-2007-fig1c.png"/></a></p>
<p>The conclusion of the paper was, then, that the rate of prion conversion must not depend on [PrP<sup>Sc</sup>], or at least not on <em>total</em> [PrP<sup>Sc</sup>]. Perhaps, instead, it depends on an infectious subset of PrP<sup>Sc</sup>. Perhaps (and I&#8217;m extrapolating here) the &#8220;heterodimer&#8221; model isn&#8217;t always strictly correct in terms of stoichiometry &#8211; perhaps PrP<sup>Sc</sup> often comes in larger fibrils or aggregates where new PrP<sup>C</sup> molecules can only be added to either end and not to the middle. Thus stationary cells might accumulate 2x more PrP<sup>Sc</sup> than dividing cells and yet have barely more <em>infectious</em> material than dividing cells do.  Alternately, it might be some other factor such as &#8220;<a href="/2013/08/16/is-protein-x-the-basis-of-dominant-negative-effects-in-prion-disease/" title="Is Protein X the basis of dominant negative effects in prion disease?">Protein X</a>&#8221; availability or space in lipid rafts which becomes the limiting reactant.</p>
<p>In any case, that&#8217;s a useful observation, because otherwise you end up with a very unstable system. Suppose, as Ghaemmaghami proposed, that [PrP<sup>C</sup>] is constant. Then:</p>
<p><img src="//s0.wp.com/latex.php?latex=%5Cfrac%7BdPrP%5E%7BSc%7D%7D%7Bdt%7D+%3D+%28k%5E%2A_%7Bconversion%7D+-+k_%7Bdegradation%7D+-+k_%7Bdivision%7D%29%5BPrP%5E%7BSc%7D%5D&#038;bg=ffffff&#038;fg=000&#038;s=0" alt="&#92;frac{dPrP^{Sc}}{dt} = (k^*_{conversion} - k_{degradation} - k_{division})[PrP^{Sc}]" title="&#92;frac{dPrP^{Sc}}{dt} = (k^*_{conversion} - k_{degradation} - k_{division})[PrP^{Sc}]" class="latex"/></p>
<p>If we just define a constant:</p>
<p><img src="//s0.wp.com/latex.php?latex=%5Cvarphi+%3D+k%5E%2A_%7Bconversion%7D+-+k_%7Bdegradation%7D+-+k_%7Bdivision%7D&#038;bg=ffffff&#038;fg=000&#038;s=0" alt="&#92;varphi = k^*_{conversion} - k_{degradation} - k_{division}" title="&#92;varphi = k^*_{conversion} - k_{degradation} - k_{division}" class="latex"/></p>
<p>Then the solution to the above differential equation is:</p>
<p><img src="//s0.wp.com/latex.php?latex=PrP%5E%7BSc%7D%28t%29+%3D+PrP%5E%7BSc%7D_0+e%5E%7B%5Cvarphi+t%7D&#038;bg=ffffff&#038;fg=000&#038;s=0" alt="PrP^{Sc}(t) = PrP^{Sc}_0 e^{&#92;varphi t}" title="PrP^{Sc}(t) = PrP^{Sc}_0 e^{&#92;varphi t}" class="latex"/></p>
<p>From this equation, we can see that the initial quantity PrP<sup>Sc</sup><sub>0</sub> will blow up exponentially without bound if &phi; &gt; 0 and will be degraded to nothing if &phi; &lt; 0.  There is but one razor-thin unstable steady state at &phi; = 0 where PrP<sup>Sc</sup><sub>0</sub> will be maintained indefinitely.</p>
<p>That&#8217;s actually not a bad model to fit the empirical observations of PrP<sup>Sc</sup>&#8216;s behavior in <em>some</em> experimental paradigms. For instance, human cell lines have never been successfully stably infected with prions &#8211; they always eventually clear the infection (&phi; &lt; 0), while <em>in vivo</em> prion infection leads pretty much inevitably to an exponential rise in PrP<sup>Sc</sup> and ultimately to death (&phi; &gt; 0). So maybe there&#8217;s really something to this model! But in N2a cells, prions have been propagated stably for about 25 years now, since [<a href="http://www.ncbi.nlm.nih.gov/pubmed/3282080">Butler 1988</a>], and no one&#8217;s going to convince me that this has been achieved simply by skating along the &phi; = 0 tightrope. It simply must be that, at least in this cell line, there is a more generous window of possible steady state values. </p>
<p>How, then, do we model the kinetics here? If we knock down mRNA by 50%, in my original <a href="/2013/08/01/how-compound-incubation-time-affects-power-in-anti-prion-drug-discovery-screens/" title="How compound incubation time affects power in anti-prion drug discovery screens">simulation</a> and <a href="/2013/12/02/analytical-solutions-for-a-model-of-prion-protein-degradation/" title="Analytical solutions for a model of prion protein degradation">equations</a> that means, asymptotically, a 50% reduction in PrP<sup>Sc</sup>. However, if this knockdown also reduces &#8220;PrP<sup>Sc</sup><sub>infectious</sub>&#8221; then it could potentially reduce PrP<sup>Sc</sup> by <em>more</em> than 50%.  Unfortunately, I&#8217;m not aware of any empirical observations that would allow us to quantify this relationship.</p>
<p>At a minimum, we could at least account for the effects of cell division. So I went back and took another look at the citation for a ~30h half life of PrP<sup>Sc</sup> in N2a cells [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11507642">Peretz 2001</a> (<a href="http://www.nature.com/nature/journal/v412/n6848/full/412739a0.html">ft</a>)].  Peretz never actually advances the 30h number, but rather cites [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1968466">Borcheldt 1990</a>] as evidence that the half-life is &gt; 24h and notes that the time required for Fab D18 (an antibody fragment) to reduce PrP<sup>Sc</sup> by 50% was 28h. As far as I can tell this experiment was done in continuously dividing cells, however, what was measured was <em>total</em> PrP<sup>Sc</sup> rather than PrP<sup>Sc</sup> per unit total protein, thus this half-life estimate would appear <em>not</em> to include the effects of cell division. </p>
<p>Therefore the best model would probably be one which assumes that the total clearance rate &nu; of PrP<sup>Sc</sup> is equal to the measured half life &nu;<sub>0</sub> = ln(2)/30h = .023h<sup>-1</sup> plus the cell division parameter &xi; = .029h<sup>-1</sup>.  In order to accommodate the assumption that we&#8217;re beginning at steady state, then, we must also assume that the default value of &beta; in untreated cells is equal to that new and improved value of &nu;. I&#8217;ve updated the <a href="https://gist.github.com/ericminikel/7755706">gist</a> from the last post to reflect this change. For now, I&#8217;ve also included &xi; in the mRNA and PrP<sup>C</sup> clearance parameters &lambda; and &mu; respectively, though it&#8217;s not clear whether or not this is correct.</p>
<p>I welcome any comments on this approach.</p>
