---
layout: post
title: "Proteolytic shedding of prion protein"
author: ericminikel
date: 2013-06-29 15:51:05
---
<p><strong>shedding</strong></p>
<p>&#8220;Shedding&#8221; of PrP refers to release of PrP from the cell surface.  This can be achieved experimentally with PIPLC, an enzyme that cleaves GPI anchors, but I haven&#8217;t read any reports that cleavage of PrP&#8217;s GPI anchor is a mechanism for shedding in vivo.  Instead, as we&#8217;ll see, shedding appears to result from cleavage of PrP itself, at a peptide bond near the C terminus.  Here&#8217;s a diagram of these two phenomena.  As I&#8217;ll explain later, the exact cleavage site in human PrP is not known.</p>
<p><a href="/wp-content/uploads/2013/06/prp-shedding.png"><img class="alignnone size-full wp-image-2539" alt="prp-shedding" src="/wp-content/uploads/2013/06/prp-shedding.png"/></a></p>
<p>Shedding of PrPC was first explored in detail in cultured embryonic Syrian hamster neural tissue [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7691278">Borcheldt 1993</a>].  Cells were pulsed with 35S methionine and within 24 hours about 10 &#8211; 20% of the radiolabeled PrPC had been shed into the culture medium.  This raised the question of where anchored PrPC was being cleaved &#8211; was the protein itself cleaved at a peptide bond between two amino acids, or was the GPI anchor being cleaved?  To understand this question better, I had to get a grasp on GPI structure.  Based on <a href="http://www.helsinki.fi/~vakeva/thesis.html">this</a>, <a href="http://www.springerimages.com/Images/RSS/1-10.1385_0-89603-547-6_187-0">this</a>, <a href="http://en.wikipedia.org/wiki/Glycophosphatidylinositol">this</a> and <a href="http://en.wikipedia.org/wiki/Phosphoinositide_phospholipase_C">this</a>, my best understanding of the GPI anchor is as follows:</p>
<p><a href="/wp-content/uploads/2013/06/gpi-anchor-and-piplc.png"><img class="alignnone size-full wp-image-2540" alt="gpi-anchor-and-piplc" src="/wp-content/uploads/2013/06/gpi-anchor-and-piplc.png"/></a></p>
<p>The C terminus of the protein is attached to ethanolamine phosphate is attached to a glycan is attached to inositol is attached to phosphate is attached to fatty acid chains.  From inositol on downward it&#8217;s the same as <a href="http://en.wikipedia.org/wiki/Phosphatidylinositol">PI</a>, a component of the <a href="/2013/02/10/cell-biology-02-the-plasma-membrane/">plasma membrane</a>, so that part of the anchor stays embedded in the membrane just like any other phospholipid.</p>
<p>PI-specific phospholipase C (PIPLC), an enzyme which specifically cleaves GPI anchors, cleaves them between the phosphate and the fatty acid chains, thus leaving phosphate, inositol, the glycan chain and ethanolamine still attached to the C terminus of the shed protein.  Hydrofluoric acid (HF) can remove these remaining portions of the GPI anchor following PIPLC cleavage.  Therefore if you treat cells with PIPLC, run the shed PrPC on a gel, then treat the PrPC with HF and run it again, you&#8217;ll find that HF removes 2-4 kDa of PrPC&#8217;s molecular weight.  That is your sign that the remnants of the GPI anchor were initially attached, but are no longer after HF treatment.</p>
<p>By contrast, Borcheldt found that the shed PrPC&#8217;s molecular weight was not affected by HF treatment, meaning the GPI anchor was already gone.  Yet the shed PrPC still stained positive for the most C-terminal antibodies to PrP available at that time, suggesting most of the protein was intact.  This meant that some proteolytic enzyme was cleaving PrP at a peptide bond very close to the C terminus.  (People still use PIPLC as an experimental tool for releasing PrP from the cell surface but I haven&#8217;t read any reports claiming that GPI anchor cleavage is a mechanism for PrP shedding in vivo).</p>
<p>The exact site and mechanism of this proteolytic cleavage were later explored in more detail in HEK and N2a cells (human and mouse respectively) [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2755666/">Taylor 2009</a>].  GPI-anchored PrP ends in QAYYQRGS in human, and QAYYDGRRS in mouse.  The DG at this site appears fairly unique to rodents:</p>
<p><a href="/wp-content/uploads/2013/06/prp-c-terminus-rodents-dg.png"><img class="alignnone size-full wp-image-2541" alt="prp-c-terminus-rodents-dg" src="/wp-content/uploads/2013/06/prp-c-terminus-rodents-dg.png"/></a></p>
<p><em>Note: I wasn&#8217;t happy with <a href="/wp-content/uploads/2013/06/prp-multalign-mammals.html">how T-Coffee aligned the C-terminus of mammal PrP</a> so for this graphic I took the liberty of rearranging the way I think they line up best.  I consider it more likely that the Q in most mammals&#8217; fourth-to-last position corresponds to the G in rodents, rather than the D, since you can get between G and Q in one base pair change, while D to Q takes two changes.</em></p>
<p>Taylor found that the cleavage site in recombinant MoPrP is between the G and R (i.e. QAYYDG | RRS), a site that doesn&#8217;t exist in HuPrP.  I couldn&#8217;t find any reports confirming the exact sheddase cleavage site is in HuPrP, but shedding is indeed observed in HuPrP just as in SHaPrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7691278">Borcheldt 1993</a>].  For the purposes of the diagram at the top of this post, I&#8217;ve drawn it as QAYYQ | RGS, but I haven&#8217;t actually read any evidence for this being the case.</p>
<p>Taylor also provided a variety of in vitro evidence implicating ADAM9 and <a href="http://en.wikipedia.org/wiki/ADAM10">ADAM10</a> as being responsible for this proteolytic cleavage (indirectly and directly, respectively) [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2755666/">Taylor 2009</a>].  Of note, both of those are also believed to be involved in <a title="Proteolysis of prion protein in the central region: alpha and beta cleavage" href="/2013/06/26/proteolysis-of-prion-protein-in-the-central-region-alpha-and-beta-cleavage/">alpha cleavage of PrP</a> &#8211; cleavage in the center of the protein, on either side of H111.</p>
<p><strong>relation to bank voles?</strong></p>
<p>How does <a href="http://en.wikipedia.org/wiki/ADAM10">ADAM10</a> &#8220;know&#8221; where to cleave PrP?  Borcheldt speculated that PrP might be processed by &#8220;a molecule like the secretase that cleaves Alzheimer&#8217;s precursor protein (APP)&#8230; This secretase exhibits little amino acid specificity and appears to cleave APP at a defined distance from the membrane&#8221;.  Two decades later, we know that the alpha-secretase that cleaves APP right above the membrane is none other than ADAM10 itself [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2944055/">Kuhn 2010</a>] &#8211; the same sheddase that cleaves PrP.  A review of known ADAM10 cleavage sites plus a battery of experiments with synthetic peptides [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2774824/">Caescu 2009</a>, see esp. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2774824/table/T2/">Table 2</a> and <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2774824/figure/F1/">Figure 1</a>] failed to find any obvious consensus sequence for ADAM10 cleavage &#8211; but amino acid sequence does appear to matter somewhat.  Interestingly, if you look at the ADAM10 portion of the heat map in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2774824/figure/F1/">Figure 1</a>, ADAM10 appears to &#8216;prefer&#8217; E in the P2 position (two amino acids N-terminal of the cleavage site) over D.</p>
<p>This is of interest because bank voles are the only known mammal with an E in that position.  Backstory: <a href="http://en.wikipedia.org/wiki/Bank_vole">bank voles</a> have almost no transmission barrier to infection by any prion strain [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1383487/">Nonno 2006</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2453331/">Agrimi 2008</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2798418/">Heisey 2009</a>], and overexpressing wild-type bank vole PrP in mice causes a spontaneous transmissible prion disease [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3295307/">Watts 2012</a>].  At Prion2013, Joel Watts presented his findings that mice expressing BvPrP, as it&#8217;s known, are highly susceptible to every prion strain tested.  They have short to medium incubation times at first passage and very short (~35 day) incubation times on second passage.  Weirdly, the BvPrP mice show almost no PrP-res, even when terminally ill of prion strains that normally produce PrP-res in other mammals.  One interpretation of this is that BvPrP is especially prone to produce PrPL &#8211; the hypothesized [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27848/">Hill 2000</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/21350487">Sandberg 2011</a>] lethal by-product of prion conversion, and that therefore it kills BvPrP mice earlier on, before they&#8217;ve had a chance to accumulate as much PrP-res.</p>
<p>I had the chance to talk to Joel Watts later that day and I chatted him up about the differences between bank vole PrP and mouse PrP.  He said they differ in 9 amino acids [see his <a href="http://www.pnas.org/content/suppl/2012/02/08/1121556109.DCSupplemental/pnas.201121556SI.pdf#nameddest=SF1">mouse / bank vole alignment</a>], but 8 of those changes have been seen in other mammals.   The only sequence change that is (as far as we know) absolutely unique to bank voles is the E near the C terminus:</p>
<p><a href="/wp-content/uploads/2013/06/prp-c-terminus-only-bank-voles-have-e.png"><img class="alignnone size-full wp-image-2542" alt="prp-c-terminus-only-bank-voles-have-e" src="/wp-content/uploads/2013/06/prp-c-terminus-only-bank-voles-have-e.png"/></a></p>
<p>Watts opined that the exceptional instability of BvPrP could be due to this change, but that this would be surprising, since D→E is a fairly conservative amino acid change and since the location of this change is so C-terminal, far from the prion conversion action at the core region.  Alternately, the unique properties of BvPrP could owe to bank voles&#8217; particular combination of other amino acid changes.</p>
<p>The E could potentially matter if it influenced shedding.  It&#8217;s not clear how much the E would really increase ADAM10&#8242;s affinity for the C terminus of PrP, though, considering that the overall sequence (EGRSS) still doesn&#8217;t appear to be highly favored by ADAM10 [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2774824/">Caescu 2009</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2774824/table/T2/">Table 2</a> and <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2774824/figure/F1/">Figure 1</a>].  It&#8217;s also not clear how a hypothetical increase in shedding would affect prion disease progression. When <a href="http://www.biochemistry.unimelb.edu.au/research/res_hill.html">Andy Hill</a> presented at Prion2013, he cited three major mechanisms &#8211; exosomes, cell-to-cell contact, and tunneling nanotubes &#8211; that allow prion infection to spread from cell to cell.  There was no mention of shed PrP as a mechanism for the spread of regular prion diseases &#8211; though anchorless PrP does cause a different disease, cerebral amyloid angiopathy &#8211; more on this below.  Anyway, even if shedding does help prions spread more rapidly, that wouldn&#8217;t explain other strange features of BvPrP &#8211; notably, the relative absence of PrP-res.</p>
<p><strong>therapeutic value of ADAM10?</strong></p>
<p>One study created transgenic mice overexpressing ADAM10, and found that they had extended survival times after scrapie infection [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19632330/">Endres 2009</a>].  The overexpressers lived to an average of 153 days, controls lived to 134 days &#8211; a 12% delay, p &lt; .0001 according to the author.  There are at least three possible ways that ADAM10 overexpression may have affected the course of disease in these mice.  First, Endres found that PrP mRNA was reduced by about 30% in the ADAM10 overexpressers &#8211; the transgene had apparently suppressed transcription of Prnp.  The mechanism for that is not at all clear.  Second, we know ADAM10 is involved in alpha cleavage of PrP, which is believed to render the remaining C1 fragment incapable of converting to PrPSc, so that would be expected to slow down prion disease progression.  Probably either of these first two explanations &#8211; mRNA reduction or alpha cleavage &#8211; could be sufficient to explain a 12% extension of survival time in Endres&#8217;s mice.  Third, ADAM10 is involved in shedding of PrP [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2755666/">Taylor 2009</a>], which might be good or bad for prion disease, as discussed in the next section.</p>
<p><strong>therapeutic implications of shedding and anchorless PrP</strong></p>
<p>What implication does shedding have for prion disease progression?  Is there a potential therapeutic angle here?  It&#8217;s not totally clear.</p>
<p>There is a school of thought that cell surface localization of PrP is essential for disease.  For instance, the recent <a href="/2013/04/04/tacrolimus-astemizole-and-a-new-screening-approach-for-drugs-that-reduce-prp/">Scripps FRET assay</a> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23576755">Karapetyan &amp; Sferrazza 2013</a>] was conceived with the goal of detecting compounds that reduce cell surface PrP (though <a href="/2013/04/04/tacrolimus-astemizole-and-a-new-screening-approach-for-drugs-that-reduce-prp/#lasmezasqa">in practice the design of the assay</a> would not have detected compounds that increased shedding).  The main argument for cell surface localization being crucial to disease is that when ScN2a (scrapie-infected mouse neuroblastoma) cells are treated with PIPLC for 3 days, they are &#8220;cured&#8221; of scrapie infection [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC55414/">Enari 2001</a>].  There has been a bit of other work in this area as well &#8211; glimepiride, a drug that promotes GPI cleavage, can also reduce PrPSc formation in cell culture [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2784943/">Bate 2009</a>].</p>
<p>But the kinetics of prion infection in rapidly dividing N2a cells are quite different than the kinetics of prion infection in post-mitotic neurons in vivo.  Cell division constantly dilutes prions in ScN2a cells, and the sum of dilution plus degradation allows the cells to keep pace with prion formation, avoiding a toxic buildup of prions [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2084281/">Ghaemmaghami 2007</a>].  In Enari&#8217;s experiment the cells were not split or passaged, meaning that cell division was somewhat limited and shed PrPSc was not washed away but rather stayed with the cells.  That makes Enari&#8217;s result a bit more like in vivo conditions, but still, cell cultures aren&#8217;t animals, and they don&#8217;t have blood vessels where PrPSc plaques could build up.</p>
<p>In humans, premature stop codons resulting in truncated, anchorless PrP cause a form of <a href="http://en.wikipedia.org/wiki/Cerebral_amyloid_angiopathy">cerebral amyloid angiopathy</a> (CAA).  This form of CAA is a prion disease characterized by intense amyloid deposition and an Alzheimer&#8217;s-like clinical presentation with a long disease duration (~ 4 years) and no spongiform change.  This disease can result from Y145X [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8097911">Kitamoto 1993</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/8570627/">Ghetti 1996</a>], Y163X [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2844092">Revesz 2009</a>], Y226X, Q227X [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2808512/">Jansen 2010</a>] and probably also the recently reported frameshift mutation at codon 178 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23577609">Matsuzono 2013</a>].</p>
<p>Based on the human data alone, it is impossible to disentangle whether anchorless PrP is <em>inherently</em> bad or if these mutations cause disease only because PrP is also truncated.  The first mice expressing anchorless PrP were originally reported to have no spontaneous disease on their own, but to be capable of producing amyloid plaques upon prion infection, albeit without fatal disease and with a very different histopathological phenotype than prion-infected wild-type mice, similar to CAA [<a href="http://www.ncbi.nlm.nih.gov/pubmed/15933194/">Chesebro 2005</a>].  This was later revised when it was found that these mice do have behavioral deficits after prion infection [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2566293/">Trifilo 2008</a>] and then revised again when it was discovered that homozygous anchorless PrP mice, when inoculated with prions, did indeed get a fatal disease dubbed &#8220;fatal transmissible amyloid encephalopathy&#8221; [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832701/">Chesebro 2010</a>].</p>
<p>The course of prion disease in Chesebro&#8217;s homozygous anchorless mice differs from wild-type mice in two ways: first, the phenotype is characterized by plaque deposition, much like the human CAA diseases, and second, the incubation time for prion inoculation is two or three times longer than for wild-type mice: 300 &#8211; 480 days instead of 150 days [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832701/">Chesebro 2010</a>].  That latter point sounded promising &#8211; if we could find drugs to promote PrP shedding, could we &#8220;convert&#8221; prion disease into CAA and thus slow the disease down by three-fold?  At Prion2013 I asked Bruce Chesebro whether this longer incubation time was an inherent property of anchorless PrP or was simply due to reduced expression, as anchorless PrP tends to be degraded before reaching the cell surface &#8211; Chesebro&#8217;s mice express just ~0.1x wild-type levels of PrP, which could be sufficient to explain the long incubation times.  From that particular mouse model, it&#8217;s impossible to tell.</p>
<p>But another anchorless mouse model expressing ~1x PrP is not only susceptible to prions but even develops a spontaneous prion disease eventually [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3248514/">Stohr 2011</a>].  And the disease is even worse if you co-express anchored PrP with anchorless PrP.  Thus it seems that promoting GPI anchor cleavage and/or proteolytic shedding of PrP <em>in vivo</em> might well do more harm than good.</p>
