---
layout: post
title: "How to calculate heritability"
author: ericminikel
date: 2013-02-04 22:21:57
---
<p>Heritability is the proportion of variance in a particular trait, in a particular population, that is due to genetic factors, as opposed to environmental influences or stochastic variation.</p>
<p>That&#8217;s just a general definition to give you a feel for it.  Actually we need to be more rigorous than that.  There are two definitions of heritability.  A common simplification in all sorts of genetic studies and models is to assume that all alleles and all genotypes act independently of each other &#8211; this is called an &#8216;additive model.&#8217;  So for instance, if one allele of a particular SNP gives you a 1 cm increase in height, then being homozygous for that SNP should give you a 2 cm increase in height.  Clearly, this model doesn&#8217;t allow for dominant or recessive effects, even though we know these abound.  It also doesn&#8217;t allow for gene-gene interactions, where maybe that SNP only gives you a 1 cm increase in height if paired with another SNP.  For these reasons, the additive model is a huge simplification, but a useful one.  Now for the two definitions of heritability:</p>
<ul>
<li>&#8216;narrow sense heritability&#8217; (h<sup>2</sup>) is defined as the proportion of trait variance that is due to additive genetic factors</li>
<li>&#8216;broad sense heritability&#8217; (H<sup>2</sup>) is defined as the proportion of trait variance that is due to all genetic factors including dominance and gene-gene interactions.</li>
</ul>
<p>Both kinds of heritability are incredibly tricky to estimate and to interpret.  In terms of estimation, a big problem is that people who share parts of their genome tend to share parts of their environment too.  One simple way you might think to estimate heritability is to <a title="parent-offspring regression, on Wikipedia" href="http://en.wikipedia.org/wiki/Heritability#Parent-offspring_regression">plot children&#8217;s traits against the average of their parents</a>, as shown in this example from <a href="http://www.ncbi.nlm.nih.gov/pubmed/18319743">Visscher 2008</a>: <img class="alignnone size-full wp-image-1596" title="visscher-2008-fig2" src="/wp-content/uploads/2013/02/visscher-2008-fig2.jpg"/></p>
<p>In the example above, the slope is taken to be the heritability.  The problem with this is that parent and child share a lot else besides half their genome.</p>
<p>One approach to calculating heritability which largely avoids the confounding of genotype with shared environment is to compare the phenotypic concordance of monozygotic (MZ, identical) twins versus dizygotic (DZ, fraternal) twins.  <em>Both</em> types of twins are expected to share virtually all environmental factors, including while in the womb, which is why this is a better study design than just comparing MZ twins to siblings.  Comparing MZ to DZ twins lets you isolate the contribution of that marginal half shared genome to phenotypic concordance.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/18319743">Visscher 2008</a>, citing <a href="http://www.ncbi.nlm.nih.gov/pubmed/16721405">Deary 2006</a> (<a href="http://www.nature.com/ejhg/journal/v14/n6/full/5201588a.html">ft</a>), discusses the example of IQ, where MZ twins have concordance of .86 and DZ twins have concordance of .60.  &#8221;Concordance&#8221; in these studies seems to refer to a <a href="http://en.wikipedia.org/wiki/Pearson_product-moment_correlation_coefficient">Pearson&#8217;s correlation</a> or similar, so something like an r or ρ.  <em>(Why r and not slope, like in parent-offspring regression?  See <a title="Slope or correlation, not variance explained, allow estimation of heritability" href="/2013/04/30/slope-or-correlation-not-variance-explained-allow-estimation-of-heritability/">this post</a> for further discussion).</em></p>
<p>At first glance it is not clear how to convert these numbers &#8211; .86 and .60 &#8211; into an estimate of heritability.  After all, both of these figures include both genetic and environmental factors.  The key observation is that sharing a <em>marginal</em> half genome with your twin explains an additional .86-.60 = 26%, so in theory, sharing a full genome explains 2*26% = 52%.</p>
<p>This is called <a href="http://en.wikipedia.org/wiki/Falconer%27s_formula">Falconer&#8217;s formula</a>:</p>
<p>heritability = 2(r<sub>MZ</sub> - r<sub>DZ</sub>)</p>
<p>I wrote &#8220;heritabiliity&#8221; on the left side of the equation, instead of h<sup>2</sup> or H<sup>2</sup>, because it is debatable what this estimate is really reflecting.  The wiki on <a href="http://en.wikipedia.org/wiki/Falconer%27s_formula">Falconer&#8217;s formula</a> claims that it estimates H<sup>2</sup>, broad-sense heritability.  Indeed: since MZ twins share virtually all their genotypes (there will be just a few chance mutations here and there that make them differ), they share dominant / recessive effects and gene-gene interactions, which DZ twins are not expected to share.  Yet the notion that you can just double the 26% marginal variance explained by a half genome in order to extrapolate to a whole genome seems to assume all additive effects.  The 26% <em>marginal</em> variance explained from sharing a whole genome as opposed to a half genome might in fact be partitioned between additive effects (which we could fairly double to extrapolate to a whole genome), gene-gene interactions (which we should perhaps multiply by 4/3, since DZ twins sharing half their genome only share 1/4 of their possible gene-gene pairings, so the MZ twins are capturing an extra 3/4) and dominance effects (which we might also argue to multiply by 4/3 since DZ twins share <em>both</em> alleles at only 25% of sites, so again, MZ twins add a marginal 3/4).</p>
<p>Accordingly, though the <a href="http://en.wikipedia.org/wiki/Falconer%27s_formula">wiki on Falconer&#8217;s formula</a> claims it calculates H<sup>2</sup>, the <a href="http://en.wikipedia.org/wiki/Twin_study#Methods">wiki on twin studies</a> claims it estimates h<sup>2</sup>.  To my view, it&#8217;s not a perfect estimate of either of these.  And frustratingly, grand sweeping reviews of the concept of heritability (such as <a href="http://www.ncbi.nlm.nih.gov/pubmed/18319743">Visscher 2008</a>) are long on talk and short on formulae.</p>
<p>I also ran across an old school paper by <a href="http://www.ncbi.nlm.nih.gov/pubmed/6626665">Jacquard 1983</a> (<a href="http://www.biostat.washington.edu/~bsweir/BIOST551/Jacquard1983.pdf">ft</a>) which presents a formula something like this:</p>
<p>heritability = (ρ<sub>MZ</sub> - ρ<sub>DZ</sub>)/(1-ρ<sub>DZ</sub>)</p>
<p>So for the IQ example, heritability = (.86-.60)/(1-.60) = .26/.40 = 65%.  This will often give pretty different answers than <a href="http://en.wikipedia.org/wiki/Falconer%27s_formula">Falconer&#8217;s formula</a>; I don&#8217;t quite understand the logic of it, though one nice property of it is that it never rises above 100%.  But I can&#8217;t find any evidence that this alternative formulation is still in use today.</p>
<p>Besides the H<sup>2</sup> vs. h<sup>2</sup> debate, there are other conceptual issues with calculating heritability from twin studies as well.  For instance, MZ twins may actually share more environmental factors than DZ twins since being similar makes people treat them similarly.  It&#8217;s entirely possible to find that ρ<sub>MZ</sub> - ρ<sub>DZ</sub> &gt; 50%, in which case your estimate of heritability will be &gt; 100%. Oops.  Also, it is assumed that DZ twins share exactly half their genome, but in fact, due to random segregation of alleles, there is variance in what fraction of alleles siblings actually share &#8211; more on this shortly.</p>
<p>There are plenty of other study designs as well.  Whereas MZ vs. DZ twin studies look at the effect of sharing 100% IBD instead of ~50%, sibling vs. adopted sibling studies look at the effect of ~50% IBD instead of 0% IBD, while theoretically controlling for shared environment (though obviously it can&#8217;t account for factors in the womb).  Here, h<sup>2 </sup>=  2(ρ<sub>sib</sub> - ρ<sub>adoptee</sub>)  <em>(*The <a href="http://en.wikipedia.org/wiki/Falconer's_formula">Falconer&#8217;s Formula wiki</a> says not to double this quantity, i.e. h<sup>2 </sup>=  ρ<sub>sib</sub> - ρ<sub>adoptee</sub> &#8211; this seems incorrect to me, but if you know otherwise, please leave me a note)</em></p>
<p>Twin pairs and sibling/adoptee pairs are all well and good when you&#8217;re dealing with a trait like height, which you can measure for absolutely anyone.  But consider the phenotype of <a title="Background on heritability in Huntington’s Disease age of onset" href="/2012/11/28/background-on-heritability-in-huntingtons-disease-age-of-onset/">residual age of onset in Huntington&#8217;s Disease</a>.  This phenotype can only be assessed for people who have HD, which is already a very rare disease; if you were to also limit yourself to twin pairs you&#8217;d have an n pretty close to zero.  In that case, you&#8217;ll take what you can get, such as <a href="http://en.wikipedia.org/wiki/Heritability#Sibling_comparison">correlation between sibling pairs</a>: 2*r<sub>sib</sub> is an upper limit for heritability.  It might be a pretty loose upper bound, and if r<sub>sib</sub> &gt; 50%, then it&#8217;s no upper bound at all.  If you have other relationships in your dataset as well &#8211; parent-offspring pairs, avuncular pairs, cousin pairs &#8211; then you can try to make a few more inferences, though it is pretty hard to disentangle genes and environment because unlike in the MZ/DZ and sibling/adoptee comparisons you don&#8217;t have any pairs of pairs where environment is shared equally between the two pairs but genotype is shared unequally.  In <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC373491/">U.S.–Venezuela Collaborative Research Project 2004</a>, the most oft-cited study of heritability in HD age of onset, the siblings had concordance of .42 (suggesting an upper bound for heritability of 84%) while parent-offspring were only .10, avuncular .07 and cousin .15 [see <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC373491/table/tbl4/">Table 4</a>].  Under an additive model (narrow-sense heritability), the parent-offspring correlation would suggest heritability of no more than 2*.10 = 20%, the avuncular would suggest 4*.07 = 28%, and the cousins no upper limit at all because 8*.15 &gt; 100%.  In short, the data are all over the place.  The authors assumed that only siblings share an environment, and then used some model (details never stated; see my commentary <a title="Heritability and genetic modifiers in Mendelian diseases" href="/2013/03/05/heritability-and-genetic-modifiers-in-mendelian-diseases/">here</a>) to integrate all these pieces of information into a single estimate of 38% heritability.  This should probably be interpreted as a pretty rough estimate.</p>
<p>Here&#8217;s a thought exercise: suppose the Venezuelan HD pedigree has some <a href="http://en.wikipedia.org/wiki/Consanguinity">consanguinity</a>, which means that relatives often share more IBD than their nominal relations to each other would suggest.  Does that bias the heritability estimate?  I am still undecided on my answer.  At first I thought the answer was <em>no</em>, because if you think of heritability as (variance explained by genes) / (total variance), then both the numerator and denominator are affected by consanguinity.  Yes, first degree relatives share &#8216;extra&#8217; IBD and so correlate better than they &#8216;should&#8217;, but so does everyone in that dataset.  However, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1413498/">Visscher 2006</a> presents formulas for controlling for parent inbreeding, implying consanguinity does matter.  Leave me a comment if you have the answer.</p>
<p>In talking about consanguinity, my concern is with excess IBD.  But you might also ask whether excess identity-by-state (IBS) matters for heritability calculations.  After all, even if you only look at SNPs that are polymorphic within my ethnic group, I&#8217;m still going to share plenty of alleles with any other random person just by chance. For a C/T SNP with minor allele frequency 50%, there are three possible genotypes CC, CT and TT, and so me and some random person will have a 50% chance (.25^2 + .5^2 + .25^2) of sharing a genotype and a 87.5% chance of sharing at least one allele (1-2*.25^2).  And most SNPs are relatively uncommon, with an average minor allele frequency around 10 &#8211; 15% in many studies, which makes those odds even higher.   Accordingly I&#8217;ll also share way more than half my alleles with my sibling just by chance.  So does that mess up the heritability calculations?  Again, since it affects both the numerator and denominator &#8211; you have extra IBS with your siblings and with random people in the population &#8211; I believe the answer should be no.</p>
<p>However, leaving consanguinity behind now, the fact is that different sibling pairs do share different amounts of IBD, and different unrelated individuals do share different amounts of IBS.  This variability has enabled a couple of very cool modern approaches to calculating heritability.</p>
<p>The first of these is sibling IBD regression.<strong> </strong><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1413498/">Visscher 2006</a> presents an excellent (and perhaps the first with any considerable sample size?) analysis of heritability of height using this approach.  Due to random segregation of parental alleles, siblings don&#8217;t always share <em>exactly</em> 50% of alleles IBD.  The mean is 50%, standard deviation ± 4% &#8211; a fair number of sibling pairs share as little as 40% of alleles or as many as 60%, as shown in Visscher&#8217;s histogram from Figure 1:</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1413498/figure/pgen-0020041-g001/"><img class="alignnone size-full wp-image-1799" title="visscher-2006-fig1-sibling-ibd-histo" src="/wp-content/uploads/2013/02/visscher-2006-fig1-sibling-ibd-histo.jpg"/></a></p>
<p>The fact that some siblings are more similar to each other than other pairs are &#8211; yet we assume they all have an equal degree of shared environment &#8211; gives us a new way to estimate heritability, <em>while controlling for environment</em>, without having twins available.  That&#8217;s pretty cool!  Visscher&#8217;s formulas are under <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1413498/#s4title">Materials and Methods</a>; there is a lot of fancy stuff you&#8217;ll need to know to implement it and estimate standard error, correct for inbreeding if present, etc., but the core concept is just to regress siblings&#8217; genotypic concordance (% shared IBD) against their phenotypic concordance.  This is done as follows (these formulas assume exactly 2 sibs per family):</p>
<ul>
<li>Let Y<sub>i1</sub> be the (quantitative) phenotype of sibling 1 in family i</li>
<li>Let π<sub>i</sub> be the percent IBD between siblings 1 and 2 in family i</li>
<li>Let σ̂<sub>p</sub><sup>2</sup> be the estimate of total phenotypic variance in the population</li>
<li>α and β are parameters to be estimated</li>
<li>ĥ<sup>2</sup> will be your estimate of the additive genetic (narrow sense) heritability</li>
</ul>
<p>Then:</p>
<ul>
<li>Use this formula to estimate β: (Y<sub>i1</sub>-Y<sub>i2</sub>)2 = α + βπ<sub>i</sub></li>
<li>Then plug your estimated β̂ into this formula: ĥ<sup>2</sup> = β̂/(2σ̂<sub>p</sub><sup>2</sup>)</li>
</ul>
<p>The Achilles heel of this approach, as Visscher points out, is that the standard errors are really high.  That&#8217;s because the range of sibling IBD is relatively narrow (not many sib pairs outside the .4 to .6 range).  Visscher&#8217;s simulations suggest that when the true h<sup>2</sup> is 0.8 and the sample size is 2500 sib pairs, the standard error of h<sup>2</sup> is 0.2.  That gives you a pretty wide range, but I&#8217;d point out that different studies give widely differing estimates of heritability anyway, at least for some traits (for instance <a href="http://www.ncbi.nlm.nih.gov/pubmed/18319743">Visscher 2008</a> cites IQ heritability estimates ranging from 0.5 to 0.8).  Estimating heritability, even in the best of circumstances, is not an exact science.  Visscher&#8217;s estimate of height heritability obtained via sibling IBD regression is 0.8, which is consistent with the estimates obtained by other methods.</p>
<p>If you can exploit the variation in IBD among siblings to estimate heritability, why not exploit the variation in IBS among unrelated members of the general population?  Even nominally unrelated individuals will vary in how many alleles they happen to share, and you can measure this using SNP chip genotyping data.  For any trait believed to have complex genetic etiology &#8211; many loci each contributing small effects &#8211; more genotype sharing should mean greater phenotypic concordance.  This, to my understanding, is the principle behind Visscher&#8217;s latest tool, <a href="http://www.complextraitgenomics.com/software/gcta/index.html">GCTA</a> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3014363/">Yang 2011</a>].  It&#8217;s offered as a Unix command line tool that you can run out of the box with <a href="http://pngu.mgh.harvard.edu/~purcell/plink/">PLINK</a> pedigree files haplotyped using <a href="http://www.sph.umich.edu/csg/abecasis/MACH/">MaCH</a>.  I don&#8217;t fully understand all the math yet &#8211; I&#8217;ll post an update if I get my head around it &#8211; but I believe the basic principle is regressing unrelated individuals&#8217; genotypic concordance against phenotypic concordance.  Because unrelated individuals don&#8217;t share a household environment, you again have a sort of &#8216;control&#8217; that lets you begin to separate out the effects of environment vs. genetics.  Admittedly, it gets messy if you consider that some genotypes correlate with some environmental factors, etc. &#8211; e.g. SNPs that predispose you to smoking, which you of course inherited from your parents, mean you&#8217;re more likely to have grown up with second-hand smoke in the house.  A couple of caveats are that (1) this only works well for common variations, since rare variations are less well tagged by SNPs on your SNP chip, and (2) because the level of genotypic concordance among unrelateds is so much smaller and less variable than between siblings, the standard errors are even higher than for sibling IBD regression. So you need huge sample sizes.  Still, this is pretty cool stuff.</p>
<p>But: don&#8217;t be fooled by all this fancy math into thinking that the genetics field is super advanced and sophisticated on precisely calculating heritability.  There are a ton of issues with how to interpret heritability estimates.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/18319743">Visscher 2008</a> does a good job of addressing these.  One important point is that heritability depends on the estimate of phenotypic variance in a particular population at a particular moment in time.  Americans today are both taller and more obese than their ancestors 100 years ago, even though (at a population level, within ethnic groups, and to a first approximation, etc. etc.) their genes haven&#8217;t changed.  We think height is about 80% heritable [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18319743">Visscher 2008</a>], but that&#8217;s just under today&#8217;s conditions &#8211; if you compared height across the whole of human history, you would be adding a ton of additional non-genetic variance, and the proportion explained by genetics &#8211; the heritability &#8211; would accordingly shrink.  So just because a trait is highly heritable doesn&#8217;t mean it&#8217;s genetically deterministic.</p>
<p>A gross estimate of heritability also tells you nothing about the architecture of heritability.  A trait that is 80% heritable could be caused by one locus that explains 80% of variance, or 80 loci that each explain 1% of variance.  So just because a trait is highly heritable doesn&#8217;t mean there will be any individual genetic variants of large effect size.</p>
<p>An additional challenge in interpreting heritability estimates is that economic incentives bias which figures get reported.  For any given trait, there will be a range of different estimates of heritability in the literature &#8211; say 0.5 to 0.8 &#8211; and even within any one study, there will probably be a range of possible estimates depending on the exact methodology chosen.  In general, the highest estimate will be the one that researchers prefer to cite, because high heritability means justification for grant applications to fund GWAS and sequencing projects to identify the genes that drive heritability.  So part of the &#8216;missing heritability&#8217; probably lies in the fact that, for a huge range of human traits, the estimates of heritability that we hear most often are a bit, well, optimistic.</p>
