---
layout: post
title: "Prion protein N-linked glycosylation: review and assessment of therapeutic potential"
author: ericminikel
date: 2013-05-05 18:59:11
---
<p>This post will review what is known about PrP glycosylation and its role (if any) in prion diseases, and will examine whether glycosylation represents a potential therapeutic target for treating these diseases.</p>
<p><strong>what is glycosylation?</strong></p>
<p><a href="http://en.wikipedia.org/wiki/Glycosylation">Glycosylation</a> is the attachment of sugar chains to proteins.  Proteins with sugar chains attached are called glycoproteins, and the sugar chains are called <a href="http://en.wikipedia.org/wiki/Glycan">glycans</a>.  There are a few different ways that a sugar chain can be attached to a protein; of relevance here is <a href="http://en.wikipedia.org/wiki/N-linked_glycosylation">N-linked glycosylation</a>, where the glycan is covalently bound to a nitrogen (N) atom in an asparagine (N) amino acid in the protein.</p>
<p>Glycans attached to proteins can be quite diverse.  This image conveys just a few of the major groups of glycans.  (<a href="http://en.wikipedia.org/wiki/File:Glycosylation.jpg">adapted from Berninsone, P.M.</a>):</p>
<p><img title="n-linked-glycosylation-vertebrates" alt="" src="/wp-content/uploads/2013/05/n-linked-glycosylation-vertebrates.png"/></p>
<p>Glycans can differ on several dimensions: the length, branching structure, and the particular sugar molecules at each position.</p>
<p><strong>brief review of PrP cell biology</strong></p>
<p>To understand PrP&#8217;s glycosylation, it&#8217;s useful to have a quick background on PrP&#8217;s cell biology.  For more details, see my <a title="Cell Biology 04: The Secretory Pathway" href="/2013/02/24/cell-biology-04-the-secretory-pathway/">secretory pathway notes</a> or [<a href="http://www.ncbi.nlm.nih.gov/pubmed/14522850">Harris 2003</a> (<a href="http://bmb.oxfordjournals.org/content/66/1/71.full">ft</a>)].  Here we go in brief.</p>
<p><img alt="" src="https://docs.google.com/drawings/d/135_6ExvL5D5tUyAB1sZXReymBZfBTD1zmYgDyTs7aDM/pub?w=558&amp;h=360"/></p>
<p>Human PrP is 253 amino acids long, but the first 22 are a <a title="an amino acid sequence which causes a protein to get moved to the ER" href="http://en.wikipedia.org/wiki/Signal_peptides">signal peptide</a> which causes the mRNA/ribosome complex to get moved, mid-translation, to the <a href="http://en.wikipedia.org/wiki/Endoplasmic_reticulum">endoplasmic reticulum</a> (this is &#8216;cotranslational translocation&#8217;).   Those 22 amino acids (MANLGCWMLVLFVATWSDLGLC) get cleaved off by <a title="an enzyme that removes signal peptides from ER proteins" href="http://en.wikipedia.org/wiki/Signal_peptidase">signal peptidase</a> and the protein continues to be translated directly into the ER.  While it&#8217;s still being translated, <a href="http://en.wikipedia.org/wiki/Oligosaccharyltransferase">oligosaccharyl transferase</a> (OST) adds glycan chains at 0, 1 or 2 of 2 possible sites.  In general, OST recognizes two amino acid motifs &#8211; NXS and NXT &#8211; as its sites for attaching glycans.  In human PrP these occur at codon 181-183 (NIT) and 197-199 (NFT).  But OST&#8217;s hit rate is not 100%.  Any given molecule of PrP might end up with glycan chains attached at codon 181, codon 197, both, or neither.</p>
<p>The C-terminus of PrP is a 23 amino acid sequence (SMVLFSSPPVILLISFLIFLIVG) which signals for the addition of a GPI anchor.  In this process, called <a href="http://en.wikipedia.org/wiki/Glypiation">glypiation</a>, these 23 amino acids are cleaved off and replaced with <a href="http://en.wikipedia.org/wiki/Glycophosphatidylinositol">GPI</a>, which is itself another kind of sugar chain, which embeds in the membrane, thus anchoring PrP to the ER membrane.  Depending who you ask, glypiation <a href="http://en.wikipedia.org/wiki/Glycosylation#Formation_of_GPI_anchors_.28glypiation.29">may also be called</a> &#8217;glycosylation&#8217;, but is not the subject of this post.  This post is about the N-linked glycosylation at codons 181 and 197.  Once the signal peptide and GPI signal have both been cleaved, the initial 253 amino acid protein is just 208 amino acids.</p>
<p>From its beginnings in the ER, PrP transits through the secretory pathway, acquiring a disulfide bond between its two cysteines (C180 and C214) and undergoing several modifications to its N-linked glycan chains.  Secretory pathway proteins with a DXE signal, called a &#8216;di-acidic&#8217; signal since D is aspartic acid and E is glutamic acid, get <a title="they travel in vesicles from the Golgi to fuse with the cell's membrane" href="http://en.wikipedia.org/wiki/Exocytosis">exocytosed</a>.  PrP has a DYE at codons 144-146, and thus travels from the Golgi to the cell surface in an exocytic vesicle.  Once there it remains GPI-anchored to the exoplasmic (outside) surface of the plasma membrane.</p>
<p><strong>how people study PrP glycosylation</strong></p>
<p>Any two molecules of PrP could differ from one another in glycosylation state, in two different ways.</p>
<p>First, as mentioned above, PrP can be glycosylated at N181, N197, both, or neither.  That&#8217;s <em>four</em> different states.  But the way that these can be studied in the lab is by separating them by molecular mass in <a href="http://en.wikipedia.org/wiki/Gel_electrophoresis">gel electrophoresis</a>.  When this is done, you only see <em>three</em> bands: diglycosylated, monoglycosylated, and unglycosylated.  Among the monoglycosylated species of PrP, it&#8217;s not possible to distinguish the N181- from the N197-glycosylated molecules.</p>
<p><em>Below: Gel electrophoresis from <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2464735/">Levavasseur 2008</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2464735/figure/pone-0002786-g001/">Fig 1</a>. PrP (in this case, PrP-res) separates as three bands corresponding to di-, mono- and unglycosylated PrP.</em></p>
<p><a href="/wp-content/uploads/2013/05/levavasseur-2008-fig1.png"><img title="" alt="" src="/wp-content/uploads/2013/05/levavasseur-2008-fig1.png"/></a></p>
<p>Second, even if two molecules of PrP are both, say, diglycosylated, they might still differ in the types of glycan chains attached (again: length, branching structure, types of sugar molecules included).  This is hard to study too.  The enzyme <a href="https://www.neb.com/products/p0704-pngase-f">PNGase F</a> can be used to cleave the glycan chains off of PrP, and then you can identify the exact glycan chains themselves using mass spectrometry.  But you won&#8217;t know whether each glycan chain you see came from N181 or N197 <strong>*See <a href="/2013/05/05/prion-protein-n-linked-glycosylation-review-and-assessment-of-therapeutic-potential/#comment-367">point 3 in Dr. Kukuskhkin&#8217;s comment</a></strong>.  Or you can do a two-dimensional gel electrophoresis, where you leave the glycans atatched to PrP and then separate PrP molecules by size along one axis (thus separating di-, mono- and unglycosylated, and probably also separating out different cleavage products) and by <a href="http://en.wikipedia.org/wiki/Isoelectric_point">isoelectric point</a> on the other axis.   Isoelectric point is the pH at which a molecule is neutrally charged, and this differs for many glycan chains, though unlike mass spec, you can&#8217;t precisely identify every glycan chain.</p>
<p><em>Below: 2D gel electrophoresis from <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3184161/">Hopf 2011</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3184161/figure/pone-0025722-g004/">Fig 4</a>.  Proteins (in this image, a bacterial protein, not PrP) are separated by molecular weight on the y axis and isoelectric point on the x axis.</em></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3184161/figure/pone-0025722-g004/"><img title="" alt="" src="/wp-content/uploads/2013/05/hopf-2011-fig4.png"/></a></p>
<p>From my reading of the literature, the term &#8216;glycoform&#8217; seems to be used loosely to refer to either the di/mono/un-glycosylated distinction or to the which-glycan-chain distinction, or to both.  I&#8217;ll use it loosely here too.</p>
<p><strong>is glycosylation how prion strain information is encoded? (answer: no)</strong></p>
<p>In humans and in other animals, there are different strains of prions, distinguishable pathologically by the specific disease symptoms, incubation times and the brain regions most heavily affected, and distinguishable biochemically by the size of cleavage products and, as we&#8217;ll see shortly, the ratio of di-, mono- and un-glycosylated species.  These properties are maintained over serial transmission between animals, in the absence of any amino acid sequence difference.</p>
<p>Ever since the notion of infectious proteins was proposed [<a href="http://www.ncbi.nlm.nih.gov/pubmed/4964084">Griffith 1967</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/6801762">Prusiner 1982</a>], a central mystery has been how proteins could encode different strains.  Before the advent of the prion hypothesis, all of the infectious agents that people were used to seeing had nucleic acids, and so differences in DNA or RNA sequence could code for different strains.  You might ask: could different strains simply be different glycoforms of PrP?</p>
<p>A considerable body of evidence says the answer is no.  The first study to show that glycosylation is not necessary for strain information considered two strains of hamster prions, named hyper (HY) and drowsy (DY) for the different behavioral phenotypes observed in infected hamsters [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7791905">Bessen 1995</a>].  Bessen discovered that even when PrP<sup>Sc</sup> from each strain was first amplified in a cell-free conversion assay [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7913989">Kocisko 1994</a>] using purely <em>un</em>glycosylated PrP as a substrate, strain properties were still preserved on transmission to a new set of hamsters.  The finding that unglycosylated PrP can preserve strain information has since been replicated in mice with the RML and 301C prion strains [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19297485">Piro 2009</a>].  <del>No one has yet been able to induce <em>spontaneous </em>PrP<sup>C</sup> &gt; PrP<sup>Sc</sup> conversion in a cell-free system with wild-type PrP &#8211; though several efforts have come very close [<a href="http://www.ncbi.nlm.nih.gov/pubmed/15286374">Legname 2004</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/15851027/">Castilla 2005</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/20660771">Edgeworth 2010</a>, reviewed in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2807694/">Benetti &amp; Legname 2009</a>].</del>  <strong>update 2013-06-08: we learned at <a href="http://www.prionalliance.org/2013/06/02/highlights-of-prion2013/">Prion2013</a> that <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2893558/">Wang 2010</a> is now accepted as the first successful creation of true <em>de novo</em> prions in a test tube.  </strong><del>But</del> we do <del>now</del> also know from studies of yeast prions that conformation alone can be sufficient to encode strain information [<a href="http://www.ncbi.nlm.nih.gov/pubmed/15029196">Tanaka 2004</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16810177">Tanaka 2006</a>].</p>
<p>All that is pretty convincing evidence that glycosylation is not<em> </em><em>necessary</em> for the existence of prion strains.  That doesn&#8217;t by definition rule out the possibility that glycosylation might still <em>contribute</em> to <em>certain</em> strains, and one recent study has sought to show that passage of prion strains through mice expressing glycosylation mutants does indeed change the strain properties [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23395905">Cancellotti 2013</a>].  A challenge in interpreting the results of this (and many other <em>in vivo</em> studies, as we&#8217;ll see below) is that in order to abolish one or both of PrP&#8217;s glycosylation sites, the investigators had to mutate the NXT sequences, thus confounding the effects of glycosylation changes with the effects of amino acid sequence changes.</p>
<p>As an interesting aside, it has recently been shown strain properties can be changed depending upon the cofactors available for prion conversion.  Using a version of the protein misfolding cyclic amplification (PMCA) assay [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11459061">Saborio 2001</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/15851027/">Castilla 2005</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/20377181/">Deleault 2010</a>], it was shown that the presence or absence of different cofactors can affect the faithfulness of prion strain propagation &#8211; specifically, when <a href="http://en.wikipedia.org/wiki/Phosphatidylethanolamine">phosphatidylethanolamine</a> was the <em>only </em>cofactor available, three different prion strains converged into one [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22711839/">Deleault 2012</a>].  This is still consistent with the idea that PrP<sup>Sc</sup> strain information is (or at least can be) encoded in conformation alone, but that cofactors are necessary for the templating of that strain information onto new PrP<sup>C</sup> molecules.  We may eventually learn that glycosylation, similarly, is necessary for the propagation of certain strains, but this hasn&#8217;t been shown convincingly yet.  But whether or not it&#8217;s strictly <em>necessary</em>, there is good evidence that glycosylation does affect the <em>efficiency</em> of prion conversion for various strains, as we&#8217;ll see later in this post.</p>
<p><strong>different strains have different glycosylation patterns</strong></p>
<p>Although glycosylation does not appear to be necessary to encode strain information, strains do have clearly different glycosylation patterns.</p>
<p>In humans, different strains of sporadic and acquired Creutzfeldt-Jakob Disease exhibit different relative proportions of un-, mono-, and di-glycosylated species of PrP, and these differences are stable over passage through mice [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8878476">Collinge 1996</a>].  Similarly reproducible differences in the extent of glycosylation were also observed for several rodent strains of PrP<sup>Sc</sup> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9121579">Somerville 1997</a>].  In that latter study, a limited amount of within-strain variation in glycosylation was still observed, and available experimental techniques were not sufficiently precise to uniquely identify <em>all</em> strains based <em>solely</em> on glycosylation state [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9121579">Somerville 1997</a>].   Still, it was clear that strains have characteristic glycoform ratios.  Indeed, the identical glycoform ratio observed in human variant Creutzfeldt-Jakob Disease (vCJD) and bovine spongiform encephalopathy (BSE) upon transmission of each to mice was used as one of several lines of evidence demonstrating that these two strains are one and the same, and that the vCJD epidemic in the U.K. arose from the transmission of BSE or ‘mad cow disease’ to humans [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9333232">Hill 1997</a>].</p>
<p>In an even more extreme example of characteristic glycoform patterns, it has recently been shown that two human prion diseases &#8211; familial Creutzfeldt-Jakob Disease caused by the V180I mutation, and a newly characterized sporadic prion disease dubbed <a href="/2012/10/06/vpspr-another-strain-of-sporadic-prion-disease/">variably protease-sensitive prionopathy</a> (VPSPr) [<a href="http://www.ncbi.nlm.nih.gov/pubmed/20695009/">Zou 2010</a>] &#8211; contain <em>no</em> PrP<sup>Sc</sup> molecules glycosylated at N181, and therefore no diglycosylated PrP<sup>Sc</sup> molecules at all [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23527023">Xiao 2013</a>].  This feature had previously been observed in a familial form of Creutzfeldt-Jakob Disease caused by the T183A mutation, but in that case, the amino acid substitution abolishes the N181 glycosylation site, which requires an NXS or NXT motif.  In contrast, VPSPr is a sporadic disease not associated with any amino acid substitution, and cells expressing V180I PrP <em>do</em> produce PrP<sup>C</sup> glycosylated at N181.  Therefore it appears that PrP glycosylated at N181 does exist in both of these diseases, but simply cannot be converted to the particular strain of PrP<sup>Sc</sup> that is present.</p>
<p>PrP<sup>Sc</sup> of different strains may differ not only on the di/mono/unglycosylated dimension, but also in the nature of glycan chains attached.  One study has shown a difference in glycan chains between PrP<sup>Sc</sup> in two different human prion diseases: sporadic fatal insomnia (sFI) and the M/M2 subtype of sporadic Creutzfeldt-Jakob Disease (sCJD) [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11489910">Pan 2001</a>].  These diseases exhibit different phenotypes in the absence of any amino acid substitutions in PrP, and all subjects examined (n=2 for each disease, i.e. n=4 total) were homozygous MM at codon 129.  The relative abundance of di-, mono- and un-glycosylated forms was indistinguishable between the two diseases, but sCJD appeared to contain a greater diversity of glycan chains than sFI.  2D gel electrophoresis revealed several dots (corresponding to different isoelectric points) present only in sCJD and not in sFI.  These disappeared when the PrP<sup>Sc</sup> was deglycosylated prior to electrophoresis, confirming that it was the glycan chains that caused the different isoelectric points.  And no such differences were observed in PrP<sup>C</sup> isolates, suggesting the observed differences were the product of prion infection and not just differences between the patients&#8217; original sets of glycoforms.</p>
<p><strong>the preferential conversion hypothesis or &#8216;selection model&#8217;</strong></p>
<p>The finding that glycosylation states differ between strains led Collinge to propose two hypotheses: (1) that each PrP<sup>Sc</sup> conformation may have an easier time converting certain glycoforms of PrP<sup>C</sup> rather than others, and (2) that the selective vulnerability of different brain regions to different prion strains may therefore owe in whole or part to the local abundance of compatible glycosylation species [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8878476">Collinge 1996</a>].</p>
<p>Hypothesis #1, which I will call the preferential conversion hypothesis and which some authors have called the &#8216;selection model&#8217;, would imply that a given strain (defined by its conformation alone) might efficiently convert, say, monoglycosylated PrP but inefficiently convert diglycosylated PrP.  (Or the efficiency might vary by the type of glycan chains attached).  This might happen because the glycan chains stabilize a particular structure, or get in the way of conversion, or get in the way of a particular folding conformation, thus raising the energy barrier that has to be crossed to convert from PrP<sup>C</sup> to a particular conformation of PrP<sup>Sc</sup>.</p>
<p>If this is true, then PrP<sup>Sc</sup> glycoforms should differ from PrP<sup>C</sup> glycoforms in the same brain. <a href="http://www.ncbi.nlm.nih.gov/pubmed/11489910">Pan 2001</a>&#8216;s and <a href="http://www.ncbi.nlm.nih.gov/pubmed/23527023">Xiao 2013</a>&#8216;s results both seem to support this notion, for glycan chain types and for di/mono/unglycosylated PrP molecules respectively.  But not all investigators have agreed.  Mass spectrometry of glycans isolated from uninfected and infected Syrian hamster brains revealed 52 different glycan chains attached to PrP, every one of which was present in both PrP<sup>C</sup> and PrP<sup>Sc</sup> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10557270">Rudd 1999</a>].  The relative abundance of some glycans differed, with PrP<sup>Sc</sup> enriched for tri- and tetra-antennary and depleted for bi-antennary structures relative to PrP<sup>C</sup>, but this was speculatively attributed to a global disruption of GnTIII enzyme activity in the course of prion-mediated neurodegeneration and thus considered a consequence of PrP<sup>Sc</sup> infection rather than an intrinsic property of PrP<sup>Sc</sup>.</p>
<p>Perhaps a more testable implication of the preferential conversion model is that prion strains should have trouble infecting mice expressing incompatible glycoforms of PrP.  For instance, a strain with a &#8216;preference&#8217; for diglycosylated PrP should cause mild or no disease in mice that don&#8217;t express diglycosylated PrP.  As mentioned above, the troubling confounder here is that in order to abolish glycosylation at one of PrP&#8217;s N-linked glycosylation sites, you have to mutate either the N or the T in the NXT signal.</p>
<p>That amino acid change matters a lot, because prion propagation efficiency depends on compatibility between the PrP<sup>Sc</sup> amino acid sequence and the target PrP<sup>C</sup> amino acid sequence [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pubmed/17991853">Collinge &amp; Clarke 2007</a>].  Prion strains have different &#8216;species barriers&#8217; which tend to go away after the strain is passaged in the new recipient species or when the recipient animal expresses a transgene of the original species&#8217; PrP sequence [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1977523">Prusiner 1990</a>] and humans with homozygous codon 129 genotypes are much more vulnerable to sporadic CJD [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1677164">Palmer 1991</a>] as well as vCJD and have earlier onset in some genetic prion diseases [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3226040/">Lukic &amp; Mead 2011</a>].  So it&#8217;s to be expected that if you mutate an N or a T in mice to some other amino acid, that might affect vulnerability to prion strains regardless of the resulting glycosylation changes.</p>
<p>Keep that in mind while interpreting the results from mouse studies.  <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2292751/">Tuzi 2008</a> created mice with the Ns mutated to Ts in the first, second or both glycosylation sites of PrP (called G1, G2 and G3 mice respectively).  The resulting mice were then infected with 79A prions (a low glycosylation strain) or ME7 prions (a medium glycosylation strain).  The incubation times of the different strains differed dramatically among the different mouse mutants and versus wild-type mice.  Perhaps most convincingly, the G3 mice (with entirely unglycosylated PrP) were capable of sustaining a 79A infection, but they got sick very late in life if ever (only 4 of 21 mice died of prion disease), and they appeared wholly incapable of sustaining an ME7 infection.  The G1 mice also could not support ME7 infection, suggesting the first glycosylation site was indispensible for ME7 infection.</p>
<p>Tuzi&#8217;s results could all be explained by amino acid sequence changes, or could be explained by glycosylation.  Indeed, the story is quite plausible if you think of it in terms of glycosylation: 79A, the &#8216;low glycosylation&#8217; strain, could infect any of the mutant mice (though with extended incubation times), while ME7, the &#8216;medium glycosylation&#8217; strain, seemed to require glycosylation at the first site but could do without glycosylation at the second.</p>
<p>The same problem &#8211; is it amino acid sequence or glcyosylation that matters &#8211; affects interpretation of similar work in cell culture [<a href="http://www.ncbi.nlm.nih.gov/pubmed/21248032">Salamat 2011</a>] and of the later work showing changes in strain properties after passage through Tuzi&#8217;s mice [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23395905">Cancellotti 2013</a>].  Of note, Salamat discovered that totally unglycosylated mutants of PrP are not properly trafficked to the cell surface, which could account for some of the extended incubation times that Tuzi observed, though it still cannot account for the differences in those effects between the 79A and ME7 prions.</p>
<p>To my knowledge, no study has yet really convincingly separated the effects of glycosylation from amino acid sequence in these mutant PrP models.  Therefore these studies cannot be taken as <em>proof</em> of the preferential conversion hypothesis, though overall they do seem to support it.</p>
<p>Rather, the most convincing evidence for the preferential conversion model comes from the fact that prion strains maintain their own characteristic glycoform ratios even in cell-free conversion assays, provided that all the cofactors and a diversity of PrP<sup>C</sup> glycoforms are available as substrates [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2567411/">Castilla 2008</a>].  In Castilla&#8217;s <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2567411/figure/f1/">Fig 1</a> you can see that the respective glycoform ratios of four different prion strains are virtually identical before and after PMCA &#8211; and consistently different from each other.</p>
<p>Following Collinge&#8217;s original hypotheses, Somerville had proposed two other models to explain the glycoform differences of prion strains, suggesting that strains might differentially affect the glycosylation of PrP at the point of synthesis (similar, but not identical, to <a href="http://www.ncbi.nlm.nih.gov/pubmed/10557270">Rudd 1999</a>&#8216;s suggestion that enzyme disruption in the disease process led to differences in PrP<sup>Sc</sup> vs. PrP<sup>C</sup> glycosylation) or that different glycoforms might be differentially deglycosylated by other enzymes during infection by different strains [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10423157">Somerville 1999</a>].  Neither synthesis nor degradation is occuring in the PMCA assay, so preferential conversion seems to be the only model to explain the maintenance of glycoform ratios.  In interpreting his own data, Castilla concludes that the selection model must be correct [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2567411/">Castilla 2008</a>].</p>
<p><strong>selective vulnerability of different brain regions</strong></p>
<p>If strains do exhibit a &#8216;preference&#8217; for converting different glycoforms of PrP, that could go a long ways towards explaining why strains preferentially affect different brain regions, a phenomenon called strain-specific <em>neurotropism</em>.</p>
<p>Total PrP<sup>C</sup> levels differ across the brain, being highest in the thalamus [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10499442">DeArmond 1999</a>].  But that&#8217;s true regardless of what strain an animal is infected with, so differences in host PrP expression could explain at most <em>one </em>strain&#8217;s neurotropism (brain region preference).  Some other brain region difference is needed to explain the neurotropic differences between strains.  And indeed, PrP<sup>C</sup> glycosylation <em>does</em> differ across brain regions.  2D gel electrophoresis has been used to separate PrP<sup>C</sup> molecules both by size and by isoelectric point, showing differences in the glycan chains of diglycosylated PrP<sup>C</sup> across several different mouse brain regions [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10499442">DeArmond 1997</a>].   Later studies using gel electrophoresis or antibodies with differential affinity for di-, mono- or un-glycosylated species of PrP were also able to show differences in the prevalence of these species across different brain tissues both in mice [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10423157">Somerville 1999</a> (<a href="http://vir.sgmjournals.org/content/80/7/1865.full.pdf">ft</a>), <a href="http://www.ncbi.nlm.nih.gov/pubmed/12821516">Beringue 2003</a>] and in humans [<a href="http://www.ncbi.nlm.nih.gov/pubmed/17466020">Kuczius 2007</a>]. All of these findings for PrP specifically are consistent with the well-established knowledge that, for glycoproteins generally, glycosylation patterns are tissue-specific [<a href="http://www.ncbi.nlm.nih.gov/pubmed/3052290">Rademacher 1988</a>].</p>
<p>Together these results suggest sufficient heterogeneity in PrP glycoforms across different brain regions to potentially account, at least in part, for selective vulnerability.</p>
<p>But to my knowledge no one has really been able to show this experimentally.  To do so would seem to require somehow changing the glycoform ratios (and <em>not</em> changing anything else) in different brain regions and then showing that this alters which brain region is affected by which strain.  That&#8217;s a tall order, especially when we don&#8217;t have the tools to even convincingly separate glycosylation changes from amino acid changes in the <em>whole</em> organism. <strong>*See <a href="/2013/05/05/prion-protein-n-linked-glycosylation-review-and-assessment-of-therapeutic-potential/#comment-367">point 5 below</a>.</strong></p>
<p>On the other hand, in a quick thought experiment, I find it hard to talk myself into the idea that glycosylation doesn&#8217;t matter for neurotropism.   As discussed above, it&#8217;s clear that the relative ratios of glycoforms of PrP<sup>Sc</sup> collected from brains of humans or other animals do differ between strains, sometimes dramatically so.  That&#8217;s consistent with the idea that strains have preferences for certain glycoforms, and that those preferences drive them to accumulate in different brain regions where their preferred glycoforms are abundant.  One alternate interpretation of this data would be that different strains affect different brain regions for some other, unobserved, non-glycosylation-related reason, and that then <em>because</em> they are active in different brain regions, they end up converting the glycoforms that are locally available.  That alternate interpretation is hard to reconcile with the existence of glycosylation preferences <em>in vitro</em> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2567411/">Castilla 2008</a>].</p>
<p>But that&#8217;s not proof, and in any event, glycosylation could contribute to strain-specific neurotropism without explaining all of it.  For instance, since it&#8217;s clear that cofactors matter too [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3396481/">Deleault 2012</a>], the availability of different cofactors across brain regions could also contribute to explaining it.</p>
<p>All this is assuming an implicit causal link: that the amount of PrP<sup>Sc</sup> produced in each brain region determines the extent of pathology there.  The couple of studies I was able to find do suggest that PrP<sup>Sc</sup> levels and pathology are pretty strongly correlated across brain regions, both in CJD [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8651649">Parchi 1996</a>] and FFI [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9222180">Cortelli 1997</a>].  Interpretation of this is complicated by the fact that when we say &#8220;PrP<sup>Sc</sup>&#8220;, what is usually being measured is actually proteinase K resistant PrP (PrP-res), or PrP plaque staining, both of which appear to be just proxies for disease and not direct measures of what is toxic, and FFI doesn&#8217;t produce much PrP-res anyway [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19709627">Jackson 2009</a>].</p>
<p><strong>summary of the literature</strong></p>
<p>Based on my above read-through of the literature, the following points seem very well-supported by available evidence:</p>
<ol>
<li>Glycosylation is not necessary for encoding strain information in prions.</li>
<li>Prion strains exhibit reproducible glycoform ratios which are faithfully maintained during propagation both <em>in vitro</em> and <em>in vivo</em>.</li>
<li>Prion strains have different &#8216;preferences&#8217; for converting different PrP glycoforms.</li>
<li>Ratios of PrP<sup>C</sup> glycoforms differ across brain regions.</li>
</ol>
<p>All this suggests a role for glycobiology in explaining strain-specific neurotropism, but this has never been experimentally demonstrated.</p>
<p><strong>is glycosylation a potential therapeutic target in prion disease?</strong></p>
<p>At first glance, based on the above evidence, it would seem that targeting glycosylation <em>could</em> be a therapeutic strategy, but not an ideal one.  For one, since glycosylation &#8216;preferences&#8217; differ between prion strains, glycosylation-based strategies seem likely to have strain-specific efficacy, which would limit their usefulness.  Second, with a few exceptions (VPSPr, V180I CJD [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23527023">Xiao 2013</a>] and ME7 in G3 mice [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2292751/">Tuzi 2008</a>]) the glycoform &#8216;preferences&#8217; of prion strains do not appear to be absolute.  That is, a strain may &#8216;prefer&#8217; to convert diglycosylated PrP but still <em>can </em>convert un- or monoglycosylated PrP.  That suggests that changing the glycosylation state of PrP<sup>C</sup> might delay prion disease a bit, but not forever.</p>
<p>Two studies that examined prion replication in the presence of compounds that alter glycosylation seem to support these conjectures [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3220511/">Browning 2011</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3567182/">Oelschlegel &amp; Weissmann 2013</a>].  These studies were not exactly therapeutic-oriented, but rather, used glycosylation-altering compounds as tools to probe the hypothesis that each prion strain is actually a quasi-species of several substrains, which can be selected for or against by different drugs.  In addition to their biochemical and phenotypic properties, prion strains can now be distinguished by their relative ability to infect different cell lines, and their relative resistance to different strain-specific drugs such as <a title="Curcumin as prion therapeutic" href="/2012/11/24/curcumin-as-prion-therapeutic/">curcumin</a> and <a title="cpd-B as a prion therapeutic" href="/2012/09/25/cpd-b-as-a-prion-therapeutic/">cpd-B</a>, in a &#8216;cell panel assay&#8217; [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2409240/">Mahal 2007</a>].  Scrapie-infected cells were treated with a variety of glycosylation-modifying drugs and other small molecules, including kifunensine (kifu), <a href="http://en.wikipedia.org/wiki/Castanospermine">castanospermine</a> (CST) and swainsonine (swa).  In support of the &#8216;quasi-species&#8217; or &#8216;many substrains&#8217; hypothesis, some strains of prions were initially sensitive to some drugs, but could develop drug resistance or drug &#8216;dependence&#8217; (increased efficiency of propagation in presence of the drug) after being cultured in the presence of the drug [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3220511/">Browning 2011</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3567182/">Oelschlegel &amp; Weissmann 2013</a>].</p>
<p>The strain specificity of these compounds, and the related fact that they may merely select for resistant substrains like <a title="Quinacrine" href="/2012/10/23/quinacrine/">quinacrine</a> does [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2777304/">Ghaemmagami &amp; Ahn 2009</a>], indeed seems to suggest they wouldn&#8217;t be ideal therapeutics, though these studies did not examine <em>in vivo</em> efficacy.</p>
<p>But another property of glycosylation-altering drugs is their ability to in some cases increase the degradation of glycoproteins, reducing the total amount of the protein produced.  That could fit into the therapeutic strategy of <a title="The case for depleting PrP as a therapeutic strategy against prion diseases" href="/2013/04/08/the-case-for-depleting-prp-as-a-therapeutic-strategy-against-prion-diseases/">depleting PrP</a>, a possibility which I&#8217;ll consider briefly here.</p>
<p>My interest in learning about PrP glycosylation and writing this post was triggered by a conversation I had with glycobiologist <a href="http://hms-harvard.academia.edu/NikolayKukushkin">Nikolay Kukushkin</a>.  He pointed out that <a href="http://en.wikipedia.org/wiki/Alpha-glucosidase_inhibitor">alpha glucosidase inhibitors</a>, a class which includes several approved drugs (and the experimental compound CST), can not only alter glycosylation but also reduce the total amount of some glycoproteins produced.  As one example we discussed a drug called <a href="http://pubchem.ncbi.nlm.nih.gov/summary/summary.cgi?sid=24892070">n-butyl-deoxynorijimycin</a> (NB-DNJ; commonly known as <a href="http://en.wikipedia.org/wiki/Miglustat">Miglustat</a> and commercially known as <a href="http://www.zavesca.com/">Zavesca</a>; shown below), which is prescribed for <a title="Gaucher disease" href="http://en.wikipedia.org/wiki/Gaucher_disease">Gaucher disease</a> and <a href="http://en.wikipedia.org/wiki/Niemann%E2%80%93Pick_disease,_type_C">Niemann-Pick Type C</a>.</p>
<p><a href="http://pubchem.ncbi.nlm.nih.gov/summary/summary.cgi?sid=24892070"><img title="n-butyl-deoxynorijimycin" alt="" src="/wp-content/uploads/2013/05/n-butyl-deoxynorijimycin.png"/></a></p>
<p>This drug&#8217;s mechanism of action is fascinating.  Its therapeutic value in the storage diseases Gaucher and Niemann-Pick is due to its inhibition of <a href="https://en.wikipedia.org/wiki/Ceramide_glucosyltransferase">ceramide glucosyltranferase</a>, thus reducing the amount of certain glycolipids produced and lessening the overall burden of storage material.  This approach is called &#8216;substrate reduction therapy&#8217;.  However, many of the glycan-processing enzymes have similar enough binding sites that one drug may inhibit several of them.  It so happens <strong>[sic, see <a href="/2013/05/05/prion-protein-n-linked-glycosylation-review-and-assessment-of-therapeutic-potential/#comment-367">point 6</a>]</strong> that NB-DNJ also inhibits alpha glucosidases [see for instance <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC189444/">Fischer 1995</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/15234963">Tian 2004</a>], enzymes which normally trim the glycan chains of a freshly synthesized, immature glycoprotein in the ER, allowing that glycoprotein to interact with folding chaperones <a href="http://en.wikipedia.org/wiki/Calnexin">calnexin</a>/<a href="http://en.wikipedia.org/wiki/Calreticulin">calreticulin</a>.  The protein will fold in the ER with their help, and then later in the Golgi, it will have its glycan chains further extended and modified by other enzymes.  When glycans are <em>not</em> trimmed (due to NB-DNJ), the glycoprotein cannot interact with calnexin/calreticulin, and remains in its initial unfolded state with immature glycan chains attached.  That state doesn&#8217;t last forever: our cells have redundant mechanisms for trimming the glycan chains later in the Golgi, and so eventually the glycan chains will be extended and modified like always.  But this redundancy isn&#8217;t perfect, so NB-DNJ does result in changes to the final glycosylation state of some proteins, and more importantly, the added time that the glycoprotein spends in an immature and unfolded state in the ER gives it more chances to be degraded (via <a title="ER-associated degradation" href="http://en.wikipedia.org/wiki/Endoplasmic-reticulum-associated_protein_degradation">ERAD</a>) before it ever passages through the Golgi.  This means that the total amount of some glycoproteins is reduced by NB-DNJ.</p>
<p>NB-DNJ&#8217;s inhibition of alpha glucosidases leads to totally non-specific downstream effects &#8211; it affects <em>all</em> glycoproteins &#8211; and raises the interesting question of how it would affect PrP.  I did PubMed and Google Scholar searches for &#8216;prion n-butyl-deoxynorijimycin&#8217; and similar search terms and could not find any studies that have examined this question. NB-DNJ/Miglustat is also not included in MSDI&#8217;s <a href="http://www.msdiscovery.com/spectrum.html">Spectrum Collection</a> recently used in <a href="/2012/10/06/screen-reveals-antiprion-compounds-that-act-indirectly/">the Ghaemmagami lab&#8217;s ELISA assay for PrP-res inhibitors</a> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3149362/">Poncet-Montagne 2011</a>] nor in MSDI&#8217;s <a href="http://www.msdiscovery.com/usdrugs.html">US Drug Collection</a> used in <a href="/2013/04/04/tacrolimus-astemizole-and-a-new-screening-approach-for-drugs-that-reduce-prp/">the Lasmezas lab&#8217;s FRET assay for PrP reducers</a> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23576755">Karapetyan &amp; Sferrazza 2013</a>], so there doesn&#8217;t seem to be any available high-throughput screening evidence on its effects either.</p>
<p>We can make a reasonable conjecture that any glycosylation-altering effects of NB-DNJ would result in strain-specific inhibitory effects, as seen for CST [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3220511/">Browning 2011</a>].  But the experiments in these studies were run on cell lines expressing wild-type PrP.  It is interesting to consider the possibility that in mutant cell lines, alpha glucosidase inhibitors might increase the degradation of the mutant protein.</p>
<p>That bit of speculation is based on a study looked at the metabolism and secretion of PrP in cell lines transfected with DNA expressing either of the D178N mutants: D178N cis 129M FFI and D178N cis 129V CJD [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8647879">Petersen 1996</a>].  In both cases, Petersen found that the unglycosylated form of the mutant (but not wild-type) PrP was almost completely degraded before reaching the cell surface.  At the time of synthesis in the ER, unglycosylated PrP appeared to account for ~20% of total PrP in both the mutant and wild-type varieties, but by the time it reached the cell surface, unglycosylated D178N 129M PrP was undetectable, and unglycosylated D178N 129V PrP had dropped to 0.6% of total PrP. Petersen then looked at postmortem brains of FFI patients, which confirmed that the same phenomenon is present <em>in vivo</em>, albeit to a somewhat lesser extent than in cell culture.  Unglycosylated mutant PrP was under-represented compared to unglycosylated PrP from the patient&#8217;s wild-type allele, suggesting that it might be preferentially degraded.</p>
<p>Petersen&#8217;s results can be taken to suggest that in the absence of glycan chains, the mutant PrP misfolds in some way that triggers a cellular response to degrade it. <strong>*Or not &#8211; see <a href="/2013/05/05/prion-protein-n-linked-glycosylation-review-and-assessment-of-therapeutic-potential/#comment-367">point 7</a>.</strong>  Could a similar effect be achieved through alpha glucosidase inhibitors, promoting early degradation in the ER?  It&#8217;s a risky proposition: by promoting misfolding of some sort, the drugs might accelerate prion disease.  Indeed, <a href="http://en.wikipedia.org/wiki/Tunicamycin">tunicamycin</a>, a toxic, non-drug experimental compound which blocks glycosylation altogether is used as a positive control for ER stress [for instance in <a href="http://www.ncbi.nlm.nih.gov/pubmed/22622579">Moreno 2012</a>] because it causes such a dramatic accumulation of misfolded proteins.  It&#8217;s also not clear if ER-associated degradation could be achieved: Petersen found evidence that the degradation of unglycosylated mutant PrP in some late endosomal/lysosomal compartment, rather than ERAD.  More recent work has agreed that mutant PrP degradation appears to take place in acidic compartments past the Golgi, rather than in the ER [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2691595/">Ashok &amp; Hedge 2009</a>].</p>
<p>To my knowledge, no research has yet examined whether alpha glucosidase inhibitors can promote early degradation of mutant PrP before it reaches the cell surface.  A first step would be to test the effects of alpha glucosidase inhibitors in cell lines expressing mutant PrP and check for reduced expression of the mutant allele.  If expression indeed appeared to be reduced, effects <em>in vivo</em> could be determined in knock-in mice.  If it happened to have therapeutic value, that would be exciting -NB-DNJ has good blood-brain barrier penetrance [<a href="http://www.ncbi.nlm.nih.gov/pubmed/17689147">Patterson 2007</a>] and is an approved drug with an apparently decent safety record [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2504062/">Ficicioglu 2008</a>].</p>
<p><strong>conclusions</strong></p>
<p>N-linked glycosylation turns out to be a fascinating part of prion biology.  Different strains of prions exhibit different characteristic glycoform patterns, a phenomenon which has now been shown, fairly conclusively, to be due to the &#8216;preference&#8217; of different strains for converting certain glycoforms of PrP<sup>C</sup>.  This &#8216;preference&#8217; might even contribute to explaining why strains of prions affect different brain regions more dramatically.</p>
<p>Glycosylation is not an ideal therapeutic target in prion diseases, because its effects would be likely to be strain-dependent, and because for the most part the &#8216;preferences&#8217; of different strains are not absolute, so even at best, altering glycosylation could only slow, and not stop, prion disease.  Still, it would be interesting to know whether alpha glucosidase inhibitors, a class of drugs which can alter glycosylation and/or increase degradation of some proteins, could reduce expression of mutant PrP.  These have apparently never been tested for their effects on genetic mutants of PrP.</p>
