---
layout: post
title: "Cell Biology 05: The Cytoskeleton Part I: Actin"
tag: bios-e-16
author: ericminikel
date: 2013-03-02 23:02:44
---
<p>These are notes from lecture 5 of <a href="http://www.extension.harvard.edu/courses/22958">Harvard Extension’s Cell Biology course</a>.</p>
<p>The <a href="http://en.wikipedia.org/wiki/Cytoskeleton"><strong>cytoskeleton</strong></a> is many things to the cell: a structural scaffold giving the cell shape, an intracellular transport system, a driver of cell motility, and a mediator of cell division, to name a few of the most important.  Accordingly the cytoskeleton gets two lectures, devoted to its two most important pieces: this week on <a href="http://en.wikipedia.org/wiki/Actin">actin</a> (red below) and next week on <a href="http://en.wikipedia.org/wiki/Microtubule">microtubules</a> (green below; the blue are nuclei).</p>
<p><a href="http://en.wikipedia.org/wiki/File:FluorescentCells.jpg"><img src="http://upload.wikimedia.org/wikipedia/commons/0/09/FluorescentCells.jpg" alt="File:FluorescentCells.jpg"/></a></p>
<p>The cytoskeleton is a dynamic 3D protein network connected to the membrane and some organelles.  There are three elements to the cytoskeleton, each a chain of specific protein units:</p>
<ol>
<li><a href="http://en.wikipedia.org/wiki/Microfilament">microfilaments</a> made of <a href="http://en.wikipedia.org/wiki/Actin">actin</a> monomers</li>
<li><a href="http://en.wikipedia.org/wiki/Microtubule">microtubules</a> made of <a href="http://en.wikipedia.org/wiki/Tubulin">tubulin</a> dimers, an</li>
<li><a href="http://en.wikipedia.org/wiki/Intermediate_filament">intermediate filaments</a> made of various other proteins.</li>
</ol>
<p>Because the cytoskeleton is modular, consisting of polymers of individual proteins, it can expand or contract to change cell shape, move and respond to stimuli.  Cytoskeletal systems vary greatly across cell types, in part responding to signals received from outside the cell.</p>
<p>Today&#8217;s lecture will focus on microfilaments but briefly here is a brief introduction to the other two things.  <a href="http://en.wikipedia.org/wiki/Microtubule">Microtubules</a> extend throughout the cell, providing an organizational framework for organelles.  They give shape to <a href="http://en.wikipedia.org/wiki/Cilium">cilia</a> and <a href="http://en.wikipedia.org/wiki/Flagella">flagella</a> in eukaryotes, and they form the spindles that pull sister <a href="http://en.wikipedia.org/wiki/Chromatid">chromatids</a> apart during <a href="http://en.wikipedia.org/wiki/Mitosis">mitosis</a>.  <a href="http://en.wikipedia.org/wiki/Intermediate_filament">Intermediate filaments</a> are tissue-specific.  They support the nucleus and nuclear membrane, provide cell structural support and promote barrier formation in hair, skin and nail cells.  To our knowledge, they have no transport role &#8211; i.e. they are <em>not </em>substrates for motor proteins to walk upon.  By contrast, microfilaments and microtubules both serve as transport systems.  <a href="http://en.wikipedia.org/wiki/Kinesin">Kinesin</a> and <a href="http://en.wikipedia.org/wiki/Dynein">dynein</a>, powered by ATP, walk on microtubules (anterograde and retrograde respectively), as will be covered in more detail next lecture.  <a href="http://en.wikipedia.org/wiki/Myosin">Myosin</a> walks on actin microfilaments, the focus of today&#8217;s lecture.</p>
<p><a href="http://en.wikipedia.org/wiki/Microfilament">Microfilaments</a> are polymers of <a href="http://en.wikipedia.org/wiki/Actin">actin</a>.  They can form complex structures by being held together in different ways by a dizzying variety of <a href="http://en.wikipedia.org/wiki/Actin-binding_protein">actin-binding proteins</a>.  Microfilaments organize the <a title="Cell Biology 02: The Plasma Membrane" href="/2013/02/10/cell-biology-02-the-plasma-membrane/">plasma membrane</a>: animal cells have a &#8216;<a href="http://en.wikipedia.org/wiki/Cell_cortex">cell cortex</a>&#8216; which is an actin-rich layer just under the membrane.  Actin also forms the contractile ring which cinches the cell into two during <a href="http://en.wikipedia.org/wiki/Mitosis">mitosis</a>.</p>
<p>Actin microfilament formation is portrayed in the Inner Life of the Cell video, although it is perhaps my least favorite part of this really awesome video.  Susan Lindquist pointed out that this video makes two major sacrifices in order to make what&#8217;s going on in the cell visually intelligible: (1) concentration and (2) speed.  Unlike the elegant negative space of the video, all the molecules in a cell are packed in right next to each other, sliding past one another and interacting constantly.  And molecules at body temperature move at about 25 miles per hour, while the cell is only 10-30 μm across [<a href="http://en.wikibooks.org/wiki/Cell_Biology/Introduction/Cell_size">wikibooks</a>], meaning that every passing second provides an astronomical number of opportunities for molecular interactions.  These two factors &#8211; speed and density &#8211; are crucial for everything that happens in the cell, and omitting them is a necessary sacrifice for any video depiction, but to me it is especially problematic in showing actin formation.  This video makes it look as though the actin proteins just know to come together by magic to form filaments:</p>
<p><iframe src="http://www.youtube.com/embed/zrXykvorybo?start=155&amp;end=180" frameborder="0" width="420" height="315"></iframe></p>
<p>Actin microfilaments also form lamellipodia and filapodia, two structures critical for the forward mobility of <a title="cells that move" href="http://en.wikipedia.org/wiki/Motility">motile</a> cells.  <a href="http://en.wikipedia.org/wiki/Lamellipodium">Lamellipodia</a> are a mesh-like network of actin filaments inside the cytoplasm that extend a smooth leading edge at the &#8216;front&#8217; of migrating cells.  <a href="http://en.wikipedia.org/wiki/Filopodia">Filapodia</a> are finger-like, spiky protrusions of actin filaments that extend more of a jagged edge.  Both are required for proper cell migration in some cells, and both are composed of actin &#8211; just with different organizing proteins giving rise to different superstructures of microfilaments.  Actin is also a structural component of <a href="http://en.wikipedia.org/wiki/Microvillus">microvilli</a>, another feature seen only in some cell types (intestines and ova), and <a href="http://en.wikipedia.org/wiki/Stress_fiber">stress fibers</a> important for muscle contraction, and <a href="http://en.wikipedia.org/wiki/Adherens_junction">adherens junctions</a> that link cells together (epithelial cells and cardiomyocytes).</p>
<p>Actin comes in three main isoforms (and a few minor ones we won&#8217;t cover here).  The big three are predictably named <a href="http://en.wikipedia.org/wiki/ACTA1">alpha-actin</a>, <a href="http://en.wikipedia.org/wiki/Beta-actin">beta-actin</a> and <a href="http://en.wikipedia.org/wiki/ACTG1">gamma-actin</a> and coded for by the three genes ACTA1, ACTB and ACTG1.  Each isoform is ~375 amino acids, only ~25 of which vary between these three isoforms &#8211; and only 20% of which vary across species from algae to humans, indicating strong conservation.  But that little bit of variability gives rise to great differences in function between isoforms.  Alpha is important in contractile structures such as the contractile ring for mitosis; beta is important in the cell cortex and at the leading edge of motile cells (lamellipodia and filapodia) and gamma is important in stress fibers.   Collectively these make up 10% of cell mass in muscle cells [according to class; <a href="http://en.wikipedia.org/wiki/Actin#Structure">20% according to Wikipedia</a>].</p>
<p>Regardless of isoform (alpha/beta/gamma), actin is called G-actin (for <span style="text-decoration: underline;">g</span>lobular) when it&#8217;s a monomer and F-actin (for <span style="text-decoration: underline;">f</span>ilamentous) when it&#8217;s a polymer.  Actin monomers are polar with a &#8216;barbed&#8217; (+) end and a &#8216;pointed&#8217; (-) end.  In the spokes of the cytoskeleton, the (+) end points to the cell surface and the (-) points to the center of the cell.  The (-) end has an ATP binding cleft, and actin must bind either ADP or ATP in order to polymerize at all, with much higher inclination towards polymerization when ATP-bound than when ADP-bound.  Actins usually bind an ion as well (most often Mg<sup>2+</sup>, sometimes K<sup>+</sup> or Na<sup>+</sup>) and ion concentrations determine whether filament formation or dissociation is favored.</p>
<p>Filament formation is said to take place in three steps:</p>
<ol>
<li><strong>Nucleation</strong>.  This is the rate limiting step &#8211; actin monomers have limited affinity to form oligomers with one another but higher affinity to join an existing filament.  It&#8217;s like how your first million is your hardest to make, or how no one wants to come to the party until they know who else will be there.  Specifically, actin dimers dissociate very easily, but once a trimer forms, it is relatively stable and can act as a seed to attract other G-actin molecules.  Apparently it takes three to tango.</li>
<li><strong>Elongation</strong>.  Once nucleation has happened, and provided that ion and G-actin concentrations favor it, the filament will elongate rapidly.  G-actin monomers prefer to join the (+) end of a filament, so that end will grow faster, but in many conditions the (-) end will grow as well.</li>
<li><strong>Steady state</strong>.  Eventually the filament will grow to where the ambient concentrations of G-actin and requisite ions have been depleted enough that the rate of actin monomer dissociation from the filament equals the rate of elongation.  The dissociation happens more at the (-) end and elongation more at the (+) end, so even in equilibrium, the filament is still dynamic and has a &#8216;treadmilling&#8217; appearance.</li>
</ol>
<p>The above is a slight simplification because filament nucleation and the equilibrium between elongation and dissociation also depend upon <a href="http://en.wikipedia.org/wiki/Actin-binding_protein">actin-binding proteins</a>.  <a href="http://en.wikipedia.org/wiki/Profilin">Profilins</a> (genes: PFN1-4) are small proteins that promote swapping out ADP for ATP in the actin cleft: they bind to the (+) end of ADP-bound G-actin when ATP is more abundant and release the ADP, allowing a swap to occur.  Profilin also blocks (-) end growth, thus further promoting &#8216;treadmilling&#8217;.  <a href="http://en.wikipedia.org/wiki/Cofilin">Cofilins</a> (genes: CFL1, CFL2, DSTN) twist and destabilize filaments, promoting dissociation.  <a href="http://en.wikipedia.org/wiki/Thymosin_beta-4">Thymosin beta 4</a> (gene: TMSB4X) is another small protein that binds ATP-bound actin and physically blocks addition to the filament at either end &#8211; thus it prevents excessive elongation of filaments in conditions of abundant ATP.</p>
<p>Other proteins can completely &#8216;cap&#8217; the ends of a filament, preventing assembly or disassembly.  <a href="http://en.wikipedia.org/wiki/CapZ">CapZ</a> (genes: CAPZA1-3, CAPZB) are (+) end specific, <a href="http://en.wikipedia.org/wiki/Tropomodulin">tropomodulins</a> (genes: TMOD1-4) are (-) end specific and are found in non-motile cells that, since they don&#8217;t move, want stable, non-expanding non-contracting actin filaments, and <a href="http://en.wikipedia.org/wiki/Gelsolin">gelsolin</a> (gene: GSN) is (+) end specific and has a Ca<sup>2+</sup>-dependent ability to sever filaments.</p>
<p>Nucleation is promoted by &#8216;actin nucleating proteins&#8217; that help bring monomers together. Chiefly, <a href="http://en.wikipedia.org/wiki/Formins">formins</a> (many genes) help to form long, straight filaments and the dimer <a href="http://en.wikipedia.org/wiki/Arp2/3_complex">Arp2/3</a> (genes: ARPC2 &amp; ARPC3) helps to form branched filaments, for instance in <a href="http://en.wikipedia.org/wiki/Lamellipodium">lamellipodia</a>.</p>
<p>Formins have two domains (called FH1 and FH2 for formin homology 1 &amp; 2) that form a donut-shaped complex that promotes elongation at the (+) end.  Specifically, FH1 is rich in proline, and <a href="http://en.wikipedia.org/wiki/Profilin">profilin</a> (see above) is attracted to proline-rich domains.  So formin hangs out at the (+) end of F-actin, and recruits profilin which attracts ATP-bound G-actin, which can then be added to the filament.  Formin thus effectively acts to increase the <em>local </em>concentration of G-actin above the critical threshold for elongation to occur.  It also prevents (+) end capping.</p>
<p>Formin itself is regulated by <a title="a whole big family of G-proteins" href="http://en.wikipedia.org/wiki/Rho_family_of_GTPases">Rho</a> G-proteins.  Remember, G-protein means <a href="http://en.wikipedia.org/wiki/GTPase">GTPase</a>, a protein that can bind GTP and hydrolyze it to GDP as an energy source.  Certain extracellular signals can activate Rho-<a title="GEFs are proteins that swap out GDP for GTP in G-proteins" href="http://en.wikipedia.org/wiki/Guanine_nucleotide_exchange_factor">GEF</a> (any of a bunch of different proteins with <a title="the RhoGEF domain" href="http://en.wikipedia.org/wiki/RhoGEF_domain">one shared domain</a>) to load Rho with GTP.  When (and only when) Rho is GTP-bound, it &#8216;opens&#8217; formin into the donut configuration so it can do its job.</p>
<p>The Arp in <a href="http://en.wikipedia.org/wiki/Arp2/3_complex">Arp2/3</a> stands for actin-related protein.   To do its job of promoting filament branching, it needs NPF (nucleation promoting factor) as well as <a href="http://en.wikipedia.org/wiki/Wiskott%E2%80%93Aldrich_syndrome_protein">WASp</a> (Wiskott-Aldrich Syndrome protein).  When Arp and WASp both bind to the <em>side</em> of a filament (i.e. not at the (+) or (-) end), they allow a new filament to form off to that side, with its (-) end pointed into the trunk of the original filament.  This process is also regulated by a Rho protein, <a href="http://en.wikipedia.org/wiki/CDC42">Cdc42</a> (gene: CDC42) which (when bound to GTP) regulates WASp which in turn activates Arp2/3.</p>
<p>A genus of bacteria called <a href="http://en.wikipedia.org/wiki/Listeria"><em>Listeria</em></a> have figured out how to hijack our actin systems for their own benefit.  Listeria has its own actin protein, <a href="http://en.wikipedia.org/wiki/Actin_assembly-inducing_protein">ActA</a>, which uses your own cells&#8217; Arp2/3 to recruit your own actin monomers to form a &#8216;comet&#8217; that propels it around the cell, as shown in this video:</p>
<p><iframe src="http://www.youtube.com/embed/40PgA0GaAwY?rel=0" frameborder="0" width="420" height="315"></iframe></p>
<p>To study actin, researchers often use:</p>
<ul>
<li><a href="http://en.wikipedia.org/wiki/Phalloidin">phalloidin</a> from <a href="http://en.wikipedia.org/wiki/Amanita"><em>Amanita</em></a> death caps, which binds to actin filaments (and stabilizes them, preventing depolymerization) &#8211; that&#8217;s both why it&#8217;s toxic and why it&#8217;s useful for imaging to see where actin is</li>
<li><a href="http://en.wikipedia.org/wiki/Latrunculin">latrunculin</a> from sponges, which binds G-actin to prevent filament formation, blocking cell motility and cytokinesis</li>
<li><a href="http://en.wikipedia.org/wiki/Cytochalasin_D">cytochalasin D</a> from molds, which caps the (+) end of filaments, preventing elongation</li>
<li>jaspakinolide from sponges [see <a href="http://www.jbc.org/content/275/7/5163.full">Bubb 2000</a>] which promotes nucleation by stabilizing actin dimers, making it more likely they&#8217;ll stay together long enough to form a trimer at which point they can start elongating into a filament.  By doing so, it effectively lowers the critical concentration of G-actin required for filament formation.</li>
</ul>
<p>Of course, the actin filaments are themselves organized into various superstructures.  Cross-linking proteins help to determine what structure will form.  In addition to Arp2/3, which as mentioned earlier promotes branching, also of note are the following:</p>
<ul>
<li><a href="http://en.wikipedia.org/wiki/Fimbrin">fimbrin</a> binds two close-together filaments with the same polarity (i.e. &#8216;pointed&#8217; in the same direction), important for forming those jagged spikes in filapodia.</li>
<li><a href="http://en.wikipedia.org/wiki/Actinin,_alpha_1">alpha-actinin</a> (gene: ACTN1), a larger protein, can bind two filaments that are further apart than fimbrin.</li>
<li><a href="http://en.wikipedia.org/wiki/Filamin">filamin</a> (genes: FLNA, FLNB, FLNC) is a sort of &#8216;hinge&#8217;, binding two actin filaments at a large angle and thus helping to form a mesh of actin filaments.</li>
<li>the <a href="http://en.wikipedia.org/wiki/ERM_protein_family">ERM proteins</a> (<a title="Ezrin" href="http://en.wikipedia.org/wiki/Ezrin">ezrin</a>, <a title="Radixin" href="http://en.wikipedia.org/wiki/Radixin">radixin</a> and <a title="Moesin" href="http://en.wikipedia.org/wiki/Moesin">moesin</a>) link parallel actin filaments, with their (+) ends all pointed outward into the tips of <a href="http://en.wikipedia.org/wiki/Microvillus">microvilli</a>.</li>
<li><a href="http://en.wikipedia.org/wiki/Spectrin">spectrin</a> is a tetramer formed of 2 alpha (genes: SPTA1, SPTAN1) and 2 beta (genes: SPTB, SPTBN1,2,4,5) subunits pointed in opposite directions.  The spectrin complex binds one actin filament in each end and is a critical part of forming the triangles that make up hexagons in the cell cortex (see below).</li>
</ul>
<p><a href="http://en.wikipedia.org/wiki/File:Cytoskeleton_(Elliptocytosis).png"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/3/34/Cytoskeleton_%28Elliptocytosis%29.png/640px-Cytoskeleton_%28Elliptocytosis%29.png" alt=""/></a></p>
<p>(thanks to Wikimedia Commons user kupirijo)</p>
<p>This spectrin tetramer is depicted in Inner Life of the Cell:</p>
<p><iframe src="http://www.youtube.com/embed/yKW4F0Nu-UY?start=118&amp;end=138" frameborder="0" width="420" height="315"></iframe></p>
<p>So far we&#8217;ve talked about the structural role of the microfilaments.  They also have a transport role, serving as sort of a set of train tracks for ATP-powered motor proteins to move along.  <a href="http://en.wikipedia.org/wiki/Myosin">Myosin</a> is the main motor protein that operates on microfilaments.  It has a &#8216;head&#8217; which both binds actin and binds ATP, a &#8216;neck&#8217; which acts as a lever &#8211; its length determines the stride at which myosin walks &#8211; and a &#8216;tail&#8217; which binds cargo to be transported.  The tail is highly variable, as its sequence determines what cargo will be bound.  The head must burn one ATP molecule for every &#8216;step&#8217; it takes.  Myosins are a huge family of proteins that come in <a href="http://en.wikipedia.org/wiki/Myosin#Myosin_classes">13 different classes</a> (genes starting with <a href="http://en.wikipedia.org/wiki/Myosin#Genes_in_humans">MYO, MYH and MYL</a> &#8211; the H and L are for heavy and light chain components).  Here are just a few of the classes:</p>
<ul>
<li>Class I myosins couple actin filaments to the cell membrane and are involved in endocytosis.  They have just one &#8216;head&#8217; that binds actin and ATP.</li>
<li>Class II do muscle contraction. They have two &#8216;heads&#8217; allowing them to &#8216;walk&#8217; by alternating which head binds.</li>
<li>Class V are two-headed and &#8216;walk&#8217; toward the (+) end of filaments, but they seem to spend most of their time just walking on the treadmill: they don&#8217;t transport vesicles so much as just keep them in the cell periphery.  In yeast it organizes the organelles by virtue of its long neck region and 6 light chains and tails.  But in animals, microtubules (topic for next week) deal with organelle organization.</li>
<li>Class VI are two-headed and are the class that can &#8216;walk&#8217; toward the (-) end, performing the retrograde movement of taking endocytic vesicles deeper into the cell.</li>
</ul>
<p>People have studied myosin movements in vitro via a filament sliding assay: tether myosin to a glass side and then add ATP and fluorescently labeled actin filaments and see what direction the <em>filaments</em> move.  These are the experiments that revealed that the majority of myosins (all but Class VI) move toward the (+) end, which you can see because the myosins push the (-) end outward when the myosin is tethered and the filament free-moving.</p>
<p>When you look at muscle fibers under a microscope you see two main parts: thick filaments, which are myosin II, and thin filaments which are actin.  When motor neurons fire onto the muscle cells, post-synaptic pathways signal for the sarcoplasmic reticulum (a special word for the ER in muscle cells) to release its reserves of Ca<sup>2+</sup>.  Myosin II is Ca<sup>2+</sup>-dependent, so when calcium rushes into the cytosol, myosin II starts burning ATP to try to &#8216;climb&#8217; toward the (+) end of actin, but since it&#8217;s doing this <em>from both ends</em>, it doesn&#8217;t actually move, but instead pulls the actin filaments closer together.  That&#8217;s what muscle contraction is at the molecular level.   Here it is in video form:</p>
<p><iframe src="http://www.youtube.com/embed/j8F5GGPACkQ?rel=0" frameborder="0" width="420" height="315"></iframe></p>
<p>And a bit more zoomed out:</p>
<p><iframe src="http://www.youtube.com/embed/zQocsLRm7_A?rel=0" frameborder="0" width="420" height="315"></iframe></p>
<p>Cell <a href="http://en.wikipedia.org/wiki/Motility">motility</a> is important for finding food and avoiding prey in single-celled organisms, and for embryonic development, wound closure and infection/disease control in multicellular organisms.  To move, cells have to have a &#8216;front&#8217; and a &#8216;back&#8217;.  At the front, actin filaments extend the cell forward via extending lamellipodia and filapodia.  These cells also have a sort of &#8216;bottom&#8217; where membrane proteins such as integrin adhere to the extracellular matrix (or a glass slide, etc. in the lab) relative to which the movement is occurring.  At the &#8216;back&#8217; of the cell, myosin II / stress fiber contraction pulls the cell away from old adhesion sites &#8211; often the adhesion proteins snap, with tails still bound to the old adhesion surface and the heads endocytosed into the cell.   At the front and back, adapter proteins (Arp2/3 and formin respectively) join the actin network to the plasma membrane via membrane-anchored G proteins: at the front, Arp2/3 binds to membrane-anchored G proteins.  Extracellular signals often induce the cell to move by signaling through G proteins, chiefly the Rho family proteins including <a href="http://en.wikipedia.org/wiki/Rac_(GTPase)">Rac</a>, <a href="http://en.wikipedia.org/wiki/CDC42">Cdc42</a>, <a href="http://en.wikipedia.org/wiki/RHO_protein_GDP_dissociation_inhibitor">GDI</a> (GDP displacement/dissociation inhibitor).</p>
<p><strong>discussion section</strong></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3328382/">Suraneni 2012</a> studied mouse fibroblasts homozygous for ARPC3 knockout, thus having no functional Arp2/3 complex.  The cells were unable to form lamellipodia.  They still formed something that looked an awful lot like filopodia but which Suraneni did not characterize at the molecular level.  The cells still moved around, but lacked directionality &#8211; they moved more like a random walk, and when a gradient of <a href="http://en.wikipedia.org/wiki/Epidermal_growth_factor">epidermal growth factor</a> (EGF) was added they moved faster but failed to move up the gradient as they are supposed to.</p>
