---
layout: post
title: "Four flavors of CRISPR knockout screens"
author: ericminikel
date: 2014-07-18 04:16:38
---
<p>About a year and a half ago, George Church&#8217;s lab demonstrated the utility of the CRISPR/Cas9 system for genome engineering in human cells [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23287722">Mali &#038; Yang 2013</a>]. A short guide RNA (sgRNA) complementary to a 20bp sequence in the genome complexes with the Cas9 nuclease and leads it to create a targeted double-strand break, thus creating an opportunity for homologous recombination (for knock-ins) or non-homologous end joining (for knockouts). In cultured cells, the technique is fairly efficient at creating knock-ins by homologous recombination (Mali describes rates of 3 to 8% ) and incredibly efficient at creating knockouts (we&#8217;ve heard rumors of efficiencies of up to 80% homozygous deletion for specific sgRNAs).</p>
<p>While Mali and Yang targeted a specific gene of interest (<em>PPP1R12C</em>) as a demonstration, they bioinformatically predicted 190,000 sgRNAs for targeting loci across the human genome, foreshadowing the use of CRISPR for genome-wide screens. One year later (this spring), four papers appeared in the literature almost synchronously, each demonstrating the use of pooled CRISPR libraries for conducting genetic screens in mammalian cells [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24336569">Wang 2014</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24336571">Shalem &#038; Sanjana 2014</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24535568">Koike-Yusa &#038; Li 2014</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24717434">Zhou 2014</a>].</p>
<p>Genetic screens involve examining in parallel the effect of a wide variety of genetic mutations on a particular cellular phenotype. These have long been popular in yeast [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11533715">Forsburg 2001</a>] but until recently were difficult to achieve in mammalian cells, which are diploid and don&#8217;t mate with one another in culture. The solution in mammalian cells has been to use RNAi screens &#8211; transfect cells with thousands of different siRNAs and see which ones alter the phenotype. But the 50 or 80% knockdown often achievable with mass-produced siRNAs is often insufficient to confer a strong phenotypic change. CRISPR/Cas9, then, finally provided an opportunity to examine the phenotypic effects of total knockout. Design a library of sgRNAs that create indels via non-homologous end joining in early constitutive exons, and you can knock out most every gene.</p>
<p>What is striking about the first four papers to develop this approach is just how similar their strategies are, despite being developed in parallel.</p>
<p>Note, first, that even highly efficient sgRNAs don&#8217;t create homozygous deletions in 100% of cells. Therefore if you expose a batch of cells to the CRISPR reagents and then measure their phenotype in bulk, you will be measuring some mix of the homozygous knockout phenotype, heterozygous knockout phenotype, and unaffected cells. This &#8220;arrayed CRISPR&#8221; approach, at least when paired with bulk phenotype measurement, would not necessarily be much more powerful than RNAi screens. Instead, the power of CRISPR lies in the ability to select clones that <em>do</em> have homozygous deletions. Therefore each of these four papers used a &#8220;pooled CRISPR library&#8221; approach: transform a batch of cells with viral vectors containing a mix of different sgRNAs, then progressively select the cells for phenotypes of interest, then measure (using next-generation sequencing) which sgRNAs you&#8217;ve thereby depleted or enriched.</p>
<p>How, then, to select for phenotypes of interest? All four of these studies used cell death as the phenotype. First, you clone the Cas9 nuclease into a lentivirus and transform cells with the lentivirus so that Cas9 is stably integrated into their genome. Then you expose the cells to lentiviruses expressing the myriad sgRNAs, and give them anywhere from a couple days to a couple weeks for the sgRNAs to be expressed and create indels in target genes. Now you&#8217;ve got a collection of cells &#8211; all mixed together in one flask, mind you &#8211; that have thousands of different genes knocked out. Then you expose the cells to something that normally kills them &#8211; say, the chemotherapy drugs 6-thioguanine, etoposide [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24336569">Wang 2014</a>] or vemurafenib [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24336571">Shalem &#038; Sanjana 2014</a>], or anthrax or diphtheria toxin [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24717434">Zhou 2014</a>], or <em>Clostridium septicum</em> alpha toxin [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24535568">Koike-Yusa &#038; Li 2014</a>]. </p>
<p>You use these toxins to kill off the cells day by day, sequencing as you go. These are all toxins that will kill any normal cell, and most genes have no effect at all on this toxicity, so overwhelmingly you find that the sgRNAs are under <em>negative</em> selection &#8211; the cells with those genes knocked out are getting killed off. By contrast, if there are a few genes that are on-pathway, that actually mediate the toxicity, then knocking them out will render the cells resistant, and the sgRNAs that cause these knockouts will be under massive positive selection. For instance, after a few days of exposure to anthrax toxin, most of the several hundred sgRNAs tested by [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24717434">Zhou 2014</a>] were virtually eliminated, while predictably enough, sgRNAs targeting the anthrax toxin receptor (<em>ANTXR1</em>) were enriched 1000-fold.</p>
<p>Similarly, the other authors also report that the most obviously expected genes for each toxin were indeed selected. For instance, DNA topoisomerase II (<em>TOP2A</em>) knockout was favored under conditions of etoposide poisoning [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24336569">Wang 2014</a>]. Two studies each found that sgRNAs targeting four genes in the DNA mismatch repair pathway were enriched when cells were treated with 6-thioguanine [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24336569">Wang 2014</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24535568">Koike-Yusa &#038; Li 2014</a>], i.e., knockouts that allowed the cells to ignore their own DNA damage encouraged the cells to proliferate. In addition to showing that known biology could be recapitulated using CRISPR libraries, these studies also reported novel validated hits, revealing new genes involved in, for example, vemurafenib resistance [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24336571">Shalem &#038; Sanjana 2014</a>].</p>
<p>The broad strokes of the approaches used in these papers are so similar that it takes a very close reading to see what&#8217;s different. One study used mouse embryonic stem cells [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24535568">Koike-Yusa &#038; Li 2014</a>], the others used human cell lines [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24336569">Wang 2014</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24336571">Shalem &#038; Sanjana 2014</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24717434">Zhou 2014</a>]. One study was circumspect, validating the approach first in the &#8220;near-haploid&#8221; human cell line KBM7 before moving on to diploid cells [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24336569">Wang 2014</a>], while the others plunged straight into diploid cell editing. One group created a small (&lt;1000 sgRNAs) library targeting a few hundred candidate genes [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24717434">Zhou 2014</a>], while the others went genome-wide, or nearly so. The libraries contained an average of about 3 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24336571">Shalem &#038; Sanjana 2014</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24717434">Zhou 2014</a>], 5 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24535568">Koike-Yusa &#038; Li 2014</a>], or 10 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24336569">Wang 2014</a>] sgRNAs per gene being targeted, and partly as a consequence, used different statistical tests to assess which genes were enriched &#8211; for instance, [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24717434">Zhou 2014</a>]] used <a href="http://www.bioconductor.org/packages/release/bioc/html/DESeq2.html">DESeq2</a> which assumes a negative binomial distribution of read count data; [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24336569">Wang 2014</a>] simply applied a <a href="http://en.wikipedia.org/wiki/Kolmogorov%E2%80%93Smirnov_test">Kolmogorov-Smirnov test</a> to the counts of the 10 sgRNAs for each gene before and after selection. Some of the studies reported ways to improve the CRISPR library approach &#8211; for instance, [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24336569">Wang 2014</a>] targeted essential ribosomal proteins without which cells cannot live, and then figured that the selection of sgRNAs under this paradigm is correlated with knockout efficiency, and thus reported tips for optimal sgRNA design. One study [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24535568">Koike-Yusa &#038; Li 2014</a>] provided what was (for me) a helpful explanation of why multiple sgRNAs are needed per gene. Genes where only <em>one</em> sgRNA is enriched are likely to be false positives &#8211; for instance, they might represent off-target effects of that sgRNA, or they might represent &#8220;passenger sgRNAs&#8221; in cells where a different &#8220;driver sgRNA&#8221; confers the resistance phenotype. I am no expert on this material, so if I&#8217;ve missed another important distinction between some of these papers, please leave me a comment below.</p>
<p>Any way you slice it, pooled CRISPR libraries are an incredibly powerful new tool for genetic screens. The genome-wide libraries have all been <a href="http://www.addgene.org/crispr/libraries/">deposited in addgene</a>, so now you too can join in the fun and start screening for genes whose knockout confers a phenotype of your choosing. Though all of these studies used cell death for selection, there seems to be in principle no reason you couldn&#8217;t employ, say, FACS sorting to select based on immunoreactivity or reporter gene expression. As time goes on it will be interesting to see what new variations on this method appear in the literature.</p>
