---
layout: post
title:  "Biolit 02: protein targeting"
date:   2014-09-11 12:00:00
author: ericminikel
tag: bbs-230
location: Boston, MA
---

*These are my notes for week 02 of Harvard's BBS 230 "Analysis of the biological literature" course (September 9-11, 2014). This includes my own reading and analysis of the papers, as well as my notes from both section meetings.*

### [Bankaitis 1986]: Isolation of yeast mutants defective in protein targeting to the vacuole

#### Background

In 1986, it was already known that different proteins localized to different parts of the cell, thanks to fractionation and [autoradiography](http://isites.harvard.edu/fs/docs/icb.topic1050994.files/CH156_Lecture5%20Autoradiography.pdf) experiments. But it was largely unknown how proteins were targeted to different subcellular locations. [Randy Schekman](http://en.wikipedia.org/wiki/Randy_Schekman) had recently been characterizing the secretory pathway [[Schekman 1985]], and people were starting to realize there were signal peptides, but no one knew what proteins recognized these signals. Protein localization was therefore a hot research area. Bankaitis sought to create a screen that could identify proteins involved in trafficking to the vacuole.

#### Terms

+ **Vacuole** is the yeast equivalent of the mammalian lysosome.
+ **Periplasm** is the space between the yeast inner membrane and cell wall. The yeast secretory pathway releases proteins into this space, from which most (including invertase) can freely move through the porous cell wall.
+ **Zymogen** means an inactive proenzyme.
+ **CPY** is carboxypeptidase Y, a serine protease which was known to be targeted to the lysosome.
    + *PRC1* is the yeast gene which encodes CPY.
    + proCPY is the zymogen form of CPY. The paper states that when it reaches the vacuole, an 8,000 kDa prodomain is cleaved by a protease called PEP4 to yield mCPY (mature CPY). This is almost certainly a typo. The [entire amino acid sequence](http://www.uniprot.org/uniprot/P00729) of CPY is only 532 residues long, so there is no way the prodomain could be 8,000 kDa. More likely they meant 8 kDa / 8,000 Da.
+ **Invertase** is an enzyme that breaks down sucrose. It is essential for yeast to survive when sucrose is their sole carbon source.
    + *SUC2* is the yeast gene which encodes invertase.

#### What they did and why

The authors sought to create a screen which would allow them to select for mutants that are deficient in targeting their proteins to the vacuole. They used yeast strains deficient in endogenous *SUC2* so that, by default, they were unable to survive on a diet of sucrose alone. Then they transformed these yeast with a construct expressing a CPY-Invertase fusion protein which they call pCYI433. It contains 433 N-terminal amino acids of proCPY (about 80% of the total peptide sequence) fused to the entirety of invertase. By default, this fusion protein is targeted to the vacuole due to its CPY domain. If and only if the cell has a mutation which precludes effective targeting of proteins to the vacuole, the fusion protein will end up in the periplasm instead. Invertase can only function in the periplasm, so when these yeast are grown on media with sucrose as the sole carbon source, *only* the mutants that lack effective vacuole targeting will survive. This paper develops this screen, uses it to isolate several mutants, and then characterizes them.

#### Step by step

+ Figure 1A shows the construct used to express pCYI433 (explained above). The choice to fuse *PRC1* and *SUC2* may have been motivated in part by the existence of a convenient restriction site allowing in-frame fusion of the two genes.
+ The authors expressed pCYI433 initially in the yeast strain SEY2101 (see table below). In Figure 1B, they characterize the behavior of this fusion protein in this yeast strain. These yeast strains (SEY2101 and SEY2101.1) *do* have endogenous CPY (see table of strains below) but its size is smaller (~50 kDa) so you don't see it on this gel. They radiolabeled sulfur in the cells, immunoprecipitated with anti-CPY or anti-invertase (data not shown for the latter) antisera, and then ran SDS-PAGE and looked at the radioactive signal. The difference between the lanes with and without tunicamycin (a glycosylation inhibitor) shows that the fusion protein gets glycosylated (as expected, since CPY is glycosylated), and the difference between the *PEP4+* and *&Delta;pep4* genotypes shows that (unlike wild-type CPY) the pCYI433 prodomain is *not* being cleaved in the vacuole. Together, the facts that they can detect pCYI433, that it seems to undergo some normal processing (glycosylation) but is not cleaved (thus we don't need to worry about whether its invertase activity changes upon cleavage) all validate that this protein is sufficiently well-behaved that we can study it.
+ Next they wished to confirm their expectaton that pCYI433 would be targeted to the vacuole. To do this, they measure enzymatic activity of whole cell lysate versus vacuole fractions isolated via centrifugation. In Table 1, they show that &alpha;-mannosidase (a control known to go entirely to the vacuole) and that the fusion protein are both found in the same fraction - i.e. 41-43% of each is found in the vacuole fraction. This number is <100% because some is in the ER/Golgi on its way to the vacuole, and some is lost in the lysis prep, etc. So this 41% recovery figure may actually reflect 100% trafficking to the vacuole, and even if it didn't, it might still be a high enough percentage to give a growth phenotype on sucrose. This is considered evidence that the fusion protein localizes to the vacuole, and not to one of the places those other proteins go (&alpha;-glucosidase is cytosolic, and NADPH-cytochrome c reductase is found in the ER). It would be nice to have >1 replicate here, though, and to know that you get *consistently* 41%. Note that the enzymes are being quantified by *enzymatic activity*. It would have been nice to have wild-type CPY as a comparison, but because it is a protease, measuring its activity would have been harder than measuring the activity of these other enzymes.
    + If this were done today, one might create a CPY-GFP-invertase fusion and look for co-localization with vacuole markers. However, even today the microscopy for this is non-trivial because yeast are small and have small organelles. [Immunogold labeling](http://en.wikipedia.org/wiki/Immunogold_labelling) would be another option.
+ In the text on p. 9077 (left column), they state that they used their screen to isolate 100 mutants which fell into 8 [complementation groups](/2014/09/08/genetics-03/). In principle, it is possible that mutations in the construct (say, mutating the CPY portion of pCYI433 so that it is no longer targeted to the vacuole) could also cause the sucrose survival phenotype. However, the authors say they sequenced the mutants and that the construct was the same in all of them. Moreover, if there are 8 complementation groups, then at most 1 of those could represent mutations in the construct.
    + Antimycin A blocks oxidative phosphorylation, forcing the yeast to perform glycolysis in order to survive. If not for antimycin A, the yeast might have burned the amino acids present in the media as energy sources, circumventing the need to break down sucrose to survive.
    + To validate that the mutants indeed failed to traffic pCYI433 to the vacuole, they measured the percentage of total invertase activity that was secreted.
+ Next the authors wanted to characterize how wild-type CPY would behave in these strains. They therefore re-introduced CPY into yeast strains SEY2108 and SEY2109. For instance, in some mutants it is found in a "p2" partially glycosylated, immature 69 kDa form, implying it is retained in some part of the secretory pathway and never gets cleaved by PEP4. Data is not shown for most of these characterizations.
+ In Figure 2, they perform a pulse-chase to observe the kinetics of CPY maturation in four mutant strains. This is a critical experiment for deciphering the mechanism by which different mutants disrupt protein targeting to the vacuole. 
    + The results, however, are difficult to interpret. For example, in the wild-type (first row), the amount of mCPY increases between 15 min and 30 min, yet no pCPY is seen at 15 min. So where does the *more* of mCPY come from at 30 min? The authors also claim that the trafficking to the vacuole has a half-life of 6 minutes, implying ~3/4 of mCPY should already be observed by the 15 min mark, which does not seem to be the case. The authors would have done well to include a 6 minute timepoint in their pulse-chase. There is no loading control, so the answer to some of these questions may simply be that there was more protein in the 30 min sample. Perhaps they left the 15 minute sample on the bench while continuing to collect samples for the pulse-chase, and some of the protein got degraded. 
    + The mutants behave differently than one another - for example, in *vpt3-2*, you see exclusively pCPY, none of it matures. *vpt6-2* has pCPY and mCPY.
    + Importantly, though, in all mutants, pCPY is *present* and just doesn't fully mature (presumably because it doesn't get to the vacuole). If not for this pulse-chase experiment, one might wonder whether some of the mutants destabilized pCPY rather than affecting its trafficking. In fact, the authors assert that the mutants *do* affect stability, though no one in my discussion section seemed to agree that the blot actually supports this conclusion.
+ Figure 3 provides further characterization of wild-type CPY in a subset of strains, simply fractioning the cells and showing that CPY is indeed located either at the periplasm or secreted into the media, rather than localizing in the vacuole. This confirms that the mutants are defective in targeting.
    + Surprisingly, in the wild-type, the p1 and p2 bands are present in whole cell lysate but absent from the spheroplast fraction. One would expect them to be there; they may be missing just because there is less protein.

#### Some details

The strains listed in the Methods are as follows:

| strain name | genotypes | what is it for |
| ---- | ----- | ---- |
| SEY2101 | *MATa* *ura3-52* *leu2-3,-112* *suc2-&Delta;9* *ade2-1* | Base strain lacking functional invertase (*suc2-&Delta;9*) so deficient in growing on sucrose. Possesses endogenous CPY. *ura3-52* and *leu2-3,-112* are probably just selection markers. The *ade2-1* is of unknown significance (also see below). |
| SEY2102 | *MAT&alpha;* *ura3-52* *leu2-3,-112* *suc2-&Delta;9* *his4-519* | Same as SEY2101 but &alpha; instead of a. Except that it has *his4-519* instead of *ade2-1*. Nobody in our section could figure out why this is; possibly these were just the strains they could purchase commercially. |
| SEY2108 | *MAT&alpha;* *ura3-52* *leu2-3,-112* *suc2-&Delta;9* *&Delta;prc1::LEU2+* | SEY2102 with *PRC1* deleted, and *LEU2* introduced in the *PRC1* locus for selection. They re-introduced CPY into this strain for Figures 2-3. |
| SEY2109 | *MATa* *ura3-52* *leu2-3,-112* *suc2-&Delta;9* *&Delta;prc1::LEU2+* | Same as SEY2108 but a instead of &alpha;. They re-introduced CPY into this strain for Figures 2-3. |
| SEY2101.1 | *MATa* *ura3-52* *leu2-3,-112* *suc2-&Delta;9* *ade2-1* *&Delta;pep4::LEU2+* | For Figure 1, to show that the fusion protein behaves the same whether or not PEP4, the CPY peptidase, is present, i.e. PEP4 does not cleave the fusion protein. |

Even in 1986, it was possible to use PCR to see if a gene is in the correct locus - I think you use a primer in the gene and a primer nearby but outside the gene, and you get the right size product only if it is in the right locus.

Some notes on methods:

+ Lithium acetate permeabilizes the cell wall to allow transformation.
+ The media needs to lack other carbon sources besides sucrose. For pulse-chase it needs to lack other sources of sulfur.
+ Why [bromocresol purple](http://en.wikipedia.org/wiki/Bromocresol_purple)?

#### Take-home messages of the paper

This can be considered a "springboard" paper. It establishes an assay, putting forth a method for studying protein localization. And it isolates mutants which this group can go on to characterize in order to learn which proteins are involved in vacuole localization. 

### [Katzmann 2001]: Ubiquitin-Dependent Sorting into the Multivesicular Body Pathway Requires the Function of a Conserved Endosomal Protein Sorting Complex, ESCRT-I

#### Background

At the time, it was known that cell surface receptors were sorted for recycling or degradation via the endocytic pathway, but for most proteins no one knew what determined the fate of a given receptor. Degradation of EGFR, however, was known to require a kinase, a cytosolic tail of EGFR, and a ubiquitin ligase. Absent any of these things, it recycled continuously and caused cancer. This was a hint that ubiquitin was important for its degradation.

By this time there were also different classes of yeast mutants with vacuole sorting defects. For instance, the Class E mutants had enlarged endosomes.

This paper shows that three proteins &mdash; Vps23, Vps28 and Vps37 &mdash; formed a complex called ESCRT-I (pronounced "escort") which is responsible for ensuring that ubiquitinated proteins end up bound to vesicle membranes in multivesicular bodies (MVBs). These proteins therefore do *not* get degraded in the lysosome, whereas if they were un-ubiquitinated they would end up in the lumen and therefore be degraded.

This paper is by the same PI, [Scott Emr](http://emr.wicmb.cornell.edu/), as the first one.

#### Terms

+ **Multi-vesicular bodies** or MVBs are one possible stage in the secretory pathway. As parts of the Golgi bud off to form early endosomes and then late endosomes, the membrane of these can invaginate to form membrane-bound vesicles inside the endosomes. An endosome containing such other vesicles is an MVB. MVBs eventually fuse to the lysosome (vacuole in yeast). Membrane proteins cannot be degraded if they are bound to the vacuole's limiting membrane, so their turnover requires that they bud off to form vesicles that float in the lumen of the vacuole.
+ **Vps** stands for vacuole protein sorting and is the name given to a series of numbered proteins that manage other proteins' localization to the vacuole.
+ **Ubiquitination** is a big topic. Ubiquitin proteins are covalently attached to target proteins at lysines on the latter, using activation, conjugation and ligation steps performed by E1, E2 and E3. This can be a signal for a variety of sorting and processing events.

#### Step by step

+ This study used yeast cells, a fact which is surprisingly obscure - it is buried in the Methods.
+ The authors already knew of a Class E (see above) yeast Vps protein called Vps23, which had a mammalian orthologue, and they knew that Vps23 was involved in turnover of cell surface receptors. They therefore used this protein as a point of entry into figuring out what else was involved, and how. They fused Vps23 to something called "protA" because they had antibodies to the latter (analogous to a Flag tag or HA). They expressed this fusion protein in yeast deficient in endogenous Vps23, which cannot grow well without Vps23.
+ In Figure 1A, they immunoprecipitate Vps23 using antibodies to protA, "clear the lysate" (which means centrifuge and then use only the supernatant, thus removing debris) and they characterize the other proteins they pull down. ProtA itself binds many proteins, so to see what binds Vps23, you need to compare the left lane to the right lane; bands found only in the right lane are specific to Vps23-protA and thus probably bind Vps23. There were two bands unique to the right lane. They purified the protein from these, trypsinized them into small peptides and then measured the mass/charge ratio using MALDI mass spec. Today you would look up the mass/charge ratio in UniProt to figure out what protein the peptide came from; in 2001 UniProt existed but it is not clear how complete its catalog of yeast proteins was. The authors don't give any more detail than to say that the mass spec revealeed the two bands to be Vps28 and Vps37, hypothesized to be members of a complex with Vps23.
+ In Figure 1B, they perform "gel filtration analysis" - basically chromatography - and each lane on the gel represents one fraction of eluent coming off the machine. Elution from a column is sort of a random walk, which is why any given molecule will elute over a few minutes and not all at once - hence the complex is seen in multiple fractions (multiple lanes). The horizontal position of the bands nevertheless approximates the mass of the complex. Once the complex was eluted, they denatured proteins and ran SDS-PAGE with antibdoies to Vps23 and Vps28. The main conclusion here is that Vps23 is never seen monomerically - it exists solely as part of the ESCRT-I complex. They also use an error-prone PCR mutagenesis screen to look for LoF mutations in Vps23, and they isolate a mutation in Vps23<sup>M85T</sup> which is non-functional but associates with the others to form a complex. This mutation is located in a domain predicted to have something to do with ubiquitination, supporting their idea that ubiquitin will somehow prove to be involved.
+ They next sought to create a mutant of Vps23, so they did error-prone PCR. In 2014 you can buy polymerases that have high error rates precisely for this purpose; in 2001 people used simpler methods such as supplying an imbalance of dNTPs for PCR. Their mutagenesis screen revealed an M85T mutant which was "found to be functionally inactive" (they don't say how they found this).
+ Figure 2 seeks to show that Vps23 (fused to GFP for visualization) co-localizes with Snf7, a marker of endosomes. In Figure 2B, there is a lot of green, so under the null hypothesis that Vps23 localizes nowhere in particular (say it's cytosolic), of course the authors would have been able to find *two* cells in which red Snf7 localized in a subset of places where green Vps23 was found. Even the two cherry-picked examples here do not imply Vps23 is exclusively endosomal. This figure on its own is fairly weak evidence for the subcellular localization of Vps23, though the conclusion is nevertheless correct. In the *vps4&Delta;* strain, Vps23 does not localize properly, because Vps4 is responsible for localization of the other Vps proteins.
    + The Methods state that they used [color deconvolution](/2012/05/13/color-transformations-for-segmenting-complex-images/) [[Ruifrok 2001]] for analyzing the images, which would imply that they had to separate the GFP and the red Snf7 signals from a straight RGB image capture, rather than just imaging with separate channels to start with.
+ Armed with the hypothesis that ESCRT-I is important for turnover of receptors, the authors studied a vacuolar hydrolase called [carboxypeptidase S (CPS)](http://www.uniprot.org/uniprot/P27614).
+ In Figure 3, they immunoprecipitate CPS and then probe with antibodies to either CPS or ubiquitin. This blot shows many different things. First, CPS is ubiquitinated. Second, its ubiquitination can be affected by a series of different mutants. At the time, the pathways indicated in blue text and yellow bubbles in Figure 3B had already been worked out, so the authors used mutations in these proteins to figure out *where* CPS was being ubiquitinated and de-ubiquitinated.

    The mutants they chose all affect different steps in the sorting pathway:
    
    | mutant | what step it messes up |
    | ------ | ----------------- |
    | &Delta;gga1 | Golgi &rarr; early endosome |
    | &Delta;pep12 | early endosome &rarr; late endosome |
    | &Delta;doa4 | late endosome |
    | &Delta;vam3 | late endosome &rarr; multi-vesicular body |

    Doa4's function is to de-ubiquitylate things, so the *&Delta;doa4* strain in lane 2 has CPS that is more highly ubiquitylated. When they mutate Gga1, they get *no* ubiquitination at all (empty-looking second-to-last lane), implying that ubiqutination happens after the Golgi. When they knock out Pep12, do get ubiquitination, implying ubiquitination has occurred in the early endosome. Doa4 is known to act in the late endosome, and knocking it out causes CPS to stay heavily ubiquitinated. Vam3 is known to manage trafficking from late endosomes to MVBs, so knocking it out gives CPS more time to interact with Doa4, hence it is fully de-ubiquitinated. Double knockout of Vam3 and Doa4 rescues this and allows CPS to be ubiquitinated.

    From the blot in Figure 3A they are able to deduce that CPS gets ubiquitinated in the early endosome and de-ubiquitinated (by Doa4) in the late endosome, and that it is degraded upon fusion with the lysosome if and only if it is not ubiquitinated. All this is diagrammed in Figure 3B.
+ In Figure 4, they seek to determine at which residue CPS is ubiquitinated. In Figure 4A they IP CPS and then probe with anti-Ub. The mutation K8R nearly abolishes Ub signal, suggesting that most ubiquitination is on K8. The K12R mutation is less severe. Accordingly, in Figure 4B they fuse the mutant CPSs to GFP and show that K8R is not properly preserved in MVBs. This is also true in K8R K12R, but not in the strain with K12R alone. Therefore ubiquitination of K8 in particular is essential for proper sorting of CPS.
    + To create mutations like this, a common method is site-directed mutagenesis. You put the desired mutations into primers, and then PCR amplify the whole plasmid, and then you can select for the new PCR product by, for instance, digesting with an enzyme that only cuts methylated DNA (plasmid DNA is methylated, your new PCR product is not).
    + When polyubiquitination occurs, the ubiquitin itself may be ubiquitinated at K6, K11, K27, K29, K33, K48, or K63; each of these may constitute a different signal - for instance, K48 is the canonical signal for proteasomal degradation. This paper doesn't decipher which Ub lysine is responsible for polyubiquitination in this pathway.
+ In Figure 5, they express a GFP with only the 7 residues of the ubiquitination site of CPS and show that this is sufficient to get GFP into the MVBs. This suggests that ubiquitination is sufficient for CPS sorting to MVBs.
+ In Figure 6 they show that Vps23 binds ubiquitinated substrates, that the Vps23 M85T mutant is unable to do so, and that Vps23 is unable to grab CPS that hasn't been ubiquitinated due to the K8R and K12R mutations.
+ Figure 7 diagrams the pathway they have arrived at from all their experiments. ESCRT-I is responsible for sorting ubiquitinated proteins into invaginating vesicles. Doa4 removes the Ub tag before invagination is complete. The proteins sorted in this manner will stay bound to the membrane and not be degraded in the lysosome. 

#### Take home message of the paper

Ubiquitin is a signal for the sorting of membrane proteins. Those that are ubiquitinated will be sorted by ESCRT-I into invaginating vesicles in MVBs, though the Ub itself will be removed along the way by Doa4. The sorting into MVBs means that the tagged membrane proteins will be degraded, whereas un-ubiquitinated ones will stay in the limiting membrane and not be degraded. This provides a mechanism for deciding which membrane proteins (receptors, etc.) get degraded and which get recycled to the cell surface.

[Bankaitis 1986]: http://www.ncbi.nlm.nih.gov/pubmed/3538017 "Bankaitis VA, Johnson LM, Emr SD. Isolation of yeast mutants defective in protein targeting to the vacuole. Proc Natl Acad Sci U S A. 1986 Dec;83(23):9075-9. PubMed PMID: 3538017; PubMed Central PMCID: PMC387077."

[Katzmann 2001]: http://www.ncbi.nlm.nih.gov/pubmed/11511343 "Katzmann DJ, Babst M, Emr SD. Ubiquitin-dependent sorting into the multivesicular body pathway requires the function of a conserved endosomal protein sorting complex, ESCRT-I. Cell. 2001 Jul 27;106(2):145-55. PubMed PMID: 11511343."

[Schekman 1985]: http://www.ncbi.nlm.nih.gov/pubmed/3916315 "Schekman R. Protein localization and membrane traffic in yeast. Annu Rev Cell  Biol. 1985;1:115-43. Review. PubMed PMID: 3916315."

[Ruifrok 2001]: http://www.ncbi.nlm.nih.gov/pubmed/11531144 "Ruifrok AC, Johnston DA. Quantification of histochemical staining by color deconvolution. Anal Quant Cytol Histol. 2001 Aug;23(4):291-9. PubMed PMID: 11531144."