---
layout: post
title:  "The conformation holds the information"
date:   2014-09-30 12:00:00
author: ericminikel
location: Boston, MA
thumb120: http://www.cureffi.org/media/2014/09/breakable-filament-thumbnail.png
summary200: "A backgrounder and deep read of Motomasa Tanaka and Jonathan Weissman's breakthrough 2004 paper showing that in [PSI+] yeast prions, strain information is encoded in protein conformation."
---

As a first-year biology PhD student, I am taking a [class](/tag/bbs-230) called "Analysis of the Biological Literature", otherwise known as "how to read papers". One of this week's papers is the classic prion paper "Conformational variations in an infectious protein determine prion strain differences" by Motamasa Tanaka and Jonathan Weissman [[Tanaka 2004]]. This landmark work demonstrated that, at least for one yeast prion, strain information is encoded in the physical conformation of the prions. This post is intended to provide a historical context and a full unpacking of the findings of this paper.

### Historical context: the state of the prion in 2004

#### A brief history of the prion hypothesis

When Stanley Prusiner first coined the term *prion* in 1982, many biologists weren't happy. He invented this term, a contraction of "proteinaceous infectious particle", to describe the unique nature of the pathogens that cause a class of diseases such as scrapie, kuru and Creutzfeldt-Jakob disease. The pathogenic agent contained, said Prusiner, "a protein that is required for infectivity" [[Prusiner 1982]].

In 1982, Prusiner's principal lines of evidence for the uniquely proteinaceous nature of the pathogen all revolved around what it took to destroy the its infectivity. The infectivity of scrapie, a model prion disease that Prusiner studied in hamsters, could be destroyed by a variety of chemical and enzymatic treatments known to destroy proteins, but was unaffected by a wide range of chemical and enzymatic treatments (and UV light) known to destroy nucleic acids.

Despite initial skepticism, over the next succeeding years, Prusiner and others accumulated a dizzying number of additional lines of evidence which collectively made the existence of prions indisputable. When purified, the pathogen looked like an amyloid made of a polymerized protein [[Prusiner 1983]]. This protein was dubbed "prion protein" (PrP). Determination of part of its amino acid sequence by Edman degradation [[Prusiner 1984]] allowed cloning of the gene that encoded it (now called *PRNP* in humans) [[Oesch 1985]]. PrP in healthy, uninfected hamsters could be readily degraded by proteinase K but PrP from prion-infected hamsters was partially protease-resistant, suggesting that a post-translational change in this protein might underlie prion pathogenesis [[Oesch 1985]]. This change appeared to be strictly conformational: the &alpha;-helices of healthy "cellular" PrP (PrP<sup>C</sup>) converted to &beta;-sheets in the "scrapie" form (PrP<sup>Sc</sup>) [[Pan 1993]]. Mouse lines in which prion disease lasted for different lengths of time proved to have different PrP amino acid sequences [[Carlson 1986], [Westaway 1987]], and PrP knockout mice could not be infected with prions at all [[Bueler 1993]]. The conformational change in PrP could be reproduced in a cell-free reaction [[Kocisko 1994]], meaning that the conformational change wasn't merely secondary to a viral infection or something else. Families with Mendelian forms of prion disease proved to have missense mutations in *PRNP* that segregated with the disease [[Hsiao 1989], [Hsiao 1991], [Medori 1992]] - this existence of a genetic disease that was also experimentally transmissible was unprecedented.

Yet even as the prion concept gained acceptance, the existence of different *strains* of prions remained difficult to explain. For years this led some researchers to try to propose a role for a nucleic acid in prion disease [e.g. [Weissmann 1991]], though no such nucleic acid was ever found. By the mid-1990s, it started to seem very likely that prion strain differences were encoded in different conformations of PrP<sup>Sc</sup>. One hint came from the fact that PrP<sup>C</sup> could acquire the protease resistance of PrP<sup>Sc</sup> in a cell-free reaction [[Kocisko 1994]], suggesting that prion replication took place via nucleated polymerization, analogous to "ice nine" in Kurt Vonnegut's *Cat's Cradle* [[Lansbury & Caughey 1995]]. This mechanism finally offered a means by which one could imagine strain information encoded in conformation being transmitted from protein to protein, from organism to organism.

Indeed, by 2004, there were at least a few separate lines of evidence directly arguing that prion strain differences were encoded in different conformations of PrP<sup>Sc</sup>:

1. A segment of ~15 amino acids (residues 82-97) in the N terminus of PrP was accessible to proteinase K digestion in prions of some strains but not others. This strain difference could be maintained faithfully even on cell-free templating of PrP<sup>Sc</sup> onto PrP<sup>C</sup>, suggesting that only prion conformation could be responsible [[Bessen 1995]].
2. Strain differences in protease accessibility and neuropathological profile could also arise spontaneously from different *PRNP* missense mutations in genetic prion disease, and yet these strain difference were maintained upon transmission of prions to mice [[Telling 1996]].
3. The 3F4 antibody binds to an epitope that is accessible in PrP<sup>C</sup> and in denatured PrP but buried in all known strains of PrP<sup>Sc</sup>. The amount of guanidine hydrochloride (GdnHCl) required to expose the 3F4 epitope in different strains of PrP<sup>Sc</sup> differs, implying that the resistance to denaturation encoded in the conformation of PrP<sup>Sc</sup> in the different strains must differ [[Safar 1998]].

But the die-hard skeptics still argued that these experiments all simply proved that differences in conformation were *associated* with different prion strains, and not that differences in conformation were *causal* of different strain properties, or in other words, that the conformation *was* the strain. To such skeptics, the final proof of the conformation-is-strain hypothesis would require generating different conformations of prions *de novo*, *in vitro*, from purified protein, and then showing that these prions behaved as different strains *in vivo*. Indeed, to do so might be hailed as not only the final proof that strain is encoded in conformation, but also the final-final-final proof that prions are comprised of protein. Yet as of spring 2004, it had so far proven impossible to generate *de novo* prions made of PrP *at all*, let alone to generate multiple different strains. Luckily, by that time it had also turned out that PrP wasn't the only prion in nature.

#### Yeast have prions too

The prion hypothesis turned out to offer a compelling resolution to some longstanding conundrums in the field of yeast genetics.

In the 1960s, through [suppressor screens](/2014/09/17/genetics-06/) against various yeast mutants, a most peculiar suppressor had been identified [see review and historical narrative in [Liebman & Derkatch 1999]]. This heritable element, whatever it was, could suppress the phenotypes of several different mutations, mostly with mutations in nutrient biosynthesis pathways [[Hawthorne & Mortimer 1963]]. Its ability to suppress so many different mutant phenotypes led it to be called a "super-suppressor". The super-suppressor arose spontaneously in about 1 in every 100,000 cells even absent any mutagenesis, and it occasionally disappeared spontaneously too. It was dominant, and was [inherited cytoplasmically](/2014/09/22/genetics-07/), yet could never be linked to mitochondrial DNA, nor to viruses, nor to any other known cytoplasmic element.

A popular system for studying the "super-suppressor" was *ade1*, an *S. cerevisiae* mutation disabling a gene in the adenine biosynthesis pathway. *ade1* yeast couldn't grow on adenine-deficient media, which meant you could easily select for suppressors of the mutation. Even when they were grown on adenine-containing media, *ade1* yeast were bright red because they accumulated P-ribosylaminoimidazole, a precursor in the adenine biosynthesis pathway (commonly called aminoimidazoleribotide or AIR for short). The ability to do selection plus a phenotype visible to the naked eye meant that *ade1* lent itself well to research.

Yeast geneticist Brian Cox used yeast with both the *ade1* mutation and the "super-suppressor" element to screen for *suppressors of the super-suppressor*. It was he who dubbed the whole super-suppressor system "&Psi;" (Psi) [[Cox 1965], [full text](http://www.nature.com/hdy/journal/v20/n4/pdf/hdy196565a.pdf)]. The intact super-suppressor was called &Psi;<sup>+</sup> and mutants that suppressed the super-suppressor were called &Psi;<sup>-</sup> mutants. Over time the nomenclature morphed into [*PSI<sup>+</sup>*] and [*psi<sup>-</sup>*], and the meaning changed slightly too (see below).

Cox posited that the super-suppressor-positive cells "contain a self-replicating particle, or a system of reactions sustaining itself by positive feedback". If that sounds like a description of a prion, it is. But it took 30 more years and all of the foundations that Prusiner and others had laid studying PrP before the true nature of [*PSI*<sup>+</sup>] began to reveal itself. Yeast geneticist Reed Wickner, studying the peculiar properties of the cytoplasmically inherited [*URE3*] element, presented evidence that [*URE3*] must be a prion conformation of the Ure2 protein [[Wickner 1994]]. He reasoned that if yeast had prions, then [*PSI<sup>+</sup>*] was probably a prion too.

The truth that eventually revealed itself was as follows. Sup35 is a yeast protein which acts as a **translation termination factor** (also called a eukaryotic translation release factor), meaning it helps to release the ribosome from an mRNA after the ribosome hits a stop codon. In [*PSI<sup>+</sup>*] yeast, Sup35 has aggregated into amyloid fibrils, leaving a dearth of functional Sup35 monomers in the cytoplasm and thus allowing ribosomes to read right through stop codons. The *ade1* mutation, it turns out, was a nonsense codon in the middle of the *ADE1* gene, and read-through of the stop codon when Sup35 is sequestered in an amyloid state restores translation of a functional protein product. The term [*psi<sup>-</sup>*] eventually came to simply refer to any yeast which have soluble monomeric Sup35, not necessarily due to a suppressor-of-the-suppressor mutation.

![](/media/2014/09/psi-prion-diagram.png)

*Credit: the photographs of yeast in the above diagram are from [Figure 1](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3561065/figure/pgen-1003257-g001/) of [[Bateman & Wickner 2013]].*

As an aside, you might ask: but isn't read-through of stop codons *bad*? Whether prions, including [*PSI<sup>+</sup>*], are good or bad for yeast remains controversial. Reed Wickner advocates the view that yeast prions harm their hosts, and that prion domains have evolved divergently to erect prion transmission barriers between different yeast strains [[Wickner 2011]], while Susan Lindquist champions the notion of yeast prions as being adaptive, and argues that evolution has in fact acted to conserve certain proteins' ability to form prions [[Halfmann & Jarosz 2012]].

In either case, the [*PSI<sup>+</sup>*] prion proved a facile model system for exploring aspects of the prion hypothesis that had proven fairly intractable for PrP. Years before scientists would succeed at generating synthetic PrP prions, UCSF's Jonathan Weissman had already in 2000 managed to create [*PSI<sup>+</sup>*] prions *in vitro* from recombinant Sup35 and had successfully infected yeast with them [[Sparrer 2000]], and had used this system to explore the biochemical basis for "strain barriers" which inhibit transmission of prion states between proteins that are not identical. By 2004, Weissman and his postdoc Motomasa Tanaka were able to generate synthetic prions of different strains, thus finally proving that, for yeast prions at least, the conformation holds the information [[Tanaka 2004]].

### Contributions of this paper

#### Efficient infection of yeast with synthetic prions

Weissman's group had previously generated synthetic prions [[Sparrer 2000]], but had only been able to infect 1-2% of yeast that were exposed, and their protocol didn't allow them to examine the properties of the synthetic prions they had produced before introducing them into yeast. Their breakthroughs in [[Tanaka 2004]] relied on a new, higher-efficiency method for introducing synthetic prions into yeast.

The generation of prions was surprisingly simple. They expressed in bacteria a fragment of Sup35 containing the prion-forming domain - this required only the <u>N</u> terminus and the <u>m</u>iddle of the protein, thus the fragment was called Sup35-NM. They denatured recombinant Sup35-NM protein with guanidine hydrochloride (GdnHCl), and then incubated it in the right buffer conditions and isolated spontaneous amyloid fibrils by centrifugation. 

Such a gentle procedure, on its own, tends to create long fibrils with relatively few *ends* for recruiting new monomers. According to the **breakable filament model** of prion formation (diagrammed below), PrP and Sup35 prions alike are composed of linear fibrils, where prion propagation requires two steps: **elongation**, where new monomers are recruited to the ends of fibrils, and **fragmentation**, where the fibrils are shattered, creating new ends.

![](/media/2014/09/breakable-filament-model.png)

The breakable filament model, diagrammed above, is still considered a hypothesis rather than fact, but it is highly consistent with a lot of observations. For instance, efficient [cell-free reactions with PrP](/2013/10/28/the-evolution-of-cell-free-prion-conversion-assays/) usually require sonication or shaking, which are thought to provide the energy for the fragmentation step. In yeast, [*PSI<sup>+</sup>*] prion propagation requires having the right amount of the chaperone Hsp104 to cleave the Sup35 fibrils [[Chernoff 1995]].

Therefore, after creating Sup35-NM fibrils by incubation, Weissman and Tanaka also sheared up their fibrils via sonication, creating many more "ends" for creating new prion infections in yeast.

To introduce the prions into yeast, they first prepared yeast [spheroplasts](http://en.wikipedia.org/wiki/Spheroplast), meaning they removed most of the cell wall, and then they permeablizied the yeast cell membranes with a buffer containing Tris (a detergent to disrupt membrane integrity) and polyethylene glycol (whose purpose [remains unclear](http://biology.stackexchange.com/questions/1995/why-is-peg-important-for-efficient-yeast-transformation), at least to me). They guessed that only a fraction of the yeast cells were being successfully permeabilized, and so at the same time as introducing their synthetic Sup35 prions, they also introduced plasmids encoding the *URA3* gene. By starting with *ura3<sup>-</sup>* and later selecting for yeast that could grow without uracil, they effectively selected for yeast that had been permeabilized and were capable of taking up materials from the surrounding environment - meaning these yeast had at least had an opportunity to become infected with the synthetic Sup35 prions.

This procedure is diagrammed in Fig 1A, and Fig 1B shows what the yeast looked like after plating: negative controls treated with only the *URA3* plasmid or with the *URA3* plasmid and monomeric denatured Sup35 were all red, while among the cells exposed to *URA3* and the synthetic prions, some were white, indicating they had become [*PSI<sup>+</sup>*]. Thanks to the *URA3* selection procedure, they were able to see a clear dose-response (Fig 1C) - when they added synthetic prions at a Sup35 concentration of 1 &mu;M, they got about 20% of cells to become [*PSI<sup>+</sup>*], and when they upped the concentration to 10 &mu;M they got nearly 100% of cells infected. Fig 1C also contains another important control. As noted above, the [*PSI<sup>+</sup>*] state can arise spontaneously in yeast at a rate of about 1 in 100,000, but this only occurs when the yeast are already positive for another prion state called [*PIN<sup>+</sup>*]. [*pin<sup>-</sup>*] yeast are incapable of spontaneously forming [*PSI<sup>+</sup>*] prions. In Fig 1C, the authors show that synthetic prion infection was equally efficient in [*pin<sup>-</sup>*] and [*PIN<sup>+</sup>*] yeast, thus confirming that the [*PSI<sup>+</sup>*] transformants they isolated had really been infected with synthetic prions and weren't just the few cells that had happened to form spontaneous prions.

The authors also showed that their synthetic prions obeyed the laws which Weissman's group had already worked out for the "species barrier" in yeast prion transmission. It had long been known that PrP prions made of one PrP amino acid sequence had difficulty recruiting PrP monomers of a different amino acid sequence. For instance, it was hard to infect mice with hamster prions, but this barrier could be overcome if the mice expressed hamster PrP transgenes [[Scott 1989]]. On similar lines, it was thankfully hard to infect humans with cow prions - this is believed to be one reason why only 200 people became infected with variant Creutzfeldt-Jakob disease after exposure to "mad cow" prions in the food supply, even though millions and millions of people in the U.K. were exposed.

The same is true in yeast: *Candida albicans* Sup35 prions cannot infect *Saccharomyces cerevisiae*, and vice versa, and one year earlier, Weissman's group had confirmed that this "species barrier" depended upon the amino acid sequence of the Sup35 [[Chien 2003]]. In Fig 1E they show positive and negative controls to illustrate their ability to reproduce the species barrier in their lab: prion extracts from *S. cerevisiae* expressing Sup35 with the *S. cerevisiae* prion domain amino acid sequence could infect *S. cerevisiae* expressing the same, but could not infect *S. cerevisiae* expressing Sup35 with the *Candida albicans* prion domain. Fig 1D shows that the same is true of the new synthetic prions made of *S. cerevisiae* Sup35 expressed in bacteria: they can infect *S. cerevisiae* expressing wild-type Sup35, but cannot infect *Candida albicans*, and cannot infect *S. cerevisiae* expressing Sup35 with the *Candida albicans* prion domain.

#### Generation of multiple strains *in vitro*

When Weissman and Tanaka picked colonies of transformed yeast and streaked them out on plates, they found a full spectrum of colors, from bright white to light pink to nearly red (Fig 2A, right panel). By 2004, it was well-known that [*PSI<sup>+</sup>*] came in different strains, with "weak" strains recruiting &lt;100% of the monomeric Sup35 into amyloid fibrils, thus allowing some translation-termination to occur. These "weak" strains accumulated intermediate amounts of AIR, and thus appeared pink. Indeed, Fig 2B provides a positive control repeating this experiment with yeast-derived prions of different known strains. The "strong" and "weak" strains in Fig 2B each produce only about 50% successful infections, with some of the streaks being dark red and thus presumably [*psi*<sup>-</sup>]. But among the successfully infected colonies, the color is relatively consistent - a bright white in the "strong" strain and a pastel pink in the "weak" strain. Therefore, the fact that the color of colonies transformed with *synthetic* prions in Fig 2A was so variable indicates that the synthetic prions came in different strains.

#### Characterization of synthetic prion strains

The fact that the synthetic prions were formed in a cell-free buffer and isolated by centrifugation made it possible to vary the conditions of their formation and then characterize the resulting amyloids before infecting yeast. The researchers found that incubating the denatured Sup35-NM monomers at different temperatures resulted in different biochemical properties. Amyloid aggregates are partially resistant to dissociation by heat and SDS, so when they performed SDS-PAGE on the synthetic Sup35-NM fibrils, the amount of Coomassie-stained monomeric Sup35-NM seen at the expected molecular weight for monomers reflected the degree to which the amyloid had been disrupted. When they formed amyloid fibrils at 4&deg;C, it was easy to get some monomers back simply by heating the fibrils to about 40-60&deg;C in SDS. In contrast, the amyloid fibrils formed at 23&deg;C or 37&deg;C were much more heat-resistant, and you had to heat them to 70&deg;C in order to see any appreciable amount of monomers coming off of the fibrils (Fig 3A). 

The resistance of the fibrils to heat and SDS was sort of an indirect measurement of their conformation, analogous to the differential accessibility to proteases or resistance to GdnHCl denaturation seen for different PrP prion strains, as discussed above. Tanaka and Weissman also managed to measure some conformational properties of their amyloid strains directly, using [electron paramagnetic resonance (EPR)](http://en.wikipedia.org/wiki/Electron_paramagnetic_resonance). I don't understand all the physics of this technique, other than to say it is analogous to NMR, except that it is electrons and not nuclei that are being excited. Like NMR, EPR can give you a readout of how much conformational flexibility different atoms in your molecule have. This technique was used to show that, while the various Sup35-NM amyloids shared broadly similar structures, they had pronounced differences in flexibility at certain residues (Fig 3B-C).

Together, the differences in heat resistance and EPR spectra between the different fibrils argued that the different temperatures had resulted in different prion conformations.

#### Different conformations result in different strain properties

In Figure 3, the authors had shown that prions formed at different temperatures had different conformations. Next they needed to show that these different conformations constituted different strains - meaning they behaved differently *in vivo*. They used their protocol again to infect yeast spheroplasts with prions formed at the three temperatures of interest, and they found compelling differences in the results. In Fig 4A, they have streaked out transformants exposed to the 4&deg;C, 23&deg;C, and 37&deg;C synthetic prions, and color is being used as a readout of strain properties. Within the plate for each temperature, there are differences in color reflecting the fact that <100% of clones were infected. But among the infected streaks, notice that the 4&deg;C prions induced many bright white [*PSI<sup>+</sup>*] colonies, while the 23&deg;C and 37&deg;C prions resulted in more pink colonies. Some of the 23&deg;C and 37&deg;C colonies are also "sectored", meaning you can see that part of the streak is pink and part has reverted to red, meaning that the prions had spontaneously disappeared (also referred to as having been "cured") in some cells. The differential abundance of "pink" versus "white" colonies is quantified in Fig 4B. Together, the pinker color and occasional "curing" of the prions from 23&deg;C and 37&deg;C argued that the prions formed at these higher temperatures were "weak" strains. In addition, [Figure S8](http://www.nature.com/nature/journal/v428/n6980/extref/nature02392-s8.jpg) shows that it was easier to cure the "weak" strains by over-expressing Hsp104. Though Hsp104 is essential for [*PSI<sup>+</sup>*] prion propagation in yeast, its overexpression cleaves the fibrils up so efficiently that it effectively cuts them back into monomers, "curing" the prion state.

One possible objection was that perhaps the titer of the 4&deg;C prions had been higher, and so the yeast had simply been infected with *more* prions, and not with a *stronger* strain of prions. To rule this out, they performed two additional experiments. First, they varied the concentration of prions. They repeated the 4&deg;C infection at 1/10 the original concentration of Sup35 amyloid, and while fewer colonies became [*PSI<sup>+</sup>*], the colonies that did do so were still bright white (Fig 4C, left). In contrast, when they repeated the 37&deg;C prion infection at a 10-fold *higher* concentration, they still got only pink colonies out (Fig 4C, right). Second, they showed that the ratios of pink and white phenotypes generated upon synthetic prion infection were maintained upon serial transmission of prions from yeast to yeast (Fig 4D). The original concentration of synthetic prions could only be expected to influence the phenotype in the first generation of infected yeast, and not in subsequent passages. A note added in proof indicates that they also found that the differences in thermal stability of the prion aggregates were also maintained on serial passage *in vivo*. Together, these experiments argued that conformation, and not prion titer, determined the strain phenotype. 

#### Conclusions and comparison to other work

This paper provided conclusive evidence that, at least for yeast Sup35 prions, the conformation of a prion encodes its strain properties. For some, this constituted the final evidence for protein-based inheritance, a definitive violation of the central dogma of biology.

Progress in the yeast prion field at that time was rapid, and it is worth noting the distinctions between different breakthroughs. In the same issue of *Nature*, another group published a paper entitled "Protein-only transmission of three yeast prion strains" [[King & Diaz-Avalos 2004]]. In that work, the researchers purified prions of different strains from yeast, transmitted the strain properties from those authentic, *in vivo*-derived yeast prions onto bacterially expressed recombinant Sup35, and then introduced the recombinant Sup35 prions *back* into yeast, and showed that the strain properties had been maintained. That paper was evidence for (as the title says) the protein-only *transmission* of strain properties, in contrast to the protein-only *generation* of distinct strains described by [[Tanaka 2004]].

### Epilogue

This paper resolved a longstanding question in the prion field, by conclusively demonstrating that differences in protein conformation alone were sufficient to generate different strain properties *in vivo* in yeast. It also raised just as many questions as it answered, including most prominently: what exactly *were* the conformational differences between the weak and strong strains? Tanaka and Weissman were back two years later with a partial answer to this question [[Tanaka 2006]]. They presented evidence that, surprisingly, the "weak" strains were actually *too stable* for their own good. Weak strains were really good at the "elongation" step diagrammed above, but were bad at "fragmentation". Because they were hard to break, they produced fewer new "ends" to recruit additional Sup35 monomers. Strong strains elongated slowly but were more brittle, and their increased propensity to fragmentation made them propagate more efficiently on the net.

Surprisingly, though, whether conformation is the *sole* determinant of PrP prion strain has remained controversial. A few months after Tanaka's paper was published, the Prusiner lab published its first success at generating synthetic PrP prions [[Legname 2004]], and several years later, the Prusiner lab validated its own results by performing experiments closely analogous to those of [[Tanaka 2004]]: they generated PrP amyloids under different urea, pH and temperature conditions and then showed that these synthetic prions gave rise to different strain properties when injected into mice [[Colby 2009]]. Still, many prion scientists objected to Prusiner's interpretation that prion infectivity and strain properties are contained solely in protein, in part because Prusiner's synthetic prions, which were prepared from nothing but protein, had very, very low amounts of infectivity. A prion-infected mouse brain, at the time of terminal illness, contains about 10<sup>9</sup> median infectious units (ID<sub>50</sub>s), where one ID<sub>50</sub> represents the quantity of prions required to produce a lethal prion infection in half of mice exposed. The synthetic prion preparations reported in [[Legname 2004]] contained roughly on the order of 1 ID<sub>50</sub>, meaning they were many logs less infectious than authentic, *in vivo*-derived prions. When other prion scientists eventually created protocols capable of generating tens of thousands of median infectious units of PrP prions, they did so by sonicating their purified PrP together with polyanion and/or phospholipid co-factors [[Deleault 2007], [Wang 2010]]. At least *in vitro*, such co-factors appear to be essential for creating high titers of prions, as well as for encoding strain properties [[Deleault 2012a], [Deleault 2012b]], suggesting that different PrP prion strains may differ not only in their conformation but in what other co-factor molecules they are complexed with. A final resolution to this controversy continues to prove elusive, and may await the determination of a high-resolution structure of infectious PrP<sup>Sc</sup>.

[Tanaka 2004]: http://www.ncbi.nlm.nih.gov/pubmed/15029196 "Tanaka M, Chien P, Naber N, Cooke R, Weissman JS. Conformational variations in an infectious protein determine prion strain differences. Nature. 2004 Mar 18;428(6980):323-8. PubMed PMID: 15029196."

[Hawthorne & Mortimer 1963]: http://www.ncbi.nlm.nih.gov/pubmed/13953232/ "HAWTHORNE DC, MORTIMER RK. Super-suppressors in yeast. Genetics. 1963 Apr;48:617-20. PubMed PMID: 13953232; PubMed Central PMCID: PMC1210498."

[Cox 1965]: http://www.nature.com/hdy/journal/v20/n4/abs/hdy196565a.html "Psi, A CYTOPLASMIC SUPRESOR OF SUPER-SUPRESOR IN YEAST"

[Prusiner 1982]: http://www.ncbi.nlm.nih.gov/pubmed/6801762 "Prusiner SB. Novel proteinaceous infectious particles cause scrapie. Science.  1982 Apr 9;216(4542):136-44. PubMed PMID: 6801762."

[Prusiner 1983]: http://www.ncbi.nlm.nih.gov/pubmed/6418385 "Prusiner SB, McKinley MP, Bowman KA, Bolton DC, Bendheim PE, Groth DF, Glenner GG. Scrapie prions aggregate to form amyloid-like birefringent rods. Cell. 1983 Dec;35(2 Pt 1):349-58. PubMed PMID: 6418385."

[Prusiner 1984]: http://www.ncbi.nlm.nih.gov/pubmed/6432339 "Prusiner SB, Groth DF, Bolton DC, Kent SB, Hood LE. Purification and structural studies of a major scrapie prion protein. Cell. 1984 Aug;38(1):127-34. PubMed PMID: 6432339."

[Oesch 1985]: http://www.ncbi.nlm.nih.gov/pubmed/2859120 "Oesch B, Westaway D, Wälchli M, McKinley MP, Kent SB, Aebersold R, Barry RA, Tempst P, Teplow DB, Hood LE, et al. A cellular gene encodes scrapie PrP 27-30 protein. Cell. 1985 Apr;40(4):735-46. PubMed PMID: 2859120."

[Carlson 1986]: http://www.ncbi.nlm.nih.gov/pubmed/3015416 "Carlson GA, Kingsbury DT, Goodman PA, Coleman S, Marshall ST, DeArmond S, Westaway D, Prusiner SB. Linkage of prion protein and scrapie incubation time genes. Cell. 1986 Aug 15;46(4):503-11. PubMed PMID: 3015416."

[Westaway 1987]: http://www.ncbi.nlm.nih.gov/pubmed/2890436 "Westaway D, Goodman PA, Mirenda CA, McKinley MP, Carlson GA, Prusiner SB. Distinct prion proteins in short and long scrapie incubation period mice. Cell. 1987 Nov 20;51(4):651-62. PubMed PMID: 2890436."

[Pan 1993]: http://www.ncbi.nlm.nih.gov/pubmed/7902575/ "Pan KM, Baldwin M, Nguyen J, Gasset M, Serban A, Groth D, Mehlhorn I, Huang Z, Fletterick RJ, Cohen FE, et al. Conversion of alpha-helices into beta-sheets features in the formation of the scrapie prion proteins. Proc Natl Acad Sci U S A. 1993 Dec 1;90(23):10962-6. PubMed PMID: 7902575; PubMed Central PMCID: PMC47901."

[Wickner 1994]: http://www.ncbi.nlm.nih.gov/pubmed/7909170 "Wickner RB. [URE3] as an altered URE2 protein: evidence for a prion analog in  Saccharomyces cerevisiae. Science. 1994 Apr 22;264(5158):566-9. PubMed PMID: 7909170."

[Bateman & Wickner 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23382698 "Bateman DA, Wickner RB. The [PSI+] prion exists as a dynamic cloud of variants. PLoS Genet. 2013;9(1):e1003257. doi: 10.1371/journal.pgen.1003257. Epub 2013 Jan 31. PubMed PMID: 23382698; PubMed Central PMCID: PMC3561065."

[Prusiner 1983]: http://www.ncbi.nlm.nih.gov/pubmed/6418385 "Prusiner SB, McKinley MP, Bowman KA, Bolton DC, Bendheim PE, Groth DF, Glenner GG. Scrapie prions aggregate to form amyloid-like birefringent rods. Cell. 1983 Dec;35(2 Pt 1):349-58. PubMed PMID: 6418385."

[Prusiner 1984]: http://www.ncbi.nlm.nih.gov/pubmed/6432339 "Prusiner SB, Groth DF, Bolton DC, Kent SB, Hood LE. Purification and structural studies of a major scrapie prion protein. Cell. 1984 Aug;38(1):127-34. PubMed PMID: 6432339."

[Oesch 1985]: http://www.ncbi.nlm.nih.gov/pubmed/2859120 "Oesch B, Westaway D, Wälchli M, McKinley MP, Kent SB, Aebersold R, Barry RA, Tempst P, Teplow DB, Hood LE, et al. A cellular gene encodes scrapie PrP 27-30 protein. Cell. 1985 Apr;40(4):735-46. PubMed PMID: 2859120."

[Bueler 1993]: http://www.ncbi.nlm.nih.gov/pubmed/8100741 "Büeler H, Aguzzi A, Sailer A, Greiner RA, Autenried P, Aguet M, Weissmann C. Mice devoid of PrP are resistant to scrapie. Cell. 1993 Jul 2;73(7):1339-47. PubMed PMID: 8100741."

[Kocisko 1994]: http://www.ncbi.nlm.nih.gov/pubmed/7913989 "Kocisko DA, Come JH, Priola SA, Chesebro B, Raymond GJ, Lansbury PT, Caughey B. Cell-free formation of protease-resistant prion protein. Nature. 1994 Aug 11;370(6489):471-4. PubMed PMID: 7913989."

[Prusiner 1998]: http://www.ncbi.nlm.nih.gov/pubmed/9811807 "Prusiner SB. Prions. Proc Natl Acad Sci U S A. 1998 Nov 10;95(23):13363-83. Review. PubMed PMID: 9811807; PubMed Central PMCID: PMC33918."

[Weissmann 1991]: http://www.ncbi.nlm.nih.gov/pubmed/1876183 "Weissmann C. A 'unified theory' of prion propagation. Nature. 1991 Aug 22;352(6337):679-83. Review. PubMed PMID: 1876183."

[Bessen 1995]: http://www.ncbi.nlm.nih.gov/pubmed/7791905 "Bessen RA, Kocisko DA, Raymond GJ, Nandan S, Lansbury PT, Caughey B. Non-genetic propagation of strain-specific properties of scrapie prion protein. Nature. 1995 Jun 22;375(6533):698-700. PubMed PMID: 7791905."

[Telling 1996]: http://www.ncbi.nlm.nih.gov/pubmed/8953038 "Telling GC, Parchi P, DeArmond SJ, Cortelli P, Montagna P, Gabizon R, Mastrianni J, Lugaresi E, Gambetti P, Prusiner SB. Evidence for the conformation  of the pathologic isoform of the prion protein enciphering and propagating prion  diversity. Science. 1996 Dec 20;274(5295):2079-82. PubMed PMID: 8953038."

[Safar 1998]: http://www.ncbi.nlm.nih.gov/pubmed/9771749 "Safar J, Wille H, Itri V, Groth D, Serban H, Torchia M, Cohen FE, Prusiner SB. Eight prion strains have PrP(Sc) molecules with different conformations. Nat Med. 1998 Oct;4(10):1157-65. PubMed PMID: 9771749."

[Legname 2004]: http://www.ncbi.nlm.nih.gov/pubmed/15286374 "Legname G, Baskakov IV, Nguyen HO, Riesner D, Cohen FE, DeArmond SJ, Prusiner  SB. Synthetic mammalian prions. Science. 2004 Jul 30;305(5684):673-6. PubMed PMID: 15286374."

[Sparrer 2000]: http://www.ncbi.nlm.nih.gov/pubmed/10915616 "Sparrer HE, Santoso A, Szoka FC Jr, Weissman JS. Evidence for the prion hypothesis: induction of the yeast [PSI+] factor by in vitro- converted Sup35 protein. Science. 2000 Jul 28;289(5479):595-9. PubMed PMID: 10915616."

[Chien 2003]: http://www.ncbi.nlm.nih.gov/pubmed/12931190 "Chien P, DePace AH, Collins SR, Weissman JS. Generation of prion transmission  barriers by mutational control of amyloid conformations. Nature. 2003 Aug 21;424(6951):948-51. PubMed PMID: 12931190."

[Liebman & Derkatch 1999]: http://www.ncbi.nlm.nih.gov/pubmed/9880481 "Liebman SW, Derkatch IL. The yeast [PSI+] prion: making sense of nonsense. J Biol Chem. 1999 Jan 15;274(3):1181-4. Review. PubMed PMID: 9880481."

[King & Diaz-Avalos 2004]: http://www.ncbi.nlm.nih.gov/pubmed/15029195 "King CY, Diaz-Avalos R. Protein-only transmission of three yeast prion strains. Nature. 2004 Mar 18;428(6980):319-23. PubMed PMID: 15029195."

[Scott 1989]: http://www.ncbi.nlm.nih.gov/pubmed/2574076 "Scott M, Foster D, Mirenda C, Serban D, Coufal F, Wälchli M, Torchia M, Groth  D, Carlson G, DeArmond SJ, Westaway D, Prusiner SB. Transgenic mice expressing hamster prion protein produce species-specific scrapie infectivity and amyloid plaques. Cell. 1989 Dec 1;59(5):847-57. PubMed PMID: 2574076."

[Chernoff 1995]: http://www.ncbi.nlm.nih.gov/pubmed/7754373 "Chernoff YO, Lindquist SL, Ono B, Inge-Vechtomov SG, Liebman SW. Role of the chaperone protein Hsp104 in propagation of the yeast prion-like factor [psi+]. Science. 1995 May 12;268(5212):880-4. PubMed PMID: 7754373."

[Tanaka 2006]: http://www.ncbi.nlm.nih.gov/pubmed/16810177 "Tanaka M, Collins SR, Toyama BH, Weissman JS. The physical basis of how prion  conformations determine strain phenotypes. Nature. 2006 Aug 3;442(7102):585-9. Epub 2006 Jun 28. PubMed PMID: 16810177."

[Colby 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19915150 "Colby DW, Giles K, Legname G, Wille H, Baskakov IV, DeArmond SJ, Prusiner SB.  Design and construction of diverse mammalian prion strains. Proc Natl Acad Sci U  S A. 2009 Dec 1;106(48):20417-22. doi: 10.1073/pnas.0910350106. Epub 2009 Nov 13. PubMed PMID: 19915150; PubMed Central PMCID: PMC2787151."

[Deleault 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17535913/ "Deleault NR, Harris BT, Rees JR, Supattapone S. Formation of native prions from minimal components in vitro. Proc Natl Acad Sci U S A. 2007 Jun 5;104(23):9741-6. Epub 2007 May 29. Erratum in: Proc Natl Acad Sci U S A. 2008 Aug 26;105(34):12636. PubMed PMID: 17535913; PubMed Central PMCID: PMC1887554."

[Wang 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20110469 "Wang F, Wang X, Yuan CG, Ma J. Generating a prion with bacterially expressed recombinant prion protein. Science. 2010 Feb 26;327(5969):1132-5. doi: 10.1126/science.1183748. Epub 2010 Jan 28. PubMed PMID: 20110469; PubMed Central  PMCID: PMC2893558."

[Deleault 2012a]: http://www.ncbi.nlm.nih.gov/pubmed/22586108 "Deleault NR, Piro JR, Walsh DJ, Wang F, Ma J, Geoghegan JC, Supattapone S. Isolation of phosphatidylethanolamine as a solitary cofactor for prion formation  in the absence of nucleic acids. Proc Natl Acad Sci U S A. 2012 May 29;109(22):8546-51. doi: 10.1073/pnas.1204498109. Epub 2012 May 14. PubMed PMID:  22586108; PubMed Central PMCID: PMC3365173."

[Deleault 2012b]: http://www.ncbi.nlm.nih.gov/pubmed/22711839 "Deleault NR, Walsh DJ, Piro JR, Wang F, Wang X, Ma J, Rees JR, Supattapone S.  Cofactor molecules maintain infectious conformation and restrict strain properties in purified prions. Proc Natl Acad Sci U S A. 2012 Jul 10;109(28):E1938-46. doi: 10.1073/pnas.1206999109. Epub 2012 Jun 18. PubMed PMID: 22711839; PubMed Central PMCID: PMC3396481."

[Hsiao 1989]: http://www.ncbi.nlm.nih.gov/pubmed/2564168 "Hsiao K, Baker HF, Crow TJ, Poulter M, Owen F, Terwilliger JD, Westaway D, Ott J, Prusiner SB. Linkage of a prion protein missense variant to Gerstmann-Sträussler syndrome. Nature. 1989 Mar 23;338(6213):342-5. PubMed PMID:  2564168."

[Medori 1992]: http://www.ncbi.nlm.nih.gov/pubmed/1346338 "Medori R, Tritschler HJ, LeBlanc A, Villare F, Manetto V, Chen HY, Xue R, Leal S, Montagna P, Cortelli P, et al. Fatal familial insomnia, a prion disease with a mutation at codon 178 of the prion protein gene. N Engl J Med. 1992 Feb 13;326(7):444-9. PubMed PMID: 1346338."

[Lansbury & Caughey 1995]: http://www.ncbi.nlm.nih.gov/pubmed/9383397 "Lansbury PT Jr, Caughey B. The chemistry of scrapie infection: implications of the 'ice 9' metaphor. Chem Biol. 1995 Jan;2(1):1-5. Review. PubMed PMID: 9383397."

[Wickner 2011]: http://www.ncbi.nlm.nih.gov/pubmed/22052353 "Wickner RB, Edskes HK, Bateman D, Kelly AC, Gorkovskiy A. The yeast prions [PSI+] and [URE3] are molecular degenerative diseases. Prion. 2011 Oct-Dec;5(4):258-62. doi: 10.4161/pri.17748. Epub 2011 Oct 1. Review. PubMed PMID: 22052353; PubMed Central PMCID: PMC4012404."

[Halfmann & Jarosz 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22337056 "Halfmann R, Jarosz DF, Jones SK, Chang A, Lancaster AK, Lindquist S. Prions are a common mechanism for phenotypic inheritance in wild yeasts. Nature. 2012 Feb 15;482(7385):363-8. doi: 10.1038/nature10875. PubMed PMID: 22337056; PubMed Central PMCID: PMC3319070."

[Hsiao 1991]: http://www.ncbi.nlm.nih.gov/pubmed/2008182 "Hsiao K, Meiner Z, Kahana E, Cass C, Kahana I, Avrahami D, Scarlato G, Abramsky O, Prusiner SB, Gabizon R. Mutation of the prion protein in Libyan Jews  with Creutzfeldt-Jakob disease. N Engl J Med. 1991 Apr 18;324(16):1091-7. PubMed  PMID: 2008182."