---
layout: post
title:  "Chemical biology 16: infectious disease"
date:   2015-11-05 12:00:00
author: ericminikel
tag: chem-101
location: Cambridge, MA
summary200: ""
---

*These are my notes from lecture 16 of Harvard's Chemistry 101: Chemical Biology Towards Precision Medicine course, taught by Dr. Stuart Schreiber on November 5, 2015. I missed class so this is based on the slides and notes.*

Key points for today:

+ The discovery of penicillin marked the beginning of a new era in human medicine, and a new challenge for synthetic chemists.
+ Drug-resistant microbes continue to pose a major public health threat.
+ Microbial genomics has offered new solutions, but the diversity of screening collections remains a limitation. 

The discovery of penicillin, a beta lactam antibiotic, marked a revolution in the treatment of infectious disease, and triggered new efforts to synthesize antibiotics. R.B. Woodward won the [1965 Nobel Prize in chemistry](http://www.nobelprize.org/nobel_prizes/chemistry/laureates/1965/) for synthesis of beta lactam antibiotics including cephalosporin C. However, despite initial hopes that antibiotics would spell an end to bacterial infections, it turns out that the proportion of deaths worldwide that are attributed to infectious disease is about the same now as it was before the discovery of penicillin (~25%).

Analysis of 30,000 year old permafrost-preserved bacteria reveals that the genes encoding resistance to beta lactams, tetracyclines, and glycopeptides were already present back then [[D'Costa 2011]]; modern antibiotics merely applied selective pressure on these genes. Antibiotic resistance is achieved by just about every mechanism you can imagine. For instance, to achieve resistance to vancomycin, which is considered a last line of defense, bacteria make a fundamental change in their cell wall composition, changing an amide bond to an ester bond. 

Genomics offers powerful new tools for the study of microbes and their drug resistance. For instance, selection and sequencing of resistant colonies and comparison to drug-sensitive parental colonies can reveal the genes that encode resistance. Also, genome-wide mutagenesis screens can reveal which genes are essential and non-essential in microbes. Yet, high throughput screening campaigns against the new targets nominated by such screens have had only limited success. Right now, novel antibiotics are simply not being discovered. This may be in large part due to a lack of sufficient chemical diversity, and established rules for drug discovery such as Lipinsky's rule of five (R05) may be especially problematic: the set of existing approved antibiotics have very little overlap with R05-compliant chemical space.

In antimicrobials more than any other class of drugs, chemical diversity is the major bottleneck to new drug discovery. Live/dead phenotypic screening coupled with sequencing of resistant colonies provides a clear path forward, and the fact that pathogen targets are not usually present in the host means that toxicity is somewhat less of a roadblock than when you are screening against host targets. Yet hitting these new targets requires better chemistries.




[D'Costa 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21881561 "D'Costa VM, King CE, Kalan L, Morar M, Sung WW, Schwarz C, Froese D, Zazula G, Calmels F, Debruyne R, Golding GB, Poinar HN, Wright GD. Antibiotic resistance is ancient. Nature. 2011 Aug 31;477(7365):457-61. doi: 10.1038/nature10388. PubMed PMID: 21881561."



