---
layout: post
title:  "Genetics in medicine 08: Inflammatory bowel disease"
date:   2015-03-27 12:01:00
author: ericminikel
tag: genetics-228
location: Cambridge, MA
summary200: ""
---

*These are my notes from week 8 of Harvard's [Genetics 228: Genetics in Medicine from Bench to Bedside](http://www2.massgeneral.org/bbs/gen228/gen228_syllabus_material.asp) course, held on March 27, 2015. Lectures by [Mark Daly, chief of the Analytical and Translational Genetics Unit at MGH](https://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/mark-daly) and [Ramnik Xavier, chief of Gastroenterology at MGH](https://www.broadinstitute.org/bios/ramnik-xavier), and a journal club presentation by me, Rebecca S. Fine and Vinita Joshi.*

### Background

For journal club today, my group is presenting the paper *Genetic and epigenetic fine mapping of causal autoimmune disease variants* [[Farh & Marson 2015]]. Here's a quick backgrounder.

#### Successes and limitations of genome-wide association studies

The concept of a genome-wide association study (GWAS) is introduced [here](/2014/11/21/genetics-25/). The first GWAS is sometimes credited to be a study of 116,000 SNPs in 96 cases and 50 controls for age-related macular degeneration in 2005 [[Klein 2005]]. But it was not until a couple of years later that GWAS really hit their stride, reaching the scale needed for locus discovery in common diseases, when the Wellcome Trust Case Control Consortium genotyped 17,000 people on 500,000 SNPs [[WTCCC 2007]; see also historical review in [Visscher 2012]]. Since then, over 1,700 GWAS have been published, reporting associations for over 11,000 SNPs [[Welter 2014]].

A few general trends have emerged. Only about 11% of GWAS loci are in protein-coding regions [[Gusev 2014]], which is a dramatic enrichment (as only about 1% of the genome is protein-coding), but still means that the vast majority of causal variants must be non-coding. 57% of GWAS hits lie in open chromatin (DNAse I hypersensitivity regions) [[Gusev 2014]], and expression quantatitive trait loci (eQTLs; variants that affect gene expression) are enriched among GWAS hits [[Nicolae 2010]]. Thus, much of the functional impact of causal variants might lie in their effects on gene expression.

Still, there are only rare cases where people have been able to confidently identify a single variant as driving the GWAS association at a particular locus. The reason that GWAS work at all is also their greatest weakness. Recombination hotspots effectively divide the human genome into haplotype blocks of 2-200kb [[International HapMap Consortium 2003] and also see [here](/2014/11/19/genetics-24/)], within which genetic variants almost always segregate together. Suppose a particular 20kb haplotype block contains 20 SNPs, one of which is causally associated with your phenotype of interest. Then genotyping *any* SNP in that block (or even nearby) will reveal that association. That's why GWAS work - you don't need to genotype the causal SNP to discover an association. But this also makes it incredibly difficult to tell *which* of those 20 SNPs is causal. And until you know the causal variant, it can be hard to reach the sorts of biological insights that will lead to therapeutic avenues or fundamental basic science insights.

Therefore people have devised a lot of strategies for trying to at least narrow down the set of causal variants at a phenotype-associated locus. One approach relevant to this paper is **fine-mapping**. This is when you re-genotype the same people (or more people, if you have them!) with a chip that has a much denser array of probes at the causal locus. For instance, the [ExomeChip](http://genome.sph.umich.edu/wiki/Exome_Chip_Design), [Immunochip](http://www.ibdgenetics.org/projects.html) or [PsychChip](http://www.med.unc.edu/pgc/psychchip). Why didn't we just put all of those SNPs on the GWAS array in the first place, you ask? Capacity on chips isn't free or infinite, and GWAS discovery chips tend to prioritize uniform coverage of the genome and very high quality, readily genotyped SNPs. These fine-mapping chips instead try to exhaustively capture variants found in sequencing studies such as 1000 Genomes or ESP, even if it means some of the probes are of lower quality and do not uniformly cover the genome. Even with fine-mapping, you'll still end up with a lot of variants at one locus all associated with your phenotype, so this is by no means the end of the story. If you can't afford to re-genotype your study participants or don't have any DNA left, you can try to *guess* what other SNPs they have in between SNPs you've genotyped, based on the haplotype structure of 1000 Genomes participants - this is called **imputation**. This is a substitute for fine-mapping and has all the same limitations, plus its own limitations - you won't guess genotypes correctly 100% of the time.

So, as a very simplified example to show how these things work, consider this imaginary locus (code to generate this plot is [here](https://github.com/ericminikel/cureffi/blob/gh-pages/media/2015/03/gwas_visual_aids.r)):

![](/media/2015/03/gwas_visual_aid.png)

You're interested in figuring out, of all the genetic variants in the population (gray bars), which are associated with your phenotype. As a first pass, you genotype a bunch of cases and controls for only a few, relatively uniformly distributed, well-genotyped SNPs (black bars). You find a few SNPs nominally associated, and two SNPs that are significantly associated with your phenotype - they're above the genome-wide significance threshold of p = 5&times;10<sup>-8</sup>. So then you go back with a fine-mapping chip, and you genotype much more (but still not all) of the variation here (orange bars), and you find a bunch more SNPs that are associated, including several that are above the genome-wide threshold and a few that are more significant than your original GWAS hits. But you still don't know which, if any, of the SNPs you've genotyped is causal, so how can you make any biological inferences? That question is where this paper picks up.

#### The genetics of autoimmunity

Of all the diseases and phenotypes that have been subjected to GWAS, autoimmune diseases must rank among either the most challenging or the most successful, depending on your viewpoint. I haven't found an updated figure for 2015, but as of three years ago, there were already 193 independent genomic loci associated with inflammatory bowel disease (IBD) [[Jostins 2012]]. This degree of polygenicity is so vexing that it has motivated whole new statistical approaches to determine whether the associations are largely real or result from confounding effects of population stratification [[Bulik-Sullivan 2015]]. Specific functional variants have been identified in a handful of loci - see for instance *CARD9* [[Rivas 2011]] - but for many or most of these loci, we don't know how the genetic variation impacts gene function, and we don't know if increased or decreased gene expression would associate with disease risk.

### [Mark Daly](https://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/mark-daly) - Crohn's disease: gene discovery and beyond

Dr. Daly phrases the goal of human genetics as:

> To provide durable insights into human disease biology that ultimately impact clinical care.

In most areas of epidemiology, correlation does not imply causation. The power of human genetics lies in the fact that correlation does imply causation - you are born with your genome and get the disease later; the disease does not change your genome.

A secondary focus of human genetics might be to deliver individualized medical insights - diagnosis in severe genetic disorders, identification of individuals predicted to respond besst to a therapy, prediction of adverse drug responses. The potential for these sorts of personalized insight has been dramatically oversold for years, and for the most part has not lived up to the hype, though we may now be nearing the time when some of these promises do become reality.

Linkage mapping was developed by Alfred Sturtevant and Thomas Hunt Morgan in 1913, as a purely mathematical concept decades before we knew the structure of DNA, or the genetic code, or had any tools for experimental manipulation of DNA. It was also recognized early on that certain human diseases - Huntington disease, cystic fibrosis, muscular dystrophies - were inherited in a Mendelian fashion. In the early 1980s it became possible to track the co-segregation of DNA markers with these diseases, and by the late 1980s this began to enable the identification of causal genetic variants in many of these diseases. This was possible because there is a deterministic relationship whereby genotype causes disease state.

By 2002 there were ~1800 Mendelian diseases for which the causal variant had been found, but fewer than 10 known complex trait genes [[Glazier 2002]]. It had been known for a century that there are many traits that are highly heritable - height, for instance, may be ~80% heritable. Yet for many traits, finding the genes that determine them was nearly impossible. In 1918, the statistician R.A. Fisher [put forth](http://en.wikipedia.org/wiki/The_Correlation_between_Relatives_on_the_Supposition_of_Mendelian_Inheritance) the possible explanation that such traits might be controlled by a large number of genes that did segregate in a Mendelian fashion but each had only a small effect size [[Fisher 1918](http://l.academicdirect.org/Horticulture/GAs/Refs/Fisher_1918_Correlation.pdf)].

In the past decade, a new approach has become feasible. We can systematically compare DNA variation between two groups of samples. For instance, in microbiology we can compare antibiotic resistant and sensitive bacteria to identify the variants that confer resistance. In cancer we can compare tumor and normal DNA to identify mutations that drive cancer. And in common diseases and traits, we can compare individuals with or without the disease to find the variants that control the trait.

Soon after the Human Genome Project was completed, it was recognized that human heterozygosity is relatively limited (~1 base pair in 1000), and most of it could be attributed to *common* genetic variants [[Gabriel 2002]]. When you discover a SNP that is heterozygous in one individual (differs between two chromosomes) and then go out and genotype that SNP in a thousand other people, about 90% of the time you discover that the variant has a frequency of >1% in the population.

It turns out that humans are not very different from one another. In fact, we are among the least diverse organisms on the planet. Almost all species ever studied have a greater amount of DNA variation than we do. We began to catalog the structure and frequency of genetic variation in humans. There are only about 10 million common genetic variants in humans, and most of them arose from a single mutational event in our history. The data also resoundingly confirmed the anthropologists' "out of Africa" hypothesis that humankind had originated in Africa.  The mapping of Mendelian disease genes actually relies on common variants. The evidence that the cystic fibrosis gene was on chromosome 7 came from the observation that over half of CF patient chromosomes had an identical common variant haplotype of about 500kb. Usefully, recombination events tend to occur at specific hotspots, thus creating haplotype "blocks" of tens or sometimes hundreds of kb, within which there is almost never recombination [[Daly 2001]] - thus, common variants within a block segregate together.

Once all of these discoveries were made, three fundamental changes came about that enabled common disease genetics:

1. Large consortia - first the SNP Consortium, then HapMap and then 1000 Genomes
2. Advances in technology - first genotyping arrays, then sequencing
3. Collaborations among disease researchers - more on this later

Inflammatory bowel disease appeared to have many advantages for common trait genetics. It is highly heritable: it has 15-35x risk in siblings of affected individuals, much higher monozygotic than dizygotic twin concordance, and 5-10% of affected people have a 1st degree relative who is also affected.

Early linkage studies enabled the 2001 discovery of *NOD2* at 5q31 which explained 2-3% of the heritability of inflammatory bowel disease. Then all of the years from 2002 to 2008 were spent on candidate gene studies which yielded no new validated hits. Finally, by 2008, genome-wide association studies (GWAS) became possible. However, it soon became clear that GWAS were different from anything that had come before them in molecular biology. In all of molecular biology up to that point, you got a gel or some sort of readout that you could look at with the naked eye and decide whether you believed the data. With GWAS you could now get a volume of data that no one human would ever look at with the naked eye. Thus, we had to develop:

+ Technical QC - removing failed SNPs and samples
+ Genetic QC - Mendelian segregation, Hardy-Weinberg equilibrium, estimating relatedness and gender, controlling for population structure
+ Analytical QC - checking for inflation of test statistics, biases toward missing data, etc

Those QC efforts were 95% of the labor. The actual effort of testing for association is remarkably simple. You basically just make a contingency table of allele A vs. allele C in cases vs. controls and do a Chi-square test.

In the first Crohn's disease Gwas, after about 500 cases and 500 controls, the highest-ranked SNP had a p value on the order of 1e-7. At first, this looked good because in linkage studies in families, a LOD score of ~3 was considered significant, because there are only so many arms of chromosomes, only so many recombination events in one family. But quickly they realized that because large populations have had many recombination events and have a 3.2Gb genome broken into, say, 3.2kb blocks, we are really doing more on the order of one million independent statistical tests in a genome. Thus, p = .05 &times; 1/(1 million) = 5e-8 became the standard threshold for genome-wide significance. Moreover, people decided that they almost never believe a single association study - only when an independent technology and independent collection of cases and controls have been used to replicate the result does it become real. 

That original hit in Crohn's disease was not real, but soon they had 1000 cases and 1000 controls, and found two genuine genome-wide significant peaks, one at *NOD2* which was already known since 2002, and one at *IL23R* which turned out to be a coding variation.

Two plots became standard in conveying the results of genome-wide association studies: the Manhattan plot and the Q-Q plot. The Manhattan plot is mostly just eye candy and (usually) doesn't tell you much about the strength of statistical evidence for association with disease. The Q-Q plot is important because if the associations are real, it will show you a distribution of p values perfectly consistent with expectation (no inflation), except at the very high end where your real hits are.

To get the numbers needed to find a large number of hits, you really needed more samples. Thus, the third fundamental change (above) was that disease communities - which used to compete to find the *one* causal gene in Mendelian disease - realized that there were many genes to be found and they could afford to collaborate to find them all. Therefore, people started to combine data from multiple GWAS studies in different case and control cohorts genotyped on different technologies. The different technologies would genotype a different set of SNPs, which meant that to compare them it was necessary to impute the missing SNPs on each chip using the HapMap reference panel. Once this was done, the first meta-analysis in 2008 revealed that simply by combining data from three already-published studies, you could find many new loci that were not significant in any one study.

The findings of GWAS have proven impressively durable. All 32 of the GWAS hits from the first IBD meta-GWAS [[Barrett 2008]] have been confirmed and are actually more significant in the 2012 GWAS data [[Jostins 2012]].

Does it matter that all of the GWAS hits have very small influences on the disease traits? No, because the (small) effects of common variation do not put a limit on the (large) potential effects of a drug. For instance, common variation in *HMGCR* has only a very small (~5%) influence on LDL levels, yet inhibition of HGMCR with statins has a large (~30%) effect on LDL. Human genetics is constrained by natural selection - common variants (say, *HMGCR*) have to have small effects, while variants of large effect (say, *LDLR* mutations) are very rare. But drugs are under no such constraint - you can develop a drug with a large effect, and give it to a large number of people.

The major challenge coming from GWAS is how to identify the exact variants that drive GWAS associations and, mechanistically, how they affect disease risk. There are a few success stories - for instance, null alleles of *FUT2* confer norovirus resistance but also increase IBD risk - but more often we do not know why a given locus affects disease risk, or even which gene is involved.

Many of the most valuable insights from GWAS have been from the overlap of associations between one disease and another - for instance, the overlap of IBD loci with leprosy GWAS and with Mendelian primary immunodeficiency genes has told us a lot about the biological pathways shared between these conditions. Similarly, it has been useful to discover the concordance (or lack thereof) between distinct but clinically similar diseases. Ulcerative colitis and Crohn's disease share many but not all of their loci, and a polygenic "risk score" based on genotypes at these loci adds a considerable amount of diagnostic accuracy (when combined with the conventional smoking and serological status meausres) at discriminating the two diseases. The patients that had been mis-classfied were almost categorically those who had responded the least well to treatment, so re-assessing their diagnosis using genomics is important.

Promisingly, exome and genome sequencing now make it possible both to find rare, more highly penetrant, genetic variants, and to identify causal common variants at GWAS loci. An example success story was the discovery of homozygous essential splice site mutations in *DGAT1* in two children with a congenital severe protein-shedding enteropathy. These sorts of insights are valuable both personally - for genetic testing and in some cases for guided therapies - and also for fundamental biology. *DGAT1* is an example where the knockout phenotype is very divergent in mice and in humans. *Dgat1* knockout mice have very low lipids or something, so millions of dollars had been invested in developing drugs to inhibit DGAT1, but these caused a very high rate of adverse side effects. The discovery of this severe *DGAT1* loss-of-function disease in humans explained why that was the case. If we had been guided by human genetics to begin with, we wouldn't have made that mistake.

Indeed, being guided by human genetics can lead to promising new drug targets. *SCN9A* (Nav1.9) knockout causes congenital indifference to pain [[Cox 2006]], and gain-of-function mutations cause extreme pain disorder. There are now small molecule inhibitors of Nav1.9 [[Yang 2013]]. Another great example is [*PCSK9*](/2014/11/19/primer-on-pcsk9-genetics/). In IBD, the most promising genetic result is that a rare splice variant that abolishes part of the protein [[Rivas 2011]] confers dramatically reduced risk of IBD, motivating an effort to find a drug that would mimic the effect of that genetic variant.

#### Q&A

Q. Doesn't it seem like hits that are just barely above the 5e-8 threshold turn out to be false more often than 1 in 20 times?

A. Actually, hits that are just *below* 5e-8 later turn out to be *real* much more often than you'd think. This is for a Bayesian reason. Now that we know that many of the diseases we study in GWAS are so polygenic, there is a strong prior that a real association would be the one that "happens" to be at the high end of the null distribution, at say 1e-7 or something. However, we chose the 5e-8 threshold to be conservative because back in the linkage era we hadn't been conservative enough. Linkage studies are only powered to find variants of large effect, yet people had applied them to diseases where the prior for common variants of large effect was extremely small. Therefore thousands of people would do those linkage studies, and 1 in 20 would find (and publish) a positive result, all of which were false.

### [Ramnik Xavier](https://www.broadinstitute.org/bios/ramnik-xavier) - Inflammatory bowel disease

IBD is divided into ulcerative colitis (UC), Crohn's disease (CD), and "intermediate colitis" (intermediate phenotype, 10-20% of patients). Each can be divided into several subtypes and it is possible that we will eventually consider them distinct diseases. Each of these diseases is rapidly progressive, with inflammation spreading across the colon in a matter of 2 weeks. UC typically presents first with bleeding, while CD presents first with abdominal pain resulting from stricture preventing flow through the bowels. In some cases, ulceration can reach all the way through to the skin, resulting in visible fistulas.

Ulcerative colitis usually has the same histological profile, can follow approximately four very different disease courses [[Solberg 2009]] - some patients have one bout, then a remission of >10 years; others have recurrence every 5 years; others have constant disease; and so on.

Treatment outcomes are quite poor. IBD does not respond to steroids (unlike other inflammation). Children experiencing their first bout do have longer remission if treated with 6-mercaptopurine. There are several known environmental factors. Smoking makes ulcerative colitis better but Crohn's disease worse. 

IBD is an interesting disease because of the interaction between genetics, immunity and microbial flora. Before 2002, people had theories that we'd find a specific gut pathogen that triggers IBD - that never happened, but we have now gained a more nuanced understanding of the ways that the gut microbiome does influence disease risk. 

The RISK cohort was a combination of 28 pediatric IBD centers to sample gut microbiomes of 1300 children with Crohn's before treatment, during their first attack. Microbes are identified by sequencing their 16S RNA. The "good" bugs (especially *Clostridiales*) were decreased in cases vs. controls [[Gevers 2014]]. Sadly, treatment with antibiotics - which many parents of kids with IBD demand and physicians give in - further depletes the "good" bugs and results in far worse outcomes. There are now efforts to develop a therapeutic microbe supplement full of the good bugs.

One key pathway that has emerged from IBD GWAS is the TH17 pathway, specific to TH17 cells. GWAS loci at *IL23R*, *RORC*, *CCR6* , *STAT3*, and Aiolos (what is that??) all point to this pathway. An early hypothesis to explain the *IL23R* cells was that the protective variant reduced the number of circulating TH17 cells. Instead, it turns out the number of cells is the same but they secrete less TH17 protein.

Another revelation from IBD genetics has been that IBD genes fall into three very broad categories: those that control homeostasis, those that control inflammation, and those that control healing (affecting recovery after an attack of inflammation).

An ongoing effort is to do extreme phenotype exome sequencing - for instance, looking for protective rare variants in people who have a high polygenic risk score based on common variants but who do not have the disease. 

Prior to 2007, there were no references on Crohn's disease and autophagy. Beginning in 2007, two GWAS identified *ATG16L1* T300A [[Hampe 2007]] and a 20-kb promoter deletion upstream of *IRGM* as being associated with IBD. Both of these are autophagy genes. Atg16 (encoded by *ATG16L1*) is a core component of the autophagosome machinery, but the T300A missense variant is in the C-terminal domain which is evolutionarily more recent. The N-terminus of *ATG16L1* has a homologue in yeast, but the C-terminal domain does not and thus probably is not involved in the protein's core conserved activities, but rather in some higher-order function. It turns out that the risk allele of *ATG16L1* is much less effective at clearing bacteria in culture [[Kuballa 2008]], and T300A knock-in mice have fewer Paneth cells and enlarged goblet cells [[Lassen 2014]].

It is increasingly clear that abnormal Paneth cells are granuloma-poor and have a reduced ability to contain pathogens, and these abnormal cells are much more often found in patients with *NOD2* or *ATG16L1* risk variants, possibly defining a new subtype of Crohn's disease [[VanDussen 2014]]. 



[Hindorff 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19474294 "Hindorff LA, Sethupathy P, Junkins HA, Ramos EM, Mehta JP, Collins FS, Manolio TA. Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. Proc Natl Acad Sci U S A. 2009 Jun 9;106(23):9362-7. doi: 10.1073/pnas.0903103106. Epub 2009 May 27. PubMed PMID: 19474294; PubMed Central PMCID: PMC2687147."

[Gusev 2014]: http://www.ncbi.nlm.nih.gov/pubmed/25439723 "Gusev A, Lee SH, Trynka G, Finucane H, Vilhjálmsson BJ, Xu H, Zang C, Ripke S, Bulik-Sullivan B, Stahl E; Schizophrenia Working Group of the Psychiatric Genomics Consortium; SWE-SCZ Consortium, Kähler AK, Hultman CM, Purcell SM, McCarroll SA, Daly M, Pasaniuc B, Sullivan PF, Neale BM, Wray NR, Raychaudhuri S, Price AL; Schizophrenia Working Group of the Psychiatric Genomics Consortium; SWE-SCZ Consortium. Partitioning heritability of regulatory and cell-type-specific variants across 11 common diseases. Am J Hum Genet. 2014 Nov 6;95(5):535-52. doi: 10.1016/j.ajhg.2014.10.004. Epub 2014 Nov 6. PubMed PMID: 25439723; PubMed Central PMCID: PMC4225595."

[Trynka 2011]: http://www.ncbi.nlm.nih.gov/pubmed/22057235 "Trynka G, Hunt KA, Bockett NA, Romanos J, Mistry V, Szperl A, Bakker SF, Bardella MT, Bhaw-Rosun L, Castillejo G, de la Concha EG, de Almeida RC, Dias KR, van Diemen CC, Dubois PC, Duerr RH, Edkins S, Franke L, Fransen K, Gutierrez J, Heap GA, Hrdlickova B, Hunt S, Plaza Izurieta L, Izzo V, Joosten LA, Langford C,  Mazzilli MC, Mein CA, Midah V, Mitrovic M, Mora B, Morelli M, Nutland S, Núñez C, Onengut-Gumuscu S, Pearce K, Platteel M, Polanco I, Potter S, Ribes-Koninckx C, Ricaño-Ponce I, Rich SS, Rybak A, Santiago JL, Senapati S, Sood A, Szajewska H, Troncone R, Varadé J, Wallace C, Wolters VM, Zhernakova A; Spanish Consortium on  the Genetics of Coeliac Disease (CEGEC); PreventCD Study Group; Wellcome Trust Case Control Consortium (WTCCC), Thelma BK, Cukrowska B, Urcelay E, Bilbao JR, Mearin ML, Barisani D, Barrett JC, Plagnol V, Deloukas P, Wijmenga C, van Heel DA. Dense genotyping identifies and localizes multiple common and rare variant association signals in celiac disease. Nat Genet. 2011 Nov 6;43(12):1193-201. doi: 10.1038/ng.998. PubMed PMID: 22057235; PubMed Central PMCID: PMC3242065."

[Cortes 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21345260 "Cortes A, Brown MA. Promise and pitfalls of the Immunochip. Arthritis Res Ther. 2011 Feb 1;13(1):101. doi: 10.1186/ar3204. Review. PubMed PMID: 21345260; PubMed Central PMCID: PMC3157635."

[Parkes 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23917628 "Parkes M, Cortes A, van Heel DA, Brown MA. Genetic insights into common pathways and complex relationships among immune-mediated diseases. Nat Rev Genet. 2013 Sep;14(9):661-73. doi: 10.1038/nrg3502. Epub 2013 Aug 6. Review. PubMed PMID: 23917628."

[Farh & Marson 2015]: http://www.ncbi.nlm.nih.gov/pubmed/25363779 "Farh KK, Marson A, Zhu J, Kleinewietfeld M, Housley WJ, Beik S, Shoresh N, Whitton H, Ryan RJ, Shishkin AA, Hatan M, Carrasco-Alfonso MJ, Mayer D, Luckey CJ, Patsopoulos NA, De Jager PL, Kuchroo VK, Epstein CB, Daly MJ, Hafler DA, Bernstein BE. Genetic and epigenetic fine mapping of causal autoimmune disease variants. Nature. 2015 Feb 19;518(7539):337-43. doi: 10.1038/nature13835. Epub 2014 Oct 29. PubMed PMID: 25363779; PubMed Central PMCID: PMC4336207."

[Klein 2005]: http://www.ncbi.nlm.nih.gov/pubmed/15761122/ "Klein RJ, Zeiss C, Chew EY, Tsai JY, Sackler RS, Haynes C, Henning AK, SanGiovanni JP, Mane SM, Mayne ST, Bracken MB, Ferris FL, Ott J, Barnstable C, Hoh J. Complement factor H polymorphism in age-related macular degeneration. Science. 2005 Apr 15;308(5720):385-9. Epub 2005 Mar 10. PubMed PMID: 15761122; PubMed Central PMCID: PMC1512523."

[WTCCC 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17554300/ "Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. Nature. 2007 Jun 7;447(7145):661-78. PubMed PMID: 17554300; PubMed Central PMCID: PMC2719288."

[Visscher 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22243964/ "Visscher PM, Brown MA, McCarthy MI, Yang J. Five years of GWAS discovery. Am J Hum Genet. 2012 Jan 13;90(1):7-24. doi: 10.1016/j.ajhg.2011.11.029. Review. PubMed PMID: 22243964; PubMed Central PMCID: PMC3257326."

[Welter 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24316577 "Welter D, MacArthur J, Morales J, Burdett T, Hall P, Junkins H, Klemm A, Flicek P, Manolio T, Hindorff L, Parkinson H. The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. Nucleic Acids Res. 2014 Jan;42(Database issue):D1001-6. doi: 10.1093/nar/gkt1229. Epub 2013 Dec 6. PubMed PMID: 24316577; PubMed Central PMCID: PMC3965119."

[International HapMap Consortium 2003]: http://www.ncbi.nlm.nih.gov/pubmed/14685227 "International HapMap Consortium. The International HapMap Project. Nature. 2003 Dec 18;426(6968):789-96. PubMed PMID: 14685227."

[Nicolae 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20369019 "Nicolae DL, Gamazon E, Zhang W, Duan S, Dolan ME, Cox NJ. Trait-associated SNPs are more likely to be eQTLs: annotation to enhance discovery from GWAS. PLoS Genet. 2010 Apr 1;6(4):e1000888. doi: 10.1371/journal.pgen.1000888. PubMed PMID:  20369019; PubMed Central PMCID: PMC2848547."

[Jostins 2012]: http://www.ncbi.nlm.nih.gov/pubmed/23128233 "Jostins L, Ripke S, Weersma RK, Duerr RH, McGovern DP, Hui KY, Lee JC, Schumm  LP, Sharma Y, Anderson CA, Essers J, Mitrovic M, Ning K, Cleynen I, Theatre E, Spain SL, Raychaudhuri S, Goyette P, Wei Z, Abraham C, Achkar JP, Ahmad T, Amininejad L, Ananthakrishnan AN, Andersen V, Andrews JM, Baidoo L, Balschun T, Bampton PA, Bitton A, Boucher G, Brand S, Büning C, Cohain A, Cichon S, D'Amato M, De Jong D, Devaney KL, Dubinsky M, Edwards C, Ellinghaus D, Ferguson LR, Franchimont D, Fransen K, Gearry R, Georges M, Gieger C, Glas J, Haritunians T, Hart A, Hawkey C, Hedl M, Hu X, Karlsen TH, Kupcinskas L, Kugathasan S, Latiano A, Laukens D, Lawrance IC, Lees CW, Louis E, Mahy G, Mansfield J, Morgan AR, Mowat C, Newman W, Palmieri O, Ponsioen CY, Potocnik U, Prescott NJ, Regueiro M,  Rotter JI, Russell RK, Sanderson JD, Sans M, Satsangi J, Schreiber S, Simms LA, Sventoraityte J, Targan SR, Taylor KD, Tremelling M, Verspaget HW, De Vos M, Wijmenga C, Wilson DC, Winkelmann J, Xavier RJ, Zeissig S, Zhang B, Zhang CK, Zhao H; International IBD Genetics Consortium (IIBDGC), Silverberg MS, Annese V,  Hakonarson H, Brant SR, Radford-Smith G, Mathew CG, Rioux JD, Schadt EE, Daly MJ, Franke A, Parkes M, Vermeire S, Barrett JC, Cho JH. Host-microbe interactions have shaped the genetic architecture of inflammatory bowel disease. Nature. 2012  Nov 1;491(7422):119-24. doi: 10.1038/nature11582. PubMed PMID: 23128233; PubMed Central PMCID: PMC3491803."

[Rivas 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21983784 "Rivas MA, Beaudoin M, Gardet A, Stevens C, Sharma Y, Zhang CK, Boucher G, Ripke S, Ellinghaus D, Burtt N, Fennell T, Kirby A, Latiano A, Goyette P, Green T, Halfvarson J, Haritunians T, Korn JM, Kuruvilla F, Lagacé C, Neale B, Lo KS, Schumm P, Törkvist L; National Institute of Diabetes and Digestive Kidney Diseases Inflammatory Bowel Disease Genetics Consortium (NIDDK IBDGC); United Kingdom Inflammatory Bowel Disease Genetics Consortium; International Inflammatory Bowel Disease Genetics Consortium, Dubinsky MC, Brant SR, Silverberg MS, Duerr RH, Altshuler D, Gabriel S, Lettre G, Franke A, D'Amato M, McGovern DP, Cho JH, Rioux JD, Xavier RJ, Daly MJ. Deep resequencing of GWAS loci identifies independent rare variants associated with inflammatory bowel disease. Nat Genet.  2011 Oct 9;43(11):1066-73. doi: 10.1038/ng.952. PubMed PMID: 21983784; PubMed Central PMCID: PMC3378381."

[Bulik-Sullivan 2015]: http://www.ncbi.nlm.nih.gov/pubmed/25642630 "Bulik-Sullivan BK, Loh PR, Finucane HK, Ripke S, Yang J; Schizophrenia Working Group of the Psychiatric Genomics Consortium, Patterson N, Daly MJ, Price AL, Neale BM. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies. Nat Genet. 2015 Mar;47(3):291-5. doi: 10.1038/ng.3211. Epub 2015 Feb 2. PubMed PMID: 25642630."

[Glazier 2002]: http://www.ncbi.nlm.nih.gov/pubmed/12493905 "Glazier AM, Nadeau JH, Aitman TJ. Finding genes that underlie complex traits.  Science. 2002 Dec 20;298(5602):2345-9. Review. PubMed PMID: 12493905."

[Gabriel 2002]: http://www.ncbi.nlm.nih.gov/pubmed/12029063 "Gabriel SB, Schaffner SF, Nguyen H, Moore JM, Roy J, Blumenstiel B, Higgins J, DeFelice M, Lochner A, Faggart M, Liu-Cordero SN, Rotimi C, Adeyemo A, Cooper R,  Ward R, Lander ES, Daly MJ, Altshuler D. The structure of haplotype blocks in the human genome. Science. 2002 Jun 21;296(5576):2225-9. Epub 2002 May 23. PubMed PMID: 12029063."

[Rosenberg 2002]: http://www.ncbi.nlm.nih.gov/pubmed/12493913 "Rosenberg NA, Pritchard JK, Weber JL, Cann HM, Kidd KK, Zhivotovsky LA, Feldman MW. Genetic structure of human populations. Science. 2002 Dec 20;298(5602):2381-5. PubMed PMID: 12493913."

[Daly 2001]: http://www.ncbi.nlm.nih.gov/pubmed/11586305 "Daly MJ, Rioux JD, Schaffner SF, Hudson TJ, Lander ES. High-resolution haplotype structure in the human genome. Nat Genet. 2001 Oct;29(2):229-32. PubMed PMID: 11586305."

[Barrett 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18587394/ "Barrett JC, Hansoul S, Nicolae DL, Cho JH, Duerr RH, Rioux JD, Brant SR, Silverberg MS, Taylor KD, Barmada MM, Bitton A, Dassopoulos T, Datta LW, Green T, Griffiths AM, Kistner EO, Murtha MT, Regueiro MD, Rotter JI, Schumm LP, Steinhart AH, Targan SR, Xavier RJ; NIDDK IBD Genetics Consortium, Libioulle C, Sandor C, Lathrop M, Belaiche J, Dewit O, Gut I, Heath S, Laukens D, Mni M, Rutgeerts P, Van Gossum A, Zelenika D, Franchimont D, Hugot JP, de Vos M, Vermeire S, Louis E; Belgian-French IBD Consortium; Wellcome Trust Case Control Consortium, Cardon LR, Anderson CA, Drummond H, Nimmo E, Ahmad T, Prescott NJ, Onnie CM, Fisher SA, Marchini J, Ghori J, Bumpstead S, Gwilliam R, Tremelling M, Deloukas P, Mansfield J, Jewell D, Satsangi J, Mathew CG, Parkes M, Georges M, Daly MJ. Genome-wide association defines more than 30 distinct susceptibility loci for Crohn's disease. Nat Genet. 2008 Aug;40(8):955-62. doi: 10.1038/ng.175. Epub 2008 Jun 29. PubMed PMID: 18587394; PubMed Central PMCID: PMC2574810."

[Yang 2013]: http://www.ncbi.nlm.nih.gov/pubmed/24082113 "Yang S, Xiao Y, Kang D, Liu J, Li Y, Undheim EA, Klint JK, Rong M, Lai R, King GF. Discovery of a selective NaV1.7 inhibitor from centipede venom with analgesic efficacy exceeding morphine in rodent pain models. Proc Natl Acad Sci U S A. 2013 Oct 22;110(43):17534-9. doi: 10.1073/pnas.1306285110. Epub 2013 Sep 30. PubMed PMID: 24082113; PubMed Central PMCID: PMC3808613."

[Cox 2006]: http://www.ncbi.nlm.nih.gov/pubmed/17167479 "Cox JJ, Reimann F, Nicholas AK, Thornton G, Roberts E, Springell K, Karbani G, Jafri H, Mannan J, Raashid Y, Al-Gazali L, Hamamy H, Valente EM, Gorman S, Williams R, McHale DP, Wood JN, Gribble FM, Woods CG. An SCN9A channelopathy causes congenital inability to experience pain. Nature. 2006 Dec 14;444(7121):894-8. PubMed PMID: 17167479."

[Solberg 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19101844 "Solberg IC, Lygren I, Jahnsen J, Aadland E, Høie O, Cvancarova M, Bernklev T,  Henriksen M, Sauar J, Vatn MH, Moum B; IBSEN Study Group. Clinical course during  the first 10 years of ulcerative colitis: results from a population-based inception cohort (IBSEN Study). Scand J Gastroenterol. 2009;44(4):431-40. doi: 10.1080/00365520802600961. PubMed PMID: 19101844."

[Gevers 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24629344 "Gevers D, Kugathasan S, Denson LA, Vázquez-Baeza Y, Van Treuren W, Ren B, Schwager E, Knights D, Song SJ, Yassour M, Morgan XC, Kostic AD, Luo C, González  A, McDonald D, Haberman Y, Walters T, Baker S, Rosh J, Stephens M, Heyman M, Markowitz J, Baldassano R, Griffiths A, Sylvester F, Mack D, Kim S, Crandall W, Hyams J, Huttenhower C, Knight R, Xavier RJ. The treatment-naive microbiome in new-onset Crohn's disease. Cell Host Microbe. 2014 Mar 12;15(3):382-92. doi: 10.1016/j.chom.2014.02.005. PubMed PMID: 24629344; PubMed Central PMCID: PMC4059512."

[Kuballa 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18852889 "Kuballa P, Huett A, Rioux JD, Daly MJ, Xavier RJ. Impaired autophagy of an intracellular pathogen induced by a Crohn's disease associated ATG16L1 variant. PLoS One. 2008;3(10):e3391. doi: 10.1371/journal.pone.0003391. Epub 2008 Oct 13.  PubMed PMID: 18852889; PubMed Central PMCID: PMC2566595."

[Lassen 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24821797 "Lassen KG, Kuballa P, Conway KL, Patel KK, Becker CE, Peloquin JM, Villablanca EJ, Norman JM, Liu TC, Heath RJ, Becker ML, Fagbami L, Horn H, Mercer J, Yilmaz OH, Jaffe JD, Shamji AF, Bhan AK, Carr SA, Daly MJ, Virgin HW, Schreiber SL, Stappenbeck TS, Xavier RJ. Atg16L1 T300A variant decreases selective autophagy resulting in altered cytokine signaling and decreased antibacterial defense. Proc Natl Acad Sci U S A. 2014 May 27;111(21):7741-6. doi: 10.1073/pnas.1407001111. Epub 2014 May 12. PubMed PMID: 24821797; PubMed Central PMCID: PMC4040621."

[Hampe 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17200669 "Hampe J, Franke A, Rosenstiel P, Till A, Teuber M, Huse K, Albrecht M, Mayr G, De La Vega FM, Briggs J, Günther S, Prescott NJ, Onnie CM, Häsler R, Sipos B, Fölsch UR, Lengauer T, Platzer M, Mathew CG, Krawczak M, Schreiber S. A genome-wide association scan of nonsynonymous SNPs identifies a susceptibility variant for Crohn disease in ATG16L1. Nat Genet. 2007 Feb;39(2):207-11. Epub 2006 Dec 31. PubMed PMID: 17200669."

[VanDussen 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24076061 "VanDussen KL, Liu TC, Li D, Towfic F, Modiano N, Winter R, Haritunians T, Taylor KD, Dhall D, Targan SR, Xavier RJ, McGovern DP, Stappenbeck TS. Genetic variants synthesize to produce paneth cell phenotypes that define subtypes of Crohn's disease. Gastroenterology. 2014 Jan;146(1):200-9. doi: 10.1053/j.gastro.2013.09.048. Epub 2013 Sep 25. PubMed PMID: 24076061; PubMed Central PMCID: PMC3899786."


