---
layout: post
title:  "The quest for the structure of PrPSc"
date:   2015-01-22 12:00
author: ericminikel
location: Boston, MA
thumb120: http://www.cureffi.org/media/2015/01/quest-for-prpsc-structure-thumbnail.png
summary200: "30 years after infectious prions were first purified, their structure is still unknown. A host of different lower-resolution methods have been used to obtain constraints on possible models of PrPSc."
---

Three years into my quest to understand prions, I have remained too ignorant about structural biology. While I don't expect to become a structural biologist myself, I decided I wanted to at least become literate in the subject - what is known, what is unknown, what are the methods, what are the challenges. So a few months ago, I wrote to Holger Wille to ask if he could recommend some reviews to get me acquainted with the subject. This led to my post last month about the [structures of PrP<sup>C</sup>](/2014/12/21/structures-of-prpc/), and now with some more time spent reading, I've written this post about efforts to find the structure(s) of PrP<sup>Sc</sup>.

I started by reading the three review articles Holger suggested [[Surewicz & Apostol 2011], [Diaz-Espinoza & Soto 2012], [Requena & Wille 2014]]. It immediately seemed to me that the story of the quest for a structure of PrP<sup>Sc</sup> is a story of methods. For soluble, crystallizable proteins we've had X-ray crystallography and solution NMR for decades, techniques which have made it possible to get high-resolution structures. For insoluble, non-crystalline solids like PrP<sup>Sc</sup>, we just don't have a method in the toolkit today for completely solving them. Instead, to understand the state of the science of these structures is to understand the methods available, what they can and cannot achieve, and what pieces of information we have learned from each. I've therefore structured this post around the various methods used to obtain structural information about PrP<sup>Sc</sup>.

As I'm new to this subject, my omissions and factual errors are sure to be both numerous and egregious. Corrections are very welcome.

### Proteinase K digestion

![](/media/2015/01/schoch-2006-figure-2.png)

*Above: prion strains differ in whether residues 82-97 are accessible to proteinase K, so PK-digested samples result in different molecular weights. From [[Schoch 2006]], [Figure 2](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1316067/figure/pmed-0030014-g002/).*

Some of the earliest data indicating that prion strain properties were encoded in conformation came from limited digestion with proteinase K. If you digest PrP<sup>Sc</sup> in high concentrations of PK at biological temperatures for a long time, PK will digest all of the protein and abolish infectivity. But under limiting conditions (say, 20 &mu;g/mL for 1h at 37&deg;C), PK will only digest the readily protease-accessible N terminus of PrP, which has a different extent in different strains of prions. You can then run the digested protein on a Western blot to figure out its size and what antibodies it will react with. In so-called type 1 strains, limited PK digestion leaves a 21 kDa fragment behind, while in so-called type 2 strains, a 19 kDa fragment remains. While the exact cut sites are variable, type 1 seems to correspond to cleavage at residue G82, and type 2 to cleavage at S97 (human numbering) [[Parchi 2000]]. This suggests that the segment from residues 82-97 is protected in some strains but exposed in others. The facts that (1) these properties can be transmitted faithfully to new PrP molecules in a cell-free reaction [[Bessen 1995]] and (2) these properties can be generated by genetic mutations (D178N and E200K) and then transmitted faithfully to mice expressing PrP without these mutations [[Telling 1996]] were two important pieces of evidence for the idea that protein conformation holds strain information. However, these observations are very low-resolution and we still don't know much about the nature of the structural differences that give rise to differential protease accessibility of residues 82-97.

Besides the 21 and 19 kDa bands of canonical type 1 and type 2 strains, many other PK-resistant PrP fragments have been observed over time. For instance, many *PRNP* mutations with a GSS phenotype give rise to an 8 kDa protease-resistant fragment [[Parchi 1998]] and one prion strain generated *de novo* in PMCA has a 15 kDa fragment [[Wang 2010]]. Over ten different exact PK cleavage sites have been mapped in hamster 263K and drowsy (DY) prions [[Sajnani 2008]] and in anchorless prions [[Vazquez-Fernandez 2012]], which begins to give more detailed conformational information.

### Antibody reactivity

The 3F4 epitope (KTNMKHM, residues 106-112 in human or hamster PrP) is accessible to the 3F4 antibody in natively folded PrP<sup>C</sup> and in GdnHCl-denatured PrP, but buried and inaccessible in PrP<sup>Sc</sup>. Different prion strains require different amounts of GdnHCl to denature them enough to expose this epitope, which is the basis of the [conformation-dependent immunoassay](/2013/07/29/introduction-to-conformation-dependent-immunoassay/) and was one additional piece of evidence that strain information is encoded in conformation [[Safar 1998]]. However, this doesn't give very high resolution information about the structure of PrP<sup>Sc</sup>.

There have also been efforts to use other antibodies to figure out which parts of PrP are buried or exposed in PrP<sup>Sc</sup>. At [Prion2014 Day 2](/2014/05/28/prion2014-day-2/) Surachai Supattapone said his lab is using this technique to probe the structural differences of infectious vs. "protein-only" PMCA-propagated PrP<sup>Sc</sup> - still unpublished.

### Fourier transform infrared spectroscopy (FTIR)

![](/media/2015/01/ftir-in-action.png)

*Above: drying protein onto the surface of a diamond for FTIR.*

For FTIR, a pure preparation of the protein of interest is suspended in a small amount (say, 2 &mu;L) of water. I say "suspended" rather than dissolved, because in many cases, such as PrP<sup>Sc</sup>, the protein is insoluble. This suspension is then pipetted onto the flat surface of a synthetic diamond, which in the above photo appears as a tiny hole recessed in the middle of the steel disc. Next you want to dry the protein onto the surface of the diamond. You could do this by blowing air onto the surface, but then you'd be contaminating your protein with particulates, so instead you blow pure N<sub>2</sub> across the surface until the protein is dry. Now you want to see how much light the protein absorbs at a variety of wavelengths. Different protein secondary structures have bonds which absorb light at different wavelengths. Which wavelengths are relevant? Well, by convention, people refer to [wavenumber](http://en.wikipedia.org/wiki/Wavenumber) rather than wavelength in IR, and they use the unit cm<sup>-1</sup>. In those units, the range used in FTIR is 1500 to 1800 cm<sup>-1</sup>. To put this in perspective, 1800 cm<sup>-1</sup> is a wavelength of (1/100)/1800 m &asymp; 5.6 &mu;m, and 1500 cm<sup>-1</sup> is ~6.7 &mu;m. Recall that the [visible spectrum](http://en.wikipedia.org/wiki/Visible_spectrum) is about 400 to 700 nm - so for comparison, the IR spectra used in FTIR are about 5600 to 6700 nm, which is what people call the ["mid-wavelength infrared"](http://en.wikipedia.org/wiki/Infrared#Regions_within_the_infrared).

Now, you might think that an intuitive way to measure absorbance at all of these different wavelenghts would be to first shine a 5600 nm light at the sample, and measure how much is absorbed, then a 5601 nm light, then a 5602 nm light, and so on. In fact, FTIR manages to achieve essentially the same measurement - how much light is absorbed at each wavelength - in a different way: instead of emitting each wavelength separately, it emits a number of different *mixtures* of wavelengths, then uses a Fourier transform to back into how much light was being absorbed at each individual wavelength. While conceptually complicated, this is not computationally intensive, and the computer connected to the IR machine will display the absorbance at different wavelengths (or actually, by wavenumbers in cm<sup>-1</sup>) in real time as you run it.

How does one interpret absorbance in different infrared wavelengths to give information about protein structure? The go-to reference seems to be [[Surewicz 1993]]. As Surewicz notes, the interpretation is non-trivial. The classical view has been that &alpha;-helices absorb a lot of light in the 1650-1658 cm<sup>-1</sup> range, while &beta;-sheets absorb in the 1620 to 1640 cm<sup>-1</sup> range. The intensity of these bands can even be construed to give quantitative information about the percentage of &beta;-sheet and &alpha;-helical content. However, there are notable exceptions - proteins whose high-resolution structure has been solved by NMR or X-ray crystallography and contain no alpha helices or no beta sheets, yet which appear to have bands in the 1650-1658 or 1620-1640 range respectively.

Indeed, though we don't have a high-resolution structure for PrP<sup>Sc</sup>, it may be one such exception. The structure of PrP<sup>Sc</sup> - in this case, hamster brain-derived 263K prion fibrils - was first probed by FTIR by [[Caughey 1991]]. Figure 5 from that paper shows the FTIR data interpreted in terms of turns, alpha helices and beta sheets:

![](/media/2015/01/caughey-1991-figure-5.png)

The peak at 1657 cm<sup>-1</sup> falls right with in the canonical &alpha;-helix range and is interpreted to mean that PrP<sup>Sc</sup> has ~17% alpha helical content. The Prusiner group completed similar studies a couple of years later and similarly concluded that PrP<sup>Sc</sup> contains 21% alpha helix [[Pan 1993]].

However, the consensus today is that PrP<sup>Sc</sup> is in fact entirely devoid of alpha helices. Byron Caughey [noted this](/2014/05/30/prion2014-day-4/) in his talk at Prion2014, and a review by Jesus Requena and Holger Wille explains that the 1657 cm<sup>-1</sup> band is now believed to correspond simply to turns between beta sheets [[Requena & Wille 2014]]. This shift in thinking occurred in part because other techniques such as hydrogen/deuterium exchange indicate the absence of any alpha helices [[Smirnovas & Baron 2011]]. It is also supported by studies of synthetic amyloid fibrils made of recombinant PrP - although the structure of these fibrils is also unsolved at any high resolution, they are believed to have a wholly beta sheet structure, and yet they, like PrP<sup>Sc</sup> preparations, also contain a peak on FTIR corresponding to ~1657 cm<sup>-1</sup> [[Cobb 2007], [Lu 2007], [Smirnovas & Baron 2011]].

### Circular dichroism

![](/media/2015/01/circular-dichroism-machine.png)

*Above: a machine for performing circular dichroism.*

[Circular dichroism](http://en.wikipedia.org/wiki/Circular_dichroism) means differential absorptions of left vs. right-handed polarized light. The CD machine emits left and then right-circularized light through a sample and measures how much of each is absorbed. The underlying principle is that polarized light interacts differently with molecules of different chirality. To understand this I had to think back to the [story of how chirality was discovered](http://en.wikipedia.org/wiki/Circular_dichroism) by Louis Pasteur, looking at crystals of tartaric or paratartaric acid in wine - two small molecules with identical molecular formulas but opposite chirality, which polarize light differently. In the case of proteins, all of the individual amino acids are levorotary no matter what, but the protein can take on different chiralities depending on its secondary structure and conformation. The relevant range of wavelengths for CD analysis of proteins is the near ultraviolet, and specifically, alpha helices have negative bands at 222 and 208 nm and a positive band at 193 nm, while beta sheets give a negative band at 218 nm and a positive band at 195 nm [[Greenfield 2006]]. These signals are, however, confounded with the fact that aromatic amino acids (F, W and Y) themselves have differential absorbance at various wavelengths. Therefore it is customary to just plug your CD spectra into a program that deconvolutes the contribution of different signals.

Jiri Safar first applied this technique to PrP<sup>Sc</sup>, either undigested or PK-digested to PrP 27-30, and with or without denaturing in various amounts of GdnHCl, to probe not only the structure of PrP<sup>Sc</sup> but also its folding and unfolding [[Safar 1993a]]. The CD spectra for undigested PrP<sup>Sc</sup>, for instance, are shown in Figure 2:

![](/media/2015/01/safar-1993a-fig-2.png)

These data were interpreted to indiate that PrP<sup>Sc</sup> was 34% beta sheet, 20% alpha helix, and 46% turns and disordered parts. However, as explained by [[Requena & Wille 2014]], this interpretation was later rejected because recombinant prion protein amyloids, which were shown by other experimental techniques to lack alpha helices, had apparently similar spectra [[Ostapchenko 2010]].

### Hydrogen/deuterium exchange

In hydrogen/deuterium exchange (variously abbreviated H/D exchange or HX), you measure the rate at which hydrogen atoms in your protein are exchanged for deuterium in the surrounding solution. This tells you which hydrogens are exposed to solvent (as opposed to buried inside the folds of the protein), and to some extent their secondary structure (hydrogens in beta sheets exchange less readily than alpha helices). After taking your protein (made with mostly normal hydrogen) and putting it into a D<sub>2</sub>O solution for several days, there are a number of ways you can then go back and figure out which hydrogens underwent exchange [reviewed in [Engen 2009]]. Deuterium changes the NMR, FTIR and CD spectra of proteins, so any of those methods can be used in combination with HX, but perhaps the highest-resolution method is HX with mass spectrometry (HXMS). In HXMS, you let the protein sit in deuterated water, then trypsinize it and perform mass spec to see which peptides acquired deuterium. This approach was used by [[Smirnovas & Baron 2011]] and showed that, in mouse brain-derived 22L PrP<sup>Sc</sup>, the entire stretch from residue ~81 to ~167 and possibly all the way to ~224 exhibited very little deuterium exchange, consistent with all of those residues belonging to beta sheets. At [Prion2014 Day 2](/2014/05/28/prion2014-day-2/), Surachai Supattapone also mentioned that his group is using HX to characterize the structure of prions propagated in PMCA with the minimal cofactor phosphatidylethanolamine. That work is still unpublished.

### Chemical modification and mass spectrometry or antibody binding

In this approach, you can use various small molecule agents to covalently modify exposed (but not buried) amino acids), then use mass spec to figure out which residues were modified. In this sense, it is somewhat analogous to HXMS (above) but with chemical changes, rather than deuterium exchange, providing the difference in mass (or charge) to be picked up on mass spec. In the reported studies using this appraoch, tetranitromethane has been used to modify tyrosines [[Gong 2011]], acetic anhydride to modify lysines [[Gong 2011]], and bis(sulfosuccinimidyl) suberate to cross-link exposed N-termini [[Onisko 2005]]. 

In addition to changing the mass of tryptic peptides in mass spec, chemical reactions also abolish antibody reactivity, and this property has been used to discriminate prion strains based on which epitopes are exposed (and therefore can be chemically modified) [[Silva 2012]].

### Electron microscopy

![](/media/2015/01/groveman-kraus-2015-figure-4f.png)

*Above: negative stain transmission electron microscopy image of amyloid fibrils formed from mutant recombinant PrP. From [[Groveman & Kraus 2015]], [Figure 4F](http://www.jbc.org/content/290/2/1119/F4.expansion.html).*

Electron microscopy comes in several forms, all of which share the principle of throwing electrons at a sample. Because electrons have smaller wavelengths, they can achieve much higher resolution than photons. One relevant form of EM is transmission electron microscopy with negative staining. In this procedure, electrons are shot upwards from underneath a sample, and a sensor above the sample detects the electrons that get through. Larger atoms are better at scattering electrons, so taking your biological sample (which consists mostly of small-ish atoms like C, N, O and H) and surrounding it with heavier atoms (such as tungsten or uranium) is one way to get contrast between the biological material (which transmits most electrons) and the "negative stain" (which blocks most electrons from getting straight through).

The first hints as to the ultrastructure of PrP<sup>Sc</sup> fibrils came from [electron microscopy](http://en.wikipedia.org/wiki/Electron_microscope) [[Merz 1981], [Prusiner 1983]]. Prusiner purified PrP<sup>Sc</sup> using the protocol described [here](/2013/10/17/how-prion-protein-was-first-purified/) and, using EM, found that prion fibrils had a diameter of ~25 nm and a length of 100 to 200 nm. Like the fibrils that by that point had already been seen in Alzheimer's brains, these fibrils were "[birefringent](http://en.wikipedia.org/wiki/Birefringence)" and would stain with Congo red, thus meeting the functional definition of amyloid. Also, the fibrils were disordered and somewhat heterogeneous, unlike crystallized viruses, which are highly ordered.

That heterogeneity is part of what makes it difficult to get high resolution information out of EM. Recently, some investigators have started focusing on so-called anchorless or &Delta;GPI prions, as these are still infectious yet lack the GPI anchor and are unglycosylated, thus consisting of a more uniform pure protein. At [Prion2014 Day 3](/2014/05/29/prion2014-day-3/), Jesus Requena described his and Holger Wille's efforts to get structural information by averaging the dimensions and periodicity of negative staining in hundreds of transmission electron microscopy images from anchorless prions. That work is still unpublished.

When you perform transmission electron microscopy on 2D crystals of a protein, that's called electron crystallography. Although PrP<sup>Sc</sup> does not form 3D crystals that would be amenable to X-ray crystallography, it was observed that when you purify PrP 27-30 you sometimes get 2D hexagonal crystals in addition to amyloid rods [[Wille 2002], see esp. [Figure 1](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC122563/figure/F1/)]. Given that these are clearly a different ultrastructure than the rods, and that both occur together, it's not clear to me how we know that these crystals are actually infectious and not just a non-infectious off-pathway aggregate. But in any case, the advantage of 2D crystals is that, although EM still doesn't have the power to resolve individual atoms, the structure is regular and so through image processing and Fourier transforms you can derive higher-resolution structural information. This approach has given some of the highest resolution (~7&Aring;) information on PrP<sup>Sc</sup> available right now [[Wille 2002], [Wille 2007]]. By comparing PrP 27-30 to the 106-residue "miniprions" [[Supattapone 1999]] it was possible to get an idea of where residues 141-176 (which miniprions lack) and the glycan chains are located.

### X-ray fiber diffraction and small angle X ray scattering

For proteins that are able to form 3D crystals, the highest-resolution structural information can often be obtained through X-ray crystallography, in which the diffraction of X-rays in the crystal is used to infer where each atom is. For proteins that don't form 3D crystals, such as PrP<sup>Sc</sup>, this isn't possible, but because PrP<sup>Sc</sup> does form fibrils, one can still get some information from X-ray [fiber diffraction](http://en.wikipedia.org/wiki/Fiber_diffraction). In this technique, the fibril rod is modeled as being a cylinder, with the axis along the height of the cylinder being called the **meridian** and the axis through the cross-sectional diameter of the cylinder being called the **equator**. By rotating your sample and seeing how X-rays diffract from different sides of the fiber, you can get a 2D image like [Figure 1](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2761340/figure/F1/) from [[Wille 2009]]. This technique performed on hamster brain-derived Sc237 PrP 27-30 and mouse brain-derived RML 27-30 precipitated with phosphotungstic acid, and one of the main conclusions seems to be that both structures contain a 19.2&Aring; repeating unit, consistent with four rungs of &beta;-strands each 4.8&Aring; in height. This has been one constraint on structural models of PrP<sup>Sc</sup>.

A related technique is to throw X-rays at the sample from a narrow range of angles and see how the scattering pattern changes; this is called [small angle X-ray scattering](http://en.wikipedia.org/wiki/Small-angle_X-ray_scattering) and, similar to EM, has given information on the dimensions of prion fibrils [[Amenitsch 2013]].

### Nuclear magnetic resonance (NMR)

![](/media/2015/01/rob-tycko-nmr-image.png)

*Above: NMR device and spectra from [Rob Tycko's website](http://www.niddk.nih.gov/about-niddk/staff-directory/intramural/robert-tycko/pages/research-images.aspx)*.

I still don't really understand how [NMR](http://en.wikipedia.org/wiki/Nuclear_magnetic_resonance_spectroscopy_of_proteins) works, except to say that you place the sample in a strong magnetic field to align all of a given atom (for instance, all of the hydrogens) with or against the field, and then you can get very high-resolution information about the structure of the molecule beacuse each atom will behave differently depending on what other atoms it is bonded to. Over 20 years ago, Kurt Wuthrich pioneered the idea of using solution NMR to find the structures of biomolecules, and this enabled the first [structure of PrP<sup>C</sup>](/2014/12/21/structures-of-prpc/) [[Riek 1996]]. But as the name implies, solution NMR requires that the biomolecule of interest be in solution, and PrP<sup>Sc</sup> is insoluble.

A more recent approach is [solid state NMR](http://en.wikipedia.org/wiki/Solid-state_nuclear_magnetic_resonance), which doesn't require the protein to be soluble. Indeed, the only prion whose structure has been solved at high resolution, the HET-s prion of [*Podospora anserina*](http://en.wikipedia.org/wiki/Podospora_anserina), was solved by solid state NMR [[Wasmer 2008]], which seems to be a good sign that maybe other prions and amyloids could prove amenable to this approach too. Many such efforts are reviewed in [[Tycko 2011]]. 

![](/media/2015/01/wasmer-2008-2rnm-het-s-amyloid.png)

*Above: the solid state NMR structure [[PDB# 2RNM](http://pdb.org/pdb/explore/explore.do?structureId=2RNM)] of HET-s amyloid, the only prion whose structure has been solved at high resolution. PyMOL code to generate this graphic: `fetch 2rnm; bg_color white; hide everything; show cartoon; spectrum;`*

As for PrP in particular, several efforts have been reported. Solid state NMR has been applied to amyloids made of trunacated recombinant PrP corresponding to PrP 106-126 [[Walsh 2009]], PrP 127-147 [[Lin 2010]], and HuPrP 23-144 [[Helmus 2008], [Helmus 2010], [Jones 2011]], but only ~30 residues of HuPrP 23-144 are immobilized enough to resolve well on NMR. Full-length recombinant PrP has been studied as well - amyloid fibrils of SHaPrP 23-231 formed under denaturing and shaking conditions [[Tycko 2010]] or in 263K-seeded RT-QuIC [[Groveman 2014]]. These efforts have yielded various constraints on the set of possible models for the structure of these amyloids, though no one has yet obtained a high-resolution structure like that of HET-s. The recombinant amyloids being studied in ssNMR are generally not infectious, and it is not yet clear how their structures must relate to those of infectious PrP<sup>Sc</sup>. For instance, both reported studies of SHaPrP23-231 fibrils [[Tycko 2010], [Groveman 2014]] have been consistent with parallel in-register beta sheet (PIRIBS) conformations, a structure that apparently conflicts with some of the measurements obtained from brain-derived PrP<sup>Sc</sup>.

![](/media/2015/01/brad-with-piribs-model.png)

*Above: Brad Groveman with a 3D printout of a theoretical PIRIBS model of RT-QuIC fibrils, proposed on the basis of solid state NMR data [[Groveman 2014]].*

There must be some reason why solid state NMR cannot be applied to brain-derived PrP<sup>Sc</sup>, or else someone would have done it, but I couldn't find an explanation in any of the reviews I read.

### Thoughts and conclusions

Given the challenges inherent in studying the structure of prions and amyloids, I reason that it must be the case that the structural biolgoists who choose to devote their careers to PrP are particularly ambitious individuals. If so, then I further speculate that they must each dream of one day writing up their manuscript entitled, "2.5&Aring; structure of an infectious mammalian prion". Is this indeed what the future holds, or is a gradual whittling-away at the structure with various low-resolution techniques the more likely road forward? After learning about the above techniques and results, I am left pondering two major questions:

1. Is there any currently imaginable route to obtaining a high-resolution structure of PrP<sup>Sc</sup>?
2. If so, then once a structure is obtained, how will one be sure that the structure you've solved is indeed the same thing that is infectious?

I welcome any comments.

[Surewicz & Apostol 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21630136 "Surewicz WK, Apostol MI. Prion protein and its conformational conversion: a structural perspective. Top Curr Chem. 2011;305:135-67. doi: 10.1007/128_2011_165. Review. PubMed PMID: 21630136."

[Diaz-Espinoza & Soto 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22472622 "Diaz-Espinoza R, Soto C. High-resolution structure of infectious prion protein: the final frontier. Nat Struct Mol Biol. 2012 Apr 4;19(4):370-7. doi: 10.1038/nsmb.2266. Review. PubMed PMID: 22472622; PubMed Central PMCID: PMC4049221."

[Requena & Wille 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24583975 "Requena JR, Wille H. The structure of the infectious prion protein: experimental data and molecular models. Prion. 2014 Jan-Feb;8(1):60-6. PubMed PMID: 24583975."

[Surewicz 1993]: http://www.ncbi.nlm.nih.gov/pubmed/8422346 "Surewicz WK, Mantsch HH, Chapman D. Determination of protein secondary structure by Fourier transform infrared spectroscopy: a critical assessment. Biochemistry. 1993 Jan 19;32(2):389-94. Review. PubMed PMID: 8422346."

[Caughey 1991]: http://www.ncbi.nlm.nih.gov/pubmed/1678278 "Caughey BW, Dong A, Bhat KS, Ernst D, Hayes SF, Caughey WS. Secondary structure analysis of the scrapie-associated protein PrP 27-30 in water by infrared spectroscopy. Biochemistry. 1991 Aug 6;30(31):7672-80. Erratum in: Biochemistry 1991 Oct 29;30(43):10600. PubMed PMID: 1678278."

[Pan 1993]: http://www.ncbi.nlm.nih.gov/pubmed/7902575 "Pan KM, Baldwin M, Nguyen J, Gasset M, Serban A, Groth D, Mehlhorn I, Huang Z, Fletterick RJ, Cohen FE, et al. Conversion of alpha-helices into beta-sheets features in the formation of the scrapie prion proteins. Proc Natl Acad Sci U S A. 1993 Dec 1;90(23):10962-6. PubMed PMID: 7902575; PubMed Central PMCID: PMC47901."

[Smirnovas & Baron 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21441913 "Smirnovas V, Baron GS, Offerdahl DK, Raymond GJ, Caughey B, Surewicz WK. Structural organization of brain-derived mammalian prions examined by hydrogen-deuterium exchange. Nat Struct Mol Biol. 2011 Apr;18(4):504-6. doi: 10.1038/nsmb.2035. Epub 2011 Mar 27. PubMed PMID: 21441913; PubMed Central PMCID: PMC3379881."

[Requena & Wille 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24583975 "Requena JR, Wille H. The structure of the infectious prion protein: experimental data and molecular models. Prion. 2014 Jan-Feb;8(1):60-6. PubMed PMID: 24583975."

[Safar 1993a]: http://www.ncbi.nlm.nih.gov/pubmed/8104185 "Safar J, Roller PP, Gajdusek DC, Gibbs CJ Jr. Conformational transitions, dissociation, and unfolding of scrapie amyloid (prion) protein. J Biol Chem. 1993 Sep 25;268(27):20276-84. PubMed PMID: 8104185."

[Safar 1993b]: http://www.ncbi.nlm.nih.gov/pubmed/7905316/ "Safar J, Roller PP, Gajdusek DC, Gibbs CJ Jr. Thermal stability and conformational transitions of scrapie amyloid (prion) protein correlate with infectivity. Protein Sci. 1993 Dec;2(12):2206-16. PubMed PMID: 7905316; PubMed Central PMCID: PMC2142321."

[Cobb 2007]: http://www.ncbi.nlm.nih.gov/pubmed/18025469 "Cobb NJ, Sönnichsen FD, McHaourab H, Surewicz WK. Molecular architecture of human prion protein amyloid: a parallel, in-register beta-structure. Proc Natl Acad Sci U S A. 2007 Nov 27;104(48):18946-51. Epub 2007 Nov 19. PubMed PMID: 18025469; PubMed Central PMCID: PMC2141888."

[Lu 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17242357 "Lu X, Wintrode PL, Surewicz WK. Beta-sheet core of human prion protein amyloid fibrils as determined by hydrogen/deuterium exchange. Proc Natl Acad Sci U S A. 2007 Jan 30;104(5):1510-5. Epub 2007 Jan 22. PubMed PMID: 17242357; PubMed Central PMCID: PMC1785245."

[Ostapchenko 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20553730 "Ostapchenko VG, Sawaya MR, Makarava N, Savtchenko R, Nilsson KP, Eisenberg D,  Baskakov IV. Two amyloid States of the prion protein display significantly different folding patterns. J Mol Biol. 2010 Jul 23;400(4):908-21. doi: 10.1016/j.jmb.2010.05.051. Epub 2010 May 27. PubMed PMID: 20553730; PubMed Central PMCID: PMC2908243."

[Greenfield 2006]: http://www.ncbi.nlm.nih.gov/pubmed/17406547 "Greenfield NJ. Using circular dichroism spectra to estimate protein secondary  structure. Nat Protoc. 2006;1(6):2876-90. PubMed PMID: 17406547; PubMed Central PMCID: PMC2728378."

[Engen 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19788312/ "Engen JR. Analysis of protein conformation and dynamics by hydrogen/deuterium  exchange MS. Anal Chem. 2009 Oct 1;81(19):7870-5. doi: 10.1021/ac901154s. PubMed  PMID: 19788312; PubMed Central PMCID: PMC2756691."

[Prusiner 1983]: http://www.ncbi.nlm.nih.gov/pubmed/6418385 "Prusiner SB, McKinley MP, Bowman KA, Bolton DC, Bendheim PE, Groth DF, Glenner GG. Scrapie prions aggregate to form amyloid-like birefringent rods. Cell. 1983 Dec;35(2 Pt 1):349-58. PubMed PMID: 6418385."

[Merz 1981]: http://www.ncbi.nlm.nih.gov/pubmed/7195134 "Merz PA, Somerville RA, Wisniewski HM, Iqbal K. Abnormal fibrils from scrapie-infected brain. Acta Neuropathol. 1981;54(1):63-74. PubMed PMID: 7195134."

[Wille 2002]: http://www.ncbi.nlm.nih.gov/pubmed/11891310/ "Wille H, Michelitsch MD, Guenebaut V, Supattapone S, Serban A, Cohen FE, Agard DA, Prusiner SB. Structural studies of the scrapie prion protein by electron crystallography. Proc Natl Acad Sci U S A. 2002 Mar 19;99(6):3563-8. Epub 2002 Mar 12. PubMed PMID: 11891310; PubMed Central PMCID: PMC122563."

[Wille 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17935686 "Wille H, Govaerts C, Borovinskiy A, Latawiec D, Downing KH, Cohen FE, Prusiner SB. Electron crystallography of the scrapie prion protein complexed with heavy metals. Arch Biochem Biophys. 2007 Nov 15;467(2):239-48. Epub 2007 Aug 23. PubMed PMID: 17935686; PubMed Central PMCID: PMC3557509."

[Supattapone 1999]: http://www.ncbi.nlm.nih.gov/pubmed/10102274 "Supattapone S, Bosque P, Muramoto T, Wille H, Aagaard C, Peretz D, Nguyen HO,  Heinrich C, Torchia M, Safar J, Cohen FE, DeArmond SJ, Prusiner SB, Scott M. Prion protein of 106 residues creates an artifical transmission barrier for prion replication in transgenic mice. Cell. 1999 Mar 19;96(6):869-78. PubMed PMID: 10102274."

[Wille 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19805070/ "Wille H, Bian W, McDonald M, Kendall A, Colby DW, Bloch L, Ollesch J, Borovinskiy AL, Cohen FE, Prusiner SB, Stubbs G. Natural and synthetic prion structure from X-ray fiber diffraction. Proc Natl Acad Sci U S A. 2009 Oct 6;106(40):16990-5. doi: 10.1073/pnas.0909006106. Epub 2009 Sep 28. PubMed PMID: 19805070; PubMed Central PMCID: PMC2761340."

[Amenitsch 2013]: http://www.ncbi.nlm.nih.gov/pubmed/24247356 "Amenitsch H, Benetti F, Ramos A, Legname G, Requena JR. SAXS structural study  of PrP(Sc) reveals ~11 nm diameter of basic double intertwined fibers. Prion. 2013 Nov-Dec;7(6):496-500. Epub 2013 Nov 18. PubMed PMID: 24247356."

[Parchi 2000]: http://www.ncbi.nlm.nih.gov/pubmed/10963679/ "Parchi P, Zou W, Wang W, Brown P, Capellari S, Ghetti B, Kopp N, Schulz-Schaeffer WJ, Kretzschmar HA, Head MW, Ironside JW, Gambetti P, Chen SG. Genetic influence on the structural variations of the abnormal prion protein. Proc Natl Acad Sci U S A. 2000 Aug 29;97(18):10168-72. PubMed PMID: 10963679; PubMed Central PMCID: PMC27779."

[Bessen 1995]: http://www.ncbi.nlm.nih.gov/pubmed/7791905 "Bessen RA, Kocisko DA, Raymond GJ, Nandan S, Lansbury PT, Caughey B. Non-genetic propagation of strain-specific properties of scrapie prion protein. Nature. 1995 Jun 22;375(6533):698-700. PubMed PMID: 7791905."

[Telling 1996]: http://www.ncbi.nlm.nih.gov/pubmed/8953038 "Telling GC, Parchi P, DeArmond SJ, Cortelli P, Montagna P, Gabizon R, Mastrianni J, Lugaresi E, Gambetti P, Prusiner SB. Evidence for the conformation  of the pathologic isoform of the prion protein enciphering and propagating prion  diversity. Science. 1996 Dec 20;274(5295):2079-82. PubMed PMID: 8953038."

[Safar 1998]: http://www.ncbi.nlm.nih.gov/pubmed/9771749 "Safar J, Wille H, Itri V, Groth D, Serban H, Torchia M, Cohen FE, Prusiner SB. Eight prion strains have PrP(Sc) molecules with different conformations. Nat Med. 1998 Oct;4(10):1157-65. PubMed PMID: 9771749."

[Parchi 1998]: http://www.ncbi.nlm.nih.gov/pubmed/9653185 "Parchi P, Chen SG, Brown P, Zou W, Capellari S, Budka H, Hainfellner J, Reyes  PF, Golden GT, Hauw JJ, Gajdusek DC, Gambetti P. Different patterns of truncated  prion protein fragments correlate with distinct phenotypes in P102L Gerstmann-Sträussler-Scheinker disease. Proc Natl Acad Sci U S A. 1998 Jul 7;95(14):8322-7. PubMed PMID: 9653185; PubMed Central PMCID: PMC20974."

[Wang 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20110469/ "Wang F, Wang X, Yuan CG, Ma J. Generating a prion with bacterially expressed recombinant prion protein. Science. 2010 Feb 26;327(5969):1132-5. doi: 10.1126/science.1183748. Epub 2010 Jan 28. PubMed PMID: 20110469; PubMed Central  PMCID: PMC2893558."

[Sajnani 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18621059/ "Sajnani G, Pastrana MA, Dynin I, Onisko B, Requena JR. Scrapie prion protein structural constraints obtained by limited proteolysis and mass spectrometry. J Mol Biol. 2008 Sep 26;382(1):88-98. doi: 10.1016/j.jmb.2008.06.070. Epub 2008 Jul 1. PubMed PMID: 18621059."

[Vazquez-Fernandez 2012]: http://www.ncbi.nlm.nih.gov/pubmed/23185550 "Vázquez-Fernández E, Alonso J, Pastrana MA, Ramos A, Stitz L, Vidal E, Dynin I, Petsch B, Silva CJ, Requena JR. Structural organization of mammalian prions as probed by limited proteolysis. PLoS One. 2012;7(11):e50111. doi: 10.1371/journal.pone.0050111. Epub 2012 Nov 20. PubMed PMID: 23185550; PubMed Central PMCID: PMC3502352."

[Riek 1996]: http://www.ncbi.nlm.nih.gov/pubmed/8700211 "Riek R, Hornemann S, Wider G, Billeter M, Glockshuber R, Wüthrich K. NMR structure of the mouse prion protein domain PrP(121-231). Nature. 1996 Jul 11;382(6587):180-2. PubMed PMID: 8700211."

[Wasmer 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18339938 "Wasmer C, Lange A, Van Melckebeke H, Siemer AB, Riek R, Meier BH. Amyloid fibrils of the HET-s(218-289) prion form a beta solenoid with a triangular hydrophobic core. Science. 2008 Mar 14;319(5869):1523-6. doi: 10.1126/science.1151839. Erratum in: Science. 2008 Apr 4;320(5872):50. PubMed PMID: 18339938."

[Helmus 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18436646/ "Helmus JJ, Surewicz K, Nadaud PS, Surewicz WK, Jaroniec CP. Molecular conformation and dynamics of the Y145Stop variant of human prion protein in amyloid fibrils. Proc Natl Acad Sci U S A. 2008 Apr 29;105(17):6284-9. doi: 10.1073/pnas.0711716105. Epub 2008 Apr 24. PubMed PMID: 18436646; PubMed Central  PMCID: PMC2359773."

[Helmus 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20121096/ "Helmus JJ, Surewicz K, Surewicz WK, Jaroniec CP. Conformational flexibility of Y145Stop human prion protein amyloid fibrils probed by solid-state nuclear magnetic resonance spectroscopy. J Am Chem Soc. 2010 Feb 24;132(7):2393-403. doi: 10.1021/ja909827v. PubMed PMID: 20121096; PubMed Central PMCID: PMC2838504."

[Tycko 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21219138/ "Tycko R. Solid-state NMR studies of amyloid fibril structure. Annu Rev Phys Chem. 2011;62:279-99. doi: 10.1146/annurev-physchem-032210-103539. Review. PubMed PMID: 21219138; PubMed Central PMCID: PMC3191906."

[Jones 2011]: http://www.ncbi.nlm.nih.gov/pubmed/22002245 "Jones EM, Wu B, Surewicz K, Nadaud PS, Helmus JJ, Chen S, Jaroniec CP, Surewicz WK. Structural polymorphism in amyloids: new insights from studies with  Y145Stop prion protein fibrils. J Biol Chem. 2011 Dec 9;286(49):42777-84. doi: 10.1074/jbc.M111.302539. Epub 2011 Oct 15. PubMed PMID: 22002245; PubMed Central  PMCID: PMC3234922."

[Tycko 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20925423 "Tycko R, Savtchenko R, Ostapchenko VG, Makarava N, Baskakov IV. The α-helical  C-terminal domain of full-length recombinant PrP converts to an in-register parallel β-sheet structure in PrP fibrils: evidence from solid state nuclear magnetic resonance. Biochemistry. 2010 Nov 9;49(44):9488-97. doi: 10.1021/bi1013134. PubMed PMID: 20925423; PubMed Central PMCID: PMC3025268."

[Walsh 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19278656/ "Walsh P, Simonetti K, Sharpe S. Core structure of amyloid fibrils formed by residues 106-126 of the human prion protein. Structure. 2009 Mar 11;17(3):417-26. doi: 10.1016/j.str.2008.12.018. PubMed PMID: 19278656."

[Lin 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20358555 "Lin NS, Chao JC, Cheng HM, Chou FC, Chang CF, Chen YR, Chang YJ, Huang SJ, Chan JC. Molecular structure of amyloid fibrils formed by residues 127 to 147 of  the human prion protein. Chemistry. 2010 May 10;16(18):5492-9. doi: 10.1002/chem.200903290. PubMed PMID: 20358555."

[Onisko 2005]: http://www.ncbi.nlm.nih.gov/pubmed/16042387 "Onisko B, Fernández EG, Freire ML, Schwarz A, Baier M, Camiña F, García JR, Rodríguez-Segade Villamarín S, Requena JR. Probing PrPSc structure using chemical cross-linking and mass spectrometry: evidence of the proximity of Gly90 amino termini in the PrP 27-30 aggregate. Biochemistry. 2005 Aug 2;44(30):10100-9. PubMed PMID: 16042387."

[Gong 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21526750 "Gong B, Ramos A, Vázquez-Fernández E, Silva CJ, Alonso J, Liu Z, Requena JR. Probing structural differences between PrP(C) and PrP(Sc) by surface nitration and acetylation: evidence of conformational change in the C-terminus. Biochemistry. 2011 Jun 7;50(22):4963-72. doi: 10.1021/bi102073j. Epub 2011 May 11. PubMed PMID: 21526750."

[Silva 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22436143 "Silva CJ. Using small molecule reagents to selectively modify epitopes based on their conformation. Prion. 2012 Apr-Jun;6(2):163-73. doi: 10.4161/pri.18795. Epub 2012 Apr 1. PubMed PMID: 22436143; PubMed Central PMCID: PMC3366355."

[Schoch 2006]: http://www.ncbi.nlm.nih.gov/pubmed/16354106 "Schoch G, Seeger H, Bogousslavsky J, Tolnay M, Janzer RC, Aguzzi A, Glatzel M. Analysis of prion strains by PrPSc profiling in sporadic Creutzfeldt-Jakob disease. PLoS Med. 2006 Feb;3(2):e14. Epub 2005 Dec 20. PubMed PMID: 16354106; PubMed Central PMCID: PMC1316067."

[Groveman 2014]: http://www.ncbi.nlm.nih.gov/pubmed/25028516 "Groveman BR, Dolan MA, Taubner LM, Kraus A, Wickner RB, Caughey B. Parallel in-register intermolecular β-sheet architectures for prion-seeded prion protein (PrP) amyloids. J Biol Chem. 2014 Aug 29;289(35):24129-42. doi: 10.1074/jbc.M114.578344. Epub 2014 Jul 15. PubMed PMID: 25028516; PubMed Central  PMCID: PMC4148845."

[Groveman & Kraus 2015]: http://www.ncbi.nlm.nih.gov/pubmed/25416779 "Groveman BR, Kraus A, Raymond LD, Dolan MA, Anson KJ, Dorward DW, Caughey B. Charge Neutralization of the Central Lysine Cluster in Prion Protein (PrP) Promotes PrPSc-like Folding of Recombinant PrP Amyloids. J Biol Chem. 2015 Jan 9;290(2):1119-28. doi: 10.1074/jbc.M114.619627. Epub 2014 Nov 21. PubMed PMID: 25416779."


