---
layout: post
title:  "What do we know about ApoE?"
date:   2016-03-02 12:01:00
author: ericminikel
location: Cambridge, MA
thumb120: http://www.cureffi.org/media/2016/03/apoe-thumbnail.png
summary200: "I'm confused about ApoE. And the more I read, the more I think I'm not alone."
---

![](/media/2016/03/APOE_2L7B_0x820BBB.png)

I'm confused about ApoE. And the more I read, the more I think I'm not alone.

Of all the known genetic risk factors for late-onset Alzheimer disease, *APOE* genotype has by far the largest effect size &mdash; a single E4 allele increases your risk by ~3X [[Harold 2009]]. If you believe, as I do, that human genetics is a valuable source of validation for therapeutic strategies, then it follows that ApoE might be considered as one possible drug target for preventing Alzheimer disease.

The trouble is, no one seems to know the mechanism by which the E4 allele increases Alzheimer risk. There is not even any agreement on whether E4 represents a gain of function or a loss of function, and therefore on whether one should strive to develop inhibitors or activators of ApoE. Accordingly, people have taken approaches that are all over the map. Some think ApoE is beneficial, and have sought to therapeutically upregulate *APOE* expression [[US Patent US6274603B1], [Jiang 2008], [Cramer 2012]]. Others view ApoE as the enemy, and look to its knockdown or its inhibition with monoclonal antibodies as potential therapies [[Kim 2011], [Kim 2012], [Bien-Ly 2012], [Choi 2015]]. Still others believe that the answer is isoform-dependent, and that one should seek to downregulate the risk-increasing E4 isoform and/or upregulate the protective E2 isoform [[Hudry 2013], [Hu 2015]].

It amazes me that there can be so little agreement on such a fundamental issue. And it's not for lack of interest. One of the studies mentioned above [[Cramer 2012]], which claimed that bexarotene upregulated *Apoe* and in turn reduced A&beta; plaque burden in a mouse model of Alzheimer's, quickly came under fire when four academic groups [failed to replicate it](http://www.nature.com/news/studies-cast-doubt-on-cancer-drug-as-alzheimer-s-treatment-1.13058), and it recently [made the news again](http://www.nature.com/news/biotech-giant-publishes-failures-to-confirm-high-profile-science-1.19269) when Amgen announced that it had also [failed to replicate it](http://f1000research.com/articles/5-138/v1).

Upon turning to the literature, the first problem that I found is that *APOE* is *too* well-studied, with [21,301 PubMed hits](http://www.ncbi.nlm.nih.gov/pubmed/?term=apoe) as of today. Paradoxically, all this literature makes my task harder, rather than easier. The literature is so crowded with different claims about *APOE* that it becomes incredibly difficult to tell what is real. For instance, in a quick Google Scholar search I spotted more than ten candidate gene studies proposing an association between *APOE* genotype and either the risk, age of onset, or phenotype of prion disease, even though in the largest prion disease GWAS to date [[Mead 2012]], there was not so much as a whiff of association at the *APOE* locus. To believe everything in the literature, it seems, would be to believe this one protein is found in every subcellular compartment, interacts with every other protein, modulates every mouse phenotype, and is a risk factor for every neurological disease. I soon developed the following postulate: for every hypothesis that you can imagine about ApoE, there exists a paper that claims it's true.

To try to sift through this morass, I figured I'd start with what I know best: human genetics. That's not to say that human genetics papers about *APOE* are any more likely to be correct than papers about mouse models or cell biology or anything else, but rather, human genetics is a body of literature that I know how to read. I don't always know a good mouse from a bad mouse or a representative image from a cherry-picked one, but I have at least a decent handle on the methods and statistical thresholds that pass for evidence in human genetics. So the question of this post, then: what does the human genetic data tell us about *APOE*? Be warned, even on this subject I'm far from an expert, so this post is very likely to contain mistakes and omissions.

### First, some general facts about *APOE* variation in humans

[*APOE*](http://exac.broadinstitute.org/gene/ENSG00000130203) consists of 3 exons located on chromosome 19. In terms of numbering, the coding sequence is 318 codons long including the stop codon, but the first 18 codons encode a signal peptide, so the numbering scheme in common use in the literature subtracts 18 from the codon number. Two variants are common worldwide, [C130R](http://exac.broadinstitute.org/variant/19-45411941-T-C) (often called C112R) and [R176C](http://exac.broadinstitute.org/variant/19-45412079-C-T) (often R158C), and another variant, [R163C](http://exac.broadinstitute.org/variant/19-45412040-C-T) (or R145C) has a frequency of 6% in Africans. And of course, as with any gene, there are loads of other less common variants &mdash; a total of [99 different protein-altering variants](http://exac.broadinstitute.org/gene/ENSG00000130203) in the current ExAC release. Other than the variants discussed above, plus one variant in the signal peptide (N14K), none of these has an allele frequency greater than 1% in any large continental population. Note that, as is convention, the amino acid codes above are given in comparison to the reference genome, but in the case of C112R, R is actually the ancestral allele, shared by [chimpanzees](http://www.uniprot.org/uniprot/Q9GJU3) and [mice](http://www.uniprot.org/uniprot/P08226) alike. This means that 112C, which is the reference and the most common allele in all major world populations, actually arose in humans and swept to its current 70-90% allele frequency before the out-of-Africa bottleneck. The R158C variant, which is found in *cis* with 112C, must therefore have arisen later on that same new haplotype.

![](/media/2016/03/apoe-allele-descent.png)

People don't often refer directly to these missense variants by their codon and resulting amino acid substitition as I've done above. Instead, one often hears a specific nomenclature for *APOE* variants, one which, as far as I can tell, arose by historical accident. ApoE protein was studied using biochemical techniques years before DNA sequencing was in common use, and it was noticed early on that different versions of the protein had different isoelectric points and could be separated on 2D gels. (I'm not sure of the earliest citation for this, but for an example gel see [[Moriyama 1992]]). The term "E3" came to refer to the electrophoretic mobility of the protein encoded by the most common allele (now the reference). "E2" referred to any ApoE protein with one fewer positive charge, and "E4" referred to any ApoE protein with one more positive charge. It turns out that most of the time, the underlying missense variants that gave rise to these differences were R158C (E2) and C112R (E4), though there were exceptions: that common African R145C variant mentioned above also has "E2" electrophoretic mobility, and there were other, rarer, variants as well, such as "E2 Fukuoka" (R224Q) [[Moriyama 1996]]. As an additional historical curiosity, "E1" referred to ApoE protein with two fewer positive charges than the reference E3 allele, and reported E1 alleles included K146E [[Moriyama 1992]] as well as G127D *cis* R158C [[Weisgraber 1984]]. Nowadays, when people say E2 or E4, they're no longer thinking of electrophoretic mobility, but instead are simply referring to R158C and C112R respectively &mdash; and for the avoidance of confusion, I will refer directly to the missense variants in the rest of this post, with parenthetical reference to the "E" term in common use.

### Blood lipids and cardiovascular disease risk

Years before *APOE* was ever associated with Alzheimer disease, its protein product was known as a component of lipoprotein particles in the bloodstream. As far as I could find, the earliest reference to what we now call ApoE may be the characterization of an "arginine-rich" (34 arginines, [we now know](http://www.uniprot.org/uniprot/P02649)) protein enriched within very low density lipoproteins (VLDL) particles in the blood of patients with a disorder called dysbetalipoproteinemia [[Havel & Kane 1973]]. Within a few years, the arginine-rich protein, christened ApoE, was known to be present not just in VLDL, but also in chylomicrons and HDL [reviewed in [Smith 1978]].

Fast-forwarding to modern day, we know that *APOE* genotype is associated with blood lipids and cardiovascular risk in a few different ways. The *APOE* locus is reproducibly associated with LDL cholesterol levels in genome-wide association studies [[Willer 2008], [Teslovich 2010], [Global Lipids Genetics Consortium 2013]]. None of those GWAS papers discussed the direction of effect nor whether the locus was also associated with coronary artery disease, but those issues were examined in detail in an enormous meta-analysis (~100,000 individuals) [[Bennet 2007]]. That study found that R158C (E2) resulted in reduced LDL and reduced risk of coronary artery disease, while C112R (E4) increased both. For LDL, R158C homozygotes (E2/E2) had 41 mg/dL lower LDL than C112R homozygotes (E4/E4). In an allelic model, compared to the reference genome (E3), R158C (E2) was associated with reduced risk of coronary artery disease (odds ratio = 0.80) while C112R (E4) was associated with increased risk (OR = 1.06).

At first glance, then, this all sounds straightforward: R158C (E2) is good, and C112R (E4) is bad. It's not quite so simple, though. Although R158C (E2) *on average* is associated with reduced risk of cardiovascular disease, there seems to be a lot of variance in people's outcomes, and the earliest studies of E2 actually examined its role in *causing* disease. It's confusing, and I only wrapped my mind around it after finding one good thorough review [[Utermann 1987]]. The R158C (E2) variant is described as being defective in receptor binding, thus causing ApoE protein, which is found on chylomicrons, VLDL, IDL, HDL, to stay in the bloodstream and circulate longer. R158C (E2) is associated with increased levels of circulating ApoE (associated with one or more of chylomicrons, VLDL, IDL, HDL) and decreased levels of circulating ApoB (associated with LDL). So although LDL is apparently always decreased in these individuals, which is good, VLDL and other lipoprotein particles can be increased, sometimes pathologically so, such that a subset of R158C homozygotes (E2/E2) actually suffer from a disorder called either dysbetalipoproteinemia or type 3 hyperlipoproteinemia. These people have very high total cholesterol, and develop xanthomas (visible deposits of cholesterol under their skin) and early-onset heart attacks. I couldn't find any reference for what proportion of R158C homozygotes (E2/E2) develop this condition, but it must be fairly small, since in the aggregate, even in a homozygous state, this variant appears to be associated with reduced risk of coronary artery disease [[Bennet 2007]].

To further add to the confusion, *APOE* loss-of-function variants also cause a Mendelian, recessive form of dysbetalipoproteinemia. Individuals lacking a functional copy of *APOE* have low LDL but high levels of all other lipoprotein particle types, especially VLDL, which is increased by several-fold. This, at first, seemed incomprehensible to me: how could the *absence* of ApoE cause the same disorder as homozygosity for R158C (E2), which is associated with *increased* circulating ApoE? Utermann speculates that any deficiency in ApoE binding to its receptors, whether due to R158C or due to total lack of ApoE, results in upregulation of the LDL receptor (thus reducing LDL) but an inability to scavenge "remnant" lipoprotein particles such as VLDL and IDL (and thus an increase in levels of those particles). To date, several families with this recessive disease of *APOE* deficiency have been reported [[Ghiselli 1981], [Mabuchi 1989], [Kurosaka 1991], [Lohse 1992], [Feussner 1996], [Mak 2014]], with casual variants including a variety of nonsense, frameshift, and splice variants. There was also one individual with an "E1" missense allele in *trans* to a frameshift [[Feussner 1992]]. Following the discovery of these families, *Apoe* knockout mice [[Zhang 1992]] became a widely used mouse model of atherosclerosis.

Is the *APOE* loss-of-function disease purely recessive, or is there a heterozygote phenotype? No one seems to know for sure. Certainly, the severe phenotypes of xanthomas and early onset atherosclerosis were only reported in double null individuals, and not in their heterozygote relatives. One proband did have an affected father, but he was deceased and his genotype was unknown, and 3 out of 6 of his children were affected [[Ghiselli 1981]], so he may well have been a homozygote himself. Some of the papers cited above do present VLDL, LDL, triglyceride and other blood lipid levels for one or two heterozygote relatives, but none of their numbers look to be glaringly outside the range in the general population, so it is not clear whether they are significantly shifted from the population distribution. One of the papers [[Feussner 1996]] phenotyped 10 heterozygotes, two of whom were even deemed to be affected, but even then, none of the lipid levels were significantly different in hets from controls after multiple testing correction. Figuring out for sure if there is a heterozygote phenotype will take a larger *N* that we don't have yet (see discussion at bottom).

### Alzheimer disease

Almost two decades after the association of ApoE to blood lipid phenotypes began to be understood, people started to notice that ApoE protein could be immunodetected in A&beta; plaques in Alzheimer disease brains [[Namba 1991]]. This made *APOE* an attractive target for candidate gene studies, and within a couple of years, it was reported that *APOE*genotype was associated with the risk of Alzheimer disease [[Strittmatter 1993], [Corder 1993]].

The amazing thing was that this was actually true. A rare success story of the candidate gene era, the association between *APOE* and Alzheimer disease has been resoundingly validated time and again, and modern-day GWAS have shown that *APOE* genotype actually has far and away the largest effect size of any common genetic variant on Alzheimer disease risk [[Naj 2011], [Hollingworth 2011], [Lambert 2013]].

Compared to the reference (E3) allele, R158C (E2) is protective, while C112R (E4) confers increased risk. I've seen a wide range of estimates of what exactly the odds ratios are [[Corder 1994], [Tsai 1994], [Genin 2011]]. The largest meta-analysis I found [[Farrer 1997]] has it as follows:

| genotype | odds ratio |
| ---- | ---- |
| E2/E2 | 0.6 |
| E2/E3 | 0.6 |
| E3/E3 | 1.0 |
| E2/E4 | 2.6 |
| E3/E4 | 3.2 |
| E4/E4 | 14.9 |

Odds ratios are a funny thing. In prion disease, which is very rare, even an odds ratio of 100 can mean a clinically negligible amount of risk [[Minikel 2016]]. But for Alzheimer disease, which is so common by old age *a priori*, some of these odds ratios verge on Mendelian: C112R (E4) homozygotes exhibit >50% penetrance of Alzheimer disease by age 85 [[Genin 2011]].

### Other phenotypes

To see if there were any genome-wide significant associations of other traits with *APOE*, I turned to the [NHGRI/EBI GWAS catalog](http://www.ebi.ac.uk/gwas/docs/downloads) and ran a couple of queries:

{% highlight bash %}
wget http://www.ebi.ac.uk/gwas/api/search/downloads/full
cat full | grep APOE | awk -v FS="\t" -v OFS="\t" '$29 > 7.3 {print $2, $8, $29}' | wc -l
cat full | grep APOE | awk -v FS="\t" -v OFS="\t" '$29 > 7.3 {print $8}' | sort | uniq -c
{% endhighlight %}

As of today, there are a total of 111 published studies reporting genome-wide significant (*P* < 5&times;10<sup>-8</sup>) associations at the *APOE* locus, nominally pertaining to 35 different traits. On closer examination, though, it seems that Alzheimer disease and blood lipids are the only two real underlying traits here. Most of the 35 GWAS traits are either synonymous or closely correlated with one of those two traits. For instance, reported associations with A&beta; levels and verbal declarative memory simply reflect the association with Alzheimer disease, and reported associations with metabolic syndrome, C-reactive protein, and blood metabolite ratios are presumably secondary to the effects on blood cholesterol. Age-related macular degeneration (AMD) also makes the list of associated traits, but here too, there is speculation that this may simply reflect an association of blood lipid profiles with AMD [[McKay 2011]]. Similarly, because heart attacks and Alzheimer disease are both major killers, it seems reasonable to suppose that the recently reported association of *APOE* with longevity [[Fortney 2015]] results from one or both of these factors.

### Are the blood lipid and Alzheimer associations related?

Above, I've suggested that all of the other *APOE*-associated traits are simply secondary to *APOE*'s associations with blood lipids and with Alzheimer disease. If so, then the question naturally arises, what about these two traits? Does *APOE* genotype affect the two traits by two separate mechanisms, or is the Alzheimer association simply a result of changes in blood lipid composition?

A link seemed unlikely *a priori*, because C112R (E4) has such a modest effect on blood lipids, with only an odds ratio of 1.06 for coronary artery disease, which makes it hard to believe that the odds ratio of >3 for Alzheimer disease is secondary to that. But I had a look anyway, and from what I see in the literature, there doesn't seem to be any evidence at all for a mechanistic link between the two. If there were, then one would expect that other genes and environmental factors that affect blood lipids would affect Alzheimer disease risk, and that doesn't seem to be the case. 

For one, blood lipids themselves have never been shown to be predictive for Alzheimer disease. This argument can be further extended to environmental risk factors and to cholesterol-lowering drugs. Diet has been linked to blood cholesterol levels for decades [[Keys 1965]], yet has never been convincingly linked to Alzheimer disease risk, no matter how great the proliferation of Alzheimer's prevention cookbooks on Amazon. And even though millions of millions of people take statins to lower their LDL levels, the claims that statins reduce Alzheimer risk [[Jick 2000]] remain unproven and controversial [[Zandi 2005]]. As Stanley Prusiner put it, when asked by an audience member at Prion2013 what people could do to reduce their risk of dementia, "It's not how many crossword puzzles you do. It's called luck." As of today, scientists have not confidently identified a single environmental risk factor for Alzheimer disease, even as hypercholesterolemia is increasingly well-understood and well-treated.

I also looked for evidence of shared genetics. At last count, there were 19 loci associated with Alzheimer disease risk [[Lambert 2013]] and 157 loci associated with blood lipid phenotypes [[Global Lipids Genetics Consortium 2013]], and *APOE* is the single point of overlap between these two lists. To double-check this conclusion, I turned again to the GWAS catalog, and ran a quick query in R:

{% highlight R %}
gwas = read.table('full',header=TRUE,sep='\t',quote='',comment.char='')
dim(gwas)
# [1] 25217    34
gwas$alzheimer = grepl('Alzheimer',gwas$DISEASE.TRAIT)
gwas$lipid = grepl('holesterol',gwas$DISEASE.TRAIT)
cholesterol_genes = unique(gwas$REPORTED.GENE.S.[gwas$lipid & gwas$PVALUE_MLOG > 7.3])
alzheimer_genes = unique(gwas$REPORTED.GENE.S.[gwas$alzheimer & gwas$PVALUE_MLOG > 7.3])
alzheimer_genes[alzheimer_genes %in% cholesterol_genes]
# [1] NA                    "APOE"                "APOE, TOMM40"        "APOC1"               "intergenic"          "APOE, TOMM40, PVRL2"
{% endhighlight %}

Sure enough, even accounting for every study in the GWAS catalog, *APOE* is still the only genomic locus associated with anything related to Alzheimer's and anything related to cholesterol. (*TOMM40*, *APOC1*, and *PVRL2* are neighboring genes that fell within *APOE*'s linkage peak in some early studies).

So it is *not* broadly the case that genetic variants that affect blood lipids also affect Alzheimer risk &mdash; only *APOE* does. None of this is to rule out the possibility that the effects of *APOE* genotype on lipids and on Alzheimer disease risk are somehow mechanistically linked, but if there is a link, it is incredibly specific to *APOE* &mdash; it's certainly not as simple as "blood cholesterol affects the risk of Alzheimer disease."

### Does *APOE* E4 increase Alzheimer risk by a gain or loss of function?

I argued above that the evidence concerning ApoE's mechanistic link to cholesterol and cardiovascular disease is complicated and confusing. R158C (E2) is associated with an increased, but still low, risk of a specific blood lipid disorder, dysbetalipoproteinemia, while still overall being associated with reduced LDL and reduced cardiovascular disease risk. C112R (E4) is associated with increased LDL and increased cardiovascular disease risk. *APOE* loss-of-function homozygotes have low LDL but wildly increased VLDL and an extremely high risk of early onset heart attacks. 

But if we do not think that there is necessarily any mechanistic link between the effects of *APOE* on blood lipids and the effects of *APOE* on Alzheimer disease risk, then all this conflicting evidence may actually have no bearing on the question at hand.

Instead, maybe we should ask a separate question: do individuals with *APOE* loss-of-function variants (whether heterozygous or homozygous) have an increased or decreased risk of Alzheimer disease compared to the general population? 

I would have thought someone would have asked this question by now, but I don't see any papers on it. Perhaps it's just because we don't have the data yet. The handful of humans who are double null for *APOE* (in the studies cited above) have very high cholesterol that apparently is hard to manage with available drugs [[Mak 2014]], so sadly, it's not clear how many of them have lived long enough that they would have developed Alzheimer's, even if their risk were high. Only a small number of heterozygote relatives were phenotyped in those studies, and it is not at all clear how detailed a family history was collected, encompassing how many individuals, and whether a family history of dementia would have been noticed and written up if it were present &mdash; particularly since some of the early studies pre-date the discovery of *APOE*'s association with Alzheimer's.

Meanwhile, *APOE* loss-of-function variants are rare enough in the population that a genotype-first approach may not yet be well-powered either as of 2016. In all of [ExAC](http://exac.broadinstitute.org/gene/ENSG00000130203) there are only 12 individuals with nominally protein-truncating variants, and most affect non-canonical transcripts or occur in the final exon, where a total loss of function is not guaranteed.

I see a few potential roads forward. If extended families of the already-identified *APOE* null individuals are keen to participate in research, one could imagine researchers following the alleles out far enough in the family tree to find a large number of heterozygotes. Or, in a few years when there are larger cohorts of individuals willing to be re-contacted &mdash; maybe the [million person cohort](http://www.nih.gov/research-training/precision-medicine-initiative/digital-health-data-million-person-precision-medicine-initiative-cohort) &mdash; some researcher will manage to re-contact enough unrelated *APOE* LoF hets and ask them whether there is dementia in their family history. Alternately, case-control cohorts may grow large enough that a significant enrichment or depletion of LoF alleles among Alzheimer disease cases may become apparent.

A paper a few years ago proposes the idea of small molecule "structure correctors", ligands that make a C112R (E4) allele fold more like a reference (E3) allele [[Chen 2012]]. The approach is intruiguing for being totally mechanism-agnostic: we don't know why E3 is better, but it is, so let's make the protein act that way. But finding a ligand that does exactly this thing is a much narrower target to hit than simply finding a ligand. So until we have a definitive answer on the gain vs. loss of function issue, the road forward is difficult indeed.

[Havel & Kane 1973]: http://www.ncbi.nlm.nih.gov/pubmed/4352966 "Havel RJ, Kane JP. Primary dysbetalipoproteinemia: predominance of a specific  apoprotein species in triglyceride-rich lipoproteins. Proc Natl Acad Sci U S A. 1973 Jul;70(7):2015-9. PubMed PMID: 4352966; PubMed Central PMCID: PMC433655."

[Smith 1978]: http://www.ncbi.nlm.nih.gov/pubmed/209732 "Smith LC, Pownall HJ, Gotto AM Jr. The plasma lipoproteins: structure and metabolism. Annu Rev Biochem. 1978;47:751-7. Review. PubMed PMID: 209732."

[Schneider 1981]: http://www.ncbi.nlm.nih.gov/pubmed/6270194/ "Schneider WJ, Kovanen PT, Brown MS, Goldstein JL, Utermann G, Weber W, Havel RJ, Kotite L, Kane JP, Innerarity TL, Mahley RW. Familial dysbetalipoproteinemia. Abnormal binding of mutant apoprotein E to low density lipoprotein receptors of human fibroblasts and membranes from liver and adrenal of rats, rabbits, and cows. J Clin Invest. 1981 Oct;68(4):1075-85. PubMed PMID: 6270194; PubMed Central PMCID: PMC370895."

[Harold 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19734902 "Harold D, Abraham R, Hollingworth P, Sims R, Gerrish A, Hamshere ML, Pahwa JS, Moskvina V, Dowzell K, Williams A, Jones N, Thomas C, Stretton A, Morgan AR, Lovestone S, Powell J, Proitsi P, Lupton MK, Brayne C, Rubinsztein DC, Gill M, Lawlor B, Lynch A, Morgan K, Brown KS, Passmore PA, Craig D, McGuinness B, Todd S, Holmes C, Mann D, Smith AD, Love S, Kehoe PG, Hardy J, Mead S, Fox N, Rossor M, Collinge J, Maier W, Jessen F, Schürmann B, Heun R, van den Bussche H, Heuser  I, Kornhuber J, Wiltfang J, Dichgans M, Frölich L, Hampel H, Hüll M, Rujescu D, Goate AM, Kauwe JS, Cruchaga C, Nowotny P, Morris JC, Mayo K, Sleegers K, Bettens K, Engelborghs S, De Deyn PP, Van Broeckhoven C, Livingston G, Bass NJ, Gurling H, McQuillin A, Gwilliam R, Deloukas P, Al-Chalabi A, Shaw CE, Tsolaki M, Singleton AB, Guerreiro R, Mühleisen TW, Nöthen MM, Moebus S, Jöckel KH, Klopp N, Wichmann HE, Carrasquillo MM, Pankratz VS, Younkin SG, Holmans PA, O'Donovan M, Owen MJ, Williams J. Genome-wide association study identifies variants at CLU and PICALM associated with Alzheimer's disease. Nat Genet. 2009 Oct;41(10):1088-93. doi: 10.1038/ng.440. Epub 2009 Sep 6. Erratum in: Nat Genet. 2009 Oct;41(10):1156. Nat Genet. 2013 Jun;45(6):712. Haun, Reinhard [added]. PubMed PMID: 19734902; PubMed Central PMCID: PMC2845877."

[de Knijff 1994]: http://www.ncbi.nlm.nih.gov/pubmed/7833947/ "de Knijff P, van den Maagdenberg AM, Frants RR, Havekes LM. Genetic heterogeneity of apolipoprotein E and its influence on plasma lipid and lipoprotein levels. Hum Mutat. 1994;4(3):178-94. Review. PubMed PMID: 7833947."

[Utermann 1987]: http://www.ncbi.nlm.nih.gov/pubmed/3544759/ "Utermann G. Apolipoprotein E polymorphism in health and disease. Am Heart J. 1987 Feb;113(2 Pt 2):433-40. Review. PubMed PMID: 3544759."

[Mead 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22210626/ "Mead S, Uphill J, Beck J, Poulter M, Campbell T, Lowe J, Adamson G, Hummerich  H, Klopp N, Rückert IM, Wichmann HE, Azazi D, Plagnol V, Pako WH, Whitfield J, Alpers MP, Whittaker J, Balding DJ, Zerr I, Kretzschmar H, Collinge J. Genome-wide association study in multiple human prion diseases suggests genetic risk factors additional to PRNP. Hum Mol Genet. 2012 Apr 15;21(8):1897-906. doi:  10.1093/hmg/ddr607. Epub 2011 Dec 30. PubMed PMID: 22210626; PubMed Central PMCID: PMC3313791."

[Mahley & Huang 2012]: http://www.ncbi.nlm.nih.gov/pubmed/23217737 "Mahley RW, Huang Y. Apolipoprotein e sets the stage: response to injury triggers neuropathology. Neuron. 2012 Dec 6;76(5):871-85. doi: 10.1016/j.neuron.2012.11.020. Review. PubMed PMID: 23217737."

[U.S. Patent WO1997016458A1]: https://patents.google.com/patent/WO1997016458A1 "Aleshkov Sergie B Zannis Vassilis I; Original Assignee: Kos Holdings Inc, Univ Boston. Apolipoprotein e2 and treatment of alzheimer's disease. US Patent WO1997016458A1"

[McKay 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21882290/ "McKay GJ, Patterson CC, Chakravarthy U, Dasari S, Klaver CC, Vingerling JR, Ho L, de Jong PT, Fletcher AE, Young IS, Seland JH, Rahu M, Soubrane G, Tomazzoli L, Topouzis F, Vioque J, Hingorani AD, Sofat R, Dean M, Sawitzke J, Seddon JM, Peter I, Webster AR, Moore AT, Yates JR, Cipriani V, Fritsche LG, Weber BH, Keilhauer CN, Lotery AJ, Ennis S, Klein ML, Francis PJ, Stambolian D, Orlin A, Gorin MB, Weeks DE, Kuo CL, Swaroop A, Othman M, Kanda A, Chen W, Abecasis GR, Wright AF, Hayward C, Baird PN, Guymer RH, Attia J, Thakkinstian A, Silvestri G. Evidence of association of APOE with age-related macular degeneration: a pooled analysis of 15 studies. Hum Mutat. 2011 Dec;32(12):1407-16. doi: 10.1002/humu.21577. Epub 2011 Sep 12. PubMed PMID: 21882290; PubMed Central PMCID: PMC3217135"

[Fortney 2015]: http://www.ncbi.nlm.nih.gov/pubmed/26677855 "Fortney K, Dobriban E, Garagnani P, Pirazzini C, Monti D, Mari D, Atzmon G, Barzilai N, Franceschi C, Owen AB, Kim SK. Genome-Wide Scan Informed by Age-Related Disease Identifies Loci for Exceptional Human Longevity. PLoS Genet.  2015 Dec 17;11(12):e1005728. doi: 10.1371/journal.pgen.1005728. eCollection 2015  Dec. PubMed PMID: 26677855; PubMed Central PMCID: PMC4683064."

[Willer 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18193043 "Willer CJ, Sanna S, Jackson AU, Scuteri A, Bonnycastle LL, Clarke R, Heath SC, Timpson NJ, Najjar SS, Stringham HM, Strait J, Duren WL, Maschio A, Busonero F, Mulas A, Albai G, Swift AJ, Morken MA, Narisu N, Bennett D, Parish S, Shen H, Galan P, Meneton P, Hercberg S, Zelenika D, Chen WM, Li Y, Scott LJ, Scheet PA, Sundvall J, Watanabe RM, Nagaraja R, Ebrahim S, Lawlor DA, Ben-Shlomo Y, Davey-Smith G, Shuldiner AR, Collins R, Bergman RN, Uda M, Tuomilehto J, Cao A, Collins FS, Lakatta E, Lathrop GM, Boehnke M, Schlessinger D, Mohlke KL, Abecasis GR. Newly identified loci that influence lipid concentrations and risk of coronary artery disease. Nat Genet. 2008 Feb;40(2):161-9. doi: 10.1038/ng.76. Epub 2008 Jan 13. PubMed PMID: 18193043."

[Global Lipids Genetics Consortium 2013]: http://www.ncbi.nlm.nih.gov/pubmed/24097068/ "Global Lipids Genetics Consortium, Willer CJ, Schmidt EM, Sengupta S, Peloso GM, Gustafsson S, Kanoni S, Ganna A, Chen J, Buchkovich ML, Mora S, Beckmann JS,  Bragg-Gresham JL, Chang HY, Demirkan A, Den Hertog HM, Do R, Donnelly LA, Ehret GB, Esko T, Feitosa MF, Ferreira T, Fischer K, Fontanillas P, Fraser RM, Freitag  DF, Gurdasani D, Heikkilä K, Hyppönen E, Isaacs A, Jackson AU, Johansson A, Johnson T, Kaakinen M, Kettunen J, Kleber ME, Li X, Luan J, Lyytikäinen LP, Magnusson PK, Mangino M, Mihailov E, Montasser ME, Müller-Nurasyid M, Nolte IM, O'Connell JR, Palmer CD, Perola M, Petersen AK, Sanna S, Saxena R, Service SK, Shah S, Shungin D, Sidore C, Song C, Strawbridge RJ, Surakka I, Tanaka T, Teslovich TM, Thorleifsson G, Van den Herik EG, Voight BF, Volcik KA, Waite LL, Wong A, Wu Y, Zhang W, Absher D, Asiki G, Barroso I, Been LF, Bolton JL, Bonnycastle LL, Brambilla P, Burnett MS, Cesana G, Dimitriou M,...iemi A, Kivimaki M, Kooner JS, Koudstaal PJ, Krauss RM, Kuh D, Kuusisto J, Kyvik KO, Laakso M, Lakka TA, Lind L, Lindgren CM,  Martin NG, März W, McCarthy MI, McKenzie CA, Meneton P, Metspalu A, Moilanen L, Morris AD, Munroe PB, Njølstad I, Pedersen NL, Power C, Pramstaller PP, Price JF, Psaty BM, Quertermous T, Rauramaa R, Saleheen D, Salomaa V, Sanghera DK, Saramies J, Schwarz PE, Sheu WH, Shuldiner AR, Siegbahn A, Spector TD, Stefansson K, Strachan DP, Tayo BO, Tremoli E, Tuomilehto J, Uusitupa M, van Duijn CM, Vollenweider P, Wallentin L, Wareham NJ, Whitfield JB, Wolffenbuttel BH, Ordovas  JM, Boerwinkle E, Palmer CN, Thorsteinsdottir U, Chasman DI, Rotter JI, Franks PW, Ripatti S, Cupples LA, Sandhu MS, Rich SS, Boehnke M, Deloukas P, Kathiresan  S, Mohlke KL, Ingelsson E, Abecasis GR. Discovery and refinement of loci associated with lipid levels. Nat Genet. 2013 Nov;45(11):1274-83. doi: 10.1038/ng.2797. Epub 2013 Oct 6. PubMed PMID: 24097068; PubMed Central PMCID: PMC3838666."

[Mozaffarian 2016]: http://www.ncbi.nlm.nih.gov/pubmed/26811276 "Mozaffarian D, Benjamin EJ, Go AS, Arnett DK, Blaha MJ, Cushman M, Das SR, de  Ferranti S, Després JP, Fullerton HJ, Howard VJ, Huffman MD, Isasi CR, Jiménez MC, Judd SE, Kissela BM, Lichtman JH, Lisabeth LD, Liu S, Mackey RH, Magid DJ, McGuire DK, Mohler ER 3rd, Moy CS, Muntner P, Mussolino ME, Nasir K, Neumar RW, Nichol G, Palaniappan L, Pandey DK, Reeves MJ, Rodriguez CJ, Rosamond W, Sorlie PD, Stein J, Towfighi A, Turan TN, Virani SS, Woo D, Yeh RW, Turner MB; American  Heart Association Statistics Committee and Stroke Statistics Subcommittee. Executive Summary: Heart Disease and Stroke Statistics-2016 Update: A Report From the American Heart Association. Circulation. 2016 Jan 26;133(4):447-54. doi: 10.1161/CIR.0000000000000366. PubMed PMID: 26811276."

[Keys 1965]: http://www.ncbi.nlm.nih.gov/pubmed/25286460 "Keys A, Anderson JT, Grande F. Serum cholesterol response to changes in the diet: II. The effect of cholesterol in the diet. Metabolism. 1965 Jul;14(7):759-65. PubMed PMID: 25286460."

[Jick 2000]: http://www.ncbi.nlm.nih.gov/pubmed/11089820 "Jick H, Zornberg GL, Jick SS, Seshadri S, Drachman DA. Statins and the risk of dementia. Lancet. 2000 Nov 11;356(9242):1627-31. Erratum in: Lancet 2001 Feb 17;357(9255):562. PubMed PMID: 11089820."

[Zandi 2005]: http://www.ncbi.nlm.nih.gov/pubmed/15699299 "Zandi PP, Sparks DL, Khachaturian AS, Tschanz J, Norton M, Steinberg M, Welsh-Bohmer KA, Breitner JC; Cache County Study investigators. Do statins reduce risk of incident dementia and Alzheimer disease? The Cache County Study. Arch Gen Psychiatry. 2005 Feb;62(2):217-24. PubMed PMID: 15699299."

[Genin 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21556001 "Genin E, Hannequin D, Wallon D, Sleegers K, Hiltunen M, Combarros O, Bullido MJ, Engelborghs S, De Deyn P, Berr C, Pasquier F, Dubois B, Tognoni G, Fiévet N,  Brouwers N, Bettens K, Arosio B, Coto E, Del Zompo M, Mateo I, Epelbaum J, Frank-Garcia A, Helisalmi S, Porcellini E, Pilotto A, Forti P, Ferri R, Scarpini  E, Siciliano G, Solfrizzi V, Sorbi S, Spalletta G, Valdivieso F, Vepsäläinen S, Alvarez V, Bosco P, Mancuso M, Panza F, Nacmias B, Bossù P, Hanon O, Piccardi P,  Annoni G, Seripa D, Galimberti D, Licastro F, Soininen H, Dartigues JF, Kamboh MI, Van Broeckhoven C, Lambert JC, Amouyel P, Campion D. APOE and Alzheimer disease: a major gene with semi-dominant inheritance. Mol Psychiatry. 2011 Sep;16(9):903-7. doi: 10.1038/mp.2011.52. Epub 2011 May 10. PubMed PMID: 21556001; PubMed Central PMCID: PMC3162068."

[Moriyama 1992]: http://www.ncbi.nlm.nih.gov/pubmed/1356443 "Moriyama K, Sasaki J, Matsunaga A, Arakawa F, Takada Y, Araki K, Kaneko S, Arakawa K. Apolipoprotein E1 Lys-146----Glu with type III hyperlipoproteinemia. Biochim Biophys Acta. 1992 Sep 22;1128(1):58-64. PubMed PMID: 1356443."

[Moriyama 1996]: http://www.ncbi.nlm.nih.gov/pubmed/8664327 "Moriyama K, Sasaki J, Takada Y, Arakawa F, Matsunaga A, Ito Y, Arakawa K. Characterization of a novel variant of apolipoprotein E, E2 Fukuoka (Arg-224 -->  Gln) in a hyperlipidemic patient with xanthomatosis. Biochim Biophys Acta. 1996 Jun 11;1301(3):185-90. PubMed PMID: 8664327."

[Weisgraber 1984]: http://www.ncbi.nlm.nih.gov/pubmed/6323533/ "Weisgraber KH, Rall SC Jr, Innerarity TL, Mahley RW, Kuusi T, Ehnholm C. A novel electrophoretic variant of human apolipoprotein E. Identification and characterization of apolipoprotein E1. J Clin Invest. 1984 Apr;73(4):1024-33. PubMed PMID: 6323533; PubMed Central PMCID: PMC425115."

[Bennet 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17878422/ "Bennet AM, Di Angelantonio E, Ye Z, Wensley F, Dahlin A, Ahlbom A, Keavney B,  Collins R, Wiman B, de Faire U, Danesh J. Association of apolipoprotein E genotypes with lipid levels and coronary risk. JAMA. 2007 Sep 19;298(11):1300-11. Review. PubMed PMID: 17878422."

[Teslovich 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20686565/ "Teslovich TM, Musunuru K, Smith AV, Edmondson AC, Stylianou IM, Koseki M, Pirruccello JP, Ripatti S, Chasman DI, Willer CJ, Johansen CT, Fouchier SW, Isaacs A, Peloso GM, Barbalic M, Ricketts SL, Bis JC, Aulchenko YS, Thorleifsson  G, Feitosa MF, Chambers J, Orho-Melander M, Melander O, Johnson T, Li X, Guo X, Li M, Shin Cho Y, Jin Go M, Jin Kim Y, Lee JY, Park T, Kim K, Sim X, Twee-Hee Ong R, Croteau-Chonka DC, Lange LA, Smith JD, Song K, Hua Zhao J, Yuan X, Luan J, Lamina C, Ziegler A, Zhang W, Zee RY, Wright AF, Witteman JC, Wilson JF, Willemsen G, Wichmann HE, Whitfield JB, Waterworth DM, Wareham NJ, Waeber G, Vollenweider P, Voight BF, Vitart V, Uitterlinden AG, Uda M, Tuomilehto J, Thompson JR, Tanaka T, Surakka I, Stringham HM, Spector TD, Soranzo N, Smit JH, Sinisalo J, Silander K, Sijbrands EJ, Scuteri A, Scott J, Schlessinger D, Sanna S, Salomaa V, Saharinen J, Sabatti C, Ruokonen A, Rudan I, Rose LM, Roberts R,..., Gieger C, Freimer NB, Ferrucci L, Erdmann J, Elliott P, Ejebe KG, Döring A, Dominiczak AF, Demissie S,  Deloukas P, de Geus EJ, de Faire U, Crawford G, Collins FS, Chen YD, Caulfield MJ, Campbell H, Burtt NP, Bonnycastle LL, Boomsma DI, Boekholdt SM, Bergman RN, Barroso I, Bandinelli S, Ballantyne CM, Assimes TL, Quertermous T, Altshuler D, Seielstad M, Wong TY, Tai ES, Feranil AB, Kuzawa CW, Adair LS, Taylor HA Jr, Borecki IB, Gabriel SB, Wilson JG, Holm H, Thorsteinsdottir U, Gudnason V, Krauss RM, Mohlke KL, Ordovas JM, Munroe PB, Kooner JS, Tall AR, Hegele RA, Kastelein JJ, Schadt EE, Rotter JI, Boerwinkle E, Strachan DP, Mooser V, Stefansson K, Reilly MP, Samani NJ, Schunkert H, Cupples LA, Sandhu MS, Ridker PM, Rader DJ, van Duijn CM, Peltonen L, Abecasis GR, Boehnke M, Kathiresan S. Biological, clinical and population relevance of 95 loci for blood lipids. Nature. 2010 Aug 5;466(7307):707-13. doi: 10.1038/nature09270. PubMed PMID: 20686565; PubMed Central PMCID: PMC3039276."

[Ghiselli 1981]: http://www.ncbi.nlm.nih.gov/pubmed/6795720 "Ghiselli G, Schaefer EJ, Gascon P, Breser HB Jr. Type III hyperlipoproteinemia associated with apolipoprotein E deficiency. Science. 1981 Dec 11;214(4526):1239-41. PubMed PMID: 6795720."

[Mabuchi 1989]: http://www.ncbi.nlm.nih.gov/pubmed/2492364 "Mabuchi H, Itoh H, Takeda M, Kajinami K, Wakasugi T, Koizumi J, Takeda R, Asagami C. A young type III hyperlipoproteinemic patient associated with apolipoprotein E deficiency. Metabolism. 1989 Feb;38(2):115-9. PubMed PMID: 2492364."

[Kurosaka 1991]: http://www.ncbi.nlm.nih.gov/pubmed/1878006 "Kurosaka D, Teramoto T, Matsushima T, Yokoyama T, Yamada A, Aikawa T, Miyamoto Y, Kurokawa K. Apolipoprotein E deficiency with a depressed mRNA of normal size.  Atherosclerosis. 1991 May;88(1):15-20. PubMed PMID: 1878006."

[Feussner 1992]: http://www.ncbi.nlm.nih.gov/pubmed/1360898 "Feussner G, Funke H, Weng W, Assmann G, Lackner KJ, Ziegler R. Severe type III hyperlipoproteinemia associated with unusual apolipoprotein E1 phenotype and epsilon 1/'null' genotype. Eur J Clin Invest. 1992 Sep;22(9):599-608. PubMed PMID: 1360898."

[Lohse 1992]: http://www.ncbi.nlm.nih.gov/pubmed/1361196 "Lohse P, Brewer HB 3rd, Meng MS, Skarlatos SI, LaRosa JC, Brewer HB Jr. Familial apolipoprotein E deficiency and type III hyperlipoproteinemia due to a premature stop codon in the apolipoprotein E gene. J Lipid Res. 1992 Nov;33(11):1583-90. PubMed PMID: 1361196."

[Feussner 1996]: http://www.ncbi.nlm.nih.gov/pubmed/8571954 "Feussner G, Dobmeyer J, Gröne HJ, Lohmer S, Wohlfeil S. A 10-bp deletion in the apolipoprotein epsilon gene causing apolipoprotein E deficiency and severe type III hyperlipoproteinemia. Am J Hum Genet. 1996 Feb;58(2):281-91. PubMed PMID: 8571954; PubMed Central PMCID: PMC1914549."

[Mak 2014]: http://www.ncbi.nlm.nih.gov/pubmed/25111166 "Mak AC, Pullinger CR, Tang LF, Wong JS, Deo RC, Schwarz JM, Gugliucci A, Movsesyan I, Ishida BY, Chu C, Poon A, Kim P, Stock EO, Schaefer EJ, Asztalos BF, Castellano JM, Wyss-Coray T, Duncan JL, Miller BL, Kane JP, Kwok PY, Malloy MJ. Effects of the absence of apolipoprotein e on lipoproteins, neurocognitive function, and retinal function. JAMA Neurol. 2014 Oct;71(10):1228-36. doi: 10.1001/jamaneurol.2014.2011. PubMed PMID: 25111166; PubMed Central PMCID: PMC4714860."

[Zhang 1992]: http://www.ncbi.nlm.nih.gov/pubmed/1411543 "Zhang SH, Reddick RL, Piedrahita JA, Maeda N. Spontaneous hypercholesterolemia and arterial lesions in mice lacking apolipoprotein E. Science. 1992 Oct 16;258(5081):468-71. PubMed PMID: 1411543."

[Chen 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22158868 "Chen HK, Liu Z, Meyer-Franke A, Brodbeck J, Miranda RD, McGuire JG, Pleiss MA, Ji ZS, Balestra ME, Walker DW, Xu Q, Jeong DE, Budamagunta MS, Voss JC, Freedman  SB, Weisgraber KH, Huang Y, Mahley RW. Small molecule structure correctors abolish detrimental effects of apolipoprotein E4 in cultured neurons. J Biol Chem. 2012 Feb 17;287(8):5253-66. doi: 10.1074/jbc.M111.276162. Epub 2011 Dec 12. PubMed PMID: 22158868; PubMed Central PMCID: PMC3285306."

[US Patent US6274603B1]: https://patents.google.com/patent/US6274603B1 "Judes Poirer. Methods for increasing ApoE levels for the treatment of neurodegenerative disease. US09160462"

[Talbot 1994]: http://www.ncbi.nlm.nih.gov/pubmed/7910910 "Talbot C, Lendon C, Craddock N, Shears S, Morris JC, Goate A. Protection against Alzheimer's disease with apoE epsilon 2. Lancet. 1994 Jun 4;343(8910):1432-3. PubMed PMID: 7910910."

[Namba 1991]: http://www.ncbi.nlm.nih.gov/pubmed/2029618 "Namba Y, Tomonaga M, Kawasaki H, Otomo E, Ikeda K. Apolipoprotein E immunoreactivity in cerebral amyloid deposits and neurofibrillary tangles in Alzheimer's disease and kuru plaque amyloid in Creutzfeldt-Jakob disease. Brain Res. 1991 Feb 8;541(1):163-6. PubMed PMID: 2029618."

[Strittmatter 1993]: http://www.ncbi.nlm.nih.gov/pubmed/8446617 "Strittmatter WJ, Saunders AM, Schmechel D, Pericak-Vance M, Enghild J, Salvesen GS, Roses AD. Apolipoprotein E: high-avidity binding to beta-amyloid and increased frequency of type 4 allele in late-onset familial Alzheimer disease. Proc Natl Acad Sci U S A. 1993 Mar 1;90(5):1977-81. PubMed PMID: 8446617; PubMed  Central PMCID: PMC46003."

[Hollingworth 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21460840/ "Hollingworth P, Harold D, Sims R, Gerrish A, Lambert JC, Carrasquillo MM, Abraham R, Hamshere ML, Pahwa JS, Moskvina V, Dowzell K, Jones N, Stretton A, Thomas C, Richards A, Ivanov D, Widdowson C, Chapman J, Lovestone S, Powell J, Proitsi P, Lupton MK, Brayne C, Rubinsztein DC, Gill M, Lawlor B, Lynch A, Brown  KS, Passmore PA, Craig D, McGuinness B, Todd S, Holmes C, Mann D, Smith AD, Beaumont H, Warden D, Wilcock G, Love S, Kehoe PG, Hooper NM, Vardy ER, Hardy J,  Mead S, Fox NC, Rossor M, Collinge J, Maier W, Jessen F, Rüther E, Schürmann B, Heun R, Kölsch H, van den Bussche H, Heuser I, Kornhuber J, Wiltfang J, Dichgans  M, Frölich L, Hampel H, Gallacher J, Hüll M, Rujescu D, Giegling I, Goate AM, Kauwe JS, Cruchaga C, Nowotny P, Morris JC, Mayo K, Sleegers K, Bettens K, Engelborghs S, De Deyn PP, Van Broeckhoven C, Livingston G, Bass NJ, Gurling H, McQuillin A, Gwilliam R, Deloukas P, Al-Chalabi A, Shaw CE, Tsolak... Campion D, Epelbaum J, Dartigues JF, Tzourio C, Alpérovitch A, Lathrop M; EADI1 consortium, Feulner TM, Friedrich P, Riehle C, Krawczak M, Schreiber S, Mayhaus M, Nicolhaus S, Wagenpfeil S, Steinberg S, Stefansson H, Stefansson K, Snaedal J, Björnsson S, Jonsson PV, Chouraki V, Genier-Boley B, Hiltunen M, Soininen H, Combarros O, Zelenika D, Delepine M, Bullido MJ, Pasquier F, Mateo I, Frank-Garcia A, Porcellini E, Hanon  O, Coto E, Alvarez V, Bosco P, Siciliano G, Mancuso M, Panza F, Solfrizzi V, Nacmias B, Sorbi S, Bossù P, Piccardi P, Arosio B, Annoni G, Seripa D, Pilotto A, Scarpini E, Galimberti D, Brice A, Hannequin D, Licastro F, Jones L, Holmans PA,  Jonsson T, Riemenschneider M, Morgan K, Younkin SG, Owen MJ, O'Donovan M, Amouyel P, Williams J. Common variants at ABCA7, MS4A6A/MS4A4E, EPHA1, CD33 and CD2AP are associated with Alzheimer's disease. Nat Genet. 2011 May;43(5):429-35. doi: 10.1038/ng.803. Epub 2011 Apr 3. PubMed PMID: 21460840; PubMed Central PMCID: PMC3084173."

[Lambert 2013]: http://www.ncbi.nlm.nih.gov/pubmed/24162737/ "Lambert JC, Ibrahim-Verbaas CA, Harold D, Naj AC, Sims R, Bellenguez C, DeStafano AL, Bis JC, Beecham GW, Grenier-Boley B, Russo G, Thorton-Wells TA, Jones N, Smith AV, Chouraki V, Thomas C, Ikram MA, Zelenika D, Vardarajan BN, Kamatani Y, Lin CF, Gerrish A, Schmidt H, Kunkle B, Dunstan ML, Ruiz A, Bihoreau  MT, Choi SH, Reitz C, Pasquier F, Cruchaga C, Craig D, Amin N, Berr C, Lopez OL,  De Jager PL, Deramecourt V, Johnston JA, Evans D, Lovestone S, Letenneur L, Morón FJ, Rubinsztein DC, Eiriksdottir G, Sleegers K, Goate AM, Fiévet N, Huentelman MW, Gill M, Brown K, Kamboh MI, Keller L, Barberger-Gateau P, McGuiness B, Larson EB, Green R, Myers AJ, Dufouil C, Todd S, Wallon D, Love S, Rogaeva E, Gallacher  J, St George-Hyslop P, Clarimon J, Lleo A, Bayer A, Tsuang DW, Yu L, Tsolaki M, Bossù P, Spalletta G, Proitsi P, Collinge J, Sorbi S, Sanchez-Garcia F, Fox NC, Hardy J, Deniz Naranjo MC, Bosco P, Clarke R, Brayne C, Gali...s O, Younkin SG, Coto E, Hamilton-Nelson KL, Gu W, Razquin C, Pastor P, Mateo I, Owen  MJ, Faber KM, Jonsson PV, Combarros O, O'Donovan MC, Cantwell LB, Soininen H, Blacker D, Mead S, Mosley TH Jr, Bennett DA, Harris TB, Fratiglioni L, Holmes C,  de Bruijn RF, Passmore P, Montine TJ, Bettens K, Rotter JI, Brice A, Morgan K, Foroud TM, Kukull WA, Hannequin D, Powell JF, Nalls MA, Ritchie K, Lunetta KL, Kauwe JS, Boerwinkle E, Riemenschneider M, Boada M, Hiltuenen M, Martin ER, Schmidt R, Rujescu D, Wang LS, Dartigues JF, Mayeux R, Tzourio C, Hofman A, Nöthen MM, Graff C, Psaty BM, Jones L, Haines JL, Holmans PA, Lathrop M, Pericak-Vance MA, Launer LJ, Farrer LA, van Duijn CM, Van Broeckhoven C, Moskvina V, Seshadri S, Williams J, Schellenberg GD, Amouyel P. Meta-analysis of 74,046 individuals identifies 11 new susceptibility loci for Alzheimer's disease. Nat Genet. 2013 Dec;45(12):1452-8. doi: 10.1038/ng.2802. Epub 2013 Oct 27. PubMed PMID: 24162737; PubMed Central PMCID: PMC3896259."

[Naj 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21460841/ "Naj AC, Jun G, Beecham GW, Wang LS, Vardarajan BN, Buros J, Gallins PJ, Buxbaum JD, Jarvik GP, Crane PK, Larson EB, Bird TD, Boeve BF, Graff-Radford NR,  De Jager PL, Evans D, Schneider JA, Carrasquillo MM, Ertekin-Taner N, Younkin SG, Cruchaga C, Kauwe JS, Nowotny P, Kramer P, Hardy J, Huentelman MJ, Myers AJ, Barmada MM, Demirci FY, Baldwin CT, Green RC, Rogaeva E, St George-Hyslop P, Arnold SE, Barber R, Beach T, Bigio EH, Bowen JD, Boxer A, Burke JR, Cairns NJ, Carlson CS, Carney RM, Carroll SL, Chui HC, Clark DG, Corneveaux J, Cotman CW, Cummings JL, DeCarli C, DeKosky ST, Diaz-Arrastia R, Dick M, Dickson DW, Ellis WG, Faber KM, Fallon KB, Farlow MR, Ferris S, Frosch MP, Galasko DR, Ganguli M, Gearing M, Geschwind DH, Ghetti B, Gilbert JR, Gilman S, Giordani B, Glass JD, Growdon JH, Hamilton RL, Harrell LE, Head E, Honig LS, Hulette CM, Hyman BT, Jicha GA, Jin LW, Johnson N, Karlawish J, Karydas A, Kaye JA, Kim R, Koo EH, ...iniuk F, Mash DC, Masliah E, McCormick WC, McCurry SM, McDavid AN, McKee AC,  Mesulam M, Miller BL, Miller CA, Miller JW, Parisi JE, Perl DP, Peskind E, Petersen RC, Poon WW, Quinn JF, Rajbhandary RA, Raskind M, Reisberg B, Ringman JM, Roberson ED, Rosenberg RN, Sano M, Schneider LS, Seeley W, Shelanski ML, Slifer MA, Smith CD, Sonnen JA, Spina S, Stern RA, Tanzi RE, Trojanowski JQ, Troncoso JC, Van Deerlin VM, Vinters HV, Vonsattel JP, Weintraub S, Welsh-Bohmer  KA, Williamson J, Woltjer RL, Cantwell LB, Dombroski BA, Beekly D, Lunetta KL, Martin ER, Kamboh MI, Saykin AJ, Reiman EM, Bennett DA, Morris JC, Montine TJ, Goate AM, Blacker D, Tsuang DW, Hakonarson H, Kukull WA, Foroud TM, Haines JL, Mayeux R, Pericak-Vance MA, Farrer LA, Schellenberg GD. Common variants at MS4A4/MS4A6E, CD2AP, CD33 and EPHA1 are associated with late-onset Alzheimer's disease. Nat Genet. 2011 May;43(5):436-41. doi: 10.1038/ng.801. Epub 2011 Apr 3.  PubMed PMID: 21460841; PubMed Central PMCID: PMC3090745."

[Corder 1993]: http://www.ncbi.nlm.nih.gov/pubmed/8346443 "Corder EH, Saunders AM, Strittmatter WJ, Schmechel DE, Gaskell PC, Small GW, Roses AD, Haines JL, Pericak-Vance MA. Gene dose of apolipoprotein E type 4 allele and the risk of Alzheimer's disease in late onset families. Science. 1993  Aug 13;261(5123):921-3. PubMed PMID: 8346443."

[Choi 2015]: http://www.ncbi.nlm.nih.gov/pubmed/26582899 "Choi J, Gao J, Kim J, Hong C, Kim J, Tontonoz P. The E3 ubiquitin ligase Idol  controls brain LDL receptor expression, ApoE clearance, and Aβ amyloidosis. Sci Transl Med. 2015 Nov 18;7(314):314ra184. doi: 10.1126/scitranslmed.aad1904. PubMed PMID: 26582899."

[Hudry 2013]: http://www.ncbi.nlm.nih.gov/pubmed/24259049 "Hudry E, Dashkoff J, Roe AD, Takeda S, Koffie RM, Hashimoto T, Scheel M, Spires-Jones T, Arbel-Ornath M, Betensky R, Davidson BL, Hyman BT. Gene transfer  of human Apoe isoforms results in differential modulation of amyloid deposition and neurotoxicity in mouse brain. Sci Transl Med. 2013 Nov 20;5(212):212ra161. doi: 10.1126/scitranslmed.3007000. PubMed PMID: 24259049; PubMed Central PMCID: PMC4334150."

[Bien-Ly 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22492035 "Bien-Ly N, Gillespie AK, Walker D, Yoon SY, Huang Y. Reducing human apolipoprotein E levels attenuates age-dependent Aβ accumulation in mutant human  amyloid precursor protein transgenic mice. J Neurosci. 2012 Apr 4;32(14):4803-11. doi: 10.1523/JNEUROSCI.0033-12.2012. PubMed PMID: 22492035; PubMed Central PMCID: PMC3433173."

[Kim 2011]: http://www.ncbi.nlm.nih.gov/pubmed/22159114 "Kim J, Jiang H, Park S, Eltorai AE, Stewart FR, Yoon H, Basak JM, Finn MB, Holtzman DM. Haploinsufficiency of human APOE reduces amyloid deposition in a mouse model of amyloid-β amyloidosis. J Neurosci. 2011 Dec 7;31(49):18007-12. doi: 10.1523/JNEUROSCI.3773-11.2011. PubMed PMID: 22159114; PubMed Central PMCID: PMC3257514."

[Kim 2012]: http://www.ncbi.nlm.nih.gov/pubmed/23129750 "Kim J, Eltorai AE, Jiang H, Liao F, Verghese PB, Kim J, Stewart FR, Basak JM,  Holtzman DM. Anti-apoE immunotherapy inhibits amyloid accumulation in a transgenic mouse model of Aβ amyloidosis. J Exp Med. 2012 Nov 19;209(12):2149-56. doi: 10.1084/jem.20121274. Epub 2012 Nov 5. PubMed PMID: 23129750; PubMed Central PMCID: PMC3501350."

[Hu 2015]: http://www.ncbi.nlm.nih.gov/pubmed/25871773 "Hu J, Liu CC, Chen XF, Zhang YW, Xu H, Bu G. Opposing effects of viral mediated brain expression of apolipoprotein E2 (apoE2) and apoE4 on apoE lipidation and Aβ metabolism in apoE4-targeted replacement mice. Mol Neurodegener. 2015 Mar 5;10:6. doi: 10.1186/s13024-015-0001-3. PubMed PMID: 25871773; PubMed Central PMCID: PMC4356137."

[Jiang 2008]: http://www.ncbi.nlm.nih.gov/pubmed/18549781/ "Jiang Q, Lee CY, Mandrekar S, Wilkinson B, Cramer P, Zelcer N, Mann K, Lamb B, Willson TM, Collins JL, Richardson JC, Smith JD, Comery TA, Riddell D, Holtzman DM, Tontonoz P, Landreth GE. ApoE promotes the proteolytic degradation of Abeta.  Neuron. 2008 Jun 12;58(5):681-93. doi: 10.1016/j.neuron.2008.04.010. PubMed PMID: 18549781; PubMed Central PMCID: PMC2493297."

[Tsai 1994]: http://www.ncbi.nlm.nih.gov/pubmed/8128961/ "Tsai MS, Tangalos EG, Petersen RC, Smith GE, Schaid DJ, Kokmen E, Ivnik RJ, Thibodeau SN. Apolipoprotein E: risk factor for Alzheimer disease. Am J Hum Genet. 1994 Apr;54(4):643-9. PubMed PMID: 8128961; PubMed Central PMCID: PMC1918094."

[Coon 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17474819 "Coon KD, Myers AJ, Craig DW, Webster JA, Pearson JV, Lince DH, Zismann VL, Beach TG, Leung D, Bryden L, Halperin RF, Marlowe L, Kaleem M, Walker DG, Ravid R, Heward CB, Rogers J, Papassotiropoulos A, Reiman EM, Hardy J, Stephan DA. A high-density whole-genome association study reveals that APOE is the major susceptibility gene for sporadic late-onset Alzheimer's disease. J Clin Psychiatry. 2007 Apr;68(4):613-8. PubMed PMID: 17474819."

[Corder 1994]: http://www.ncbi.nlm.nih.gov/pubmed/7920638 "Corder EH, Saunders AM, Risch NJ, Strittmatter WJ, Schmechel DE, Gaskell PC Jr, Rimmler JB, Locke PA, Conneally PM, Schmader KE, et al. Protective effect of  apolipoprotein E type 2 allele for late onset Alzheimer disease. Nat Genet. 1994  Jun;7(2):180-4. PubMed PMID: 7920638."

[Farrer 1997]: http://www.ncbi.nlm.nih.gov/pubmed/9343467/ "Farrer LA, Cupples LA, Haines JL, Hyman B, Kukull WA, Mayeux R, Myers RH, Pericak-Vance MA, Risch N, van Duijn CM. Effects of age, sex, and ethnicity on the association between apolipoprotein E genotype and Alzheimer disease. A meta-analysis. APOE and Alzheimer Disease Meta Analysis Consortium. JAMA. 1997 Oct 22-29;278(16):1349-56. PubMed PMID: 9343467."

[Minikel 2016]: http://www.ncbi.nlm.nih.gov/pubmed/26791950 "Minikel EV, Vallabh SM, Lek M, Estrada K, Samocha KE, Sathirapongsasuti JF, McLean CY, Tung JY, Yu LP, Gambetti P, Blevins J, Zhang S, Cohen Y, Chen W, Yamada M, Hamaguchi T, Sanjo N, Mizusawa H, Nakamura Y, Kitamoto T, Collins SJ, Boyd A, Will RG, Knight R, Ponto C, Zerr I, Kraus TF, Eigenbrod S, Giese A, Calero M, de Pedro-Cuesta J, Haïk S, Laplanche JL, Bouaziz-Amar E, Brandel JP, Capellari S, Parchi P, Poleggi A, Ladogana A, O'Donnell-Luria AH, Karczewski KJ,  Marshall JL, Boehnke M, Laakso M, Mohlke KL, Kähler A, Chambert K, McCarroll S, Sullivan PF, Hultman CM, Purcell SM, Sklar P, van der Lee SJ, Rozemuller A, Jansen C, Hofman A, Kraaij R, van Rooij JG, Ikram MA, Uitterlinden AG, van Duijn  CM; Exome Aggregation Consortium (ExAC), Daly MJ, MacArthur DG. Quantifying prion disease penetrance using large population control cohorts. Sci Transl Med. 2016 Jan 20;8(322):322ra9. doi: 10.1126/scitranslmed.aad5169. PubMed PMID: 26791950."

[Cramer 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22323736 "Cramer PE, Cirrito JR, Wesson DW, Lee CY, Karlo JC, Zinn AE, Casali BT, Restivo JL, Goebel WD, James MJ, Brunden KR, Wilson DA, Landreth GE. ApoE-directed therapeutics rapidly clear β-amyloid and reverse deficits in AD mouse models. Science. 2012 Mar 23;335(6075):1503-6. doi: 10.1126/science.1217697. Epub 2012 Feb 9. PubMed PMID: 22323736; PubMed Central PMCID: PMC3651582."


