---
layout: post
title:  "Inferring Clonal Dynamics"
tags: [chromatin]
---

* DNA wrapped around histones, proteins, is called chromatin.
* The analogy is that imagine we have a garden hose that we need to figure out how to store in an organized, compact way. The histones provide the structure for DNA to wrap around and also the energy (through electrostatic interactions) to fold the DNA. 

## Histones

Histones are a small family of termed:

1. H1
1. H2A
1. H2B
1. H3
1. H4

As DNA is negatively charged, due to the phosphate groups in its phosphate-sugar backbone, its bind very strong around histones which are positively charged.

## Nucleosome: The Unit of Chromatin

* The basic repeating unit of chromatin is the nucleosome.
* Contains about 8 histone proteins and about 146 basepairs.
* 2 of H2A, H2B, H3, and H4 come together to form a histone octomer which bind and wrap 146 base pairs of DNA.
* When H1 (linker histone) is added, this forms a chromatosome and wraps another 20 basepairs.
* Each chromosome contains hundreds of thousands of nucleosomes which linked together with a DNA sequence called a "linker DNA".

## Histone Marks 

Histones can be enzymatically modified through the addition of acetyl, methyl and phosphate groups.

* H3K27 refers to a covalent modification of the 27th amino acid residue (which is Lysine as indicated by the K symbol) on the core histone H3 of a nucleosome.
* Histones can be displaced by chromatin remodeling complexes, thereby exposing underlying DNA sequences to polymerases and other enzymes (Smith & Peterson, 2005

## Chromatin Remodelling

Chromatin remodelling refers to the dynamic modification of chromatin structure which exposes DNA to the transcriptional mechanism and ultimately controls gene expression.

## Cell-free DNA and Chromatin

* Interesting link between cell-free DNA (cfDNA) and chromatin structure. 
* cfDNA size distribution strongly correlates with the amount of DNA wrapped around a nucleosome (~ 146 bp) and chromatosomes (~167 bp; nucleosome + linker histone H1).
* Snyder et al. (2016) demonstrate how by sequencing cfDNA, it is possible to produce genome-wide maps of in vivo nucleosome occupancy. Furthermore, nucleosome position is a function of cell type, and so they were able to infer the tissue of origin based on these nucleosome occupancy maps.

## References

* [DNA Packaging: Nucleosomes and Chromatin](http://www.nature.com/scitable/topicpage/dna-packaging-nucleosomes-and-chromatin-310)
